<!DOCTYPE html>
<html lang="en">
<head>
    <meta http-equiv="Content-Type" content="text/html;charset=utf-8" >

    <style rel="stylesheet">
/*!
 * Bootstrap v2.0.4
 *
 * Copyright 2012 Twitter, Inc
 * Licensed under the Apache License v2.0
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Designed and built with all the love in the world @twitter by @mdo and @fat.
 */
.clearfix{*zoom:1;}.clearfix:before,.clearfix:after{display:table;content:"";}
.clearfix:after{clear:both;}
.hide-text{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0;}
.input-block-level{display:block;width:100%;min-height:28px;-webkit-box-sizing:border-box;-moz-box-sizing:border-box;-ms-box-sizing:border-box;box-sizing:border-box;}
.tooltip{position:absolute;z-index:1020;display:block;visibility:visible;padding:5px;font-size:11px;opacity:0;filter:alpha(opacity=0);}.tooltip.in{opacity:0.8;filter:alpha(opacity=80);}
.tooltip.top{margin-top:-2px;}
.tooltip.right{margin-left:2px;}
.tooltip.bottom{margin-top:2px;}
.tooltip.left{margin-left:-2px;}
.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-left:5px solid transparent;border-right:5px solid transparent;border-top:5px solid #000000;}
.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-top:5px solid transparent;border-bottom:5px solid transparent;border-left:5px solid #000000;}
.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-left:5px solid transparent;border-right:5px solid transparent;border-bottom:5px solid #000000;}
.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-top:5px solid transparent;border-bottom:5px solid transparent;border-right:5px solid #000000;}
.tooltip-inner{max-width:200px;padding:3px 8px;color:#ffffff;text-align:center;text-decoration:none;background-color:#000000;-webkit-border-radius:4px;-moz-border-radius:4px;border-radius:4px;}
.tooltip-arrow{position:absolute;width:0;height:0;}

</style>

    <style rel="stylesheet">
/* Main things */
html, body, div, span, h1, h3, h4, h5, h6, blockquote, pre,
em, sub, sup, dl, dt, dd, ol, ul, li,
fieldset, form, label,
table, caption, tbody, tfoot, thead,
article, aside, canvas, details, figure, figcaption, footer, header, hgroup,
menu, nav, output, section, time {
    margin: 0;
    padding: 0;
    border: 0;
    font-size: 100%;
    font: inherit;
    vertical-align: baseline;
}
body {
    margin: 0;
    font-family: Lucida Grande, Verdana, Arial, sans-serif;
    font-size: 12px;
    color: black;
    padding-bottom: 30px;
}
a {
    text-decoration: none;
    /*color: #0c4f72;*/
    color: #0077B5;
    border-bottom: 1px solid #487b95;
    border-color: #487b95; /* #487b95 */
    border-color: rgba(12, 79, 114, 0.2); /* #0c4f72; */ /* #a0d2eb; */ /*rgba(0, 88, cc, 0.3);*/
}
a:visited {
    text-decoration: none;
    color: #8C66B2;
    border-bottom: 1px solid #8c66b2;
    border-color: #8c66b2;
    border-color: rgba(102, 51, 153, 0.2);
}
a:hover {
    text-decoration: none;
    color: #b33; /* #0088cc; */
    border-bottom: 1px solid #ffcccc;
    border-color: #ffcccc;
    border-color: rgba(255, 0, 0, 0.2); /* #55b2e0; */ /*#005580;*/
}
a.dark_bg_link {
    color: white; /* #0088cc; */
    border-bottom: 1px solid #cccccc;
    border-color: #cccccc;
}
a.dark_bg_link:hover {
    color: white; /* #0088cc; */
    border-bottom: 1px solid white;
}

.leading_link_dot {
    border-bottom: white solid 1px;
}

a.header_link, a.header_link:hover, a.header_link:visited {
    color: black;
    border-bottom: 1px solid #bbb;
}


.dotted-link {
    cursor: pointer;
    text-decoration: none;
    /*color: #0c4f72;*/
    color: #0077B5;
    border-bottom: 1px dashed #487b95;
    border-color: #487b95; /* #487b95 */
    border-color: rgba(12, 79, 114, 0.2);
}
.dotted-link:hover {
    text-decoration: none;
    color: #b33 !important;
    border-bottom: 1px dashed #b33 !important;
    border-color: rgba(255, 0, 0, 0.2) !important;
}
.js:visited {
    color: #0077B5;
    border-bottom: 1px dashed #487b95;
    border-color: #487b95;
    border-color: rgba(12, 79, 114, 0.2);
}

.selected-switch {
    background-color: #fafad2;
}

.json-code {
    display: none;
}
.smallcapitals {
    font-size: .875em;
    letter-spacing: .15em;
    margin-right: -.15em;
}
.highlight {
    background-color:#FFF9B5;
    padding: 0 3px;
    margin-left: -3px;
}

ul.list {
    list-style-type: none;
    padding: 0 0 0 2em;
    margin: -0.5em 0 0 0;
}
ul.list li {
    margin-bottom: 0.7em;
}

ul.mdash {
    margin: -1em 0 0 0;
    padding: 0 0 0 2em;
}
ul.mdash li {
    margin-bottom: 0.7em;
    list-style-type: none;
    text-indent: -1.3em;
    behavior: expression(!this.before ? this.before = this.innerHTML = '&mdash;&nbsp;' + this.innerHTML : '');
}
ul.mdash li:before {
    content: '\2014\a0';
}

.small_line_indent {
    height: 0;
    margin-top: -7px;
}
/************/

/* Tooltips */
.tooltip {
    font-size: 0.9em;
    margin-bottom: 5px;
}
.tooltip.in {
    opacity: 1;
}
.tooltip-inner {
    text-align: left;
}
.tooltip-link,
.tooltip-link:visited {
    text-decoration: none;
    color: #000;
    border-bottom: 1px dashed #bbb;
    cursor: hand;
}
.tooltip-link:hover {
    text-decoration: none;
    color: #b33;
    border-bottom: 1px dashed #b33;
}
/************/

/*********/

.hs {
    /* Полупробел для
  разделения разрядов чисел:
  10<span class="hs"></span>000 */
    margin-left: .2em;
}

.rhs {
    /* Полупробел на случай,
  если реальный пробел важен,
  например между ценой и валютой:
  1<span class="rhs">&nbsp;</span>$ */
    font-size: 50%;
    line-height: 1;
}

/*9490<span class="rhs"> </span>€<br/>*/
/*12<span class="hs"></span>650<span class="rhs"> </span>$<br/>*/
/*379<span class="hs"></span>000<span*/
/*class="rhs"> </span><span class="b-rub">Р</span><br/>*/
/*100<span class="hs"></span>500<span class="rhs"> </span>+*/

/* for wrapping long words with no spaces */
/* http://stackoverflow.com/questions/3247358/css-how-do-i-wrap-text-with-no-whitespace-inside-a-td */
.wrapword {
    white-space: -moz-pre-wrap !important;  /* Mozilla, since 1999 */
    white-space: -pre-wrap;      /* Opera 4-6 */
    white-space: -o-pre-wrap;    /* Opera 7 */
    white-space: pre-wrap;       /* css-3 */
    word-wrap: break-word;       /* Internet Explorer 5.5+ */
    word-break: break-all;
    white-space: normal;
}


.space_1px {
    height: 1px;
}
.space_2px {
    height: 2px;
}
.space_3px {
    height: 3px;
}
.space_4px {
    height: 4px;
}
.space_5px {
    height: 5px;
}
.space_6px {
    height: 6px;
}
.space_7px {
    height: 7px;
}
.space_8px {
    height: 8px;
}
.space_9px {
    height: 9px;
}
.space_10px {
    height: 10px;
}
.layout-table {
    width: 99%;
    border-spacing: 0;
}

.layout-table td.layout-table-td {
    vertical-align: top;
    padding: 0;
}
.layout-table td.layout-table-fst-td {
    min-width: 400px;
    padding-right: 50px;
}
.layout-table td.layout-table-snd-td {
    min-width: 200px;
}
.layout-table td.layout-table-trd-td {
}

.top-panel {
    background-color: #444;
    color: white;
    min-width: 650px;
    padding: 8px 10px 0 147px;
    min-height: 70px;
    text-shadow: -1px -1px 0 #111;
}
.top-panel span {
    color: white;
}
.top-panel a {
    color: white;
    border-bottom: 1px solid white;
}
#to_main_menu_button a {
    border-bottom: none;
}
.page_title {
    margin-top: -3px;
    margin-bottom: -3px;
    color: white;
}
.page_subtitle {
    line-height: 100%;
    white-space: nowrap;
}
.content {
    margin-left: 147px;
    width: 90%;
    /*padding-top: 20px;*/
    margin-bottom: 0;
    /*padding-bottom: 50px;*/
/*{#        padding-right: 20px;#}*/
}
.p_after_content {
    margin-left: 147px;
}
.links_after_content{
    font-size: 14px;
}

p {
    display: block;
    font-size: 12px;
}

h1 {
    display: block;
    font-size: 26px;
    padding: 0 -3px 0 5px;
    -webkit-margin-before: 5px;
    -webkit-margin-after: 0px;
    -webkit-margin-start: 0px;
    -webkit-margin-end: 0px;
    font-weight: bold;
}

#header {
    display: block;
    width: 100%;
    margin-bottom: 10px;
}

.logo_img {
    height: 40px;
    margin-bottom: -10px;
    margin-left: -6px;
    margin-right: -7px;
}
</style>

    <style rel="stylesheet">
.report-and-plots {
    margin-top: 15px;
}

/****************/
/* REPORT TABLE */
.report {    
    margin: 0 100px 50px 0;
    float: left;
    font-size: 1em;
}

#main_report {
    /*min-width: 600px;*/
}

/*#extended_link {*/
    /*margin-top: 5px;*/
/*}*/

#ref_report {
    /*min-width: 300px;*/
    margin-right: 0;
    margin-bottom: 20px;
    /*margin-right: 200px;*/
}

.report td {
    margin: 0;
    padding: 2px 0.6em 0 14px;
    /*min-width: 100px;*/
    max-width: 400px;
    white-space: nowrap;
}
.report td.number {
    text-align: right;
}

.report tr.top_row_tr td,
.report tr.group_header td {
    font-weight: bold;
    border-bottom: 1px solid #ddd;
}
.report tr.group_header td {
    height: 20px;
    vertical-align: bottom;
}

#ref_report tr.top_row_tr td,
#ref_report tr.group_header td {
    font-weight: normal;
}

.report tr td:first-child {
    padding-left: 0;
}
.report tr td:last-child {
    padding-right: 0;
}

.report tr.row_hidden, tr.group_empty {
    display: none;
}

.report tr.secondary_hidden {
    display: none;
}

/*.report td.secondary_td {*/
	/*padding-left: 10px !important;*/
/*}*/

span.heatmap_legend {
    font-size: 11px;
    display: inline-block;
}

#heatmap_header {
    margin-right: 30px;
}

#combined_reference_length {
    padding-top: 1px;
    padding-bottom: 15px;
}

span.metric-name.expandable {
    margin-left: -13px;
}

span.metric-name.secondary {
    margin-left: 13px;
}

span.metric-name.expandable.collapsed::before {
    content: '+ ';
    color: #CCC;
    cursor: pointer;
}

span.metric-name.expandable.expanded::before {
    content: '\2212\a0';  /* &minus&nbsp */
    color: #CCC;
    cursor: pointer;
}

.report_legend {
    padding-left: 15px;  /* compensate drag handles */
}

#contigs_are_ordered {
    z-index: 0;
    position: relative;
}


/*.report .report-table-header {
    font-weight: bold;
}*/

/*.report thead td {
    border-bottom: 1px solid #bbb;
}*/


/*************/
/* Draggable */
table.moving_table {
    box-shadow: -0px 0px 5px 1px #888888;
    -moz-box-shadow: -0px 0px 5px 1px #888888;
    -webkit-box-shadow: -0px 0px 5px 1px #888888;
    /*outline: 1px solid #888888;*/
    background-color: white;
    z-index: 3 !important;
}

table.draggable .drag_handle {
    padding: 2px 1px 0 1px;
    margin: 0 2px 0 1px;

    cursor: grab;
    cursor: -moz-grab;
    cursor: -webkit-grab;
}

table.draggable .drag_handle .drag_image {
    height: 9px;
    width: 9px;
    display: inline-block;
    background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAkAAAAJCAIAAABv85FHAAAACXBIWXMAAAsTAAALEwEAmpwYAAAAiElEQVQImV2OsQ3FIAxEz2AhCAukyATZf5ZMQI8okZUjhfWj6F/pd09n6b1f1wUgxkgSAACS53lqSmnfdxHBLyJCMucsc877vv201nobKSUl2VoLIeATksdxKEmf+UqOxczGGDHGP7Ztm5qZmfmkYx8upehaq/fu4P2WZK1VAdRaAbjqXghBVR9BvlQcgtQsKgAAAABJRU5ErkJggg==") no-repeat;
}

table.draggable .drag_handle:active, 
table.draggable td.cell_of_moving_table .drag_handle {
    cursor: grabbing !important;
    cursor: -moz-grabbing !important;
    cursor: -webkit-grabbing !important;
}

/* Override the styles for the first and the last columns of the table.report.
   The cloned table contains a single column which is both first and last,
   and we don't need it to inherit those styles. */
table.draggable td.cell_of_moving_table {
    padding-left: 14px !important; 
    padding-right: 0.6em !important;
}

table.draggable td.hovered_column_td {
    background-color: #cccccc;
}

table.draggable td.second_through_last_col_headers_td {
    padding-left: 0 !important;  /* compensate handle images */
}

table.draggable td.cell_of_empty_space * {
    visibility: hidden;
}

table.draggable td.cell_of_empty_space {
    border: none !important;
  /*border-left: 1px solid #888888;
    border-right: 1px solid #888888;   
    margin-right: -1px !important;
    margin-left: -1px !important;  */
}

/*table.draggable td.bottom_cell_of_empty_space {
    border-bottom: 1px solid #888888  !important;
}

table.draggable td.top_cell_of_empty_space {
    border-top: 1px solid #888888  !important;    
    /*margin-top: -1px;
}*/

table.draggable td.top_cell_of_empty_space {
}

table.draggable td.left_to_cell_of_empty_space {
    /*outline-left: 1px solid #888888;*/
}

table.draggable td.right_to_cell_of_empty_space {
    /*outline-right: 1px solid #888888;*/
}

/********/
/* META */
#main_ref_genome {

}

table#refgenome td.last_row {
    border-top: 1px solid #ddd;
}

/************/
/* PLOTTING */
.plot-switch {
    margin-left: 10px;
}

.plot-mis-type-switch {
    margin-left: 23px;
}

.plot-wrapper {
    padding-right: 80px;
}

.plot-header {
    margin-left: 0;
    font-weight: bold;
    height: 1.5em;
    display: block;
}

.plots {
    margin-left: -30px;
    margin-right: 70px;
    margin-top: 2px;
    margin-bottom: 50px;
    float: left;
}
.plots td {
    vertical-align: top;
    white-space: nowrap;
}

.plots div#plots-switches, tr#plots-into-tr p {
    padding-left: 30px;
}

#legend-placeholder {
    min-width: 80px;
    margin-top: -2px;
    margin-left: -5px;
    position: relative;
    z-index: 0;
}

.legendColorBox div {
    display: none;
}

#legend-placeholder td {
    vertical-align: top;
    white-space: nowrap;
}

#plot-placeholder {
    width: 986px;
    height: 600px;

    /* for auto sizing plots when resizing the page 
    min-width: 500px;
    min-height: 500px;
    width: 100%;
    min-height: 400px;
    height: 50%;    */

    margin-left: -55px;
    margin-right: 20px;
    float: left;

    z-index: 2;
}

#change-assembly {
    white-space: nowrap;
    width: 0;
}

/*************/
/* Plot tips */
#plot_tip {
    position: absolute;
    opacity: 1;
    padding: 2px 5px 2px 5px;

    z-index: 1;
}

#plot_tip_vertical_rule, #plot_tip_horizontal_rule {
    position: absolute;
    border: 1px solid #BBB;
    z-index: -2;
}

#plot_tip_vertical_rule {
    border-left: 1px;
}

#plot_tip_horizontal_rule {
    border-top: 1px;
}

.white_stroked {
    color: black;
    background-color: white;
    text-shadow:
        -3px -3px 0 white,
        3px -3px 0 white,
        -3px 3px 0 white,
        3px 3px 0 white;
}

img.icon_plot {
    width: 14px;
    height: 12px;
    display: inline-block;
    content: '';
    background: url("data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAANCAYAAACZ3F9/AAAACXBIWXMAAAsTAAALEwEAmpwYAAA7oGlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPD94cGFja2V0IGJlZ2luPSLvu78iIGlkPSJXNU0wTXBDZWhpSHpyZVN6TlRjemtjOWQiPz4KPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iQWRvYmUgWE1QIENvcmUgNS41LWMwMjEgNzkuMTU1NzcyLCAyMDE0LzAxLzEzLTE5OjQ0OjAwICAgICAgICAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iCiAgICAgICAgICAgIHhtbG5zOmRjPSJodHRwOi8vcHVybC5vcmcvZGMvZWxlbWVudHMvMS4xLyIKICAgICAgICAgICAgeG1sbnM6cGhvdG9zaG9wPSJodHRwOi8vbnMuYWRvYmUuY29tL3Bob3Rvc2hvcC8xLjAvIgogICAgICAgICAgICB4bWxuczp4bXBNTT0iaHR0cDovL25zLmFkb2JlLmNvbS94YXAvMS4wL21tLyIKICAgICAgICAgICAgeG1sbnM6c3RFdnQ9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9zVHlwZS9SZXNvdXJjZUV2ZW50IyIKICAgICAgICAgICAgeG1sbnM6dGlmZj0iaHR0cDovL25zLmFkb2JlLmNvbS90aWZmLzEuMC8iCiAgICAgICAgICAgIHhtbG5zOmV4aWY9Imh0dHA6Ly9ucy5hZG9iZS5jb20vZXhpZi8xLjAvIj4KICAgICAgICAgPHhtcDpDcmVhdG9yVG9vbD5BZG9iZSBQaG90b3Nob3AgQ0MgMjAxNCAoTWFjaW50b3NoKTwveG1wOkNyZWF0b3JUb29sPgogICAgICAgICA8eG1wOkNyZWF0ZURhdGU+MjAxNS0wOC0wN1QxNjowMzowNCswMzowMDwveG1wOkNyZWF0ZURhdGU+CiAgICAgICAgIDx4bXA6TW9kaWZ5RGF0ZT4yMDE1LTA4LTA3VDIxOjE2OjMyKzAzOjAwPC94bXA6TW9kaWZ5RGF0ZT4KICAgICAgICAgPHhtcDpNZXRhZGF0YURhdGU+MjAxNS0wOC0wN1QyMToxNjozMiswMzowMDwveG1wOk1ldGFkYXRhRGF0ZT4KICAgICAgICAgPGRjOmZvcm1hdD5pbWFnZS9wbmc8L2RjOmZvcm1hdD4KICAgICAgICAgPHBob3Rvc2hvcDpDb2xvck1vZGU+MzwvcGhvdG9zaG9wOkNvbG9yTW9kZT4KICAgICAgICAgPHhtcE1NOkluc3RhbmNlSUQ+eG1wLmlpZDphYjJhNzRkZi04MjcxLTQ1NjctOTgxNy0zNTVkNTAwNmQ5OTk8L3htcE1NOkluc3RhbmNlSUQ+CiAgICAgICAgIDx4bXBNTTpEb2N1bWVudElEPnhtcC5kaWQ6ZWUxMmZkZGQtZWQ3OC00MTliLTk5MDctZTQ5OTkyZDUyNjRiPC94bXBNTTpEb2N1bWVudElEPgogICAgICAgICA8eG1wTU06T3JpZ2luYWxEb2N1bWVudElEPnhtcC5kaWQ6ZWUxMmZkZGQtZWQ3OC00MTliLTk5MDctZTQ5OTkyZDUyNjRiPC94bXBNTTpPcmlnaW5hbERvY3VtZW50SUQ+CiAgICAgICAgIDx4bXBNTTpIaXN0b3J5PgogICAgICAgICAgICA8cmRmOlNlcT4KICAgICAgICAgICAgICAgPHJkZjpsaSByZGY6cGFyc2VUeXBlPSJSZXNvdXJjZSI+CiAgICAgICAgICAgICAgICAgIDxzdEV2dDphY3Rpb24+Y3JlYXRlZDwvc3RFdnQ6YWN0aW9uPgogICAgICAgICAgICAgICAgICA8c3RFdnQ6aW5zdGFuY2VJRD54bXAuaWlkOmVlMTJmZGRkLWVkNzgtNDE5Yi05OTA3LWU0OTk5MmQ1MjY0Yjwvc3RFdnQ6aW5zdGFuY2VJRD4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OndoZW4+MjAxNS0wOC0wN1QxNjowMzowNCswMzowMDwvc3RFdnQ6d2hlbj4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OnNvZnR3YXJlQWdlbnQ+QWRvYmUgUGhvdG9zaG9wIENDIDIwMTQgKE1hY2ludG9zaCk8L3N0RXZ0OnNvZnR3YXJlQWdlbnQ+CiAgICAgICAgICAgICAgIDwvcmRmOmxpPgogICAgICAgICAgICAgICA8cmRmOmxpIHJkZjpwYXJzZVR5cGU9IlJlc291cmNlIj4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OmFjdGlvbj5zYXZlZDwvc3RFdnQ6YWN0aW9uPgogICAgICAgICAgICAgICAgICA8c3RFdnQ6aW5zdGFuY2VJRD54bXAuaWlkOjQzY2E0MmJhLWQ2YmItNGQ5MC05MDQ0LTc2NDJiMjgxNGVjZTwvc3RFdnQ6aW5zdGFuY2VJRD4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OndoZW4+MjAxNS0wOC0wN1QyMTowMjo1NSswMzowMDwvc3RFdnQ6d2hlbj4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OnNvZnR3YXJlQWdlbnQ+QWRvYmUgUGhvdG9zaG9wIENDIDIwMTQgKE1hY2ludG9zaCk8L3N0RXZ0OnNvZnR3YXJlQWdlbnQ+CiAgICAgICAgICAgICAgICAgIDxzdEV2dDpjaGFuZ2VkPi88L3N0RXZ0OmNoYW5nZWQ+CiAgICAgICAgICAgICAgIDwvcmRmOmxpPgogICAgICAgICAgICAgICA8cmRmOmxpIHJkZjpwYXJzZVR5cGU9IlJlc291cmNlIj4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OmFjdGlvbj5zYXZlZDwvc3RFdnQ6YWN0aW9uPgogICAgICAgICAgICAgICAgICA8c3RFdnQ6aW5zdGFuY2VJRD54bXAuaWlkOmFiMmE3NGRmLTgyNzEtNDU2Ny05ODE3LTM1NWQ1MDA2ZDk5OTwvc3RFdnQ6aW5zdGFuY2VJRD4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OndoZW4+MjAxNS0wOC0wN1QyMToxNjozMiswMzowMDwvc3RFdnQ6d2hlbj4KICAgICAgICAgICAgICAgICAgPHN0RXZ0OnNvZnR3YXJlQWdlbnQ+QWRvYmUgUGhvdG9zaG9wIENDIDIwMTQgKE1hY2ludG9zaCk8L3N0RXZ0OnNvZnR3YXJlQWdlbnQ+CiAgICAgICAgICAgICAgICAgIDxzdEV2dDpjaGFuZ2VkPi88L3N0RXZ0OmNoYW5nZWQ+CiAgICAgICAgICAgICAgIDwvcmRmOmxpPgogICAgICAgICAgICA8L3JkZjpTZXE+CiAgICAgICAgIDwveG1wTU06SGlzdG9yeT4KICAgICAgICAgPHRpZmY6T3JpZW50YXRpb24+MTwvdGlmZjpPcmllbnRhdGlvbj4KICAgICAgICAgPHRpZmY6WFJlc29sdXRpb24+NzIwMDAwLzEwMDAwPC90aWZmOlhSZXNvbHV0aW9uPgogICAgICAgICA8dGlmZjpZUmVzb2x1dGlvbj43MjAwMDAvMTAwMDA8L3RpZmY6WVJlc29sdXRpb24+CiAgICAgICAgIDx0aWZmOlJlc29sdXRpb25Vbml0PjI8L3RpZmY6UmVzb2x1dGlvblVuaXQ+CiAgICAgICAgIDxleGlmOkNvbG9yU3BhY2U+NjU1MzU8L2V4aWY6Q29sb3JTcGFjZT4KICAgICAgICAgPGV4aWY6UGl4ZWxYRGltZW5zaW9uPjE0PC9leGlmOlBpeGVsWERpbWVuc2lvbj4KICAgICAgICAgPGV4aWY6UGl4ZWxZRGltZW5zaW9uPjEzPC9leGlmOlBpeGVsWURpbWVuc2lvbj4KICAgICAgPC9yZGY6RGVzY3JpcHRpb24+CiAgIDwvcmRmOlJERj4KPC94OnhtcG1ldGE+CiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgCiAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAKICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgICAgIAogICAgICAgICAgICAgICAgICAgICAgICAgICAgCjw/eHBhY2tldCBlbmQ9InciPz6T8SobAAAAIGNIUk0AAHolAACAgwAA+f8AAIDpAAB1MAAA6mAAADqYAAAXb5JfxUYAAAD7SURBVHjaYvz//z8DOYCJgUxAHY393d3PGRgYPqDjG4sWfnh348ZzZLUsyJz///5JwNjPP/1guPbuG4MJ63cGtn9/GQSlpfhx2sjCx/ft8apVDF/OnmF4zcjG8OL7H4YDxy8wfPcMYnjNyv0Np8bf378xiGqpMzz9/Z/h7c6dDKab5jA4mOsyPPr+n+Hhq/cMOJ3K9OMbw0cdfYY3nxkYDPlvMHxh/MbALyvH4IklcFA0sjAzc737ysAg9vc3g4CmBoOApgayNBcyhxE5ASRERDwX4OXk5GDnYPjy8xcDAyMjA8P//wy/f/1mUFRW/l5RVy+JVePgTjmAAQCr211NDit/xwAAAABJRU5ErkJggg==");
}

#sortRefsBtn label {
    display: block;
    margin-left: 20px;
}
#sortRefsBtn input {
    float: left;
    margin-left: -20px;
    margin-right: 7px;
}

#mincontig {
    margin-bottom: 0;
}
</style>


    <script type="text/javascript">
/*! jQuery v1.8.2 jquery.com | jquery.org/license */
(function(a,b){function G(a){var b=F[a]={};return p.each(a.split(s),function(a,c){b[c]=!0}),b}function J(a,c,d){if(d===b&&a.nodeType===1){var e="data-"+c.replace(I,"-$1").toLowerCase();d=a.getAttribute(e);if(typeof d=="string"){try{d=d==="true"?!0:d==="false"?!1:d==="null"?null:+d+""===d?+d:H.test(d)?p.parseJSON(d):d}catch(f){}p.data(a,c,d)}else d=b}return d}function K(a){var b;for(b in a){if(b==="data"&&p.isEmptyObject(a[b]))continue;if(b!=="toJSON")return!1}return!0}function ba(){return!1}function bb(){return!0}function bh(a){return!a||!a.parentNode||a.parentNode.nodeType===11}function bi(a,b){do a=a[b];while(a&&a.nodeType!==1);return a}function bj(a,b,c){b=b||0;if(p.isFunction(b))return p.grep(a,function(a,d){var e=!!b.call(a,d,a);return e===c});if(b.nodeType)return p.grep(a,function(a,d){return a===b===c});if(typeof b=="string"){var d=p.grep(a,function(a){return a.nodeType===1});if(be.test(b))return p.filter(b,d,!c);b=p.filter(b,d)}return p.grep(a,function(a,d){return p.inArray(a,b)>=0===c})}function bk(a){var b=bl.split("|"),c=a.createDocumentFragment();if(c.createElement)while(b.length)c.createElement(b.pop());return c}function bC(a,b){return a.getElementsByTagName(b)[0]||a.appendChild(a.ownerDocument.createElement(b))}function bD(a,b){if(b.nodeType!==1||!p.hasData(a))return;var c,d,e,f=p._data(a),g=p._data(b,f),h=f.events;if(h){delete g.handle,g.events={};for(c in h)for(d=0,e=h[c].length;d<e;d++)p.event.add(b,c,h[c][d])}g.data&&(g.data=p.extend({},g.data))}function bE(a,b){var c;if(b.nodeType!==1)return;b.clearAttributes&&b.clearAttributes(),b.mergeAttributes&&b.mergeAttributes(a),c=b.nodeName.toLowerCase(),c==="object"?(b.parentNode&&(b.outerHTML=a.outerHTML),p.support.html5Clone&&a.innerHTML&&!p.trim(b.innerHTML)&&(b.innerHTML=a.innerHTML)):c==="input"&&bv.test(a.type)?(b.defaultChecked=b.checked=a.checked,b.value!==a.value&&(b.value=a.value)):c==="option"?b.selected=a.defaultSelected:c==="input"||c==="textarea"?b.defaultValue=a.defaultValue:c==="script"&&b.text!==a.text&&(b.text=a.text),b.removeAttribute(p.expando)}function bF(a){return typeof a.getElementsByTagName!="undefined"?a.getElementsByTagName("*"):typeof a.querySelectorAll!="undefined"?a.querySelectorAll("*"):[]}function bG(a){bv.test(a.type)&&(a.defaultChecked=a.checked)}function bY(a,b){if(b in a)return b;var c=b.charAt(0).toUpperCase()+b.slice(1),d=b,e=bW.length;while(e--){b=bW[e]+c;if(b in a)return b}return d}function bZ(a,b){return a=b||a,p.css(a,"display")==="none"||!p.contains(a.ownerDocument,a)}function b$(a,b){var c,d,e=[],f=0,g=a.length;for(;f<g;f++){c=a[f];if(!c.style)continue;e[f]=p._data(c,"olddisplay"),b?(!e[f]&&c.style.display==="none"&&(c.style.display=""),c.style.display===""&&bZ(c)&&(e[f]=p._data(c,"olddisplay",cc(c.nodeName)))):(d=bH(c,"display"),!e[f]&&d!=="none"&&p._data(c,"olddisplay",d))}for(f=0;f<g;f++){c=a[f];if(!c.style)continue;if(!b||c.style.display==="none"||c.style.display==="")c.style.display=b?e[f]||"":"none"}return a}function b_(a,b,c){var d=bP.exec(b);return d?Math.max(0,d[1]-(c||0))+(d[2]||"px"):b}function ca(a,b,c,d){var e=c===(d?"border":"content")?4:b==="width"?1:0,f=0;for(;e<4;e+=2)c==="margin"&&(f+=p.css(a,c+bV[e],!0)),d?(c==="content"&&(f-=parseFloat(bH(a,"padding"+bV[e]))||0),c!=="margin"&&(f-=parseFloat(bH(a,"border"+bV[e]+"Width"))||0)):(f+=parseFloat(bH(a,"padding"+bV[e]))||0,c!=="padding"&&(f+=parseFloat(bH(a,"border"+bV[e]+"Width"))||0));return f}function cb(a,b,c){var d=b==="width"?a.offsetWidth:a.offsetHeight,e=!0,f=p.support.boxSizing&&p.css(a,"boxSizing")==="border-box";if(d<=0||d==null){d=bH(a,b);if(d<0||d==null)d=a.style[b];if(bQ.test(d))return d;e=f&&(p.support.boxSizingReliable||d===a.style[b]),d=parseFloat(d)||0}return d+ca(a,b,c||(f?"border":"content"),e)+"px"}function cc(a){if(bS[a])return bS[a];var b=p("<"+a+">").appendTo(e.body),c=b.css("display");b.remove();if(c==="none"||c===""){bI=e.body.appendChild(bI||p.extend(e.createElement("iframe"),{frameBorder:0,width:0,height:0}));if(!bJ||!bI.createElement)bJ=(bI.contentWindow||bI.contentDocument).document,bJ.write("<!doctype html><html><body>"),bJ.close();b=bJ.body.appendChild(bJ.createElement(a)),c=bH(b,"display"),e.body.removeChild(bI)}return bS[a]=c,c}function ci(a,b,c,d){var e;if(p.isArray(b))p.each(b,function(b,e){c||ce.test(a)?d(a,e):ci(a+"["+(typeof e=="object"?b:"")+"]",e,c,d)});else if(!c&&p.type(b)==="object")for(e in b)ci(a+"["+e+"]",b[e],c,d);else d(a,b)}function cz(a){return function(b,c){typeof b!="string"&&(c=b,b="*");var d,e,f,g=b.toLowerCase().split(s),h=0,i=g.length;if(p.isFunction(c))for(;h<i;h++)d=g[h],f=/^\+/.test(d),f&&(d=d.substr(1)||"*"),e=a[d]=a[d]||[],e[f?"unshift":"push"](c)}}function cA(a,c,d,e,f,g){f=f||c.dataTypes[0],g=g||{},g[f]=!0;var h,i=a[f],j=0,k=i?i.length:0,l=a===cv;for(;j<k&&(l||!h);j++)h=i[j](c,d,e),typeof h=="string"&&(!l||g[h]?h=b:(c.dataTypes.unshift(h),h=cA(a,c,d,e,h,g)));return(l||!h)&&!g["*"]&&(h=cA(a,c,d,e,"*",g)),h}function cB(a,c){var d,e,f=p.ajaxSettings.flatOptions||{};for(d in c)c[d]!==b&&((f[d]?a:e||(e={}))[d]=c[d]);e&&p.extend(!0,a,e)}function cC(a,c,d){var e,f,g,h,i=a.contents,j=a.dataTypes,k=a.responseFields;for(f in k)f in d&&(c[k[f]]=d[f]);while(j[0]==="*")j.shift(),e===b&&(e=a.mimeType||c.getResponseHeader("content-type"));if(e)for(f in i)if(i[f]&&i[f].test(e)){j.unshift(f);break}if(j[0]in d)g=j[0];else{for(f in d){if(!j[0]||a.converters[f+" "+j[0]]){g=f;break}h||(h=f)}g=g||h}if(g)return g!==j[0]&&j.unshift(g),d[g]}function cD(a,b){var c,d,e,f,g=a.dataTypes.slice(),h=g[0],i={},j=0;a.dataFilter&&(b=a.dataFilter(b,a.dataType));if(g[1])for(c in a.converters)i[c.toLowerCase()]=a.converters[c];for(;e=g[++j];)if(e!=="*"){if(h!=="*"&&h!==e){c=i[h+" "+e]||i["* "+e];if(!c)for(d in i){f=d.split(" ");if(f[1]===e){c=i[h+" "+f[0]]||i["* "+f[0]];if(c){c===!0?c=i[d]:i[d]!==!0&&(e=f[0],g.splice(j--,0,e));break}}}if(c!==!0)if(c&&a["throws"])b=c(b);else try{b=c(b)}catch(k){return{state:"parsererror",error:c?k:"No conversion from "+h+" to "+e}}}h=e}return{state:"success",data:b}}function cL(){try{return new a.XMLHttpRequest}catch(b){}}function cM(){try{return new a.ActiveXObject("Microsoft.XMLHTTP")}catch(b){}}function cU(){return setTimeout(function(){cN=b},0),cN=p.now()}function cV(a,b){p.each(b,function(b,c){var d=(cT[b]||[]).concat(cT["*"]),e=0,f=d.length;for(;e<f;e++)if(d[e].call(a,b,c))return})}function cW(a,b,c){var d,e=0,f=0,g=cS.length,h=p.Deferred().always(function(){delete i.elem}),i=function(){var b=cN||cU(),c=Math.max(0,j.startTime+j.duration-b),d=1-(c/j.duration||0),e=0,f=j.tweens.length;for(;e<f;e++)j.tweens[e].run(d);return h.notifyWith(a,[j,d,c]),d<1&&f?c:(h.resolveWith(a,[j]),!1)},j=h.promise({elem:a,props:p.extend({},b),opts:p.extend(!0,{specialEasing:{}},c),originalProperties:b,originalOptions:c,startTime:cN||cU(),duration:c.duration,tweens:[],createTween:function(b,c,d){var e=p.Tween(a,j.opts,b,c,j.opts.specialEasing[b]||j.opts.easing);return j.tweens.push(e),e},stop:function(b){var c=0,d=b?j.tweens.length:0;for(;c<d;c++)j.tweens[c].run(1);return b?h.resolveWith(a,[j,b]):h.rejectWith(a,[j,b]),this}}),k=j.props;cX(k,j.opts.specialEasing);for(;e<g;e++){d=cS[e].call(j,a,k,j.opts);if(d)return d}return cV(j,k),p.isFunction(j.opts.start)&&j.opts.start.call(a,j),p.fx.timer(p.extend(i,{anim:j,queue:j.opts.queue,elem:a})),j.progress(j.opts.progress).done(j.opts.done,j.opts.complete).fail(j.opts.fail).always(j.opts.always)}function cX(a,b){var c,d,e,f,g;for(c in a){d=p.camelCase(c),e=b[d],f=a[c],p.isArray(f)&&(e=f[1],f=a[c]=f[0]),c!==d&&(a[d]=f,delete a[c]),g=p.cssHooks[d];if(g&&"expand"in g){f=g.expand(f),delete a[d];for(c in f)c in a||(a[c]=f[c],b[c]=e)}else b[d]=e}}function cY(a,b,c){var d,e,f,g,h,i,j,k,l=this,m=a.style,n={},o=[],q=a.nodeType&&bZ(a);c.queue||(j=p._queueHooks(a,"fx"),j.unqueued==null&&(j.unqueued=0,k=j.empty.fire,j.empty.fire=function(){j.unqueued||k()}),j.unqueued++,l.always(function(){l.always(function(){j.unqueued--,p.queue(a,"fx").length||j.empty.fire()})})),a.nodeType===1&&("height"in b||"width"in b)&&(c.overflow=[m.overflow,m.overflowX,m.overflowY],p.css(a,"display")==="inline"&&p.css(a,"float")==="none"&&(!p.support.inlineBlockNeedsLayout||cc(a.nodeName)==="inline"?m.display="inline-block":m.zoom=1)),c.overflow&&(m.overflow="hidden",p.support.shrinkWrapBlocks||l.done(function(){m.overflow=c.overflow[0],m.overflowX=c.overflow[1],m.overflowY=c.overflow[2]}));for(d in b){f=b[d];if(cP.exec(f)){delete b[d];if(f===(q?"hide":"show"))continue;o.push(d)}}g=o.length;if(g){h=p._data(a,"fxshow")||p._data(a,"fxshow",{}),q?p(a).show():l.done(function(){p(a).hide()}),l.done(function(){var b;p.removeData(a,"fxshow",!0);for(b in n)p.style(a,b,n[b])});for(d=0;d<g;d++)e=o[d],i=l.createTween(e,q?h[e]:0),n[e]=h[e]||p.style(a,e),e in h||(h[e]=i.start,q&&(i.end=i.start,i.start=e==="width"||e==="height"?1:0))}}function cZ(a,b,c,d,e){return new cZ.prototype.init(a,b,c,d,e)}function c$(a,b){var c,d={height:a},e=0;b=b?1:0;for(;e<4;e+=2-b)c=bV[e],d["margin"+c]=d["padding"+c]=a;return b&&(d.opacity=d.width=a),d}function da(a){return p.isWindow(a)?a:a.nodeType===9?a.defaultView||a.parentWindow:!1}var c,d,e=a.document,f=a.location,g=a.navigator,h=a.jQuery,i=a.$,j=Array.prototype.push,k=Array.prototype.slice,l=Array.prototype.indexOf,m=Object.prototype.toString,n=Object.prototype.hasOwnProperty,o=String.prototype.trim,p=function(a,b){return new p.fn.init(a,b,c)},q=/[\-+]?(?:\d*\.|)\d+(?:[eE][\-+]?\d+|)/.source,r=/\S/,s=/\s+/,t=/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g,u=/^(?:[^#<]*(<[\w\W]+>)[^>]*$|#([\w\-]*)$)/,v=/^<(\w+)\s*\/?>(?:<\/\1>|)$/,w=/^[\],:{}\s]*$/,x=/(?:^|:|,)(?:\s*\[)+/g,y=/\\(?:["\\\/bfnrt]|u[\da-fA-F]{4})/g,z=/"[^"\\\r\n]*"|true|false|null|-?(?:\d\d*\.|)\d+(?:[eE][\-+]?\d+|)/g,A=/^-ms-/,B=/-([\da-z])/gi,C=function(a,b){return(b+"").toUpperCase()},D=function(){e.addEventListener?(e.removeEventListener("DOMContentLoaded",D,!1),p.ready()):e.readyState==="complete"&&(e.detachEvent("onreadystatechange",D),p.ready())},E={};p.fn=p.prototype={constructor:p,init:function(a,c,d){var f,g,h,i;if(!a)return this;if(a.nodeType)return this.context=this[0]=a,this.length=1,this;if(typeof a=="string"){a.charAt(0)==="<"&&a.charAt(a.length-1)===">"&&a.length>=3?f=[null,a,null]:f=u.exec(a);if(f&&(f[1]||!c)){if(f[1])return c=c instanceof p?c[0]:c,i=c&&c.nodeType?c.ownerDocument||c:e,a=p.parseHTML(f[1],i,!0),v.test(f[1])&&p.isPlainObject(c)&&this.attr.call(a,c,!0),p.merge(this,a);g=e.getElementById(f[2]);if(g&&g.parentNode){if(g.id!==f[2])return d.find(a);this.length=1,this[0]=g}return this.context=e,this.selector=a,this}return!c||c.jquery?(c||d).find(a):this.constructor(c).find(a)}return p.isFunction(a)?d.ready(a):(a.selector!==b&&(this.selector=a.selector,this.context=a.context),p.makeArray(a,this))},selector:"",jquery:"1.8.2",length:0,size:function(){return this.length},toArray:function(){return k.call(this)},get:function(a){return a==null?this.toArray():a<0?this[this.length+a]:this[a]},pushStack:function(a,b,c){var d=p.merge(this.constructor(),a);return d.prevObject=this,d.context=this.context,b==="find"?d.selector=this.selector+(this.selector?" ":"")+c:b&&(d.selector=this.selector+"."+b+"("+c+")"),d},each:function(a,b){return p.each(this,a,b)},ready:function(a){return p.ready.promise().done(a),this},eq:function(a){return a=+a,a===-1?this.slice(a):this.slice(a,a+1)},first:function(){return this.eq(0)},last:function(){return this.eq(-1)},slice:function(){return this.pushStack(k.apply(this,arguments),"slice",k.call(arguments).join(","))},map:function(a){return this.pushStack(p.map(this,function(b,c){return a.call(b,c,b)}))},end:function(){return this.prevObject||this.constructor(null)},push:j,sort:[].sort,splice:[].splice},p.fn.init.prototype=p.fn,p.extend=p.fn.extend=function(){var a,c,d,e,f,g,h=arguments[0]||{},i=1,j=arguments.length,k=!1;typeof h=="boolean"&&(k=h,h=arguments[1]||{},i=2),typeof h!="object"&&!p.isFunction(h)&&(h={}),j===i&&(h=this,--i);for(;i<j;i++)if((a=arguments[i])!=null)for(c in a){d=h[c],e=a[c];if(h===e)continue;k&&e&&(p.isPlainObject(e)||(f=p.isArray(e)))?(f?(f=!1,g=d&&p.isArray(d)?d:[]):g=d&&p.isPlainObject(d)?d:{},h[c]=p.extend(k,g,e)):e!==b&&(h[c]=e)}return h},p.extend({noConflict:function(b){return a.$===p&&(a.$=i),b&&a.jQuery===p&&(a.jQuery=h),p},isReady:!1,readyWait:1,holdReady:function(a){a?p.readyWait++:p.ready(!0)},ready:function(a){if(a===!0?--p.readyWait:p.isReady)return;if(!e.body)return setTimeout(p.ready,1);p.isReady=!0;if(a!==!0&&--p.readyWait>0)return;d.resolveWith(e,[p]),p.fn.trigger&&p(e).trigger("ready").off("ready")},isFunction:function(a){return p.type(a)==="function"},isArray:Array.isArray||function(a){return p.type(a)==="array"},isWindow:function(a){return a!=null&&a==a.window},isNumeric:function(a){return!isNaN(parseFloat(a))&&isFinite(a)},type:function(a){return a==null?String(a):E[m.call(a)]||"object"},isPlainObject:function(a){if(!a||p.type(a)!=="object"||a.nodeType||p.isWindow(a))return!1;try{if(a.constructor&&!n.call(a,"constructor")&&!n.call(a.constructor.prototype,"isPrototypeOf"))return!1}catch(c){return!1}var d;for(d in a);return d===b||n.call(a,d)},isEmptyObject:function(a){var b;for(b in a)return!1;return!0},error:function(a){throw new Error(a)},parseHTML:function(a,b,c){var d;return!a||typeof a!="string"?null:(typeof b=="boolean"&&(c=b,b=0),b=b||e,(d=v.exec(a))?[b.createElement(d[1])]:(d=p.buildFragment([a],b,c?null:[]),p.merge([],(d.cacheable?p.clone(d.fragment):d.fragment).childNodes)))},parseJSON:function(b){if(!b||typeof b!="string")return null;b=p.trim(b);if(a.JSON&&a.JSON.parse)return a.JSON.parse(b);if(w.test(b.replace(y,"@").replace(z,"]").replace(x,"")))return(new Function("return "+b))();p.error("Invalid JSON: "+b)},parseXML:function(c){var d,e;if(!c||typeof c!="string")return null;try{a.DOMParser?(e=new DOMParser,d=e.parseFromString(c,"text/xml")):(d=new ActiveXObject("Microsoft.XMLDOM"),d.async="false",d.loadXML(c))}catch(f){d=b}return(!d||!d.documentElement||d.getElementsByTagName("parsererror").length)&&p.error("Invalid XML: "+c),d},noop:function(){},globalEval:function(b){b&&r.test(b)&&(a.execScript||function(b){a.eval.call(a,b)})(b)},camelCase:function(a){return a.replace(A,"ms-").replace(B,C)},nodeName:function(a,b){return a.nodeName&&a.nodeName.toLowerCase()===b.toLowerCase()},each:function(a,c,d){var e,f=0,g=a.length,h=g===b||p.isFunction(a);if(d){if(h){for(e in a)if(c.apply(a[e],d)===!1)break}else for(;f<g;)if(c.apply(a[f++],d)===!1)break}else if(h){for(e in a)if(c.call(a[e],e,a[e])===!1)break}else for(;f<g;)if(c.call(a[f],f,a[f++])===!1)break;return a},trim:o&&!o.call("﻿ ")?function(a){return a==null?"":o.call(a)}:function(a){return a==null?"":(a+"").replace(t,"")},makeArray:function(a,b){var c,d=b||[];return a!=null&&(c=p.type(a),a.length==null||c==="string"||c==="function"||c==="regexp"||p.isWindow(a)?j.call(d,a):p.merge(d,a)),d},inArray:function(a,b,c){var d;if(b){if(l)return l.call(b,a,c);d=b.length,c=c?c<0?Math.max(0,d+c):c:0;for(;c<d;c++)if(c in b&&b[c]===a)return c}return-1},merge:function(a,c){var d=c.length,e=a.length,f=0;if(typeof d=="number")for(;f<d;f++)a[e++]=c[f];else while(c[f]!==b)a[e++]=c[f++];return a.length=e,a},grep:function(a,b,c){var d,e=[],f=0,g=a.length;c=!!c;for(;f<g;f++)d=!!b(a[f],f),c!==d&&e.push(a[f]);return e},map:function(a,c,d){var e,f,g=[],h=0,i=a.length,j=a instanceof p||i!==b&&typeof i=="number"&&(i>0&&a[0]&&a[i-1]||i===0||p.isArray(a));if(j)for(;h<i;h++)e=c(a[h],h,d),e!=null&&(g[g.length]=e);else for(f in a)e=c(a[f],f,d),e!=null&&(g[g.length]=e);return g.concat.apply([],g)},guid:1,proxy:function(a,c){var d,e,f;return typeof c=="string"&&(d=a[c],c=a,a=d),p.isFunction(a)?(e=k.call(arguments,2),f=function(){return a.apply(c,e.concat(k.call(arguments)))},f.guid=a.guid=a.guid||p.guid++,f):b},access:function(a,c,d,e,f,g,h){var i,j=d==null,k=0,l=a.length;if(d&&typeof d=="object"){for(k in d)p.access(a,c,k,d[k],1,g,e);f=1}else if(e!==b){i=h===b&&p.isFunction(e),j&&(i?(i=c,c=function(a,b,c){return i.call(p(a),c)}):(c.call(a,e),c=null));if(c)for(;k<l;k++)c(a[k],d,i?e.call(a[k],k,c(a[k],d)):e,h);f=1}return f?a:j?c.call(a):l?c(a[0],d):g},now:function(){return(new Date).getTime()}}),p.ready.promise=function(b){if(!d){d=p.Deferred();if(e.readyState==="complete")setTimeout(p.ready,1);else if(e.addEventListener)e.addEventListener("DOMContentLoaded",D,!1),a.addEventListener("load",p.ready,!1);else{e.attachEvent("onreadystatechange",D),a.attachEvent("onload",p.ready);var c=!1;try{c=a.frameElement==null&&e.documentElement}catch(f){}c&&c.doScroll&&function g(){if(!p.isReady){try{c.doScroll("left")}catch(a){return setTimeout(g,50)}p.ready()}}()}}return d.promise(b)},p.each("Boolean Number String Function Array Date RegExp Object".split(" "),function(a,b){E["[object "+b+"]"]=b.toLowerCase()}),c=p(e);var F={};p.Callbacks=function(a){a=typeof a=="string"?F[a]||G(a):p.extend({},a);var c,d,e,f,g,h,i=[],j=!a.once&&[],k=function(b){c=a.memory&&b,d=!0,h=f||0,f=0,g=i.length,e=!0;for(;i&&h<g;h++)if(i[h].apply(b[0],b[1])===!1&&a.stopOnFalse){c=!1;break}e=!1,i&&(j?j.length&&k(j.shift()):c?i=[]:l.disable())},l={add:function(){if(i){var b=i.length;(function d(b){p.each(b,function(b,c){var e=p.type(c);e==="function"&&(!a.unique||!l.has(c))?i.push(c):c&&c.length&&e!=="string"&&d(c)})})(arguments),e?g=i.length:c&&(f=b,k(c))}return this},remove:function(){return i&&p.each(arguments,function(a,b){var c;while((c=p.inArray(b,i,c))>-1)i.splice(c,1),e&&(c<=g&&g--,c<=h&&h--)}),this},has:function(a){return p.inArray(a,i)>-1},empty:function(){return i=[],this},disable:function(){return i=j=c=b,this},disabled:function(){return!i},lock:function(){return j=b,c||l.disable(),this},locked:function(){return!j},fireWith:function(a,b){return b=b||[],b=[a,b.slice?b.slice():b],i&&(!d||j)&&(e?j.push(b):k(b)),this},fire:function(){return l.fireWith(this,arguments),this},fired:function(){return!!d}};return l},p.extend({Deferred:function(a){var b=[["resolve","done",p.Callbacks("once memory"),"resolved"],["reject","fail",p.Callbacks("once memory"),"rejected"],["notify","progress",p.Callbacks("memory")]],c="pending",d={state:function(){return c},always:function(){return e.done(arguments).fail(arguments),this},then:function(){var a=arguments;return p.Deferred(function(c){p.each(b,function(b,d){var f=d[0],g=a[b];e[d[1]](p.isFunction(g)?function(){var a=g.apply(this,arguments);a&&p.isFunction(a.promise)?a.promise().done(c.resolve).fail(c.reject).progress(c.notify):c[f+"With"](this===e?c:this,[a])}:c[f])}),a=null}).promise()},promise:function(a){return a!=null?p.extend(a,d):d}},e={};return d.pipe=d.then,p.each(b,function(a,f){var g=f[2],h=f[3];d[f[1]]=g.add,h&&g.add(function(){c=h},b[a^1][2].disable,b[2][2].lock),e[f[0]]=g.fire,e[f[0]+"With"]=g.fireWith}),d.promise(e),a&&a.call(e,e),e},when:function(a){var b=0,c=k.call(arguments),d=c.length,e=d!==1||a&&p.isFunction(a.promise)?d:0,f=e===1?a:p.Deferred(),g=function(a,b,c){return function(d){b[a]=this,c[a]=arguments.length>1?k.call(arguments):d,c===h?f.notifyWith(b,c):--e||f.resolveWith(b,c)}},h,i,j;if(d>1){h=new Array(d),i=new Array(d),j=new Array(d);for(;b<d;b++)c[b]&&p.isFunction(c[b].promise)?c[b].promise().done(g(b,j,c)).fail(f.reject).progress(g(b,i,h)):--e}return e||f.resolveWith(j,c),f.promise()}}),p.support=function(){var b,c,d,f,g,h,i,j,k,l,m,n=e.createElement("div");n.setAttribute("className","t"),n.innerHTML="  <link/><table></table><a href='/a'>a</a><input type='checkbox'/>",c=n.getElementsByTagName("*"),d=n.getElementsByTagName("a")[0],d.style.cssText="top:1px;float:left;opacity:.5";if(!c||!c.length)return{};f=e.createElement("select"),g=f.appendChild(e.createElement("option")),h=n.getElementsByTagName("input")[0],b={leadingWhitespace:n.firstChild.nodeType===3,tbody:!n.getElementsByTagName("tbody").length,htmlSerialize:!!n.getElementsByTagName("link").length,style:/top/.test(d.getAttribute("style")),hrefNormalized:d.getAttribute("href")==="/a",opacity:/^0.5/.test(d.style.opacity),cssFloat:!!d.style.cssFloat,checkOn:h.value==="on",optSelected:g.selected,getSetAttribute:n.className!=="t",enctype:!!e.createElement("form").enctype,html5Clone:e.createElement("nav").cloneNode(!0).outerHTML!=="<:nav></:nav>",boxModel:e.compatMode==="CSS1Compat",submitBubbles:!0,changeBubbles:!0,focusinBubbles:!1,deleteExpando:!0,noCloneEvent:!0,inlineBlockNeedsLayout:!1,shrinkWrapBlocks:!1,reliableMarginRight:!0,boxSizingReliable:!0,pixelPosition:!1},h.checked=!0,b.noCloneChecked=h.cloneNode(!0).checked,f.disabled=!0,b.optDisabled=!g.disabled;try{delete n.test}catch(o){b.deleteExpando=!1}!n.addEventListener&&n.attachEvent&&n.fireEvent&&(n.attachEvent("onclick",m=function(){b.noCloneEvent=!1}),n.cloneNode(!0).fireEvent("onclick"),n.detachEvent("onclick",m)),h=e.createElement("input"),h.value="t",h.setAttribute("type","radio"),b.radioValue=h.value==="t",h.setAttribute("checked","checked"),h.setAttribute("name","t"),n.appendChild(h),i=e.createDocumentFragment(),i.appendChild(n.lastChild),b.checkClone=i.cloneNode(!0).cloneNode(!0).lastChild.checked,b.appendChecked=h.checked,i.removeChild(h),i.appendChild(n);if(n.attachEvent)for(k in{submit:!0,change:!0,focusin:!0})j="on"+k,l=j in n,l||(n.setAttribute(j,"return;"),l=typeof n[j]=="function"),b[k+"Bubbles"]=l;return p(function(){var c,d,f,g,h="padding:0;margin:0;border:0;display:block;overflow:hidden;",i=e.getElementsByTagName("body")[0];if(!i)return;c=e.createElement("div"),c.style.cssText="visibility:hidden;border:0;width:0;height:0;position:static;top:0;margin-top:1px",i.insertBefore(c,i.firstChild),d=e.createElement("div"),c.appendChild(d),d.innerHTML="<table><tr><td></td><td>t</td></tr></table>",f=d.getElementsByTagName("td"),f[0].style.cssText="padding:0;margin:0;border:0;display:none",l=f[0].offsetHeight===0,f[0].style.display="",f[1].style.display="none",b.reliableHiddenOffsets=l&&f[0].offsetHeight===0,d.innerHTML="",d.style.cssText="box-sizing:border-box;-moz-box-sizing:border-box;-webkit-box-sizing:border-box;padding:1px;border:1px;display:block;width:4px;margin-top:1%;position:absolute;top:1%;",b.boxSizing=d.offsetWidth===4,b.doesNotIncludeMarginInBodyOffset=i.offsetTop!==1,a.getComputedStyle&&(b.pixelPosition=(a.getComputedStyle(d,null)||{}).top!=="1%",b.boxSizingReliable=(a.getComputedStyle(d,null)||{width:"4px"}).width==="4px",g=e.createElement("div"),g.style.cssText=d.style.cssText=h,g.style.marginRight=g.style.width="0",d.style.width="1px",d.appendChild(g),b.reliableMarginRight=!parseFloat((a.getComputedStyle(g,null)||{}).marginRight)),typeof d.style.zoom!="undefined"&&(d.innerHTML="",d.style.cssText=h+"width:1px;padding:1px;display:inline;zoom:1",b.inlineBlockNeedsLayout=d.offsetWidth===3,d.style.display="block",d.style.overflow="visible",d.innerHTML="<div></div>",d.firstChild.style.width="5px",b.shrinkWrapBlocks=d.offsetWidth!==3,c.style.zoom=1),i.removeChild(c),c=d=f=g=null}),i.removeChild(n),c=d=f=g=h=i=n=null,b}();var H=/(?:\{[\s\S]*\}|\[[\s\S]*\])$/,I=/([A-Z])/g;p.extend({cache:{},deletedIds:[],uuid:0,expando:"jQuery"+(p.fn.jquery+Math.random()).replace(/\D/g,""),noData:{embed:!0,object:"clsid:D27CDB6E-AE6D-11cf-96B8-444553540000",applet:!0},hasData:function(a){return a=a.nodeType?p.cache[a[p.expando]]:a[p.expando],!!a&&!K(a)},data:function(a,c,d,e){if(!p.acceptData(a))return;var f,g,h=p.expando,i=typeof c=="string",j=a.nodeType,k=j?p.cache:a,l=j?a[h]:a[h]&&h;if((!l||!k[l]||!e&&!k[l].data)&&i&&d===b)return;l||(j?a[h]=l=p.deletedIds.pop()||p.guid++:l=h),k[l]||(k[l]={},j||(k[l].toJSON=p.noop));if(typeof c=="object"||typeof c=="function")e?k[l]=p.extend(k[l],c):k[l].data=p.extend(k[l].data,c);return f=k[l],e||(f.data||(f.data={}),f=f.data),d!==b&&(f[p.camelCase(c)]=d),i?(g=f[c],g==null&&(g=f[p.camelCase(c)])):g=f,g},removeData:function(a,b,c){if(!p.acceptData(a))return;var d,e,f,g=a.nodeType,h=g?p.cache:a,i=g?a[p.expando]:p.expando;if(!h[i])return;if(b){d=c?h[i]:h[i].data;if(d){p.isArray(b)||(b in d?b=[b]:(b=p.camelCase(b),b in d?b=[b]:b=b.split(" ")));for(e=0,f=b.length;e<f;e++)delete d[b[e]];if(!(c?K:p.isEmptyObject)(d))return}}if(!c){delete h[i].data;if(!K(h[i]))return}g?p.cleanData([a],!0):p.support.deleteExpando||h!=h.window?delete h[i]:h[i]=null},_data:function(a,b,c){return p.data(a,b,c,!0)},acceptData:function(a){var b=a.nodeName&&p.noData[a.nodeName.toLowerCase()];return!b||b!==!0&&a.getAttribute("classid")===b}}),p.fn.extend({data:function(a,c){var d,e,f,g,h,i=this[0],j=0,k=null;if(a===b){if(this.length){k=p.data(i);if(i.nodeType===1&&!p._data(i,"parsedAttrs")){f=i.attributes;for(h=f.length;j<h;j++)g=f[j].name,g.indexOf("data-")||(g=p.camelCase(g.substring(5)),J(i,g,k[g]));p._data(i,"parsedAttrs",!0)}}return k}return typeof a=="object"?this.each(function(){p.data(this,a)}):(d=a.split(".",2),d[1]=d[1]?"."+d[1]:"",e=d[1]+"!",p.access(this,function(c){if(c===b)return k=this.triggerHandler("getData"+e,[d[0]]),k===b&&i&&(k=p.data(i,a),k=J(i,a,k)),k===b&&d[1]?this.data(d[0]):k;d[1]=c,this.each(function(){var b=p(this);b.triggerHandler("setData"+e,d),p.data(this,a,c),b.triggerHandler("changeData"+e,d)})},null,c,arguments.length>1,null,!1))},removeData:function(a){return this.each(function(){p.removeData(this,a)})}}),p.extend({queue:function(a,b,c){var d;if(a)return b=(b||"fx")+"queue",d=p._data(a,b),c&&(!d||p.isArray(c)?d=p._data(a,b,p.makeArray(c)):d.push(c)),d||[]},dequeue:function(a,b){b=b||"fx";var c=p.queue(a,b),d=c.length,e=c.shift(),f=p._queueHooks(a,b),g=function(){p.dequeue(a,b)};e==="inprogress"&&(e=c.shift(),d--),e&&(b==="fx"&&c.unshift("inprogress"),delete f.stop,e.call(a,g,f)),!d&&f&&f.empty.fire()},_queueHooks:function(a,b){var c=b+"queueHooks";return p._data(a,c)||p._data(a,c,{empty:p.Callbacks("once memory").add(function(){p.removeData(a,b+"queue",!0),p.removeData(a,c,!0)})})}}),p.fn.extend({queue:function(a,c){var d=2;return typeof a!="string"&&(c=a,a="fx",d--),arguments.length<d?p.queue(this[0],a):c===b?this:this.each(function(){var b=p.queue(this,a,c);p._queueHooks(this,a),a==="fx"&&b[0]!=="inprogress"&&p.dequeue(this,a)})},dequeue:function(a){return this.each(function(){p.dequeue(this,a)})},delay:function(a,b){return a=p.fx?p.fx.speeds[a]||a:a,b=b||"fx",this.queue(b,function(b,c){var d=setTimeout(b,a);c.stop=function(){clearTimeout(d)}})},clearQueue:function(a){return this.queue(a||"fx",[])},promise:function(a,c){var d,e=1,f=p.Deferred(),g=this,h=this.length,i=function(){--e||f.resolveWith(g,[g])};typeof a!="string"&&(c=a,a=b),a=a||"fx";while(h--)d=p._data(g[h],a+"queueHooks"),d&&d.empty&&(e++,d.empty.add(i));return i(),f.promise(c)}});var L,M,N,O=/[\t\r\n]/g,P=/\r/g,Q=/^(?:button|input)$/i,R=/^(?:button|input|object|select|textarea)$/i,S=/^a(?:rea|)$/i,T=/^(?:autofocus|autoplay|async|checked|controls|defer|disabled|hidden|loop|multiple|open|readonly|required|scoped|selected)$/i,U=p.support.getSetAttribute;p.fn.extend({attr:function(a,b){return p.access(this,p.attr,a,b,arguments.length>1)},removeAttr:function(a){return this.each(function(){p.removeAttr(this,a)})},prop:function(a,b){return p.access(this,p.prop,a,b,arguments.length>1)},removeProp:function(a){return a=p.propFix[a]||a,this.each(function(){try{this[a]=b,delete this[a]}catch(c){}})},addClass:function(a){var b,c,d,e,f,g,h;if(p.isFunction(a))return this.each(function(b){p(this).addClass(a.call(this,b,this.className))});if(a&&typeof a=="string"){b=a.split(s);for(c=0,d=this.length;c<d;c++){e=this[c];if(e.nodeType===1)if(!e.className&&b.length===1)e.className=a;else{f=" "+e.className+" ";for(g=0,h=b.length;g<h;g++)f.indexOf(" "+b[g]+" ")<0&&(f+=b[g]+" ");e.className=p.trim(f)}}}return this},removeClass:function(a){var c,d,e,f,g,h,i;if(p.isFunction(a))return this.each(function(b){p(this).removeClass(a.call(this,b,this.className))});if(a&&typeof a=="string"||a===b){c=(a||"").split(s);for(h=0,i=this.length;h<i;h++){e=this[h];if(e.nodeType===1&&e.className){d=(" "+e.className+" ").replace(O," ");for(f=0,g=c.length;f<g;f++)while(d.indexOf(" "+c[f]+" ")>=0)d=d.replace(" "+c[f]+" "," ");e.className=a?p.trim(d):""}}}return this},toggleClass:function(a,b){var c=typeof a,d=typeof b=="boolean";return p.isFunction(a)?this.each(function(c){p(this).toggleClass(a.call(this,c,this.className,b),b)}):this.each(function(){if(c==="string"){var e,f=0,g=p(this),h=b,i=a.split(s);while(e=i[f++])h=d?h:!g.hasClass(e),g[h?"addClass":"removeClass"](e)}else if(c==="undefined"||c==="boolean")this.className&&p._data(this,"__className__",this.className),this.className=this.className||a===!1?"":p._data(this,"__className__")||""})},hasClass:function(a){var b=" "+a+" ",c=0,d=this.length;for(;c<d;c++)if(this[c].nodeType===1&&(" "+this[c].className+" ").replace(O," ").indexOf(b)>=0)return!0;return!1},val:function(a){var c,d,e,f=this[0];if(!arguments.length){if(f)return c=p.valHooks[f.type]||p.valHooks[f.nodeName.toLowerCase()],c&&"get"in c&&(d=c.get(f,"value"))!==b?d:(d=f.value,typeof d=="string"?d.replace(P,""):d==null?"":d);return}return e=p.isFunction(a),this.each(function(d){var f,g=p(this);if(this.nodeType!==1)return;e?f=a.call(this,d,g.val()):f=a,f==null?f="":typeof f=="number"?f+="":p.isArray(f)&&(f=p.map(f,function(a){return a==null?"":a+""})),c=p.valHooks[this.type]||p.valHooks[this.nodeName.toLowerCase()];if(!c||!("set"in c)||c.set(this,f,"value")===b)this.value=f})}}),p.extend({valHooks:{option:{get:function(a){var b=a.attributes.value;return!b||b.specified?a.value:a.text}},select:{get:function(a){var b,c,d,e,f=a.selectedIndex,g=[],h=a.options,i=a.type==="select-one";if(f<0)return null;c=i?f:0,d=i?f+1:h.length;for(;c<d;c++){e=h[c];if(e.selected&&(p.support.optDisabled?!e.disabled:e.getAttribute("disabled")===null)&&(!e.parentNode.disabled||!p.nodeName(e.parentNode,"optgroup"))){b=p(e).val();if(i)return b;g.push(b)}}return i&&!g.length&&h.length?p(h[f]).val():g},set:function(a,b){var c=p.makeArray(b);return p(a).find("option").each(function(){this.selected=p.inArray(p(this).val(),c)>=0}),c.length||(a.selectedIndex=-1),c}}},attrFn:{},attr:function(a,c,d,e){var f,g,h,i=a.nodeType;if(!a||i===3||i===8||i===2)return;if(e&&p.isFunction(p.fn[c]))return p(a)[c](d);if(typeof a.getAttribute=="undefined")return p.prop(a,c,d);h=i!==1||!p.isXMLDoc(a),h&&(c=c.toLowerCase(),g=p.attrHooks[c]||(T.test(c)?M:L));if(d!==b){if(d===null){p.removeAttr(a,c);return}return g&&"set"in g&&h&&(f=g.set(a,d,c))!==b?f:(a.setAttribute(c,d+""),d)}return g&&"get"in g&&h&&(f=g.get(a,c))!==null?f:(f=a.getAttribute(c),f===null?b:f)},removeAttr:function(a,b){var c,d,e,f,g=0;if(b&&a.nodeType===1){d=b.split(s);for(;g<d.length;g++)e=d[g],e&&(c=p.propFix[e]||e,f=T.test(e),f||p.attr(a,e,""),a.removeAttribute(U?e:c),f&&c in a&&(a[c]=!1))}},attrHooks:{type:{set:function(a,b){if(Q.test(a.nodeName)&&a.parentNode)p.error("type property can't be changed");else if(!p.support.radioValue&&b==="radio"&&p.nodeName(a,"input")){var c=a.value;return a.setAttribute("type",b),c&&(a.value=c),b}}},value:{get:function(a,b){return L&&p.nodeName(a,"button")?L.get(a,b):b in a?a.value:null},set:function(a,b,c){if(L&&p.nodeName(a,"button"))return L.set(a,b,c);a.value=b}}},propFix:{tabindex:"tabIndex",readonly:"readOnly","for":"htmlFor","class":"className",maxlength:"maxLength",cellspacing:"cellSpacing",cellpadding:"cellPadding",rowspan:"rowSpan",colspan:"colSpan",usemap:"useMap",frameborder:"frameBorder",contenteditable:"contentEditable"},prop:function(a,c,d){var e,f,g,h=a.nodeType;if(!a||h===3||h===8||h===2)return;return g=h!==1||!p.isXMLDoc(a),g&&(c=p.propFix[c]||c,f=p.propHooks[c]),d!==b?f&&"set"in f&&(e=f.set(a,d,c))!==b?e:a[c]=d:f&&"get"in f&&(e=f.get(a,c))!==null?e:a[c]},propHooks:{tabIndex:{get:function(a){var c=a.getAttributeNode("tabindex");return c&&c.specified?parseInt(c.value,10):R.test(a.nodeName)||S.test(a.nodeName)&&a.href?0:b}}}}),M={get:function(a,c){var d,e=p.prop(a,c);return e===!0||typeof e!="boolean"&&(d=a.getAttributeNode(c))&&d.nodeValue!==!1?c.toLowerCase():b},set:function(a,b,c){var d;return b===!1?p.removeAttr(a,c):(d=p.propFix[c]||c,d in a&&(a[d]=!0),a.setAttribute(c,c.toLowerCase())),c}},U||(N={name:!0,id:!0,coords:!0},L=p.valHooks.button={get:function(a,c){var d;return d=a.getAttributeNode(c),d&&(N[c]?d.value!=="":d.specified)?d.value:b},set:function(a,b,c){var d=a.getAttributeNode(c);return d||(d=e.createAttribute(c),a.setAttributeNode(d)),d.value=b+""}},p.each(["width","height"],function(a,b){p.attrHooks[b]=p.extend(p.attrHooks[b],{set:function(a,c){if(c==="")return a.setAttribute(b,"auto"),c}})}),p.attrHooks.contenteditable={get:L.get,set:function(a,b,c){b===""&&(b="false"),L.set(a,b,c)}}),p.support.hrefNormalized||p.each(["href","src","width","height"],function(a,c){p.attrHooks[c]=p.extend(p.attrHooks[c],{get:function(a){var d=a.getAttribute(c,2);return d===null?b:d}})}),p.support.style||(p.attrHooks.style={get:function(a){return a.style.cssText.toLowerCase()||b},set:function(a,b){return a.style.cssText=b+""}}),p.support.optSelected||(p.propHooks.selected=p.extend(p.propHooks.selected,{get:function(a){var b=a.parentNode;return b&&(b.selectedIndex,b.parentNode&&b.parentNode.selectedIndex),null}})),p.support.enctype||(p.propFix.enctype="encoding"),p.support.checkOn||p.each(["radio","checkbox"],function(){p.valHooks[this]={get:function(a){return a.getAttribute("value")===null?"on":a.value}}}),p.each(["radio","checkbox"],function(){p.valHooks[this]=p.extend(p.valHooks[this],{set:function(a,b){if(p.isArray(b))return a.checked=p.inArray(p(a).val(),b)>=0}})});var V=/^(?:textarea|input|select)$/i,W=/^([^\.]*|)(?:\.(.+)|)$/,X=/(?:^|\s)hover(\.\S+|)\b/,Y=/^key/,Z=/^(?:mouse|contextmenu)|click/,$=/^(?:focusinfocus|focusoutblur)$/,_=function(a){return p.event.special.hover?a:a.replace(X,"mouseenter$1 mouseleave$1")};p.event={add:function(a,c,d,e,f){var g,h,i,j,k,l,m,n,o,q,r;if(a.nodeType===3||a.nodeType===8||!c||!d||!(g=p._data(a)))return;d.handler&&(o=d,d=o.handler,f=o.selector),d.guid||(d.guid=p.guid++),i=g.events,i||(g.events=i={}),h=g.handle,h||(g.handle=h=function(a){return typeof p!="undefined"&&(!a||p.event.triggered!==a.type)?p.event.dispatch.apply(h.elem,arguments):b},h.elem=a),c=p.trim(_(c)).split(" ");for(j=0;j<c.length;j++){k=W.exec(c[j])||[],l=k[1],m=(k[2]||"").split(".").sort(),r=p.event.special[l]||{},l=(f?r.delegateType:r.bindType)||l,r=p.event.special[l]||{},n=p.extend({type:l,origType:k[1],data:e,handler:d,guid:d.guid,selector:f,needsContext:f&&p.expr.match.needsContext.test(f),namespace:m.join(".")},o),q=i[l];if(!q){q=i[l]=[],q.delegateCount=0;if(!r.setup||r.setup.call(a,e,m,h)===!1)a.addEventListener?a.addEventListener(l,h,!1):a.attachEvent&&a.attachEvent("on"+l,h)}r.add&&(r.add.call(a,n),n.handler.guid||(n.handler.guid=d.guid)),f?q.splice(q.delegateCount++,0,n):q.push(n),p.event.global[l]=!0}a=null},global:{},remove:function(a,b,c,d,e){var f,g,h,i,j,k,l,m,n,o,q,r=p.hasData(a)&&p._data(a);if(!r||!(m=r.events))return;b=p.trim(_(b||"")).split(" ");for(f=0;f<b.length;f++){g=W.exec(b[f])||[],h=i=g[1],j=g[2];if(!h){for(h in m)p.event.remove(a,h+b[f],c,d,!0);continue}n=p.event.special[h]||{},h=(d?n.delegateType:n.bindType)||h,o=m[h]||[],k=o.length,j=j?new RegExp("(^|\\.)"+j.split(".").sort().join("\\.(?:.*\\.|)")+"(\\.|$)"):null;for(l=0;l<o.length;l++)q=o[l],(e||i===q.origType)&&(!c||c.guid===q.guid)&&(!j||j.test(q.namespace))&&(!d||d===q.selector||d==="**"&&q.selector)&&(o.splice(l--,1),q.selector&&o.delegateCount--,n.remove&&n.remove.call(a,q));o.length===0&&k!==o.length&&((!n.teardown||n.teardown.call(a,j,r.handle)===!1)&&p.removeEvent(a,h,r.handle),delete m[h])}p.isEmptyObject(m)&&(delete r.handle,p.removeData(a,"events",!0))},customEvent:{getData:!0,setData:!0,changeData:!0},trigger:function(c,d,f,g){if(!f||f.nodeType!==3&&f.nodeType!==8){var h,i,j,k,l,m,n,o,q,r,s=c.type||c,t=[];if($.test(s+p.event.triggered))return;s.indexOf("!")>=0&&(s=s.slice(0,-1),i=!0),s.indexOf(".")>=0&&(t=s.split("."),s=t.shift(),t.sort());if((!f||p.event.customEvent[s])&&!p.event.global[s])return;c=typeof c=="object"?c[p.expando]?c:new p.Event(s,c):new p.Event(s),c.type=s,c.isTrigger=!0,c.exclusive=i,c.namespace=t.join("."),c.namespace_re=c.namespace?new RegExp("(^|\\.)"+t.join("\\.(?:.*\\.|)")+"(\\.|$)"):null,m=s.indexOf(":")<0?"on"+s:"";if(!f){h=p.cache;for(j in h)h[j].events&&h[j].events[s]&&p.event.trigger(c,d,h[j].handle.elem,!0);return}c.result=b,c.target||(c.target=f),d=d!=null?p.makeArray(d):[],d.unshift(c),n=p.event.special[s]||{};if(n.trigger&&n.trigger.apply(f,d)===!1)return;q=[[f,n.bindType||s]];if(!g&&!n.noBubble&&!p.isWindow(f)){r=n.delegateType||s,k=$.test(r+s)?f:f.parentNode;for(l=f;k;k=k.parentNode)q.push([k,r]),l=k;l===(f.ownerDocument||e)&&q.push([l.defaultView||l.parentWindow||a,r])}for(j=0;j<q.length&&!c.isPropagationStopped();j++)k=q[j][0],c.type=q[j][1],o=(p._data(k,"events")||{})[c.type]&&p._data(k,"handle"),o&&o.apply(k,d),o=m&&k[m],o&&p.acceptData(k)&&o.apply&&o.apply(k,d)===!1&&c.preventDefault();return c.type=s,!g&&!c.isDefaultPrevented()&&(!n._default||n._default.apply(f.ownerDocument,d)===!1)&&(s!=="click"||!p.nodeName(f,"a"))&&p.acceptData(f)&&m&&f[s]&&(s!=="focus"&&s!=="blur"||c.target.offsetWidth!==0)&&!p.isWindow(f)&&(l=f[m],l&&(f[m]=null),p.event.triggered=s,f[s](),p.event.triggered=b,l&&(f[m]=l)),c.result}return},dispatch:function(c){c=p.event.fix(c||a.event);var d,e,f,g,h,i,j,l,m,n,o=(p._data(this,"events")||{})[c.type]||[],q=o.delegateCount,r=k.call(arguments),s=!c.exclusive&&!c.namespace,t=p.event.special[c.type]||{},u=[];r[0]=c,c.delegateTarget=this;if(t.preDispatch&&t.preDispatch.call(this,c)===!1)return;if(q&&(!c.button||c.type!=="click"))for(f=c.target;f!=this;f=f.parentNode||this)if(f.disabled!==!0||c.type!=="click"){h={},j=[];for(d=0;d<q;d++)l=o[d],m=l.selector,h[m]===b&&(h[m]=l.needsContext?p(m,this).index(f)>=0:p.find(m,this,null,[f]).length),h[m]&&j.push(l);j.length&&u.push({elem:f,matches:j})}o.length>q&&u.push({elem:this,matches:o.slice(q)});for(d=0;d<u.length&&!c.isPropagationStopped();d++){i=u[d],c.currentTarget=i.elem;for(e=0;e<i.matches.length&&!c.isImmediatePropagationStopped();e++){l=i.matches[e];if(s||!c.namespace&&!l.namespace||c.namespace_re&&c.namespace_re.test(l.namespace))c.data=l.data,c.handleObj=l,g=((p.event.special[l.origType]||{}).handle||l.handler).apply(i.elem,r),g!==b&&(c.result=g,g===!1&&(c.preventDefault(),c.stopPropagation()))}}return t.postDispatch&&t.postDispatch.call(this,c),c.result},props:"attrChange attrName relatedNode srcElement altKey bubbles cancelable ctrlKey currentTarget eventPhase metaKey relatedTarget shiftKey target timeStamp view which".split(" "),fixHooks:{},keyHooks:{props:"char charCode key keyCode".split(" "),filter:function(a,b){return a.which==null&&(a.which=b.charCode!=null?b.charCode:b.keyCode),a}},mouseHooks:{props:"button buttons clientX clientY fromElement offsetX offsetY pageX pageY screenX screenY toElement".split(" "),filter:function(a,c){var d,f,g,h=c.button,i=c.fromElement;return a.pageX==null&&c.clientX!=null&&(d=a.target.ownerDocument||e,f=d.documentElement,g=d.body,a.pageX=c.clientX+(f&&f.scrollLeft||g&&g.scrollLeft||0)-(f&&f.clientLeft||g&&g.clientLeft||0),a.pageY=c.clientY+(f&&f.scrollTop||g&&g.scrollTop||0)-(f&&f.clientTop||g&&g.clientTop||0)),!a.relatedTarget&&i&&(a.relatedTarget=i===a.target?c.toElement:i),!a.which&&h!==b&&(a.which=h&1?1:h&2?3:h&4?2:0),a}},fix:function(a){if(a[p.expando])return a;var b,c,d=a,f=p.event.fixHooks[a.type]||{},g=f.props?this.props.concat(f.props):this.props;a=p.Event(d);for(b=g.length;b;)c=g[--b],a[c]=d[c];return a.target||(a.target=d.srcElement||e),a.target.nodeType===3&&(a.target=a.target.parentNode),a.metaKey=!!a.metaKey,f.filter?f.filter(a,d):a},special:{load:{noBubble:!0},focus:{delegateType:"focusin"},blur:{delegateType:"focusout"},beforeunload:{setup:function(a,b,c){p.isWindow(this)&&(this.onbeforeunload=c)},teardown:function(a,b){this.onbeforeunload===b&&(this.onbeforeunload=null)}}},simulate:function(a,b,c,d){var e=p.extend(new p.Event,c,{type:a,isSimulated:!0,originalEvent:{}});d?p.event.trigger(e,null,b):p.event.dispatch.call(b,e),e.isDefaultPrevented()&&c.preventDefault()}},p.event.handle=p.event.dispatch,p.removeEvent=e.removeEventListener?function(a,b,c){a.removeEventListener&&a.removeEventListener(b,c,!1)}:function(a,b,c){var d="on"+b;a.detachEvent&&(typeof a[d]=="undefined"&&(a[d]=null),a.detachEvent(d,c))},p.Event=function(a,b){if(this instanceof p.Event)a&&a.type?(this.originalEvent=a,this.type=a.type,this.isDefaultPrevented=a.defaultPrevented||a.returnValue===!1||a.getPreventDefault&&a.getPreventDefault()?bb:ba):this.type=a,b&&p.extend(this,b),this.timeStamp=a&&a.timeStamp||p.now(),this[p.expando]=!0;else return new p.Event(a,b)},p.Event.prototype={preventDefault:function(){this.isDefaultPrevented=bb;var a=this.originalEvent;if(!a)return;a.preventDefault?a.preventDefault():a.returnValue=!1},stopPropagation:function(){this.isPropagationStopped=bb;var a=this.originalEvent;if(!a)return;a.stopPropagation&&a.stopPropagation(),a.cancelBubble=!0},stopImmediatePropagation:function(){this.isImmediatePropagationStopped=bb,this.stopPropagation()},isDefaultPrevented:ba,isPropagationStopped:ba,isImmediatePropagationStopped:ba},p.each({mouseenter:"mouseover",mouseleave:"mouseout"},function(a,b){p.event.special[a]={delegateType:b,bindType:b,handle:function(a){var c,d=this,e=a.relatedTarget,f=a.handleObj,g=f.selector;if(!e||e!==d&&!p.contains(d,e))a.type=f.origType,c=f.handler.apply(this,arguments),a.type=b;return c}}}),p.support.submitBubbles||(p.event.special.submit={setup:function(){if(p.nodeName(this,"form"))return!1;p.event.add(this,"click._submit keypress._submit",function(a){var c=a.target,d=p.nodeName(c,"input")||p.nodeName(c,"button")?c.form:b;d&&!p._data(d,"_submit_attached")&&(p.event.add(d,"submit._submit",function(a){a._submit_bubble=!0}),p._data(d,"_submit_attached",!0))})},postDispatch:function(a){a._submit_bubble&&(delete a._submit_bubble,this.parentNode&&!a.isTrigger&&p.event.simulate("submit",this.parentNode,a,!0))},teardown:function(){if(p.nodeName(this,"form"))return!1;p.event.remove(this,"._submit")}}),p.support.changeBubbles||(p.event.special.change={setup:function(){if(V.test(this.nodeName)){if(this.type==="checkbox"||this.type==="radio")p.event.add(this,"propertychange._change",function(a){a.originalEvent.propertyName==="checked"&&(this._just_changed=!0)}),p.event.add(this,"click._change",function(a){this._just_changed&&!a.isTrigger&&(this._just_changed=!1),p.event.simulate("change",this,a,!0)});return!1}p.event.add(this,"beforeactivate._change",function(a){var b=a.target;V.test(b.nodeName)&&!p._data(b,"_change_attached")&&(p.event.add(b,"change._change",function(a){this.parentNode&&!a.isSimulated&&!a.isTrigger&&p.event.simulate("change",this.parentNode,a,!0)}),p._data(b,"_change_attached",!0))})},handle:function(a){var b=a.target;if(this!==b||a.isSimulated||a.isTrigger||b.type!=="radio"&&b.type!=="checkbox")return a.handleObj.handler.apply(this,arguments)},teardown:function(){return p.event.remove(this,"._change"),!V.test(this.nodeName)}}),p.support.focusinBubbles||p.each({focus:"focusin",blur:"focusout"},function(a,b){var c=0,d=function(a){p.event.simulate(b,a.target,p.event.fix(a),!0)};p.event.special[b]={setup:function(){c++===0&&e.addEventListener(a,d,!0)},teardown:function(){--c===0&&e.removeEventListener(a,d,!0)}}}),p.fn.extend({on:function(a,c,d,e,f){var g,h;if(typeof a=="object"){typeof c!="string"&&(d=d||c,c=b);for(h in a)this.on(h,c,d,a[h],f);return this}d==null&&e==null?(e=c,d=c=b):e==null&&(typeof c=="string"?(e=d,d=b):(e=d,d=c,c=b));if(e===!1)e=ba;else if(!e)return this;return f===1&&(g=e,e=function(a){return p().off(a),g.apply(this,arguments)},e.guid=g.guid||(g.guid=p.guid++)),this.each(function(){p.event.add(this,a,e,d,c)})},one:function(a,b,c,d){return this.on(a,b,c,d,1)},off:function(a,c,d){var e,f;if(a&&a.preventDefault&&a.handleObj)return e=a.handleObj,p(a.delegateTarget).off(e.namespace?e.origType+"."+e.namespace:e.origType,e.selector,e.handler),this;if(typeof a=="object"){for(f in a)this.off(f,c,a[f]);return this}if(c===!1||typeof c=="function")d=c,c=b;return d===!1&&(d=ba),this.each(function(){p.event.remove(this,a,d,c)})},bind:function(a,b,c){return this.on(a,null,b,c)},unbind:function(a,b){return this.off(a,null,b)},live:function(a,b,c){return p(this.context).on(a,this.selector,b,c),this},die:function(a,b){return p(this.context).off(a,this.selector||"**",b),this},delegate:function(a,b,c,d){return this.on(b,a,c,d)},undelegate:function(a,b,c){return arguments.length===1?this.off(a,"**"):this.off(b,a||"**",c)},trigger:function(a,b){return this.each(function(){p.event.trigger(a,b,this)})},triggerHandler:function(a,b){if(this[0])return p.event.trigger(a,b,this[0],!0)},toggle:function(a){var b=arguments,c=a.guid||p.guid++,d=0,e=function(c){var e=(p._data(this,"lastToggle"+a.guid)||0)%d;return p._data(this,"lastToggle"+a.guid,e+1),c.preventDefault(),b[e].apply(this,arguments)||!1};e.guid=c;while(d<b.length)b[d++].guid=c;return this.click(e)},hover:function(a,b){return this.mouseenter(a).mouseleave(b||a)}}),p.each("blur focus focusin focusout load resize scroll unload click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup error contextmenu".split(" "),function(a,b){p.fn[b]=function(a,c){return c==null&&(c=a,a=null),arguments.length>0?this.on(b,null,a,c):this.trigger(b)},Y.test(b)&&(p.event.fixHooks[b]=p.event.keyHooks),Z.test(b)&&(p.event.fixHooks[b]=p.event.mouseHooks)}),function(a,b){function bc(a,b,c,d){c=c||[],b=b||r;var e,f,i,j,k=b.nodeType;if(!a||typeof a!="string")return c;if(k!==1&&k!==9)return[];i=g(b);if(!i&&!d)if(e=P.exec(a))if(j=e[1]){if(k===9){f=b.getElementById(j);if(!f||!f.parentNode)return c;if(f.id===j)return c.push(f),c}else if(b.ownerDocument&&(f=b.ownerDocument.getElementById(j))&&h(b,f)&&f.id===j)return c.push(f),c}else{if(e[2])return w.apply(c,x.call(b.getElementsByTagName(a),0)),c;if((j=e[3])&&_&&b.getElementsByClassName)return w.apply(c,x.call(b.getElementsByClassName(j),0)),c}return bp(a.replace(L,"$1"),b,c,d,i)}function bd(a){return function(b){var c=b.nodeName.toLowerCase();return c==="input"&&b.type===a}}function be(a){return function(b){var c=b.nodeName.toLowerCase();return(c==="input"||c==="button")&&b.type===a}}function bf(a){return z(function(b){return b=+b,z(function(c,d){var e,f=a([],c.length,b),g=f.length;while(g--)c[e=f[g]]&&(c[e]=!(d[e]=c[e]))})})}function bg(a,b,c){if(a===b)return c;var d=a.nextSibling;while(d){if(d===b)return-1;d=d.nextSibling}return 1}function bh(a,b){var c,d,f,g,h,i,j,k=C[o][a];if(k)return b?0:k.slice(0);h=a,i=[],j=e.preFilter;while(h){if(!c||(d=M.exec(h)))d&&(h=h.slice(d[0].length)),i.push(f=[]);c=!1;if(d=N.exec(h))f.push(c=new q(d.shift())),h=h.slice(c.length),c.type=d[0].replace(L," ");for(g in e.filter)(d=W[g].exec(h))&&(!j[g]||(d=j[g](d,r,!0)))&&(f.push(c=new q(d.shift())),h=h.slice(c.length),c.type=g,c.matches=d);if(!c)break}return b?h.length:h?bc.error(a):C(a,i).slice(0)}function bi(a,b,d){var e=b.dir,f=d&&b.dir==="parentNode",g=u++;return b.first?function(b,c,d){while(b=b[e])if(f||b.nodeType===1)return a(b,c,d)}:function(b,d,h){if(!h){var i,j=t+" "+g+" ",k=j+c;while(b=b[e])if(f||b.nodeType===1){if((i=b[o])===k)return b.sizset;if(typeof i=="string"&&i.indexOf(j)===0){if(b.sizset)return b}else{b[o]=k;if(a(b,d,h))return b.sizset=!0,b;b.sizset=!1}}}else while(b=b[e])if(f||b.nodeType===1)if(a(b,d,h))return b}}function bj(a){return a.length>1?function(b,c,d){var e=a.length;while(e--)if(!a[e](b,c,d))return!1;return!0}:a[0]}function bk(a,b,c,d,e){var f,g=[],h=0,i=a.length,j=b!=null;for(;h<i;h++)if(f=a[h])if(!c||c(f,d,e))g.push(f),j&&b.push(h);return g}function bl(a,b,c,d,e,f){return d&&!d[o]&&(d=bl(d)),e&&!e[o]&&(e=bl(e,f)),z(function(f,g,h,i){if(f&&e)return;var j,k,l,m=[],n=[],o=g.length,p=f||bo(b||"*",h.nodeType?[h]:h,[],f),q=a&&(f||!b)?bk(p,m,a,h,i):p,r=c?e||(f?a:o||d)?[]:g:q;c&&c(q,r,h,i);if(d){l=bk(r,n),d(l,[],h,i),j=l.length;while(j--)if(k=l[j])r[n[j]]=!(q[n[j]]=k)}if(f){j=a&&r.length;while(j--)if(k=r[j])f[m[j]]=!(g[m[j]]=k)}else r=bk(r===g?r.splice(o,r.length):r),e?e(null,g,r,i):w.apply(g,r)})}function bm(a){var b,c,d,f=a.length,g=e.relative[a[0].type],h=g||e.relative[" "],i=g?1:0,j=bi(function(a){return a===b},h,!0),k=bi(function(a){return y.call(b,a)>-1},h,!0),m=[function(a,c,d){return!g&&(d||c!==l)||((b=c).nodeType?j(a,c,d):k(a,c,d))}];for(;i<f;i++)if(c=e.relative[a[i].type])m=[bi(bj(m),c)];else{c=e.filter[a[i].type].apply(null,a[i].matches);if(c[o]){d=++i;for(;d<f;d++)if(e.relative[a[d].type])break;return bl(i>1&&bj(m),i>1&&a.slice(0,i-1).join("").replace(L,"$1"),c,i<d&&bm(a.slice(i,d)),d<f&&bm(a=a.slice(d)),d<f&&a.join(""))}m.push(c)}return bj(m)}function bn(a,b){var d=b.length>0,f=a.length>0,g=function(h,i,j,k,m){var n,o,p,q=[],s=0,u="0",x=h&&[],y=m!=null,z=l,A=h||f&&e.find.TAG("*",m&&i.parentNode||i),B=t+=z==null?1:Math.E;y&&(l=i!==r&&i,c=g.el);for(;(n=A[u])!=null;u++){if(f&&n){for(o=0;p=a[o];o++)if(p(n,i,j)){k.push(n);break}y&&(t=B,c=++g.el)}d&&((n=!p&&n)&&s--,h&&x.push(n))}s+=u;if(d&&u!==s){for(o=0;p=b[o];o++)p(x,q,i,j);if(h){if(s>0)while(u--)!x[u]&&!q[u]&&(q[u]=v.call(k));q=bk(q)}w.apply(k,q),y&&!h&&q.length>0&&s+b.length>1&&bc.uniqueSort(k)}return y&&(t=B,l=z),x};return g.el=0,d?z(g):g}function bo(a,b,c,d){var e=0,f=b.length;for(;e<f;e++)bc(a,b[e],c,d);return c}function bp(a,b,c,d,f){var g,h,j,k,l,m=bh(a),n=m.length;if(!d&&m.length===1){h=m[0]=m[0].slice(0);if(h.length>2&&(j=h[0]).type==="ID"&&b.nodeType===9&&!f&&e.relative[h[1].type]){b=e.find.ID(j.matches[0].replace(V,""),b,f)[0];if(!b)return c;a=a.slice(h.shift().length)}for(g=W.POS.test(a)?-1:h.length-1;g>=0;g--){j=h[g];if(e.relative[k=j.type])break;if(l=e.find[k])if(d=l(j.matches[0].replace(V,""),R.test(h[0].type)&&b.parentNode||b,f)){h.splice(g,1),a=d.length&&h.join("");if(!a)return w.apply(c,x.call(d,0)),c;break}}}return i(a,m)(d,b,f,c,R.test(a)),c}function bq(){}var c,d,e,f,g,h,i,j,k,l,m=!0,n="undefined",o=("sizcache"+Math.random()).replace(".",""),q=String,r=a.document,s=r.documentElement,t=0,u=0,v=[].pop,w=[].push,x=[].slice,y=[].indexOf||function(a){var b=0,c=this.length;for(;b<c;b++)if(this[b]===a)return b;return-1},z=function(a,b){return a[o]=b==null||b,a},A=function(){var a={},b=[];return z(function(c,d){return b.push(c)>e.cacheLength&&delete a[b.shift()],a[c]=d},a)},B=A(),C=A(),D=A(),E="[\\x20\\t\\r\\n\\f]",F="(?:\\\\.|[-\\w]|[^\\x00-\\xa0])+",G=F.replace("w","w#"),H="([*^$|!~]?=)",I="\\["+E+"*("+F+")"+E+"*(?:"+H+E+"*(?:(['\"])((?:\\\\.|[^\\\\])*?)\\3|("+G+")|)|)"+E+"*\\]",J=":("+F+")(?:\\((?:(['\"])((?:\\\\.|[^\\\\])*?)\\2|([^()[\\]]*|(?:(?:"+I+")|[^:]|\\\\.)*|.*))\\)|)",K=":(even|odd|eq|gt|lt|nth|first|last)(?:\\("+E+"*((?:-\\d)?\\d*)"+E+"*\\)|)(?=[^-]|$)",L=new RegExp("^"+E+"+|((?:^|[^\\\\])(?:\\\\.)*)"+E+"+$","g"),M=new RegExp("^"+E+"*,"+E+"*"),N=new RegExp("^"+E+"*([\\x20\\t\\r\\n\\f>+~])"+E+"*"),O=new RegExp(J),P=/^(?:#([\w\-]+)|(\w+)|\.([\w\-]+))$/,Q=/^:not/,R=/[\x20\t\r\n\f]*[+~]/,S=/:not\($/,T=/h\d/i,U=/input|select|textarea|button/i,V=/\\(?!\\)/g,W={ID:new RegExp("^#("+F+")"),CLASS:new RegExp("^\\.("+F+")"),NAME:new RegExp("^\\[name=['\"]?("+F+")['\"]?\\]"),TAG:new RegExp("^("+F.replace("w","w*")+")"),ATTR:new RegExp("^"+I),PSEUDO:new RegExp("^"+J),POS:new RegExp(K,"i"),CHILD:new RegExp("^:(only|nth|first|last)-child(?:\\("+E+"*(even|odd|(([+-]|)(\\d*)n|)"+E+"*(?:([+-]|)"+E+"*(\\d+)|))"+E+"*\\)|)","i"),needsContext:new RegExp("^"+E+"*[>+~]|"+K,"i")},X=function(a){var b=r.createElement("div");try{return a(b)}catch(c){return!1}finally{b=null}},Y=X(function(a){return a.appendChild(r.createComment("")),!a.getElementsByTagName("*").length}),Z=X(function(a){return a.innerHTML="<a href='#'></a>",a.firstChild&&typeof a.firstChild.getAttribute!==n&&a.firstChild.getAttribute("href")==="#"}),$=X(function(a){a.innerHTML="<select></select>";var b=typeof a.lastChild.getAttribute("multiple");return b!=="boolean"&&b!=="string"}),_=X(function(a){return a.innerHTML="<div class='hidden e'></div><div class='hidden'></div>",!a.getElementsByClassName||!a.getElementsByClassName("e").length?!1:(a.lastChild.className="e",a.getElementsByClassName("e").length===2)}),ba=X(function(a){a.id=o+0,a.innerHTML="<a name='"+o+"'></a><div name='"+o+"'></div>",s.insertBefore(a,s.firstChild);var b=r.getElementsByName&&r.getElementsByName(o).length===2+r.getElementsByName(o+0).length;return d=!r.getElementById(o),s.removeChild(a),b});try{x.call(s.childNodes,0)[0].nodeType}catch(bb){x=function(a){var b,c=[];for(;b=this[a];a++)c.push(b);return c}}bc.matches=function(a,b){return bc(a,null,null,b)},bc.matchesSelector=function(a,b){return bc(b,null,null,[a]).length>0},f=bc.getText=function(a){var b,c="",d=0,e=a.nodeType;if(e){if(e===1||e===9||e===11){if(typeof a.textContent=="string")return a.textContent;for(a=a.firstChild;a;a=a.nextSibling)c+=f(a)}else if(e===3||e===4)return a.nodeValue}else for(;b=a[d];d++)c+=f(b);return c},g=bc.isXML=function(a){var b=a&&(a.ownerDocument||a).documentElement;return b?b.nodeName!=="HTML":!1},h=bc.contains=s.contains?function(a,b){var c=a.nodeType===9?a.documentElement:a,d=b&&b.parentNode;return a===d||!!(d&&d.nodeType===1&&c.contains&&c.contains(d))}:s.compareDocumentPosition?function(a,b){return b&&!!(a.compareDocumentPosition(b)&16)}:function(a,b){while(b=b.parentNode)if(b===a)return!0;return!1},bc.attr=function(a,b){var c,d=g(a);return d||(b=b.toLowerCase()),(c=e.attrHandle[b])?c(a):d||$?a.getAttribute(b):(c=a.getAttributeNode(b),c?typeof a[b]=="boolean"?a[b]?b:null:c.specified?c.value:null:null)},e=bc.selectors={cacheLength:50,createPseudo:z,match:W,attrHandle:Z?{}:{href:function(a){return a.getAttribute("href",2)},type:function(a){return a.getAttribute("type")}},find:{ID:d?function(a,b,c){if(typeof b.getElementById!==n&&!c){var d=b.getElementById(a);return d&&d.parentNode?[d]:[]}}:function(a,c,d){if(typeof c.getElementById!==n&&!d){var e=c.getElementById(a);return e?e.id===a||typeof e.getAttributeNode!==n&&e.getAttributeNode("id").value===a?[e]:b:[]}},TAG:Y?function(a,b){if(typeof b.getElementsByTagName!==n)return b.getElementsByTagName(a)}:function(a,b){var c=b.getElementsByTagName(a);if(a==="*"){var d,e=[],f=0;for(;d=c[f];f++)d.nodeType===1&&e.push(d);return e}return c},NAME:ba&&function(a,b){if(typeof b.getElementsByName!==n)return b.getElementsByName(name)},CLASS:_&&function(a,b,c){if(typeof b.getElementsByClassName!==n&&!c)return b.getElementsByClassName(a)}},relative:{">":{dir:"parentNode",first:!0}," ":{dir:"parentNode"},"+":{dir:"previousSibling",first:!0},"~":{dir:"previousSibling"}},preFilter:{ATTR:function(a){return a[1]=a[1].replace(V,""),a[3]=(a[4]||a[5]||"").replace(V,""),a[2]==="~="&&(a[3]=" "+a[3]+" "),a.slice(0,4)},CHILD:function(a){return a[1]=a[1].toLowerCase(),a[1]==="nth"?(a[2]||bc.error(a[0]),a[3]=+(a[3]?a[4]+(a[5]||1):2*(a[2]==="even"||a[2]==="odd")),a[4]=+(a[6]+a[7]||a[2]==="odd")):a[2]&&bc.error(a[0]),a},PSEUDO:function(a){var b,c;if(W.CHILD.test(a[0]))return null;if(a[3])a[2]=a[3];else if(b=a[4])O.test(b)&&(c=bh(b,!0))&&(c=b.indexOf(")",b.length-c)-b.length)&&(b=b.slice(0,c),a[0]=a[0].slice(0,c)),a[2]=b;return a.slice(0,3)}},filter:{ID:d?function(a){return a=a.replace(V,""),function(b){return b.getAttribute("id")===a}}:function(a){return a=a.replace(V,""),function(b){var c=typeof b.getAttributeNode!==n&&b.getAttributeNode("id");return c&&c.value===a}},TAG:function(a){return a==="*"?function(){return!0}:(a=a.replace(V,"").toLowerCase(),function(b){return b.nodeName&&b.nodeName.toLowerCase()===a})},CLASS:function(a){var b=B[o][a];return b||(b=B(a,new RegExp("(^|"+E+")"+a+"("+E+"|$)"))),function(a){return b.test(a.className||typeof a.getAttribute!==n&&a.getAttribute("class")||"")}},ATTR:function(a,b,c){return function(d,e){var f=bc.attr(d,a);return f==null?b==="!=":b?(f+="",b==="="?f===c:b==="!="?f!==c:b==="^="?c&&f.indexOf(c)===0:b==="*="?c&&f.indexOf(c)>-1:b==="$="?c&&f.substr(f.length-c.length)===c:b==="~="?(" "+f+" ").indexOf(c)>-1:b==="|="?f===c||f.substr(0,c.length+1)===c+"-":!1):!0}},CHILD:function(a,b,c,d){return a==="nth"?function(a){var b,e,f=a.parentNode;if(c===1&&d===0)return!0;if(f){e=0;for(b=f.firstChild;b;b=b.nextSibling)if(b.nodeType===1){e++;if(a===b)break}}return e-=d,e===c||e%c===0&&e/c>=0}:function(b){var c=b;switch(a){case"only":case"first":while(c=c.previousSibling)if(c.nodeType===1)return!1;if(a==="first")return!0;c=b;case"last":while(c=c.nextSibling)if(c.nodeType===1)return!1;return!0}}},PSEUDO:function(a,b){var c,d=e.pseudos[a]||e.setFilters[a.toLowerCase()]||bc.error("unsupported pseudo: "+a);return d[o]?d(b):d.length>1?(c=[a,a,"",b],e.setFilters.hasOwnProperty(a.toLowerCase())?z(function(a,c){var e,f=d(a,b),g=f.length;while(g--)e=y.call(a,f[g]),a[e]=!(c[e]=f[g])}):function(a){return d(a,0,c)}):d}},pseudos:{not:z(function(a){var b=[],c=[],d=i(a.replace(L,"$1"));return d[o]?z(function(a,b,c,e){var f,g=d(a,null,e,[]),h=a.length;while(h--)if(f=g[h])a[h]=!(b[h]=f)}):function(a,e,f){return b[0]=a,d(b,null,f,c),!c.pop()}}),has:z(function(a){return function(b){return bc(a,b).length>0}}),contains:z(function(a){return function(b){return(b.textContent||b.innerText||f(b)).indexOf(a)>-1}}),enabled:function(a){return a.disabled===!1},disabled:function(a){return a.disabled===!0},checked:function(a){var b=a.nodeName.toLowerCase();return b==="input"&&!!a.checked||b==="option"&&!!a.selected},selected:function(a){return a.parentNode&&a.parentNode.selectedIndex,a.selected===!0},parent:function(a){return!e.pseudos.empty(a)},empty:function(a){var b;a=a.firstChild;while(a){if(a.nodeName>"@"||(b=a.nodeType)===3||b===4)return!1;a=a.nextSibling}return!0},header:function(a){return T.test(a.nodeName)},text:function(a){var b,c;return a.nodeName.toLowerCase()==="input"&&(b=a.type)==="text"&&((c=a.getAttribute("type"))==null||c.toLowerCase()===b)},radio:bd("radio"),checkbox:bd("checkbox"),file:bd("file"),password:bd("password"),image:bd("image"),submit:be("submit"),reset:be("reset"),button:function(a){var b=a.nodeName.toLowerCase();return b==="input"&&a.type==="button"||b==="button"},input:function(a){return U.test(a.nodeName)},focus:function(a){var b=a.ownerDocument;return a===b.activeElement&&(!b.hasFocus||b.hasFocus())&&(!!a.type||!!a.href)},active:function(a){return a===a.ownerDocument.activeElement},first:bf(function(a,b,c){return[0]}),last:bf(function(a,b,c){return[b-1]}),eq:bf(function(a,b,c){return[c<0?c+b:c]}),even:bf(function(a,b,c){for(var d=0;d<b;d+=2)a.push(d);return a}),odd:bf(function(a,b,c){for(var d=1;d<b;d+=2)a.push(d);return a}),lt:bf(function(a,b,c){for(var d=c<0?c+b:c;--d>=0;)a.push(d);return a}),gt:bf(function(a,b,c){for(var d=c<0?c+b:c;++d<b;)a.push(d);return a})}},j=s.compareDocumentPosition?function(a,b){return a===b?(k=!0,0):(!a.compareDocumentPosition||!b.compareDocumentPosition?a.compareDocumentPosition:a.compareDocumentPosition(b)&4)?-1:1}:function(a,b){if(a===b)return k=!0,0;if(a.sourceIndex&&b.sourceIndex)return a.sourceIndex-b.sourceIndex;var c,d,e=[],f=[],g=a.parentNode,h=b.parentNode,i=g;if(g===h)return bg(a,b);if(!g)return-1;if(!h)return 1;while(i)e.unshift(i),i=i.parentNode;i=h;while(i)f.unshift(i),i=i.parentNode;c=e.length,d=f.length;for(var j=0;j<c&&j<d;j++)if(e[j]!==f[j])return bg(e[j],f[j]);return j===c?bg(a,f[j],-1):bg(e[j],b,1)},[0,0].sort(j),m=!k,bc.uniqueSort=function(a){var b,c=1;k=m,a.sort(j);if(k)for(;b=a[c];c++)b===a[c-1]&&a.splice(c--,1);return a},bc.error=function(a){throw new Error("Syntax error, unrecognized expression: "+a)},i=bc.compile=function(a,b){var c,d=[],e=[],f=D[o][a];if(!f){b||(b=bh(a)),c=b.length;while(c--)f=bm(b[c]),f[o]?d.push(f):e.push(f);f=D(a,bn(e,d))}return f},r.querySelectorAll&&function(){var a,b=bp,c=/'|\\/g,d=/\=[\x20\t\r\n\f]*([^'"\]]*)[\x20\t\r\n\f]*\]/g,e=[":focus"],f=[":active",":focus"],h=s.matchesSelector||s.mozMatchesSelector||s.webkitMatchesSelector||s.oMatchesSelector||s.msMatchesSelector;X(function(a){a.innerHTML="<select><option selected=''></option></select>",a.querySelectorAll("[selected]").length||e.push("\\["+E+"*(?:checked|disabled|ismap|multiple|readonly|selected|value)"),a.querySelectorAll(":checked").length||e.push(":checked")}),X(function(a){a.innerHTML="<p test=''></p>",a.querySelectorAll("[test^='']").length&&e.push("[*^$]="+E+"*(?:\"\"|'')"),a.innerHTML="<input type='hidden'/>",a.querySelectorAll(":enabled").length||e.push(":enabled",":disabled")}),e=new RegExp(e.join("|")),bp=function(a,d,f,g,h){if(!g&&!h&&(!e||!e.test(a))){var i,j,k=!0,l=o,m=d,n=d.nodeType===9&&a;if(d.nodeType===1&&d.nodeName.toLowerCase()!=="object"){i=bh(a),(k=d.getAttribute("id"))?l=k.replace(c,"\\$&"):d.setAttribute("id",l),l="[id='"+l+"'] ",j=i.length;while(j--)i[j]=l+i[j].join("");m=R.test(a)&&d.parentNode||d,n=i.join(",")}if(n)try{return w.apply(f,x.call(m.querySelectorAll(n),0)),f}catch(p){}finally{k||d.removeAttribute("id")}}return b(a,d,f,g,h)},h&&(X(function(b){a=h.call(b,"div");try{h.call(b,"[test!='']:sizzle"),f.push("!=",J)}catch(c){}}),f=new RegExp(f.join("|")),bc.matchesSelector=function(b,c){c=c.replace(d,"='$1']");if(!g(b)&&!f.test(c)&&(!e||!e.test(c)))try{var i=h.call(b,c);if(i||a||b.document&&b.document.nodeType!==11)return i}catch(j){}return bc(c,null,null,[b]).length>0})}(),e.pseudos.nth=e.pseudos.eq,e.filters=bq.prototype=e.pseudos,e.setFilters=new bq,bc.attr=p.attr,p.find=bc,p.expr=bc.selectors,p.expr[":"]=p.expr.pseudos,p.unique=bc.uniqueSort,p.text=bc.getText,p.isXMLDoc=bc.isXML,p.contains=bc.contains}(a);var bc=/Until$/,bd=/^(?:parents|prev(?:Until|All))/,be=/^.[^:#\[\.,]*$/,bf=p.expr.match.needsContext,bg={children:!0,contents:!0,next:!0,prev:!0};p.fn.extend({find:function(a){var b,c,d,e,f,g,h=this;if(typeof a!="string")return p(a).filter(function(){for(b=0,c=h.length;b<c;b++)if(p.contains(h[b],this))return!0});g=this.pushStack("","find",a);for(b=0,c=this.length;b<c;b++){d=g.length,p.find(a,this[b],g);if(b>0)for(e=d;e<g.length;e++)for(f=0;f<d;f++)if(g[f]===g[e]){g.splice(e--,1);break}}return g},has:function(a){var b,c=p(a,this),d=c.length;return this.filter(function(){for(b=0;b<d;b++)if(p.contains(this,c[b]))return!0})},not:function(a){return this.pushStack(bj(this,a,!1),"not",a)},filter:function(a){return this.pushStack(bj(this,a,!0),"filter",a)},is:function(a){return!!a&&(typeof a=="string"?bf.test(a)?p(a,this.context).index(this[0])>=0:p.filter(a,this).length>0:this.filter(a).length>0)},closest:function(a,b){var c,d=0,e=this.length,f=[],g=bf.test(a)||typeof a!="string"?p(a,b||this.context):0;for(;d<e;d++){c=this[d];while(c&&c.ownerDocument&&c!==b&&c.nodeType!==11){if(g?g.index(c)>-1:p.find.matchesSelector(c,a)){f.push(c);break}c=c.parentNode}}return f=f.length>1?p.unique(f):f,this.pushStack(f,"closest",a)},index:function(a){return a?typeof a=="string"?p.inArray(this[0],p(a)):p.inArray(a.jquery?a[0]:a,this):this[0]&&this[0].parentNode?this.prevAll().length:-1},add:function(a,b){var c=typeof a=="string"?p(a,b):p.makeArray(a&&a.nodeType?[a]:a),d=p.merge(this.get(),c);return this.pushStack(bh(c[0])||bh(d[0])?d:p.unique(d))},addBack:function(a){return this.add(a==null?this.prevObject:this.prevObject.filter(a))}}),p.fn.andSelf=p.fn.addBack,p.each({parent:function(a){var b=a.parentNode;return b&&b.nodeType!==11?b:null},parents:function(a){return p.dir(a,"parentNode")},parentsUntil:function(a,b,c){return p.dir(a,"parentNode",c)},next:function(a){return bi(a,"nextSibling")},prev:function(a){return bi(a,"previousSibling")},nextAll:function(a){return p.dir(a,"nextSibling")},prevAll:function(a){return p.dir(a,"previousSibling")},nextUntil:function(a,b,c){return p.dir(a,"nextSibling",c)},prevUntil:function(a,b,c){return p.dir(a,"previousSibling",c)},siblings:function(a){return p.sibling((a.parentNode||{}).firstChild,a)},children:function(a){return p.sibling(a.firstChild)},contents:function(a){return p.nodeName(a,"iframe")?a.contentDocument||a.contentWindow.document:p.merge([],a.childNodes)}},function(a,b){p.fn[a]=function(c,d){var e=p.map(this,b,c);return bc.test(a)||(d=c),d&&typeof d=="string"&&(e=p.filter(d,e)),e=this.length>1&&!bg[a]?p.unique(e):e,this.length>1&&bd.test(a)&&(e=e.reverse()),this.pushStack(e,a,k.call(arguments).join(","))}}),p.extend({filter:function(a,b,c){return c&&(a=":not("+a+")"),b.length===1?p.find.matchesSelector(b[0],a)?[b[0]]:[]:p.find.matches(a,b)},dir:function(a,c,d){var e=[],f=a[c];while(f&&f.nodeType!==9&&(d===b||f.nodeType!==1||!p(f).is(d)))f.nodeType===1&&e.push(f),f=f[c];return e},sibling:function(a,b){var c=[];for(;a;a=a.nextSibling)a.nodeType===1&&a!==b&&c.push(a);return c}});var bl="abbr|article|aside|audio|bdi|canvas|data|datalist|details|figcaption|figure|footer|header|hgroup|mark|meter|nav|output|progress|section|summary|time|video",bm=/ jQuery\d+="(?:null|\d+)"/g,bn=/^\s+/,bo=/<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,bp=/<([\w:]+)/,bq=/<tbody/i,br=/<|&#?\w+;/,bs=/<(?:script|style|link)/i,bt=/<(?:script|object|embed|option|style)/i,bu=new RegExp("<(?:"+bl+")[\\s/>]","i"),bv=/^(?:checkbox|radio)$/,bw=/checked\s*(?:[^=]|=\s*.checked.)/i,bx=/\/(java|ecma)script/i,by=/^\s*<!(?:\[CDATA\[|\-\-)|[\]\-]{2}>\s*$/g,bz={option:[1,"<select multiple='multiple'>","</select>"],legend:[1,"<fieldset>","</fieldset>"],thead:[1,"<table>","</table>"],tr:[2,"<table><tbody>","</tbody></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],col:[2,"<table><tbody></tbody><colgroup>","</colgroup></table>"],area:[1,"<map>","</map>"],_default:[0,"",""]},bA=bk(e),bB=bA.appendChild(e.createElement("div"));bz.optgroup=bz.option,bz.tbody=bz.tfoot=bz.colgroup=bz.caption=bz.thead,bz.th=bz.td,p.support.htmlSerialize||(bz._default=[1,"X<div>","</div>"]),p.fn.extend({text:function(a){return p.access(this,function(a){return a===b?p.text(this):this.empty().append((this[0]&&this[0].ownerDocument||e).createTextNode(a))},null,a,arguments.length)},wrapAll:function(a){if(p.isFunction(a))return this.each(function(b){p(this).wrapAll(a.call(this,b))});if(this[0]){var b=p(a,this[0].ownerDocument).eq(0).clone(!0);this[0].parentNode&&b.insertBefore(this[0]),b.map(function(){var a=this;while(a.firstChild&&a.firstChild.nodeType===1)a=a.firstChild;return a}).append(this)}return this},wrapInner:function(a){return p.isFunction(a)?this.each(function(b){p(this).wrapInner(a.call(this,b))}):this.each(function(){var b=p(this),c=b.contents();c.length?c.wrapAll(a):b.append(a)})},wrap:function(a){var b=p.isFunction(a);return this.each(function(c){p(this).wrapAll(b?a.call(this,c):a)})},unwrap:function(){return this.parent().each(function(){p.nodeName(this,"body")||p(this).replaceWith(this.childNodes)}).end()},append:function(){return this.domManip(arguments,!0,function(a){(this.nodeType===1||this.nodeType===11)&&this.appendChild(a)})},prepend:function(){return this.domManip(arguments,!0,function(a){(this.nodeType===1||this.nodeType===11)&&this.insertBefore(a,this.firstChild)})},before:function(){if(!bh(this[0]))return this.domManip(arguments,!1,function(a){this.parentNode.insertBefore(a,this)});if(arguments.length){var a=p.clean(arguments);return this.pushStack(p.merge(a,this),"before",this.selector)}},after:function(){if(!bh(this[0]))return this.domManip(arguments,!1,function(a){this.parentNode.insertBefore(a,this.nextSibling)});if(arguments.length){var a=p.clean(arguments);return this.pushStack(p.merge(this,a),"after",this.selector)}},remove:function(a,b){var c,d=0;for(;(c=this[d])!=null;d++)if(!a||p.filter(a,[c]).length)!b&&c.nodeType===1&&(p.cleanData(c.getElementsByTagName("*")),p.cleanData([c])),c.parentNode&&c.parentNode.removeChild(c);return this},empty:function(){var a,b=0;for(;(a=this[b])!=null;b++){a.nodeType===1&&p.cleanData(a.getElementsByTagName("*"));while(a.firstChild)a.removeChild(a.firstChild)}return this},clone:function(a,b){return a=a==null?!1:a,b=b==null?a:b,this.map(function(){return p.clone(this,a,b)})},html:function(a){return p.access(this,function(a){var c=this[0]||{},d=0,e=this.length;if(a===b)return c.nodeType===1?c.innerHTML.replace(bm,""):b;if(typeof a=="string"&&!bs.test(a)&&(p.support.htmlSerialize||!bu.test(a))&&(p.support.leadingWhitespace||!bn.test(a))&&!bz[(bp.exec(a)||["",""])[1].toLowerCase()]){a=a.replace(bo,"<$1></$2>");try{for(;d<e;d++)c=this[d]||{},c.nodeType===1&&(p.cleanData(c.getElementsByTagName("*")),c.innerHTML=a);c=0}catch(f){}}c&&this.empty().append(a)},null,a,arguments.length)},replaceWith:function(a){return bh(this[0])?this.length?this.pushStack(p(p.isFunction(a)?a():a),"replaceWith",a):this:p.isFunction(a)?this.each(function(b){var c=p(this),d=c.html();c.replaceWith(a.call(this,b,d))}):(typeof a!="string"&&(a=p(a).detach()),this.each(function(){var b=this.nextSibling,c=this.parentNode;p(this).remove(),b?p(b).before(a):p(c).append(a)}))},detach:function(a){return this.remove(a,!0)},domManip:function(a,c,d){a=[].concat.apply([],a);var e,f,g,h,i=0,j=a[0],k=[],l=this.length;if(!p.support.checkClone&&l>1&&typeof j=="string"&&bw.test(j))return this.each(function(){p(this).domManip(a,c,d)});if(p.isFunction(j))return this.each(function(e){var f=p(this);a[0]=j.call(this,e,c?f.html():b),f.domManip(a,c,d)});if(this[0]){e=p.buildFragment(a,this,k),g=e.fragment,f=g.firstChild,g.childNodes.length===1&&(g=f);if(f){c=c&&p.nodeName(f,"tr");for(h=e.cacheable||l-1;i<l;i++)d.call(c&&p.nodeName(this[i],"table")?bC(this[i],"tbody"):this[i],i===h?g:p.clone(g,!0,!0))}g=f=null,k.length&&p.each(k,function(a,b){b.src?p.ajax?p.ajax({url:b.src,type:"GET",dataType:"script",async:!1,global:!1,"throws":!0}):p.error("no ajax"):p.globalEval((b.text||b.textContent||b.innerHTML||"").replace(by,"")),b.parentNode&&b.parentNode.removeChild(b)})}return this}}),p.buildFragment=function(a,c,d){var f,g,h,i=a[0];return c=c||e,c=!c.nodeType&&c[0]||c,c=c.ownerDocument||c,a.length===1&&typeof i=="string"&&i.length<512&&c===e&&i.charAt(0)==="<"&&!bt.test(i)&&(p.support.checkClone||!bw.test(i))&&(p.support.html5Clone||!bu.test(i))&&(g=!0,f=p.fragments[i],h=f!==b),f||(f=c.createDocumentFragment(),p.clean(a,c,f,d),g&&(p.fragments[i]=h&&f)),{fragment:f,cacheable:g}},p.fragments={},p.each({appendTo:"append",prependTo:"prepend",insertBefore:"before",insertAfter:"after",replaceAll:"replaceWith"},function(a,b){p.fn[a]=function(c){var d,e=0,f=[],g=p(c),h=g.length,i=this.length===1&&this[0].parentNode;if((i==null||i&&i.nodeType===11&&i.childNodes.length===1)&&h===1)return g[b](this[0]),this;for(;e<h;e++)d=(e>0?this.clone(!0):this).get(),p(g[e])[b](d),f=f.concat(d);return this.pushStack(f,a,g.selector)}}),p.extend({clone:function(a,b,c){var d,e,f,g;p.support.html5Clone||p.isXMLDoc(a)||!bu.test("<"+a.nodeName+">")?g=a.cloneNode(!0):(bB.innerHTML=a.outerHTML,bB.removeChild(g=bB.firstChild));if((!p.support.noCloneEvent||!p.support.noCloneChecked)&&(a.nodeType===1||a.nodeType===11)&&!p.isXMLDoc(a)){bE(a,g),d=bF(a),e=bF(g);for(f=0;d[f];++f)e[f]&&bE(d[f],e[f])}if(b){bD(a,g);if(c){d=bF(a),e=bF(g);for(f=0;d[f];++f)bD(d[f],e[f])}}return d=e=null,g},clean:function(a,b,c,d){var f,g,h,i,j,k,l,m,n,o,q,r,s=b===e&&bA,t=[];if(!b||typeof b.createDocumentFragment=="undefined")b=e;for(f=0;(h=a[f])!=null;f++){typeof h=="number"&&(h+="");if(!h)continue;if(typeof h=="string")if(!br.test(h))h=b.createTextNode(h);else{s=s||bk(b),l=b.createElement("div"),s.appendChild(l),h=h.replace(bo,"<$1></$2>"),i=(bp.exec(h)||["",""])[1].toLowerCase(),j=bz[i]||bz._default,k=j[0],l.innerHTML=j[1]+h+j[2];while(k--)l=l.lastChild;if(!p.support.tbody){m=bq.test(h),n=i==="table"&&!m?l.firstChild&&l.firstChild.childNodes:j[1]==="<table>"&&!m?l.childNodes:[];for(g=n.length-1;g>=0;--g)p.nodeName(n[g],"tbody")&&!n[g].childNodes.length&&n[g].parentNode.removeChild(n[g])}!p.support.leadingWhitespace&&bn.test(h)&&l.insertBefore(b.createTextNode(bn.exec(h)[0]),l.firstChild),h=l.childNodes,l.parentNode.removeChild(l)}h.nodeType?t.push(h):p.merge(t,h)}l&&(h=l=s=null);if(!p.support.appendChecked)for(f=0;(h=t[f])!=null;f++)p.nodeName(h,"input")?bG(h):typeof h.getElementsByTagName!="undefined"&&p.grep(h.getElementsByTagName("input"),bG);if(c){q=function(a){if(!a.type||bx.test(a.type))return d?d.push(a.parentNode?a.parentNode.removeChild(a):a):c.appendChild(a)};for(f=0;(h=t[f])!=null;f++)if(!p.nodeName(h,"script")||!q(h))c.appendChild(h),typeof h.getElementsByTagName!="undefined"&&(r=p.grep(p.merge([],h.getElementsByTagName("script")),q),t.splice.apply(t,[f+1,0].concat(r)),f+=r.length)}return t},cleanData:function(a,b){var c,d,e,f,g=0,h=p.expando,i=p.cache,j=p.support.deleteExpando,k=p.event.special;for(;(e=a[g])!=null;g++)if(b||p.acceptData(e)){d=e[h],c=d&&i[d];if(c){if(c.events)for(f in c.events)k[f]?p.event.remove(e,f):p.removeEvent(e,f,c.handle);i[d]&&(delete i[d],j?delete e[h]:e.removeAttribute?e.removeAttribute(h):e[h]=null,p.deletedIds.push(d))}}}}),function(){var a,b;p.uaMatch=function(a){a=a.toLowerCase();var b=/(chrome)[ \/]([\w.]+)/.exec(a)||/(webkit)[ \/]([\w.]+)/.exec(a)||/(opera)(?:.*version|)[ \/]([\w.]+)/.exec(a)||/(msie) ([\w.]+)/.exec(a)||a.indexOf("compatible")<0&&/(mozilla)(?:.*? rv:([\w.]+)|)/.exec(a)||[];return{browser:b[1]||"",version:b[2]||"0"}},a=p.uaMatch(g.userAgent),b={},a.browser&&(b[a.browser]=!0,b.version=a.version),b.chrome?b.webkit=!0:b.webkit&&(b.safari=!0),p.browser=b,p.sub=function(){function a(b,c){return new a.fn.init(b,c)}p.extend(!0,a,this),a.superclass=this,a.fn=a.prototype=this(),a.fn.constructor=a,a.sub=this.sub,a.fn.init=function c(c,d){return d&&d instanceof p&&!(d instanceof a)&&(d=a(d)),p.fn.init.call(this,c,d,b)},a.fn.init.prototype=a.fn;var b=a(e);return a}}();var bH,bI,bJ,bK=/alpha\([^)]*\)/i,bL=/opacity=([^)]*)/,bM=/^(top|right|bottom|left)$/,bN=/^(none|table(?!-c[ea]).+)/,bO=/^margin/,bP=new RegExp("^("+q+")(.*)$","i"),bQ=new RegExp("^("+q+")(?!px)[a-z%]+$","i"),bR=new RegExp("^([-+])=("+q+")","i"),bS={},bT={position:"absolute",visibility:"hidden",display:"block"},bU={letterSpacing:0,fontWeight:400},bV=["Top","Right","Bottom","Left"],bW=["Webkit","O","Moz","ms"],bX=p.fn.toggle;p.fn.extend({css:function(a,c){return p.access(this,function(a,c,d){return d!==b?p.style(a,c,d):p.css(a,c)},a,c,arguments.length>1)},show:function(){return b$(this,!0)},hide:function(){return b$(this)},toggle:function(a,b){var c=typeof a=="boolean";return p.isFunction(a)&&p.isFunction(b)?bX.apply(this,arguments):this.each(function(){(c?a:bZ(this))?p(this).show():p(this).hide()})}}),p.extend({cssHooks:{opacity:{get:function(a,b){if(b){var c=bH(a,"opacity");return c===""?"1":c}}}},cssNumber:{fillOpacity:!0,fontWeight:!0,lineHeight:!0,opacity:!0,orphans:!0,widows:!0,zIndex:!0,zoom:!0},cssProps:{"float":p.support.cssFloat?"cssFloat":"styleFloat"},style:function(a,c,d,e){if(!a||a.nodeType===3||a.nodeType===8||!a.style)return;var f,g,h,i=p.camelCase(c),j=a.style;c=p.cssProps[i]||(p.cssProps[i]=bY(j,i)),h=p.cssHooks[c]||p.cssHooks[i];if(d===b)return h&&"get"in h&&(f=h.get(a,!1,e))!==b?f:j[c];g=typeof d,g==="string"&&(f=bR.exec(d))&&(d=(f[1]+1)*f[2]+parseFloat(p.css(a,c)),g="number");if(d==null||g==="number"&&isNaN(d))return;g==="number"&&!p.cssNumber[i]&&(d+="px");if(!h||!("set"in h)||(d=h.set(a,d,e))!==b)try{j[c]=d}catch(k){}},css:function(a,c,d,e){var f,g,h,i=p.camelCase(c);return c=p.cssProps[i]||(p.cssProps[i]=bY(a.style,i)),h=p.cssHooks[c]||p.cssHooks[i],h&&"get"in h&&(f=h.get(a,!0,e)),f===b&&(f=bH(a,c)),f==="normal"&&c in bU&&(f=bU[c]),d||e!==b?(g=parseFloat(f),d||p.isNumeric(g)?g||0:f):f},swap:function(a,b,c){var d,e,f={};for(e in b)f[e]=a.style[e],a.style[e]=b[e];d=c.call(a);for(e in b)a.style[e]=f[e];return d}}),a.getComputedStyle?bH=function(b,c){var d,e,f,g,h=a.getComputedStyle(b,null),i=b.style;return h&&(d=h[c],d===""&&!p.contains(b.ownerDocument,b)&&(d=p.style(b,c)),bQ.test(d)&&bO.test(c)&&(e=i.width,f=i.minWidth,g=i.maxWidth,i.minWidth=i.maxWidth=i.width=d,d=h.width,i.width=e,i.minWidth=f,i.maxWidth=g)),d}:e.documentElement.currentStyle&&(bH=function(a,b){var c,d,e=a.currentStyle&&a.currentStyle[b],f=a.style;return e==null&&f&&f[b]&&(e=f[b]),bQ.test(e)&&!bM.test(b)&&(c=f.left,d=a.runtimeStyle&&a.runtimeStyle.left,d&&(a.runtimeStyle.left=a.currentStyle.left),f.left=b==="fontSize"?"1em":e,e=f.pixelLeft+"px",f.left=c,d&&(a.runtimeStyle.left=d)),e===""?"auto":e}),p.each(["height","width"],function(a,b){p.cssHooks[b]={get:function(a,c,d){if(c)return a.offsetWidth===0&&bN.test(bH(a,"display"))?p.swap(a,bT,function(){return cb(a,b,d)}):cb(a,b,d)},set:function(a,c,d){return b_(a,c,d?ca(a,b,d,p.support.boxSizing&&p.css(a,"boxSizing")==="border-box"):0)}}}),p.support.opacity||(p.cssHooks.opacity={get:function(a,b){return bL.test((b&&a.currentStyle?a.currentStyle.filter:a.style.filter)||"")?.01*parseFloat(RegExp.$1)+"":b?"1":""},set:function(a,b){var c=a.style,d=a.currentStyle,e=p.isNumeric(b)?"alpha(opacity="+b*100+")":"",f=d&&d.filter||c.filter||"";c.zoom=1;if(b>=1&&p.trim(f.replace(bK,""))===""&&c.removeAttribute){c.removeAttribute("filter");if(d&&!d.filter)return}c.filter=bK.test(f)?f.replace(bK,e):f+" "+e}}),p(function(){p.support.reliableMarginRight||(p.cssHooks.marginRight={get:function(a,b){return p.swap(a,{display:"inline-block"},function(){if(b)return bH(a,"marginRight")})}}),!p.support.pixelPosition&&p.fn.position&&p.each(["top","left"],function(a,b){p.cssHooks[b]={get:function(a,c){if(c){var d=bH(a,b);return bQ.test(d)?p(a).position()[b]+"px":d}}}})}),p.expr&&p.expr.filters&&(p.expr.filters.hidden=function(a){return a.offsetWidth===0&&a.offsetHeight===0||!p.support.reliableHiddenOffsets&&(a.style&&a.style.display||bH(a,"display"))==="none"},p.expr.filters.visible=function(a){return!p.expr.filters.hidden(a)}),p.each({margin:"",padding:"",border:"Width"},function(a,b){p.cssHooks[a+b]={expand:function(c){var d,e=typeof c=="string"?c.split(" "):[c],f={};for(d=0;d<4;d++)f[a+bV[d]+b]=e[d]||e[d-2]||e[0];return f}},bO.test(a)||(p.cssHooks[a+b].set=b_)});var cd=/%20/g,ce=/\[\]$/,cf=/\r?\n/g,cg=/^(?:color|date|datetime|datetime-local|email|hidden|month|number|password|range|search|tel|text|time|url|week)$/i,ch=/^(?:select|textarea)/i;p.fn.extend({serialize:function(){return p.param(this.serializeArray())},serializeArray:function(){return this.map(function(){return this.elements?p.makeArray(this.elements):this}).filter(function(){return this.name&&!this.disabled&&(this.checked||ch.test(this.nodeName)||cg.test(this.type))}).map(function(a,b){var c=p(this).val();return c==null?null:p.isArray(c)?p.map(c,function(a,c){return{name:b.name,value:a.replace(cf,"\r\n")}}):{name:b.name,value:c.replace(cf,"\r\n")}}).get()}}),p.param=function(a,c){var d,e=[],f=function(a,b){b=p.isFunction(b)?b():b==null?"":b,e[e.length]=encodeURIComponent(a)+"="+encodeURIComponent(b)};c===b&&(c=p.ajaxSettings&&p.ajaxSettings.traditional);if(p.isArray(a)||a.jquery&&!p.isPlainObject(a))p.each(a,function(){f(this.name,this.value)});else for(d in a)ci(d,a[d],c,f);return e.join("&").replace(cd,"+")};var cj,ck,cl=/#.*$/,cm=/^(.*?):[ \t]*([^\r\n]*)\r?$/mg,cn=/^(?:about|app|app\-storage|.+\-extension|file|res|widget):$/,co=/^(?:GET|HEAD)$/,cp=/^\/\//,cq=/\?/,cr=/<script\b[^<]*(?:(?!<\/script>)<[^<]*)*<\/script>/gi,cs=/([?&])_=[^&]*/,ct=/^([\w\+\.\-]+:)(?:\/\/([^\/?#:]*)(?::(\d+)|)|)/,cu=p.fn.load,cv={},cw={},cx=["*/"]+["*"];try{ck=f.href}catch(cy){ck=e.createElement("a"),ck.href="",ck=ck.href}cj=ct.exec(ck.toLowerCase())||[],p.fn.load=function(a,c,d){if(typeof a!="string"&&cu)return cu.apply(this,arguments);if(!this.length)return this;var e,f,g,h=this,i=a.indexOf(" ");return i>=0&&(e=a.slice(i,a.length),a=a.slice(0,i)),p.isFunction(c)?(d=c,c=b):c&&typeof c=="object"&&(f="POST"),p.ajax({url:a,type:f,dataType:"html",data:c,complete:function(a,b){d&&h.each(d,g||[a.responseText,b,a])}}).done(function(a){g=arguments,h.html(e?p("<div>").append(a.replace(cr,"")).find(e):a)}),this},p.each("ajaxStart ajaxStop ajaxComplete ajaxError ajaxSuccess ajaxSend".split(" "),function(a,b){p.fn[b]=function(a){return this.on(b,a)}}),p.each(["get","post"],function(a,c){p[c]=function(a,d,e,f){return p.isFunction(d)&&(f=f||e,e=d,d=b),p.ajax({type:c,url:a,data:d,success:e,dataType:f})}}),p.extend({getScript:function(a,c){return p.get(a,b,c,"script")},getJSON:function(a,b,c){return p.get(a,b,c,"json")},ajaxSetup:function(a,b){return b?cB(a,p.ajaxSettings):(b=a,a=p.ajaxSettings),cB(a,b),a},ajaxSettings:{url:ck,isLocal:cn.test(cj[1]),global:!0,type:"GET",contentType:"application/x-www-form-urlencoded; charset=UTF-8",processData:!0,async:!0,accepts:{xml:"application/xml, text/xml",html:"text/html",text:"text/plain",json:"application/json, text/javascript","*":cx},contents:{xml:/xml/,html:/html/,json:/json/},responseFields:{xml:"responseXML",text:"responseText"},converters:{"* text":a.String,"text html":!0,"text json":p.parseJSON,"text xml":p.parseXML},flatOptions:{context:!0,url:!0}},ajaxPrefilter:cz(cv),ajaxTransport:cz(cw),ajax:function(a,c){function y(a,c,f,i){var k,s,t,u,w,y=c;if(v===2)return;v=2,h&&clearTimeout(h),g=b,e=i||"",x.readyState=a>0?4:0,f&&(u=cC(l,x,f));if(a>=200&&a<300||a===304)l.ifModified&&(w=x.getResponseHeader("Last-Modified"),w&&(p.lastModified[d]=w),w=x.getResponseHeader("Etag"),w&&(p.etag[d]=w)),a===304?(y="notmodified",k=!0):(k=cD(l,u),y=k.state,s=k.data,t=k.error,k=!t);else{t=y;if(!y||a)y="error",a<0&&(a=0)}x.status=a,x.statusText=(c||y)+"",k?o.resolveWith(m,[s,y,x]):o.rejectWith(m,[x,y,t]),x.statusCode(r),r=b,j&&n.trigger("ajax"+(k?"Success":"Error"),[x,l,k?s:t]),q.fireWith(m,[x,y]),j&&(n.trigger("ajaxComplete",[x,l]),--p.active||p.event.trigger("ajaxStop"))}typeof a=="object"&&(c=a,a=b),c=c||{};var d,e,f,g,h,i,j,k,l=p.ajaxSetup({},c),m=l.context||l,n=m!==l&&(m.nodeType||m instanceof p)?p(m):p.event,o=p.Deferred(),q=p.Callbacks("once memory"),r=l.statusCode||{},t={},u={},v=0,w="canceled",x={readyState:0,setRequestHeader:function(a,b){if(!v){var c=a.toLowerCase();a=u[c]=u[c]||a,t[a]=b}return this},getAllResponseHeaders:function(){return v===2?e:null},getResponseHeader:function(a){var c;if(v===2){if(!f){f={};while(c=cm.exec(e))f[c[1].toLowerCase()]=c[2]}c=f[a.toLowerCase()]}return c===b?null:c},overrideMimeType:function(a){return v||(l.mimeType=a),this},abort:function(a){return a=a||w,g&&g.abort(a),y(0,a),this}};o.promise(x),x.success=x.done,x.error=x.fail,x.complete=q.add,x.statusCode=function(a){if(a){var b;if(v<2)for(b in a)r[b]=[r[b],a[b]];else b=a[x.status],x.always(b)}return this},l.url=((a||l.url)+"").replace(cl,"").replace(cp,cj[1]+"//"),l.dataTypes=p.trim(l.dataType||"*").toLowerCase().split(s),l.crossDomain==null&&(i=ct.exec(l.url.toLowerCase())||!1,l.crossDomain=i&&i.join(":")+(i[3]?"":i[1]==="http:"?80:443)!==cj.join(":")+(cj[3]?"":cj[1]==="http:"?80:443)),l.data&&l.processData&&typeof l.data!="string"&&(l.data=p.param(l.data,l.traditional)),cA(cv,l,c,x);if(v===2)return x;j=l.global,l.type=l.type.toUpperCase(),l.hasContent=!co.test(l.type),j&&p.active++===0&&p.event.trigger("ajaxStart");if(!l.hasContent){l.data&&(l.url+=(cq.test(l.url)?"&":"?")+l.data,delete l.data),d=l.url;if(l.cache===!1){var z=p.now(),A=l.url.replace(cs,"$1_="+z);l.url=A+(A===l.url?(cq.test(l.url)?"&":"?")+"_="+z:"")}}(l.data&&l.hasContent&&l.contentType!==!1||c.contentType)&&x.setRequestHeader("Content-Type",l.contentType),l.ifModified&&(d=d||l.url,p.lastModified[d]&&x.setRequestHeader("If-Modified-Since",p.lastModified[d]),p.etag[d]&&x.setRequestHeader("If-None-Match",p.etag[d])),x.setRequestHeader("Accept",l.dataTypes[0]&&l.accepts[l.dataTypes[0]]?l.accepts[l.dataTypes[0]]+(l.dataTypes[0]!=="*"?", "+cx+"; q=0.01":""):l.accepts["*"]);for(k in l.headers)x.setRequestHeader(k,l.headers[k]);if(!l.beforeSend||l.beforeSend.call(m,x,l)!==!1&&v!==2){w="abort";for(k in{success:1,error:1,complete:1})x[k](l[k]);g=cA(cw,l,c,x);if(!g)y(-1,"No Transport");else{x.readyState=1,j&&n.trigger("ajaxSend",[x,l]),l.async&&l.timeout>0&&(h=setTimeout(function(){x.abort("timeout")},l.timeout));try{v=1,g.send(t,y)}catch(B){if(v<2)y(-1,B);else throw B}}return x}return x.abort()},active:0,lastModified:{},etag:{}});var cE=[],cF=/\?/,cG=/(=)\?(?=&|$)|\?\?/,cH=p.now();p.ajaxSetup({jsonp:"callback",jsonpCallback:function(){var a=cE.pop()||p.expando+"_"+cH++;return this[a]=!0,a}}),p.ajaxPrefilter("json jsonp",function(c,d,e){var f,g,h,i=c.data,j=c.url,k=c.jsonp!==!1,l=k&&cG.test(j),m=k&&!l&&typeof i=="string"&&!(c.contentType||"").indexOf("application/x-www-form-urlencoded")&&cG.test(i);if(c.dataTypes[0]==="jsonp"||l||m)return f=c.jsonpCallback=p.isFunction(c.jsonpCallback)?c.jsonpCallback():c.jsonpCallback,g=a[f],l?c.url=j.replace(cG,"$1"+f):m?c.data=i.replace(cG,"$1"+f):k&&(c.url+=(cF.test(j)?"&":"?")+c.jsonp+"="+f),c.converters["script json"]=function(){return h||p.error(f+" was not called"),h[0]},c.dataTypes[0]="json",a[f]=function(){h=arguments},e.always(function(){a[f]=g,c[f]&&(c.jsonpCallback=d.jsonpCallback,cE.push(f)),h&&p.isFunction(g)&&g(h[0]),h=g=b}),"script"}),p.ajaxSetup({accepts:{script:"text/javascript, application/javascript, application/ecmascript, application/x-ecmascript"},contents:{script:/javascript|ecmascript/},converters:{"text script":function(a){return p.globalEval(a),a}}}),p.ajaxPrefilter("script",function(a){a.cache===b&&(a.cache=!1),a.crossDomain&&(a.type="GET",a.global=!1)}),p.ajaxTransport("script",function(a){if(a.crossDomain){var c,d=e.head||e.getElementsByTagName("head")[0]||e.documentElement;return{send:function(f,g){c=e.createElement("script"),c.async="async",a.scriptCharset&&(c.charset=a.scriptCharset),c.src=a.url,c.onload=c.onreadystatechange=function(a,e){if(e||!c.readyState||/loaded|complete/.test(c.readyState))c.onload=c.onreadystatechange=null,d&&c.parentNode&&d.removeChild(c),c=b,e||g(200,"success")},d.insertBefore(c,d.firstChild)},abort:function(){c&&c.onload(0,1)}}}});var cI,cJ=a.ActiveXObject?function(){for(var a in cI)cI[a](0,1)}:!1,cK=0;p.ajaxSettings.xhr=a.ActiveXObject?function(){return!this.isLocal&&cL()||cM()}:cL,function(a){p.extend(p.support,{ajax:!!a,cors:!!a&&"withCredentials"in a})}(p.ajaxSettings.xhr()),p.support.ajax&&p.ajaxTransport(function(c){if(!c.crossDomain||p.support.cors){var d;return{send:function(e,f){var g,h,i=c.xhr();c.username?i.open(c.type,c.url,c.async,c.username,c.password):i.open(c.type,c.url,c.async);if(c.xhrFields)for(h in c.xhrFields)i[h]=c.xhrFields[h];c.mimeType&&i.overrideMimeType&&i.overrideMimeType(c.mimeType),!c.crossDomain&&!e["X-Requested-With"]&&(e["X-Requested-With"]="XMLHttpRequest");try{for(h in e)i.setRequestHeader(h,e[h])}catch(j){}i.send(c.hasContent&&c.data||null),d=function(a,e){var h,j,k,l,m;try{if(d&&(e||i.readyState===4)){d=b,g&&(i.onreadystatechange=p.noop,cJ&&delete cI[g]);if(e)i.readyState!==4&&i.abort();else{h=i.status,k=i.getAllResponseHeaders(),l={},m=i.responseXML,m&&m.documentElement&&(l.xml=m);try{l.text=i.responseText}catch(a){}try{j=i.statusText}catch(n){j=""}!h&&c.isLocal&&!c.crossDomain?h=l.text?200:404:h===1223&&(h=204)}}}catch(o){e||f(-1,o)}l&&f(h,j,l,k)},c.async?i.readyState===4?setTimeout(d,0):(g=++cK,cJ&&(cI||(cI={},p(a).unload(cJ)),cI[g]=d),i.onreadystatechange=d):d()},abort:function(){d&&d(0,1)}}}});var cN,cO,cP=/^(?:toggle|show|hide)$/,cQ=new RegExp("^(?:([-+])=|)("+q+")([a-z%]*)$","i"),cR=/queueHooks$/,cS=[cY],cT={"*":[function(a,b){var c,d,e=this.createTween(a,b),f=cQ.exec(b),g=e.cur(),h=+g||0,i=1,j=20;if(f){c=+f[2],d=f[3]||(p.cssNumber[a]?"":"px");if(d!=="px"&&h){h=p.css(e.elem,a,!0)||c||1;do i=i||".5",h=h/i,p.style(e.elem,a,h+d);while(i!==(i=e.cur()/g)&&i!==1&&--j)}e.unit=d,e.start=h,e.end=f[1]?h+(f[1]+1)*c:c}return e}]};p.Animation=p.extend(cW,{tweener:function(a,b){p.isFunction(a)?(b=a,a=["*"]):a=a.split(" ");var c,d=0,e=a.length;for(;d<e;d++)c=a[d],cT[c]=cT[c]||[],cT[c].unshift(b)},prefilter:function(a,b){b?cS.unshift(a):cS.push(a)}}),p.Tween=cZ,cZ.prototype={constructor:cZ,init:function(a,b,c,d,e,f){this.elem=a,this.prop=c,this.easing=e||"swing",this.options=b,this.start=this.now=this.cur(),this.end=d,this.unit=f||(p.cssNumber[c]?"":"px")},cur:function(){var a=cZ.propHooks[this.prop];return a&&a.get?a.get(this):cZ.propHooks._default.get(this)},run:function(a){var b,c=cZ.propHooks[this.prop];return this.options.duration?this.pos=b=p.easing[this.easing](a,this.options.duration*a,0,1,this.options.duration):this.pos=b=a,this.now=(this.end-this.start)*b+this.start,this.options.step&&this.options.step.call(this.elem,this.now,this),c&&c.set?c.set(this):cZ.propHooks._default.set(this),this}},cZ.prototype.init.prototype=cZ.prototype,cZ.propHooks={_default:{get:function(a){var b;return a.elem[a.prop]==null||!!a.elem.style&&a.elem.style[a.prop]!=null?(b=p.css(a.elem,a.prop,!1,""),!b||b==="auto"?0:b):a.elem[a.prop]},set:function(a){p.fx.step[a.prop]?p.fx.step[a.prop](a):a.elem.style&&(a.elem.style[p.cssProps[a.prop]]!=null||p.cssHooks[a.prop])?p.style(a.elem,a.prop,a.now+a.unit):a.elem[a.prop]=a.now}}},cZ.propHooks.scrollTop=cZ.propHooks.scrollLeft={set:function(a){a.elem.nodeType&&a.elem.parentNode&&(a.elem[a.prop]=a.now)}},p.each(["toggle","show","hide"],function(a,b){var c=p.fn[b];p.fn[b]=function(d,e,f){return d==null||typeof d=="boolean"||!a&&p.isFunction(d)&&p.isFunction(e)?c.apply(this,arguments):this.animate(c$(b,!0),d,e,f)}}),p.fn.extend({fadeTo:function(a,b,c,d){return this.filter(bZ).css("opacity",0).show().end().animate({opacity:b},a,c,d)},animate:function(a,b,c,d){var e=p.isEmptyObject(a),f=p.speed(b,c,d),g=function(){var b=cW(this,p.extend({},a),f);e&&b.stop(!0)};return e||f.queue===!1?this.each(g):this.queue(f.queue,g)},stop:function(a,c,d){var e=function(a){var b=a.stop;delete a.stop,b(d)};return typeof a!="string"&&(d=c,c=a,a=b),c&&a!==!1&&this.queue(a||"fx",[]),this.each(function(){var b=!0,c=a!=null&&a+"queueHooks",f=p.timers,g=p._data(this);if(c)g[c]&&g[c].stop&&e(g[c]);else for(c in g)g[c]&&g[c].stop&&cR.test(c)&&e(g[c]);for(c=f.length;c--;)f[c].elem===this&&(a==null||f[c].queue===a)&&(f[c].anim.stop(d),b=!1,f.splice(c,1));(b||!d)&&p.dequeue(this,a)})}}),p.each({slideDown:c$("show"),slideUp:c$("hide"),slideToggle:c$("toggle"),fadeIn:{opacity:"show"},fadeOut:{opacity:"hide"},fadeToggle:{opacity:"toggle"}},function(a,b){p.fn[a]=function(a,c,d){return this.animate(b,a,c,d)}}),p.speed=function(a,b,c){var d=a&&typeof a=="object"?p.extend({},a):{complete:c||!c&&b||p.isFunction(a)&&a,duration:a,easing:c&&b||b&&!p.isFunction(b)&&b};d.duration=p.fx.off?0:typeof d.duration=="number"?d.duration:d.duration in p.fx.speeds?p.fx.speeds[d.duration]:p.fx.speeds._default;if(d.queue==null||d.queue===!0)d.queue="fx";return d.old=d.complete,d.complete=function(){p.isFunction(d.old)&&d.old.call(this),d.queue&&p.dequeue(this,d.queue)},d},p.easing={linear:function(a){return a},swing:function(a){return.5-Math.cos(a*Math.PI)/2}},p.timers=[],p.fx=cZ.prototype.init,p.fx.tick=function(){var a,b=p.timers,c=0;for(;c<b.length;c++)a=b[c],!a()&&b[c]===a&&b.splice(c--,1);b.length||p.fx.stop()},p.fx.timer=function(a){a()&&p.timers.push(a)&&!cO&&(cO=setInterval(p.fx.tick,p.fx.interval))},p.fx.interval=13,p.fx.stop=function(){clearInterval(cO),cO=null},p.fx.speeds={slow:600,fast:200,_default:400},p.fx.step={},p.expr&&p.expr.filters&&(p.expr.filters.animated=function(a){return p.grep(p.timers,function(b){return a===b.elem}).length});var c_=/^(?:body|html)$/i;p.fn.offset=function(a){if(arguments.length)return a===b?this:this.each(function(b){p.offset.setOffset(this,a,b)});var c,d,e,f,g,h,i,j={top:0,left:0},k=this[0],l=k&&k.ownerDocument;if(!l)return;return(d=l.body)===k?p.offset.bodyOffset(k):(c=l.documentElement,p.contains(c,k)?(typeof k.getBoundingClientRect!="undefined"&&(j=k.getBoundingClientRect()),e=da(l),f=c.clientTop||d.clientTop||0,g=c.clientLeft||d.clientLeft||0,h=e.pageYOffset||c.scrollTop,i=e.pageXOffset||c.scrollLeft,{top:j.top+h-f,left:j.left+i-g}):j)},p.offset={bodyOffset:function(a){var b=a.offsetTop,c=a.offsetLeft;return p.support.doesNotIncludeMarginInBodyOffset&&(b+=parseFloat(p.css(a,"marginTop"))||0,c+=parseFloat(p.css(a,"marginLeft"))||0),{top:b,left:c}},setOffset:function(a,b,c){var d=p.css(a,"position");d==="static"&&(a.style.position="relative");var e=p(a),f=e.offset(),g=p.css(a,"top"),h=p.css(a,"left"),i=(d==="absolute"||d==="fixed")&&p.inArray("auto",[g,h])>-1,j={},k={},l,m;i?(k=e.position(),l=k.top,m=k.left):(l=parseFloat(g)||0,m=parseFloat(h)||0),p.isFunction(b)&&(b=b.call(a,c,f)),b.top!=null&&(j.top=b.top-f.top+l),b.left!=null&&(j.left=b.left-f.left+m),"using"in b?b.using.call(a,j):e.css(j)}},p.fn.extend({position:function(){if(!this[0])return;var a=this[0],b=this.offsetParent(),c=this.offset(),d=c_.test(b[0].nodeName)?{top:0,left:0}:b.offset();return c.top-=parseFloat(p.css(a,"marginTop"))||0,c.left-=parseFloat(p.css(a,"marginLeft"))||0,d.top+=parseFloat(p.css(b[0],"borderTopWidth"))||0,d.left+=parseFloat(p.css(b[0],"borderLeftWidth"))||0,{top:c.top-d.top,left:c.left-d.left}},offsetParent:function(){return this.map(function(){var a=this.offsetParent||e.body;while(a&&!c_.test(a.nodeName)&&p.css(a,"position")==="static")a=a.offsetParent;return a||e.body})}}),p.each({scrollLeft:"pageXOffset",scrollTop:"pageYOffset"},function(a,c){var d=/Y/.test(c);p.fn[a]=function(e){return p.access(this,function(a,e,f){var g=da(a);if(f===b)return g?c in g?g[c]:g.document.documentElement[e]:a[e];g?g.scrollTo(d?p(g).scrollLeft():f,d?f:p(g).scrollTop()):a[e]=f},a,e,arguments.length,null)}}),p.each({Height:"height",Width:"width"},function(a,c){p.each({padding:"inner"+a,content:c,"":"outer"+a},function(d,e){p.fn[e]=function(e,f){var g=arguments.length&&(d||typeof e!="boolean"),h=d||(e===!0||f===!0?"margin":"border");return p.access(this,function(c,d,e){var f;return p.isWindow(c)?c.document.documentElement["client"+a]:c.nodeType===9?(f=c.documentElement,Math.max(c.body["scroll"+a],f["scroll"+a],c.body["offset"+a],f["offset"+a],f["client"+a])):e===b?p.css(c,d,e,h):p.style(c,d,e,h)},c,g?e:b,g,null)}})}),a.jQuery=a.$=p,typeof define=="function"&&define.amd&&define.amd.jQuery&&define("jquery",[],function(){return p})})(window);
</script>

<script type="text/javascript">
(function(b){b.color={};b.color.make=function(d,e,g,f){var c={};c.r=d||0;c.g=e||0;c.b=g||0;c.a=f!=null?f:1;c.add=function(h,j){for(var k=0;k<h.length;++k){c[h.charAt(k)]+=j}return c.normalize()};c.scale=function(h,j){for(var k=0;k<h.length;++k){c[h.charAt(k)]*=j}return c.normalize()};c.toString=function(){if(c.a>=1){return"rgb("+[c.r,c.g,c.b].join(",")+")"}else{return"rgba("+[c.r,c.g,c.b,c.a].join(",")+")"}};c.normalize=function(){function h(k,j,l){return j<k?k:(j>l?l:j)}c.r=h(0,parseInt(c.r),255);c.g=h(0,parseInt(c.g),255);c.b=h(0,parseInt(c.b),255);c.a=h(0,c.a,1);return c};c.clone=function(){return b.color.make(c.r,c.b,c.g,c.a)};return c.normalize()};b.color.extract=function(d,e){var c;do{c=d.css(e).toLowerCase();if(c!=""&&c!="transparent"){break}d=d.parent()}while(!b.nodeName(d.get(0),"body"));if(c=="rgba(0, 0, 0, 0)"){c="transparent"}return b.color.parse(c)};b.color.parse=function(c){var d,f=b.color.make;if(d=/rgb\(\s*([0-9]{1,3})\s*,\s*([0-9]{1,3})\s*,\s*([0-9]{1,3})\s*\)/.exec(c)){return f(parseInt(d[1],10),parseInt(d[2],10),parseInt(d[3],10))}if(d=/rgba\(\s*([0-9]{1,3})\s*,\s*([0-9]{1,3})\s*,\s*([0-9]{1,3})\s*,\s*([0-9]+(?:\.[0-9]+)?)\s*\)/.exec(c)){return f(parseInt(d[1],10),parseInt(d[2],10),parseInt(d[3],10),parseFloat(d[4]))}if(d=/rgb\(\s*([0-9]+(?:\.[0-9]+)?)\%\s*,\s*([0-9]+(?:\.[0-9]+)?)\%\s*,\s*([0-9]+(?:\.[0-9]+)?)\%\s*\)/.exec(c)){return f(parseFloat(d[1])*2.55,parseFloat(d[2])*2.55,parseFloat(d[3])*2.55)}if(d=/rgba\(\s*([0-9]+(?:\.[0-9]+)?)\%\s*,\s*([0-9]+(?:\.[0-9]+)?)\%\s*,\s*([0-9]+(?:\.[0-9]+)?)\%\s*,\s*([0-9]+(?:\.[0-9]+)?)\s*\)/.exec(c)){return f(parseFloat(d[1])*2.55,parseFloat(d[2])*2.55,parseFloat(d[3])*2.55,parseFloat(d[4]))}if(d=/#([a-fA-F0-9]{2})([a-fA-F0-9]{2})([a-fA-F0-9]{2})/.exec(c)){return f(parseInt(d[1],16),parseInt(d[2],16),parseInt(d[3],16))}if(d=/#([a-fA-F0-9])([a-fA-F0-9])([a-fA-F0-9])/.exec(c)){return f(parseInt(d[1]+d[1],16),parseInt(d[2]+d[2],16),parseInt(d[3]+d[3],16))}var e=b.trim(c).toLowerCase();if(e=="transparent"){return f(255,255,255,0)}else{d=a[e]||[0,0,0];return f(d[0],d[1],d[2])}};var a={aqua:[0,255,255],azure:[240,255,255],beige:[245,245,220],black:[0,0,0],blue:[0,0,255],brown:[165,42,42],cyan:[0,255,255],darkblue:[0,0,139],darkcyan:[0,139,139],darkgrey:[169,169,169],darkgreen:[0,100,0],darkkhaki:[189,183,107],darkmagenta:[139,0,139],darkolivegreen:[85,107,47],darkorange:[255,140,0],darkorchid:[153,50,204],darkred:[139,0,0],darksalmon:[233,150,122],darkviolet:[148,0,211],fuchsia:[255,0,255],gold:[255,215,0],green:[0,128,0],indigo:[75,0,130],khaki:[240,230,140],lightblue:[173,216,230],lightcyan:[224,255,255],lightgreen:[144,238,144],lightgrey:[211,211,211],lightpink:[255,182,193],lightyellow:[255,255,224],lime:[0,255,0],magenta:[255,0,255],maroon:[128,0,0],navy:[0,0,128],olive:[128,128,0],orange:[255,165,0],pink:[255,192,203],purple:[128,0,128],violet:[128,0,128],red:[255,0,0],silver:[192,192,192],white:[255,255,255],yellow:[255,255,0]}})(jQuery);(function(c){function b(av,ai,J,af){var Q=[],O={colors:["#edc240","#afd8f8","#cb4b4b","#4da74d","#9440ed"],legend:{show:true,noColumns:1,labelFormatter:null,labelBoxBorderColor:"#ccc",container:null,position:"ne",margin:5,backgroundColor:null,backgroundOpacity:0.85},xaxis:{show:null,position:"bottom",mode:null,color:null,tickColor:null,transform:null,inverseTransform:null,min:null,max:null,autoscaleMargin:null,ticks:null,tickFormatter:null,labelWidth:null,labelHeight:null,reserveSpace:null,tickLength:null,alignTicksWithAxis:null,tickDecimals:null,tickSize:null,minTickSize:null,monthNames:null,timeformat:null,twelveHourClock:false},yaxis:{autoscaleMargin:0.02,position:"left"},xaxes:[],yaxes:[],series:{points:{show:false,radius:3,lineWidth:2,fill:true,fillColor:"#ffffff",symbol:"circle"},lines:{lineWidth:2,fill:false,fillColor:null,steps:false},bars:{show:false,lineWidth:2,barWidth:1,fill:true,fillColor:null,align:"left",horizontal:false},shadowSize:3},grid:{show:true,aboveData:false,color:"#545454",backgroundColor:null,borderColor:null,tickColor:null,labelMargin:5,axisMargin:8,borderWidth:2,minBorderMargin:null,markings:null,markingsColor:"#f4f4f4",markingsLineWidth:2,clickable:false,hoverable:false,autoHighlight:true,mouseActiveRadius:10},hooks:{}},az=null,ad=null,y=null,H=null,A=null,p=[],aw=[],q={left:0,right:0,top:0,bottom:0},G=0,I=0,h=0,w=0,ak={processOptions:[],processRawData:[],processDatapoints:[],drawSeries:[],draw:[],bindEvents:[],drawOverlay:[],shutdown:[]},aq=this;aq.setData=aj;aq.setupGrid=t;aq.draw=W;aq.getPlaceholder=function(){return av};aq.getCanvas=function(){return az};aq.getPlotOffset=function(){return q};aq.width=function(){return h};aq.height=function(){return w};aq.offset=function(){var aB=y.offset();aB.left+=q.left;aB.top+=q.top;return aB};aq.getData=function(){return Q};aq.getAxes=function(){var aC={},aB;c.each(p.concat(aw),function(aD,aE){if(aE){aC[aE.direction+(aE.n!=1?aE.n:"")+"axis"]=aE}});return aC};aq.getXAxes=function(){return p};aq.getYAxes=function(){return aw};aq.c2p=C;aq.p2c=ar;aq.getOptions=function(){return O};aq.highlight=x;aq.unhighlight=T;aq.triggerRedrawOverlay=f;aq.pointOffset=function(aB){return{left:parseInt(p[aA(aB,"x")-1].p2c(+aB.x)+q.left),top:parseInt(aw[aA(aB,"y")-1].p2c(+aB.y)+q.top)}};aq.shutdown=ag;aq.resize=function(){B();g(az);g(ad)};aq.hooks=ak;F(aq);Z(J);X();aj(ai);t();W();ah();function an(aD,aB){aB=[aq].concat(aB);for(var aC=0;aC<aD.length;++aC){aD[aC].apply(this,aB)}}function F(){for(var aB=0;aB<af.length;++aB){var aC=af[aB];aC.init(aq);if(aC.options){c.extend(true,O,aC.options)}}}function Z(aC){var aB;c.extend(true,O,aC);if(O.xaxis.color==null){O.xaxis.color=O.grid.color}if(O.yaxis.color==null){O.yaxis.color=O.grid.color}if(O.xaxis.tickColor==null){O.xaxis.tickColor=O.grid.tickColor}if(O.yaxis.tickColor==null){O.yaxis.tickColor=O.grid.tickColor}if(O.grid.borderColor==null){O.grid.borderColor=O.grid.color}if(O.grid.tickColor==null){O.grid.tickColor=c.color.parse(O.grid.color).scale("a",0.22).toString()}for(aB=0;aB<Math.max(1,O.xaxes.length);++aB){O.xaxes[aB]=c.extend(true,{},O.xaxis,O.xaxes[aB])}for(aB=0;aB<Math.max(1,O.yaxes.length);++aB){O.yaxes[aB]=c.extend(true,{},O.yaxis,O.yaxes[aB])}if(O.xaxis.noTicks&&O.xaxis.ticks==null){O.xaxis.ticks=O.xaxis.noTicks}if(O.yaxis.noTicks&&O.yaxis.ticks==null){O.yaxis.ticks=O.yaxis.noTicks}if(O.x2axis){O.xaxes[1]=c.extend(true,{},O.xaxis,O.x2axis);O.xaxes[1].position="top"}if(O.y2axis){O.yaxes[1]=c.extend(true,{},O.yaxis,O.y2axis);O.yaxes[1].position="right"}if(O.grid.coloredAreas){O.grid.markings=O.grid.coloredAreas}if(O.grid.coloredAreasColor){O.grid.markingsColor=O.grid.coloredAreasColor}if(O.lines){c.extend(true,O.series.lines,O.lines)}if(O.points){c.extend(true,O.series.points,O.points)}if(O.bars){c.extend(true,O.series.bars,O.bars)}if(O.shadowSize!=null){O.series.shadowSize=O.shadowSize}for(aB=0;aB<O.xaxes.length;++aB){V(p,aB+1).options=O.xaxes[aB]}for(aB=0;aB<O.yaxes.length;++aB){V(aw,aB+1).options=O.yaxes[aB]}for(var aD in ak){if(O.hooks[aD]&&O.hooks[aD].length){ak[aD]=ak[aD].concat(O.hooks[aD])}}an(ak.processOptions,[O])}function aj(aB){Q=Y(aB);ax();z()}function Y(aE){var aC=[];for(var aB=0;aB<aE.length;++aB){var aD=c.extend(true,{},O.series);if(aE[aB].data!=null){aD.data=aE[aB].data;delete aE[aB].data;c.extend(true,aD,aE[aB]);aE[aB].data=aD.data}else{aD.data=aE[aB]}aC.push(aD)}return aC}function aA(aC,aD){var aB=aC[aD+"axis"];if(typeof aB=="object"){aB=aB.n}if(typeof aB!="number"){aB=1}return aB}function m(){return c.grep(p.concat(aw),function(aB){return aB})}function C(aE){var aC={},aB,aD;for(aB=0;aB<p.length;++aB){aD=p[aB];if(aD&&aD.used){aC["x"+aD.n]=aD.c2p(aE.left)}}for(aB=0;aB<aw.length;++aB){aD=aw[aB];if(aD&&aD.used){aC["y"+aD.n]=aD.c2p(aE.top)}}if(aC.x1!==undefined){aC.x=aC.x1}if(aC.y1!==undefined){aC.y=aC.y1}return aC}function ar(aF){var aD={},aC,aE,aB;for(aC=0;aC<p.length;++aC){aE=p[aC];if(aE&&aE.used){aB="x"+aE.n;if(aF[aB]==null&&aE.n==1){aB="x"}if(aF[aB]!=null){aD.left=aE.p2c(aF[aB]);break}}}for(aC=0;aC<aw.length;++aC){aE=aw[aC];if(aE&&aE.used){aB="y"+aE.n;if(aF[aB]==null&&aE.n==1){aB="y"}if(aF[aB]!=null){aD.top=aE.p2c(aF[aB]);break}}}return aD}function V(aC,aB){if(!aC[aB-1]){aC[aB-1]={n:aB,direction:aC==p?"x":"y",options:c.extend(true,{},aC==p?O.xaxis:O.yaxis)}}return aC[aB-1]}function ax(){var aG;var aM=Q.length,aB=[],aE=[];for(aG=0;aG<Q.length;++aG){var aJ=Q[aG].color;if(aJ!=null){--aM;if(typeof aJ=="number"){aE.push(aJ)}else{aB.push(c.color.parse(Q[aG].color))}}}for(aG=0;aG<aE.length;++aG){aM=Math.max(aM,aE[aG]+1)}var aC=[],aF=0;aG=0;while(aC.length<aM){var aI;if(O.colors.length==aG){aI=c.color.make(100,100,100)}else{aI=c.color.parse(O.colors[aG])}var aD=aF%2==1?-1:1;aI.scale("rgb",1+aD*Math.ceil(aF/2)*0.2);aC.push(aI);++aG;if(aG>=O.colors.length){aG=0;++aF}}var aH=0,aN;for(aG=0;aG<Q.length;++aG){aN=Q[aG];if(aN.color==null){aN.color=aC[aH].toString();++aH}else{if(typeof aN.color=="number"){aN.color=aC[aN.color].toString()}}if(aN.lines.show==null){var aL,aK=true;for(aL in aN){if(aN[aL]&&aN[aL].show){aK=false;break}}if(aK){aN.lines.show=true}}aN.xaxis=V(p,aA(aN,"x"));aN.yaxis=V(aw,aA(aN,"y"))}}function z(){var aO=Number.POSITIVE_INFINITY,aI=Number.NEGATIVE_INFINITY,aB=Number.MAX_VALUE,aU,aS,aR,aN,aD,aJ,aT,aP,aH,aG,aC,a0,aX,aL;function aF(a3,a2,a1){if(a2<a3.datamin&&a2!=-aB){a3.datamin=a2}if(a1>a3.datamax&&a1!=aB){a3.datamax=a1}}c.each(m(),function(a1,a2){a2.datamin=aO;a2.datamax=aI;a2.used=false});for(aU=0;aU<Q.length;++aU){aJ=Q[aU];aJ.datapoints={points:[]};an(ak.processRawData,[aJ,aJ.data,aJ.datapoints])}for(aU=0;aU<Q.length;++aU){aJ=Q[aU];var aZ=aJ.data,aW=aJ.datapoints.format;if(!aW){aW=[];aW.push({x:true,number:true,required:true});aW.push({y:true,number:true,required:true});if(aJ.bars.show||(aJ.lines.show&&aJ.lines.fill)){aW.push({y:true,number:true,required:false,defaultValue:0});if(aJ.bars.horizontal){delete aW[aW.length-1].y;aW[aW.length-1].x=true}}aJ.datapoints.format=aW}if(aJ.datapoints.pointsize!=null){continue}aJ.datapoints.pointsize=aW.length;aP=aJ.datapoints.pointsize;aT=aJ.datapoints.points;insertSteps=aJ.lines.show&&aJ.lines.steps;aJ.xaxis.used=aJ.yaxis.used=true;for(aS=aR=0;aS<aZ.length;++aS,aR+=aP){aL=aZ[aS];var aE=aL==null;if(!aE){for(aN=0;aN<aP;++aN){a0=aL[aN];aX=aW[aN];if(aX){if(aX.number&&a0!=null){a0=+a0;if(isNaN(a0)){a0=null}else{if(a0==Infinity){a0=aB}else{if(a0==-Infinity){a0=-aB}}}}if(a0==null){if(aX.required){aE=true}if(aX.defaultValue!=null){a0=aX.defaultValue}}}aT[aR+aN]=a0}}if(aE){for(aN=0;aN<aP;++aN){a0=aT[aR+aN];if(a0!=null){aX=aW[aN];if(aX.x){aF(aJ.xaxis,a0,a0)}if(aX.y){aF(aJ.yaxis,a0,a0)}}aT[aR+aN]=null}}else{if(insertSteps&&aR>0&&aT[aR-aP]!=null&&aT[aR-aP]!=aT[aR]&&aT[aR-aP+1]!=aT[aR+1]){for(aN=0;aN<aP;++aN){aT[aR+aP+aN]=aT[aR+aN]}aT[aR+1]=aT[aR-aP+1];aR+=aP}}}}for(aU=0;aU<Q.length;++aU){aJ=Q[aU];an(ak.processDatapoints,[aJ,aJ.datapoints])}for(aU=0;aU<Q.length;++aU){aJ=Q[aU];aT=aJ.datapoints.points,aP=aJ.datapoints.pointsize;var aK=aO,aQ=aO,aM=aI,aV=aI;for(aS=0;aS<aT.length;aS+=aP){if(aT[aS]==null){continue}for(aN=0;aN<aP;++aN){a0=aT[aS+aN];aX=aW[aN];if(!aX||a0==aB||a0==-aB){continue}if(aX.x){if(a0<aK){aK=a0}if(a0>aM){aM=a0}}if(aX.y){if(a0<aQ){aQ=a0}if(a0>aV){aV=a0}}}}if(aJ.bars.show){var aY=aJ.bars.align=="left"?0:-aJ.bars.barWidth/2;if(aJ.bars.horizontal){aQ+=aY;aV+=aY+aJ.bars.barWidth}else{aK+=aY;aM+=aY+aJ.bars.barWidth}}aF(aJ.xaxis,aK,aM);aF(aJ.yaxis,aQ,aV)}c.each(m(),function(a1,a2){if(a2.datamin==aO){a2.datamin=null}if(a2.datamax==aI){a2.datamax=null}})}function j(aB,aC){var aD=document.createElement("canvas");aD.className=aC;aD.width=G;aD.height=I;if(!aB){c(aD).css({position:"absolute",left:0,top:0})}c(aD).appendTo(av);if(!aD.getContext){aD=window.G_vmlCanvasManager.initElement(aD)}aD.getContext("2d").save();return aD}function B(){G=av.width();I=av.height();if(G<=0||I<=0){throw"Invalid dimensions for plot, width = "+G+", height = "+I}}function g(aC){if(aC.width!=G){aC.width=G}if(aC.height!=I){aC.height=I}var aB=aC.getContext("2d");aB.restore();aB.save()}function X(){var aC,aB=av.children("canvas.base"),aD=av.children("canvas.overlay");if(aB.length==0||aD==0){av.html("");av.css({padding:0});if(av.css("position")=="static"){av.css("position","relative")}B();az=j(true,"base");ad=j(false,"overlay");aC=false}else{az=aB.get(0);ad=aD.get(0);aC=true}H=az.getContext("2d");A=ad.getContext("2d");y=c([ad,az]);if(aC){av.data("plot").shutdown();aq.resize();A.clearRect(0,0,G,I);y.unbind();av.children().not([az,ad]).remove()}av.data("plot",aq)}function ah(){if(O.grid.hoverable){y.mousemove(aa);y.mouseleave(l)}if(O.grid.clickable){y.click(R)}an(ak.bindEvents,[y])}function ag(){if(M){clearTimeout(M)}y.unbind("mousemove",aa);y.unbind("mouseleave",l);y.unbind("click",R);an(ak.shutdown,[y])}function r(aG){function aC(aH){return aH}var aF,aB,aD=aG.options.transform||aC,aE=aG.options.inverseTransform;if(aG.direction=="x"){aF=aG.scale=h/Math.abs(aD(aG.max)-aD(aG.min));aB=Math.min(aD(aG.max),aD(aG.min))}else{aF=aG.scale=w/Math.abs(aD(aG.max)-aD(aG.min));aF=-aF;aB=Math.max(aD(aG.max),aD(aG.min))}if(aD==aC){aG.p2c=function(aH){return(aH-aB)*aF}}else{aG.p2c=function(aH){return(aD(aH)-aB)*aF}}if(!aE){aG.c2p=function(aH){return aB+aH/aF}}else{aG.c2p=function(aH){return aE(aB+aH/aF)}}}function L(aD){var aB=aD.options,aF,aJ=aD.ticks||[],aI=[],aE,aK=aB.labelWidth,aG=aB.labelHeight,aC;function aH(aM,aL){return c('<div style="position:absolute;top:-10000px;'+aL+'font-size:smaller"><div class="'+aD.direction+"Axis "+aD.direction+aD.n+'Axis">'+aM.join("")+"</div></div>").appendTo(av)}if(aD.direction=="x"){if(aK==null){aK=Math.floor(G/(aJ.length>0?aJ.length:1))}if(aG==null){aI=[];for(aF=0;aF<aJ.length;++aF){aE=aJ[aF].label;if(aE){aI.push('<div class="tickLabel" style="float:left;width:'+aK+'px">'+aE+"</div>")}}if(aI.length>0){aI.push('<div style="clear:left"></div>');aC=aH(aI,"width:10000px;");aG=aC.height();aC.remove()}}}else{if(aK==null||aG==null){for(aF=0;aF<aJ.length;++aF){aE=aJ[aF].label;if(aE){aI.push('<div class="tickLabel">'+aE+"</div>")}}if(aI.length>0){aC=aH(aI,"");if(aK==null){aK=aC.children().width()}if(aG==null){aG=aC.find("div.tickLabel").height()}aC.remove()}}}if(aK==null){aK=0}if(aG==null){aG=0}aD.labelWidth=aK;aD.labelHeight=aG}function au(aD){var aC=aD.labelWidth,aL=aD.labelHeight,aH=aD.options.position,aF=aD.options.tickLength,aG=O.grid.axisMargin,aJ=O.grid.labelMargin,aK=aD.direction=="x"?p:aw,aE;var aB=c.grep(aK,function(aN){return aN&&aN.options.position==aH&&aN.reserveSpace});if(c.inArray(aD,aB)==aB.length-1){aG=0}if(aF==null){aF="full"}var aI=c.grep(aK,function(aN){return aN&&aN.reserveSpace});var aM=c.inArray(aD,aI)==0;if(!aM&&aF=="full"){aF=5}if(!isNaN(+aF)){aJ+=+aF}if(aD.direction=="x"){aL+=aJ;if(aH=="bottom"){q.bottom+=aL+aG;aD.box={top:I-q.bottom,height:aL}}else{aD.box={top:q.top+aG,height:aL};q.top+=aL+aG}}else{aC+=aJ;if(aH=="left"){aD.box={left:q.left+aG,width:aC};q.left+=aC+aG}else{q.right+=aC+aG;aD.box={left:G-q.right,width:aC}}}aD.position=aH;aD.tickLength=aF;aD.box.padding=aJ;aD.innermost=aM}function U(aB){if(aB.direction=="x"){aB.box.left=q.left;aB.box.width=h}else{aB.box.top=q.top;aB.box.height=w}}function t(){var aC,aE=m();c.each(aE,function(aF,aG){aG.show=aG.options.show;if(aG.show==null){aG.show=aG.used}aG.reserveSpace=aG.show||aG.options.reserveSpace;n(aG)});allocatedAxes=c.grep(aE,function(aF){return aF.reserveSpace});q.left=q.right=q.top=q.bottom=0;if(O.grid.show){c.each(allocatedAxes,function(aF,aG){S(aG);P(aG);ap(aG,aG.ticks);L(aG)});for(aC=allocatedAxes.length-1;aC>=0;--aC){au(allocatedAxes[aC])}var aD=O.grid.minBorderMargin;if(aD==null){aD=0;for(aC=0;aC<Q.length;++aC){aD=Math.max(aD,Q[aC].points.radius+Q[aC].points.lineWidth/2)}}for(var aB in q){q[aB]+=O.grid.borderWidth;q[aB]=Math.max(aD,q[aB])}}h=G-q.left-q.right;w=I-q.bottom-q.top;c.each(aE,function(aF,aG){r(aG)});if(O.grid.show){c.each(allocatedAxes,function(aF,aG){U(aG)});k()}o()}function n(aE){var aF=aE.options,aD=+(aF.min!=null?aF.min:aE.datamin),aB=+(aF.max!=null?aF.max:aE.datamax),aH=aB-aD;if(aH==0){var aC=aB==0?1:0.01;if(aF.min==null){aD-=aC}if(aF.max==null||aF.min!=null){aB+=aC}}else{var aG=aF.autoscaleMargin;if(aG!=null){if(aF.min==null){aD-=aH*aG;if(aD<0&&aE.datamin!=null&&aE.datamin>=0){aD=0}}if(aF.max==null){aB+=aH*aG;if(aB>0&&aE.datamax!=null&&aE.datamax<=0){aB=0}}}}aE.min=aD;aE.max=aB}function S(aG){var aM=aG.options;var aH;if(typeof aM.ticks=="number"&&aM.ticks>0){aH=aM.ticks}else{aH=0.3*Math.sqrt(aG.direction=="x"?G:I)}var aT=(aG.max-aG.min)/aH,aO,aB,aN,aR,aS,aQ,aI;if(aM.mode=="time"){var aJ={second:1000,minute:60*1000,hour:60*60*1000,day:24*60*60*1000,month:30*24*60*60*1000,year:365.2425*24*60*60*1000};var aK=[[1,"second"],[2,"second"],[5,"second"],[10,"second"],[30,"second"],[1,"minute"],[2,"minute"],[5,"minute"],[10,"minute"],[30,"minute"],[1,"hour"],[2,"hour"],[4,"hour"],[8,"hour"],[12,"hour"],[1,"day"],[2,"day"],[3,"day"],[0.25,"month"],[0.5,"month"],[1,"month"],[2,"month"],[3,"month"],[6,"month"],[1,"year"]];var aC=0;if(aM.minTickSize!=null){if(typeof aM.tickSize=="number"){aC=aM.tickSize}else{aC=aM.minTickSize[0]*aJ[aM.minTickSize[1]]}}for(var aS=0;aS<aK.length-1;++aS){if(aT<(aK[aS][0]*aJ[aK[aS][1]]+aK[aS+1][0]*aJ[aK[aS+1][1]])/2&&aK[aS][0]*aJ[aK[aS][1]]>=aC){break}}aO=aK[aS][0];aN=aK[aS][1];if(aN=="year"){aQ=Math.pow(10,Math.floor(Math.log(aT/aJ.year)/Math.LN10));aI=(aT/aJ.year)/aQ;if(aI<1.5){aO=1}else{if(aI<3){aO=2}else{if(aI<7.5){aO=5}else{aO=10}}}aO*=aQ}aG.tickSize=aM.tickSize||[aO,aN];aB=function(aX){var a2=[],a0=aX.tickSize[0],a3=aX.tickSize[1],a1=new Date(aX.min);var aW=a0*aJ[a3];if(a3=="second"){a1.setUTCSeconds(a(a1.getUTCSeconds(),a0))}if(a3=="minute"){a1.setUTCMinutes(a(a1.getUTCMinutes(),a0))}if(a3=="hour"){a1.setUTCHours(a(a1.getUTCHours(),a0))}if(a3=="month"){a1.setUTCMonth(a(a1.getUTCMonth(),a0))}if(a3=="year"){a1.setUTCFullYear(a(a1.getUTCFullYear(),a0))}a1.setUTCMilliseconds(0);if(aW>=aJ.minute){a1.setUTCSeconds(0)}if(aW>=aJ.hour){a1.setUTCMinutes(0)}if(aW>=aJ.day){a1.setUTCHours(0)}if(aW>=aJ.day*4){a1.setUTCDate(1)}if(aW>=aJ.year){a1.setUTCMonth(0)}var a5=0,a4=Number.NaN,aY;do{aY=a4;a4=a1.getTime();a2.push(a4);if(a3=="month"){if(a0<1){a1.setUTCDate(1);var aV=a1.getTime();a1.setUTCMonth(a1.getUTCMonth()+1);var aZ=a1.getTime();a1.setTime(a4+a5*aJ.hour+(aZ-aV)*a0);a5=a1.getUTCHours();a1.setUTCHours(0)}else{a1.setUTCMonth(a1.getUTCMonth()+a0)}}else{if(a3=="year"){a1.setUTCFullYear(a1.getUTCFullYear()+a0)}else{a1.setTime(a4+aW)}}}while(a4<aX.max&&a4!=aY);return a2};aR=function(aV,aY){var a0=new Date(aV);if(aM.timeformat!=null){return c.plot.formatDate(a0,aM.timeformat,aM.monthNames)}var aW=aY.tickSize[0]*aJ[aY.tickSize[1]];var aX=aY.max-aY.min;var aZ=(aM.twelveHourClock)?" %p":"";if(aW<aJ.minute){fmt="%h:%M:%S"+aZ}else{if(aW<aJ.day){if(aX<2*aJ.day){fmt="%h:%M"+aZ}else{fmt="%b %d %h:%M"+aZ}}else{if(aW<aJ.month){fmt="%b %d"}else{if(aW<aJ.year){if(aX<aJ.year){fmt="%b"}else{fmt="%b %y"}}else{fmt="%y"}}}}return c.plot.formatDate(a0,fmt,aM.monthNames)}}else{var aU=aM.tickDecimals;var aP=-Math.floor(Math.log(aT)/Math.LN10);if(aU!=null&&aP>aU){aP=aU}aQ=Math.pow(10,-aP);aI=aT/aQ;if(aI<1.5){aO=1}else{if(aI<3){aO=2;if(aI>2.25&&(aU==null||aP+1<=aU)){aO=2.5;++aP}}else{if(aI<7.5){aO=5}else{aO=10}}}aO*=aQ;if(aM.minTickSize!=null&&aO<aM.minTickSize){aO=aM.minTickSize}aG.tickDecimals=Math.max(0,aU!=null?aU:aP);aG.tickSize=aM.tickSize||aO;aB=function(aX){var aZ=[];var a0=a(aX.min,aX.tickSize),aW=0,aV=Number.NaN,aY;do{aY=aV;aV=a0+aW*aX.tickSize;aZ.push(aV);++aW}while(aV<aX.max&&aV!=aY);return aZ};aR=function(aV,aW){return aV.toFixed(aW.tickDecimals)}}if(aM.alignTicksWithAxis!=null){var aF=(aG.direction=="x"?p:aw)[aM.alignTicksWithAxis-1];if(aF&&aF.used&&aF!=aG){var aL=aB(aG);if(aL.length>0){if(aM.min==null){aG.min=Math.min(aG.min,aL[0])}if(aM.max==null&&aL.length>1){aG.max=Math.max(aG.max,aL[aL.length-1])}}aB=function(aX){var aY=[],aV,aW;for(aW=0;aW<aF.ticks.length;++aW){aV=(aF.ticks[aW].v-aF.min)/(aF.max-aF.min);aV=aX.min+aV*(aX.max-aX.min);aY.push(aV)}return aY};if(aG.mode!="time"&&aM.tickDecimals==null){var aE=Math.max(0,-Math.floor(Math.log(aT)/Math.LN10)+1),aD=aB(aG);if(!(aD.length>1&&/\..*0$/.test((aD[1]-aD[0]).toFixed(aE)))){aG.tickDecimals=aE}}}}aG.tickGenerator=aB;if(c.isFunction(aM.tickFormatter)){aG.tickFormatter=function(aV,aW){return""+aM.tickFormatter(aV,aW)}}else{aG.tickFormatter=aR}}function P(aF){var aH=aF.options.ticks,aG=[];if(aH==null||(typeof aH=="number"&&aH>0)){aG=aF.tickGenerator(aF)}else{if(aH){if(c.isFunction(aH)){aG=aH({min:aF.min,max:aF.max})}else{aG=aH}}}var aE,aB;aF.ticks=[];for(aE=0;aE<aG.length;++aE){var aC=null;var aD=aG[aE];if(typeof aD=="object"){aB=+aD[0];if(aD.length>1){aC=aD[1]}}else{aB=+aD}if(aC==null){aC=aF.tickFormatter(aB,aF)}if(!isNaN(aB)){aF.ticks.push({v:aB,label:aC})}}}function ap(aB,aC){if(aB.options.autoscaleMargin&&aC.length>0){if(aB.options.min==null){aB.min=Math.min(aB.min,aC[0].v)}if(aB.options.max==null&&aC.length>1){aB.max=Math.max(aB.max,aC[aC.length-1].v)}}}function W(){H.clearRect(0,0,G,I);var aC=O.grid;if(aC.show&&aC.backgroundColor){N()}if(aC.show&&!aC.aboveData){ac()}for(var aB=0;aB<Q.length;++aB){an(ak.drawSeries,[H,Q[aB]]);d(Q[aB])}an(ak.draw,[H]);if(aC.show&&aC.aboveData){ac()}}function D(aB,aI){var aE,aH,aG,aD,aF=m();for(i=0;i<aF.length;++i){aE=aF[i];if(aE.direction==aI){aD=aI+aE.n+"axis";if(!aB[aD]&&aE.n==1){aD=aI+"axis"}if(aB[aD]){aH=aB[aD].from;aG=aB[aD].to;break}}}if(!aB[aD]){aE=aI=="x"?p[0]:aw[0];aH=aB[aI+"1"];aG=aB[aI+"2"]}if(aH!=null&&aG!=null&&aH>aG){var aC=aH;aH=aG;aG=aC}return{from:aH,to:aG,axis:aE}}function N(){H.save();H.translate(q.left,q.top);H.fillStyle=am(O.grid.backgroundColor,w,0,"rgba(255, 255, 255, 0)");H.fillRect(0,0,h,w);H.restore()}function ac(){var aF;H.save();H.translate(q.left,q.top);var aH=O.grid.markings;if(aH){if(c.isFunction(aH)){var aK=aq.getAxes();aK.xmin=aK.xaxis.min;aK.xmax=aK.xaxis.max;aK.ymin=aK.yaxis.min;aK.ymax=aK.yaxis.max;aH=aH(aK)}for(aF=0;aF<aH.length;++aF){var aD=aH[aF],aC=D(aD,"x"),aI=D(aD,"y");if(aC.from==null){aC.from=aC.axis.min}if(aC.to==null){aC.to=aC.axis.max}if(aI.from==null){aI.from=aI.axis.min}if(aI.to==null){aI.to=aI.axis.max}if(aC.to<aC.axis.min||aC.from>aC.axis.max||aI.to<aI.axis.min||aI.from>aI.axis.max){continue}aC.from=Math.max(aC.from,aC.axis.min);aC.to=Math.min(aC.to,aC.axis.max);aI.from=Math.max(aI.from,aI.axis.min);aI.to=Math.min(aI.to,aI.axis.max);if(aC.from==aC.to&&aI.from==aI.to){continue}aC.from=aC.axis.p2c(aC.from);aC.to=aC.axis.p2c(aC.to);aI.from=aI.axis.p2c(aI.from);aI.to=aI.axis.p2c(aI.to);if(aC.from==aC.to||aI.from==aI.to){H.beginPath();H.strokeStyle=aD.color||O.grid.markingsColor;H.lineWidth=aD.lineWidth||O.grid.markingsLineWidth;H.moveTo(aC.from,aI.from);H.lineTo(aC.to,aI.to);H.stroke()}else{H.fillStyle=aD.color||O.grid.markingsColor;H.fillRect(aC.from,aI.to,aC.to-aC.from,aI.from-aI.to)}}}var aK=m(),aM=O.grid.borderWidth;for(var aE=0;aE<aK.length;++aE){var aB=aK[aE],aG=aB.box,aQ=aB.tickLength,aN,aL,aP,aJ;if(!aB.show||aB.ticks.length==0){continue}H.strokeStyle=aB.options.tickColor||c.color.parse(aB.options.color).scale("a",0.22).toString();H.lineWidth=aB.options.lineWidth;if(aB.direction=="x"){aN=0;if(aQ=="full"){aL=(aB.position=="top"?0:w)}else{aL=aG.top-q.top+(aB.position=="top"?aG.height:0)}}else{aL=0;if(aQ=="full"){aN=(aB.position=="left"?0:h)}else{aN=aG.left-q.left+(aB.position=="left"?aG.width:0)}}if(!aB.innermost){H.beginPath();aP=aJ=0;if(aB.direction=="x"){aP=h}else{aJ=w}if(H.lineWidth==1){aN=Math.floor(aN)+0.5;aL=Math.floor(aL)+0.5}H.moveTo(aN,aL);H.lineTo(aN+aP,aL+aJ);H.stroke()}H.beginPath();for(aF=0;aF<aB.ticks.length;++aF){var aO=aB.ticks[aF].v;aP=aJ=0;if(aO<aB.min||aO>aB.max||(aQ=="full"&&aM>0&&(aO==aB.min||aO==aB.max))){continue}if(aB.direction=="x"){aN=aB.p2c(aO);aJ=aQ=="full"?-w:aQ;if(aB.position=="top"){aJ=-aJ}}else{aL=aB.p2c(aO);aP=aQ=="full"?-h:aQ;if(aB.position=="left"){aP=-aP}}if(H.lineWidth==1){if(aB.direction=="x"){aN=Math.floor(aN)+0.5}else{aL=Math.floor(aL)+0.5}}H.moveTo(aN,aL);H.lineTo(aN+aP,aL+aJ)}H.stroke()}if(aM){H.lineWidth=aM;H.strokeStyle=O.grid.borderColor;H.strokeRect(-aM/2,-aM/2,h+aM,w+aM)}H.restore()}function k(){av.find(".tickLabels").remove();var aG=['<div class="tickLabels" style="font-size:smaller">'];var aJ=m();for(var aD=0;aD<aJ.length;++aD){var aC=aJ[aD],aF=aC.box;if(!aC.show){continue}aG.push('<div class="'+aC.direction+"Axis "+aC.direction+aC.n+'Axis" style="color:'+aC.options.color+'">');for(var aE=0;aE<aC.ticks.length;++aE){var aH=aC.ticks[aE];if(!aH.label||aH.v<aC.min||aH.v>aC.max){continue}var aK={},aI;if(aC.direction=="x"){aI="center";aK.left=Math.round(q.left+aC.p2c(aH.v)-aC.labelWidth/2);if(aC.position=="bottom"){aK.top=aF.top+aF.padding}else{aK.bottom=I-(aF.top+aF.height-aF.padding)}}else{aK.top=Math.round(q.top+aC.p2c(aH.v)-aC.labelHeight/2);if(aC.position=="left"){aK.right=G-(aF.left+aF.width-aF.padding);aI="right"}else{aK.left=aF.left+aF.padding;aI="left"}}aK.width=aC.labelWidth;var aB=["position:absolute","text-align:"+aI];for(var aL in aK){aB.push(aL+":"+aK[aL]+"px")}aG.push('<div class="tickLabel" style="'+aB.join(";")+'">'+aH.label+"</div>")}aG.push("</div>")}aG.push("</div>");av.append(aG.join(""))}function d(aB){if(aB.lines.show){at(aB)}if(aB.bars.show){e(aB)}if(aB.points.show){ao(aB)}}function at(aE){function aD(aP,aQ,aI,aU,aT){var aV=aP.points,aJ=aP.pointsize,aN=null,aM=null;H.beginPath();for(var aO=aJ;aO<aV.length;aO+=aJ){var aL=aV[aO-aJ],aS=aV[aO-aJ+1],aK=aV[aO],aR=aV[aO+1];if(aL==null||aK==null){continue}if(aS<=aR&&aS<aT.min){if(aR<aT.min){continue}aL=(aT.min-aS)/(aR-aS)*(aK-aL)+aL;aS=aT.min}else{if(aR<=aS&&aR<aT.min){if(aS<aT.min){continue}aK=(aT.min-aS)/(aR-aS)*(aK-aL)+aL;aR=aT.min}}if(aS>=aR&&aS>aT.max){if(aR>aT.max){continue}aL=(aT.max-aS)/(aR-aS)*(aK-aL)+aL;aS=aT.max}else{if(aR>=aS&&aR>aT.max){if(aS>aT.max){continue}aK=(aT.max-aS)/(aR-aS)*(aK-aL)+aL;aR=aT.max}}if(aL<=aK&&aL<aU.min){if(aK<aU.min){continue}aS=(aU.min-aL)/(aK-aL)*(aR-aS)+aS;aL=aU.min}else{if(aK<=aL&&aK<aU.min){if(aL<aU.min){continue}aR=(aU.min-aL)/(aK-aL)*(aR-aS)+aS;aK=aU.min}}if(aL>=aK&&aL>aU.max){if(aK>aU.max){continue}aS=(aU.max-aL)/(aK-aL)*(aR-aS)+aS;aL=aU.max}else{if(aK>=aL&&aK>aU.max){if(aL>aU.max){continue}aR=(aU.max-aL)/(aK-aL)*(aR-aS)+aS;aK=aU.max}}if(aL!=aN||aS!=aM){H.moveTo(aU.p2c(aL)+aQ,aT.p2c(aS)+aI)}aN=aK;aM=aR;H.lineTo(aU.p2c(aK)+aQ,aT.p2c(aR)+aI)}H.stroke()}function aF(aI,aQ,aP){var aW=aI.points,aV=aI.pointsize,aN=Math.min(Math.max(0,aP.min),aP.max),aX=0,aU,aT=false,aM=1,aL=0,aR=0;while(true){if(aV>0&&aX>aW.length+aV){break}aX+=aV;var aZ=aW[aX-aV],aK=aW[aX-aV+aM],aY=aW[aX],aJ=aW[aX+aM];if(aT){if(aV>0&&aZ!=null&&aY==null){aR=aX;aV=-aV;aM=2;continue}if(aV<0&&aX==aL+aV){H.fill();aT=false;aV=-aV;aM=1;aX=aL=aR+aV;continue}}if(aZ==null||aY==null){continue}if(aZ<=aY&&aZ<aQ.min){if(aY<aQ.min){continue}aK=(aQ.min-aZ)/(aY-aZ)*(aJ-aK)+aK;aZ=aQ.min}else{if(aY<=aZ&&aY<aQ.min){if(aZ<aQ.min){continue}aJ=(aQ.min-aZ)/(aY-aZ)*(aJ-aK)+aK;aY=aQ.min}}if(aZ>=aY&&aZ>aQ.max){if(aY>aQ.max){continue}aK=(aQ.max-aZ)/(aY-aZ)*(aJ-aK)+aK;aZ=aQ.max}else{if(aY>=aZ&&aY>aQ.max){if(aZ>aQ.max){continue}aJ=(aQ.max-aZ)/(aY-aZ)*(aJ-aK)+aK;aY=aQ.max}}if(!aT){H.beginPath();H.moveTo(aQ.p2c(aZ),aP.p2c(aN));aT=true}if(aK>=aP.max&&aJ>=aP.max){H.lineTo(aQ.p2c(aZ),aP.p2c(aP.max));H.lineTo(aQ.p2c(aY),aP.p2c(aP.max));continue}else{if(aK<=aP.min&&aJ<=aP.min){H.lineTo(aQ.p2c(aZ),aP.p2c(aP.min));H.lineTo(aQ.p2c(aY),aP.p2c(aP.min));continue}}var aO=aZ,aS=aY;if(aK<=aJ&&aK<aP.min&&aJ>=aP.min){aZ=(aP.min-aK)/(aJ-aK)*(aY-aZ)+aZ;aK=aP.min}else{if(aJ<=aK&&aJ<aP.min&&aK>=aP.min){aY=(aP.min-aK)/(aJ-aK)*(aY-aZ)+aZ;aJ=aP.min}}if(aK>=aJ&&aK>aP.max&&aJ<=aP.max){aZ=(aP.max-aK)/(aJ-aK)*(aY-aZ)+aZ;aK=aP.max}else{if(aJ>=aK&&aJ>aP.max&&aK<=aP.max){aY=(aP.max-aK)/(aJ-aK)*(aY-aZ)+aZ;aJ=aP.max}}if(aZ!=aO){H.lineTo(aQ.p2c(aO),aP.p2c(aK))}H.lineTo(aQ.p2c(aZ),aP.p2c(aK));H.lineTo(aQ.p2c(aY),aP.p2c(aJ));if(aY!=aS){H.lineTo(aQ.p2c(aY),aP.p2c(aJ));H.lineTo(aQ.p2c(aS),aP.p2c(aJ))}}}H.save();H.translate(q.left,q.top);H.lineJoin="round";var aG=aE.lines.lineWidth,aB=aE.shadowSize;if(aG>0&&aB>0){H.lineWidth=aB;H.strokeStyle="rgba(0,0,0,0.1)";var aH=Math.PI/18;aD(aE.datapoints,Math.sin(aH)*(aG/2+aB/2),Math.cos(aH)*(aG/2+aB/2),aE.xaxis,aE.yaxis);H.lineWidth=aB/2;aD(aE.datapoints,Math.sin(aH)*(aG/2+aB/4),Math.cos(aH)*(aG/2+aB/4),aE.xaxis,aE.yaxis)}H.lineWidth=aG;H.strokeStyle=aE.color;var aC=ae(aE.lines,aE.color,0,w);if(aC){H.fillStyle=aC;aF(aE.datapoints,aE.xaxis,aE.yaxis)}if(aG>0){aD(aE.datapoints,0,0,aE.xaxis,aE.yaxis)}H.restore()}function ao(aE){function aH(aN,aM,aU,aK,aS,aT,aQ,aJ){var aR=aN.points,aI=aN.pointsize;for(var aL=0;aL<aR.length;aL+=aI){var aP=aR[aL],aO=aR[aL+1];if(aP==null||aP<aT.min||aP>aT.max||aO<aQ.min||aO>aQ.max){continue}H.beginPath();aP=aT.p2c(aP);aO=aQ.p2c(aO)+aK;if(aJ=="circle"){H.arc(aP,aO,aM,0,aS?Math.PI:Math.PI*2,false)}else{aJ(H,aP,aO,aM,aS)}H.closePath();if(aU){H.fillStyle=aU;H.fill()}/*H.stroke()*/}}H.save();H.translate(q.left,q.top);var aG=aE.points.lineWidth,aC=aE.shadowSize,aB=aE.points.radius,aF=aE.points.symbol;if(aG>0&&aC>0){var aD=aC/2;H.lineWidth=aD;H.strokeStyle="rgba(0,0,0,0.1)";aH(aE.datapoints,aB,null,aD+aD/2,true,aE.xaxis,aE.yaxis,aF);H.strokeStyle="rgba(0,0,0,0.2)";aH(aE.datapoints,aB,null,aD/2,true,aE.xaxis,aE.yaxis,aF)}H.lineWidth=aG;H.strokeStyle=aE.color;aH(aE.datapoints,aB,ae(aE.points,aE.color),0,false,aE.xaxis,aE.yaxis,aF);H.restore()}function E(aN,aM,aV,aI,aQ,aF,aD,aL,aK,aU,aR,aC){var aE,aT,aJ,aP,aG,aB,aO,aH,aS;if(aR){aH=aB=aO=true;aG=false;aE=aV;aT=aN;aP=aM+aI;aJ=aM+aQ;if(aT<aE){aS=aT;aT=aE;aE=aS;aG=true;aB=false}}else{aG=aB=aO=true;aH=false;aE=aN+aI;aT=aN+aQ;aJ=aV;aP=aM;if(aP<aJ){aS=aP;aP=aJ;aJ=aS;aH=true;aO=false}}if(aT<aL.min||aE>aL.max||aP<aK.min||aJ>aK.max){return}if(aE<aL.min){aE=aL.min;aG=false}if(aT>aL.max){aT=aL.max;aB=false}if(aJ<aK.min){aJ=aK.min;aH=false}if(aP>aK.max){aP=aK.max;aO=false}aE=aL.p2c(aE);aJ=aK.p2c(aJ);aT=aL.p2c(aT);aP=aK.p2c(aP);if(aD){aU.beginPath();aU.moveTo(aE,aJ);aU.lineTo(aE,aP);aU.lineTo(aT,aP);aU.lineTo(aT,aJ);aU.fillStyle=aD(aJ,aP);aU.fill()}if(aC>0&&(aG||aB||aO||aH)){aU.beginPath();aU.moveTo(aE,aJ+aF);if(aG){aU.lineTo(aE,aP+aF)}else{aU.moveTo(aE,aP+aF)}if(aO){aU.lineTo(aT,aP+aF)}else{aU.moveTo(aT,aP+aF)}if(aB){aU.lineTo(aT,aJ+aF)}else{aU.moveTo(aT,aJ+aF)}if(aH){aU.lineTo(aE,aJ+aF)}else{aU.moveTo(aE,aJ+aF)}aU.stroke()}}function e(aD){function aC(aJ,aI,aL,aG,aK,aN,aM){var aO=aJ.points,aF=aJ.pointsize;for(var aH=0;aH<aO.length;aH+=aF){if(aO[aH]==null){continue}E(aO[aH],aO[aH+1],aO[aH+2],aI,aL,aG,aK,aN,aM,H,aD.bars.horizontal,aD.bars.lineWidth)}}H.save();H.translate(q.left,q.top);H.lineWidth=aD.bars.lineWidth;H.strokeStyle=aD.color;var aB=aD.bars.align=="left"?0:-aD.bars.barWidth/2;var aE=aD.bars.fill?function(aF,aG){return ae(aD.bars,aD.color,aF,aG)}:null;aC(aD.datapoints,aB,aB+aD.bars.barWidth,0,aE,aD.xaxis,aD.yaxis);H.restore()}function ae(aD,aB,aC,aF){var aE=aD.fill;if(!aE){return null}if(aD.fillColor){return am(aD.fillColor,aC,aF,aB)}var aG=c.color.parse(aB);aG.a=typeof aE=="number"?aE:0.4;aG.normalize();return aG.toString()}function o(){av.find(".legend").remove();if(!O.legend.show){return}var aH=[],aF=false,aN=O.legend.labelFormatter,aM,aJ;for(var aE=0;aE<Q.length;++aE){aM=Q[aE];aJ=aM.label;if(!aJ){continue}if(aE%O.legend.noColumns==0){if(aF){aH.push("</tr>")}aH.push("<tr>");aF=true}if(aN){aJ=aN(aJ,aM)}aH.push('<td class="legendColorBox"><div style="border:1px solid '+O.legend.labelBoxBorderColor+';padding:1px"><div style="width:4px;height:0;border:5px solid '+aM.color+';overflow:hidden"></div></div></td><td class="legendLabel">'+aJ+"</td>")}if(aF){aH.push("</tr>")}if(aH.length==0){return}var aL='<table style="font-size:smaller;color:'+O.grid.color+'">'+aH.join("")+"</table>";if(O.legend.container!=null){c(O.legend.container).html(aL)}else{var aI="",aC=O.legend.position,aD=O.legend.margin;if(aD[0]==null){aD=[aD,aD]}if(aC.charAt(0)=="n"){aI+="top:"+(aD[1]+q.top)+"px;"}else{if(aC.charAt(0)=="s"){aI+="bottom:"+(aD[1]+q.bottom)+"px;"}}if(aC.charAt(1)=="e"){aI+="right:"+(aD[0]+q.right)+"px;"}else{if(aC.charAt(1)=="w"){aI+="left:"+(aD[0]+q.left)+"px;"}}var aK=c('<div class="legend">'+aL.replace('style="','style="position:absolute;'+aI+";")+"</div>").appendTo(av);if(O.legend.backgroundOpacity!=0){var aG=O.legend.backgroundColor;if(aG==null){aG=O.grid.backgroundColor;if(aG&&typeof aG=="string"){aG=c.color.parse(aG)}else{aG=c.color.extract(aK,"background-color")}aG.a=1;aG=aG.toString()}var aB=aK.children();c('<div style="position:absolute;width:'+aB.width()+"px;height:"+aB.height()+"px;"+aI+"background-color:"+aG+';"> </div>').prependTo(aK).css("opacity",O.legend.backgroundOpacity)}}}var ab=[],M=null;function K(aI,aG,aD){var aO=O.grid.mouseActiveRadius,a0=aO*aO+1,aY=null,aR=false,aW,aU;for(aW=Q.length-1;aW>=0;--aW){if(!aD(Q[aW])){continue}var aP=Q[aW],aH=aP.xaxis,aF=aP.yaxis,aV=aP.datapoints.points,aT=aP.datapoints.pointsize,aQ=aH.c2p(aI),aN=aF.c2p(aG),aC=aO/aH.scale,aB=aO/aF.scale;if(aH.options.inverseTransform){aC=Number.MAX_VALUE}if(aF.options.inverseTransform){aB=Number.MAX_VALUE}if(aP.lines.show||aP.points.show||aP.dashes.show){for(aU=0;aU<aV.length;aU+=aT){var aK=aV[aU],aJ=aV[aU+1];if(aK==null){continue}if(aK-aQ>aC||aK-aQ<-aC||aJ-aN>aB||aJ-aN<-aB){continue}var aM=Math.abs(aH.p2c(aK)-aI),aL=Math.abs(aF.p2c(aJ)-aG),aS=aM*aM+aL*aL;if(aS<a0){a0=aS;aY=[aW,aU/aT]}}}if(aP.bars.show&&!aY){var aE=aP.bars.align=="left"?0:-aP.bars.barWidth/2,aX=aE+aP.bars.barWidth;for(aU=0;aU<aV.length;aU+=aT){var aK=aV[aU],aJ=aV[aU+1],aZ=aV[aU+2];if(aK==null){continue}if(Q[aW].bars.horizontal?(aQ<=Math.max(aZ,aK)&&aQ>=Math.min(aZ,aK)&&aN>=aJ+aE&&aN<=aJ+aX):(aQ>=aK+aE&&aQ<=aK+aX&&aN>=Math.min(aZ,aJ)&&aN<=Math.max(aZ,aJ))){aY=[aW,aU/aT]}}}}if(aY){aW=aY[0];aU=aY[1];aT=Q[aW].datapoints.pointsize;return{datapoint:Q[aW].datapoints.points.slice(aU*aT,(aU+1)*aT),dataIndex:aU,series:Q[aW],seriesIndex:aW}}return null}function aa(aB){if(O.grid.hoverable){u("plothover",aB,function(aC){return aC.hoverable!=false})}}function l(aB){if(O.grid.hoverable){u("plothover",aB,function(aC){return false})}}function R(aB){u("plotclick",aB,function(aC){return aC.clickable!=false})}function u(aC,aB,aD){var aE=y.offset(),aH=aB.pageX-aE.left-q.left,aF=aB.pageY-aE.top-q.top,aJ=C({left:aH,top:aF});aJ.pageX=aB.pageX;aJ.pageY=aB.pageY;var aK=K(aH,aF,aD);if(aK){aK.pageX=parseInt(aK.series.xaxis.p2c(aK.datapoint[0])+aE.left+q.left);aK.pageY=parseInt(aK.series.yaxis.p2c(aK.datapoint[1])+aE.top+q.top)}if(O.grid.autoHighlight){for(var aG=0;aG<ab.length;++aG){var aI=ab[aG];if(aI.auto==aC&&!(aK&&aI.series==aK.series&&aI.point[0]==aK.datapoint[0]&&aI.point[1]==aK.datapoint[1])){T(aI.series,aI.point)}}if(aK){x(aK.series,aK.datapoint,aC)}}av.trigger(aC,[aJ,aK])}function f(){if(!M){M=setTimeout(s,30)}}function s(){M=null;A.save();A.clearRect(0,0,G,I);A.translate(q.left,q.top);var aC,aB;for(aC=0;aC<ab.length;++aC){aB=ab[aC];if(aB.series.bars.show){v(aB.series,aB.point)}else{ay(aB.series,aB.point)}}A.restore();an(ak.drawOverlay,[A])}function x(aD,aB,aF){if(typeof aD=="number"){aD=Q[aD]}if(typeof aB=="number"){var aE=aD.datapoints.pointsize;aB=aD.datapoints.points.slice(aE*aB,aE*(aB+1))}var aC=al(aD,aB);if(aC==-1){ab.push({series:aD,point:aB,auto:aF});f()}else{if(!aF){ab[aC].auto=false}}}function T(aD,aB){if(aD==null&&aB==null){ab=[];f()}if(typeof aD=="number"){aD=Q[aD]}if(typeof aB=="number"){aB=aD.data[aB]}var aC=al(aD,aB);if(aC!=-1){ab.splice(aC,1);f()}}function al(aD,aE){for(var aB=0;aB<ab.length;++aB){var aC=ab[aB];if(aC.series==aD&&aC.point[0]==aE[0]&&aC.point[1]==aE[1]){return aB}}return -1}function ay(aE,aD){var aC=aD[0],aI=aD[1],aH=aE.xaxis,aG=aE.yaxis;if(aC<aH.min||aC>aH.max||aI<aG.min||aI>aG.max){return}var aF=aE.points.radius+aE.points.lineWidth/2;A.lineWidth=aF;A.strokeStyle=c.color.parse(aE.color).scale("a",0.5).toString();var aB=1.5*aF,aC=aH.p2c(aC),aI=aG.p2c(aI);A.beginPath();if(aE.points.symbol=="circle"){A.arc(aC,aI,aB,0,2*Math.PI,false)}else{aE.points.symbol(A,aC,aI,aB,false)}A.closePath();A.stroke()}function v(aE,aB){A.lineWidth=aE.bars.lineWidth;A.strokeStyle=c.color.parse(aE.color).scale("a",0.5).toString();var aD=c.color.parse(aE.color).scale("a",0.5).toString();var aC=aE.bars.align=="left"?0:-aE.bars.barWidth/2;E(aB[0],aB[1],aB[2]||0,aC,aC+aE.bars.barWidth,0,function(){return aD},aE.xaxis,aE.yaxis,A,aE.bars.horizontal,aE.bars.lineWidth)}function am(aJ,aB,aH,aC){if(typeof aJ=="string"){return aJ}else{var aI=H.createLinearGradient(0,aH,0,aB);for(var aE=0,aD=aJ.colors.length;aE<aD;++aE){var aF=aJ.colors[aE];if(typeof aF!="string"){var aG=c.color.parse(aC);if(aF.brightness!=null){aG=aG.scale("rgb",aF.brightness)}if(aF.opacity!=null){aG.a*=aF.opacity}aF=aG.toString()}aI.addColorStop(aE/(aD-1),aF)}return aI}}}c.plot=function(g,e,d){var f=new b(c(g),e,d,c.plot.plugins);return f};c.plot.version="0.7";c.plot.plugins=[];c.plot.formatDate=function(l,f,h){var o=function(d){d=""+d;return d.length==1?"0"+d:d};var e=[];var p=false,j=false;var n=l.getUTCHours();var k=n<12;if(h==null){h=["Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"]}if(f.search(/%p|%P/)!=-1){if(n>12){n=n-12}else{if(n==0){n=12}}}for(var g=0;g<f.length;++g){var m=f.charAt(g);if(p){switch(m){case"h":m=""+n;break;case"H":m=o(n);break;case"M":m=o(l.getUTCMinutes());break;case"S":m=o(l.getUTCSeconds());break;case"d":m=""+l.getUTCDate();break;case"m":m=""+(l.getUTCMonth()+1);break;case"y":m=""+l.getUTCFullYear();break;case"b":m=""+h[l.getUTCMonth()];break;case"p":m=(k)?("am"):("pm");break;case"P":m=(k)?("AM"):("PM");break;case"0":m="";j=true;break}if(m&&j){m=o(m);j=false}e.push(m);if(!j){p=false}}else{if(m=="%"){p=true}else{e.push(m)}}}return e.join("")};function a(e,d){return d*Math.floor(e/d)}})(jQuery);
</script>

<script type="text/javascript">
if(!document.createElement("canvas").getContext){(function(){var z=Math;var K=z.round;var J=z.sin;var U=z.cos;var b=z.abs;var k=z.sqrt;var D=10;var F=D/2;function T(){return this.context_||(this.context_=new W(this))}var O=Array.prototype.slice;function G(i,j,m){var Z=O.call(arguments,2);return function(){return i.apply(j,Z.concat(O.call(arguments)))}}function AD(Z){return String(Z).replace(/&/g,"&amp;").replace(/"/g,"&quot;")}function r(i){if(!i.namespaces.g_vml_){i.namespaces.add("g_vml_","urn:schemas-microsoft-com:vml","#default#VML")}if(!i.namespaces.g_o_){i.namespaces.add("g_o_","urn:schemas-microsoft-com:office:office","#default#VML")}if(!i.styleSheets.ex_canvas_){var Z=i.createStyleSheet();Z.owningElement.id="ex_canvas_";Z.cssText="canvas{display:inline-block;overflow:hidden;text-align:left;width:300px;height:150px}"}}r(document);var E={init:function(Z){if(/MSIE/.test(navigator.userAgent)&&!window.opera){var i=Z||document;i.createElement("canvas");i.attachEvent("onreadystatechange",G(this.init_,this,i))}},init_:function(m){var j=m.getElementsByTagName("canvas");for(var Z=0;Z<j.length;Z++){this.initElement(j[Z])}},initElement:function(i){if(!i.getContext){i.getContext=T;r(i.ownerDocument);i.innerHTML="";i.attachEvent("onpropertychange",S);i.attachEvent("onresize",w);var Z=i.attributes;if(Z.width&&Z.width.specified){i.style.width=Z.width.nodeValue+"px"}else{i.width=i.clientWidth}if(Z.height&&Z.height.specified){i.style.height=Z.height.nodeValue+"px"}else{i.height=i.clientHeight}}return i}};function S(i){var Z=i.srcElement;switch(i.propertyName){case"width":Z.getContext().clearRect();Z.style.width=Z.attributes.width.nodeValue+"px";Z.firstChild.style.width=Z.clientWidth+"px";break;case"height":Z.getContext().clearRect();Z.style.height=Z.attributes.height.nodeValue+"px";Z.firstChild.style.height=Z.clientHeight+"px";break}}function w(i){var Z=i.srcElement;if(Z.firstChild){Z.firstChild.style.width=Z.clientWidth+"px";Z.firstChild.style.height=Z.clientHeight+"px"}}E.init();var I=[];for(var AC=0;AC<16;AC++){for(var AB=0;AB<16;AB++){I[AC*16+AB]=AC.toString(16)+AB.toString(16)}}function V(){return[[1,0,0],[0,1,0],[0,0,1]]}function d(m,j){var i=V();for(var Z=0;Z<3;Z++){for(var AF=0;AF<3;AF++){var p=0;for(var AE=0;AE<3;AE++){p+=m[Z][AE]*j[AE][AF]}i[Z][AF]=p}}return i}function Q(i,Z){Z.fillStyle=i.fillStyle;Z.lineCap=i.lineCap;Z.lineJoin=i.lineJoin;Z.lineWidth=i.lineWidth;Z.miterLimit=i.miterLimit;Z.shadowBlur=i.shadowBlur;Z.shadowColor=i.shadowColor;Z.shadowOffsetX=i.shadowOffsetX;Z.shadowOffsetY=i.shadowOffsetY;Z.strokeStyle=i.strokeStyle;Z.globalAlpha=i.globalAlpha;Z.font=i.font;Z.textAlign=i.textAlign;Z.textBaseline=i.textBaseline;Z.arcScaleX_=i.arcScaleX_;Z.arcScaleY_=i.arcScaleY_;Z.lineScale_=i.lineScale_}var B={aliceblue:"#F0F8FF",antiquewhite:"#FAEBD7",aquamarine:"#7FFFD4",azure:"#F0FFFF",beige:"#F5F5DC",bisque:"#FFE4C4",black:"#000000",blanchedalmond:"#FFEBCD",blueviolet:"#8A2BE2",brown:"#A52A2A",burlywood:"#DEB887",cadetblue:"#5F9EA0",chartreuse:"#7FFF00",chocolate:"#D2691E",coral:"#FF7F50",cornflowerblue:"#6495ED",cornsilk:"#FFF8DC",crimson:"#DC143C",cyan:"#00FFFF",darkblue:"#00008B",darkcyan:"#008B8B",darkgoldenrod:"#B8860B",darkgray:"#A9A9A9",darkgreen:"#006400",darkgrey:"#A9A9A9",darkkhaki:"#BDB76B",darkmagenta:"#8B008B",darkolivegreen:"#556B2F",darkorange:"#FF8C00",darkorchid:"#9932CC",darkred:"#8B0000",darksalmon:"#E9967A",darkseagreen:"#8FBC8F",darkslateblue:"#483D8B",darkslategray:"#2F4F4F",darkslategrey:"#2F4F4F",darkturquoise:"#00CED1",darkviolet:"#9400D3",deeppink:"#FF1493",deepskyblue:"#00BFFF",dimgray:"#696969",dimgrey:"#696969",dodgerblue:"#1E90FF",firebrick:"#B22222",floralwhite:"#FFFAF0",forestgreen:"#228B22",gainsboro:"#DCDCDC",ghostwhite:"#F8F8FF",gold:"#FFD700",goldenrod:"#DAA520",grey:"#808080",greenyellow:"#ADFF2F",honeydew:"#F0FFF0",hotpink:"#FF69B4",indianred:"#CD5C5C",indigo:"#4B0082",ivory:"#FFFFF0",khaki:"#F0E68C",lavender:"#E6E6FA",lavenderblush:"#FFF0F5",lawngreen:"#7CFC00",lemonchiffon:"#FFFACD",lightblue:"#ADD8E6",lightcoral:"#F08080",lightcyan:"#E0FFFF",lightgoldenrodyellow:"#FAFAD2",lightgreen:"#90EE90",lightgrey:"#D3D3D3",lightpink:"#FFB6C1",lightsalmon:"#FFA07A",lightseagreen:"#20B2AA",lightskyblue:"#87CEFA",lightslategray:"#778899",lightslategrey:"#778899",lightsteelblue:"#B0C4DE",lightyellow:"#FFFFE0",limegreen:"#32CD32",linen:"#FAF0E6",magenta:"#FF00FF",mediumaquamarine:"#66CDAA",mediumblue:"#0000CD",mediumorchid:"#BA55D3",mediumpurple:"#9370DB",mediumseagreen:"#3CB371",mediumslateblue:"#7B68EE",mediumspringgreen:"#00FA9A",mediumturquoise:"#48D1CC",mediumvioletred:"#C71585",midnightblue:"#191970",mintcream:"#F5FFFA",mistyrose:"#FFE4E1",moccasin:"#FFE4B5",navajowhite:"#FFDEAD",oldlace:"#FDF5E6",olivedrab:"#6B8E23",orange:"#FFA500",orangered:"#FF4500",orchid:"#DA70D6",palegoldenrod:"#EEE8AA",palegreen:"#98FB98",paleturquoise:"#AFEEEE",palevioletred:"#DB7093",papayawhip:"#FFEFD5",peachpuff:"#FFDAB9",peru:"#CD853F",pink:"#FFC0CB",plum:"#DDA0DD",powderblue:"#B0E0E6",rosybrown:"#BC8F8F",royalblue:"#4169E1",saddlebrown:"#8B4513",salmon:"#FA8072",sandybrown:"#F4A460",seagreen:"#2E8B57",seashell:"#FFF5EE",sienna:"#A0522D",skyblue:"#87CEEB",slateblue:"#6A5ACD",slategray:"#708090",slategrey:"#708090",snow:"#FFFAFA",springgreen:"#00FF7F",steelblue:"#4682B4",tan:"#D2B48C",thistle:"#D8BFD8",tomato:"#FF6347",turquoise:"#40E0D0",violet:"#EE82EE",wheat:"#F5DEB3",whitesmoke:"#F5F5F5",yellowgreen:"#9ACD32"};function g(i){var m=i.indexOf("(",3);var Z=i.indexOf(")",m+1);var j=i.substring(m+1,Z).split(",");if(j.length==4&&i.substr(3,1)=="a"){alpha=Number(j[3])}else{j[3]=1}return j}function C(Z){return parseFloat(Z)/100}function N(i,j,Z){return Math.min(Z,Math.max(j,i))}function c(AF){var j,i,Z;h=parseFloat(AF[0])/360%360;if(h<0){h++}s=N(C(AF[1]),0,1);l=N(C(AF[2]),0,1);if(s==0){j=i=Z=l}else{var m=l<0.5?l*(1+s):l+s-l*s;var AE=2*l-m;j=A(AE,m,h+1/3);i=A(AE,m,h);Z=A(AE,m,h-1/3)}return"#"+I[Math.floor(j*255)]+I[Math.floor(i*255)]+I[Math.floor(Z*255)]}function A(i,Z,j){if(j<0){j++}if(j>1){j--}if(6*j<1){return i+(Z-i)*6*j}else{if(2*j<1){return Z}else{if(3*j<2){return i+(Z-i)*(2/3-j)*6}else{return i}}}}function Y(Z){var AE,p=1;Z=String(Z);if(Z.charAt(0)=="#"){AE=Z}else{if(/^rgb/.test(Z)){var m=g(Z);var AE="#",AF;for(var j=0;j<3;j++){if(m[j].indexOf("%")!=-1){AF=Math.floor(C(m[j])*255)}else{AF=Number(m[j])}AE+=I[N(AF,0,255)]}p=m[3]}else{if(/^hsl/.test(Z)){var m=g(Z);AE=c(m);p=m[3]}else{AE=B[Z]||Z}}}return{color:AE,alpha:p}}var L={style:"normal",variant:"normal",weight:"normal",size:10,family:"sans-serif"};var f={};function X(Z){if(f[Z]){return f[Z]}var m=document.createElement("div");var j=m.style;try{j.font=Z}catch(i){}return f[Z]={style:j.fontStyle||L.style,variant:j.fontVariant||L.variant,weight:j.fontWeight||L.weight,size:j.fontSize||L.size,family:j.fontFamily||L.family}}function P(j,i){var Z={};for(var AF in j){Z[AF]=j[AF]}var AE=parseFloat(i.currentStyle.fontSize),m=parseFloat(j.size);if(typeof j.size=="number"){Z.size=j.size}else{if(j.size.indexOf("px")!=-1){Z.size=m}else{if(j.size.indexOf("em")!=-1){Z.size=AE*m}else{if(j.size.indexOf("%")!=-1){Z.size=(AE/100)*m}else{if(j.size.indexOf("pt")!=-1){Z.size=m/0.75}else{Z.size=AE}}}}}Z.size*=0.981;return Z}function AA(Z){return Z.style+" "+Z.variant+" "+Z.weight+" "+Z.size+"px "+Z.family}function t(Z){switch(Z){case"butt":return"flat";case"round":return"round";case"square":default:return"square"}}function W(i){this.m_=V();this.mStack_=[];this.aStack_=[];this.currentPath_=[];this.strokeStyle="#000";this.fillStyle="#000";this.lineWidth=1;this.lineJoin="miter";this.lineCap="butt";this.miterLimit=D*1;this.globalAlpha=1;this.font="10px sans-serif";this.textAlign="left";this.textBaseline="alphabetic";this.canvas=i;var Z=i.ownerDocument.createElement("div");Z.style.width=i.clientWidth+"px";Z.style.height=i.clientHeight+"px";Z.style.overflow="hidden";Z.style.position="absolute";i.appendChild(Z);this.element_=Z;this.arcScaleX_=1;this.arcScaleY_=1;this.lineScale_=1}var M=W.prototype;M.clearRect=function(){if(this.textMeasureEl_){this.textMeasureEl_.removeNode(true);this.textMeasureEl_=null}this.element_.innerHTML=""};M.beginPath=function(){this.currentPath_=[]};M.moveTo=function(i,Z){var j=this.getCoords_(i,Z);this.currentPath_.push({type:"moveTo",x:j.x,y:j.y});this.currentX_=j.x;this.currentY_=j.y};M.lineTo=function(i,Z){var j=this.getCoords_(i,Z);this.currentPath_.push({type:"lineTo",x:j.x,y:j.y});this.currentX_=j.x;this.currentY_=j.y};M.bezierCurveTo=function(j,i,AI,AH,AG,AE){var Z=this.getCoords_(AG,AE);var AF=this.getCoords_(j,i);var m=this.getCoords_(AI,AH);e(this,AF,m,Z)};function e(Z,m,j,i){Z.currentPath_.push({type:"bezierCurveTo",cp1x:m.x,cp1y:m.y,cp2x:j.x,cp2y:j.y,x:i.x,y:i.y});Z.currentX_=i.x;Z.currentY_=i.y}M.quadraticCurveTo=function(AG,j,i,Z){var AF=this.getCoords_(AG,j);var AE=this.getCoords_(i,Z);var AH={x:this.currentX_+2/3*(AF.x-this.currentX_),y:this.currentY_+2/3*(AF.y-this.currentY_)};var m={x:AH.x+(AE.x-this.currentX_)/3,y:AH.y+(AE.y-this.currentY_)/3};e(this,AH,m,AE)};M.arc=function(AJ,AH,AI,AE,i,j){AI*=D;var AN=j?"at":"wa";var AK=AJ+U(AE)*AI-F;var AM=AH+J(AE)*AI-F;var Z=AJ+U(i)*AI-F;var AL=AH+J(i)*AI-F;if(AK==Z&&!j){AK+=0.125}var m=this.getCoords_(AJ,AH);var AG=this.getCoords_(AK,AM);var AF=this.getCoords_(Z,AL);this.currentPath_.push({type:AN,x:m.x,y:m.y,radius:AI,xStart:AG.x,yStart:AG.y,xEnd:AF.x,yEnd:AF.y})};M.rect=function(j,i,Z,m){this.moveTo(j,i);this.lineTo(j+Z,i);this.lineTo(j+Z,i+m);this.lineTo(j,i+m);this.closePath()};M.strokeRect=function(j,i,Z,m){var p=this.currentPath_;this.beginPath();this.moveTo(j,i);this.lineTo(j+Z,i);this.lineTo(j+Z,i+m);this.lineTo(j,i+m);this.closePath();this.stroke();this.currentPath_=p};M.fillRect=function(j,i,Z,m){var p=this.currentPath_;this.beginPath();this.moveTo(j,i);this.lineTo(j+Z,i);this.lineTo(j+Z,i+m);this.lineTo(j,i+m);this.closePath();this.fill();this.currentPath_=p};M.createLinearGradient=function(i,m,Z,j){var p=new v("gradient");p.x0_=i;p.y0_=m;p.x1_=Z;p.y1_=j;return p};M.createRadialGradient=function(m,AE,j,i,p,Z){var AF=new v("gradientradial");AF.x0_=m;AF.y0_=AE;AF.r0_=j;AF.x1_=i;AF.y1_=p;AF.r1_=Z;return AF};M.drawImage=function(AO,j){var AH,AF,AJ,AV,AM,AK,AQ,AX;var AI=AO.runtimeStyle.width;var AN=AO.runtimeStyle.height;AO.runtimeStyle.width="auto";AO.runtimeStyle.height="auto";var AG=AO.width;var AT=AO.height;AO.runtimeStyle.width=AI;AO.runtimeStyle.height=AN;if(arguments.length==3){AH=arguments[1];AF=arguments[2];AM=AK=0;AQ=AJ=AG;AX=AV=AT}else{if(arguments.length==5){AH=arguments[1];AF=arguments[2];AJ=arguments[3];AV=arguments[4];AM=AK=0;AQ=AG;AX=AT}else{if(arguments.length==9){AM=arguments[1];AK=arguments[2];AQ=arguments[3];AX=arguments[4];AH=arguments[5];AF=arguments[6];AJ=arguments[7];AV=arguments[8]}else{throw Error("Invalid number of arguments")}}}var AW=this.getCoords_(AH,AF);var m=AQ/2;var i=AX/2;var AU=[];var Z=10;var AE=10;AU.push(" <g_vml_:group",' coordsize="',D*Z,",",D*AE,'"',' coordorigin="0,0"',' style="width:',Z,"px;height:",AE,"px;position:absolute;");if(this.m_[0][0]!=1||this.m_[0][1]||this.m_[1][1]!=1||this.m_[1][0]){var p=[];p.push("M11=",this.m_[0][0],",","M12=",this.m_[1][0],",","M21=",this.m_[0][1],",","M22=",this.m_[1][1],",","Dx=",K(AW.x/D),",","Dy=",K(AW.y/D),"");var AS=AW;var AR=this.getCoords_(AH+AJ,AF);var AP=this.getCoords_(AH,AF+AV);var AL=this.getCoords_(AH+AJ,AF+AV);AS.x=z.max(AS.x,AR.x,AP.x,AL.x);AS.y=z.max(AS.y,AR.y,AP.y,AL.y);AU.push("padding:0 ",K(AS.x/D),"px ",K(AS.y/D),"px 0;filter:progid:DXImageTransform.Microsoft.Matrix(",p.join(""),", sizingmethod='clip');")}else{AU.push("top:",K(AW.y/D),"px;left:",K(AW.x/D),"px;")}AU.push(' ">','<g_vml_:image src="',AO.src,'"',' style="width:',D*AJ,"px;"," height:",D*AV,'px"',' cropleft="',AM/AG,'"',' croptop="',AK/AT,'"',' cropright="',(AG-AM-AQ)/AG,'"',' cropbottom="',(AT-AK-AX)/AT,'"'," />","</g_vml_:group>");this.element_.insertAdjacentHTML("BeforeEnd",AU.join(""))};M.stroke=function(AM){var m=10;var AN=10;var AE=5000;var AG={x:null,y:null};var AL={x:null,y:null};for(var AH=0;AH<this.currentPath_.length;AH+=AE){var AK=[];var AF=false;AK.push("<g_vml_:shape",' filled="',!!AM,'"',' style="position:absolute;width:',m,"px;height:",AN,'px;"',' coordorigin="0,0"',' coordsize="',D*m,",",D*AN,'"',' stroked="',!AM,'"',' path="');var AO=false;for(var AI=AH;AI<Math.min(AH+AE,this.currentPath_.length);AI++){if(AI%AE==0&&AI>0){AK.push(" m ",K(this.currentPath_[AI-1].x),",",K(this.currentPath_[AI-1].y))}var Z=this.currentPath_[AI];var AJ;switch(Z.type){case"moveTo":AJ=Z;AK.push(" m ",K(Z.x),",",K(Z.y));break;case"lineTo":AK.push(" l ",K(Z.x),",",K(Z.y));break;case"close":AK.push(" x ");Z=null;break;case"bezierCurveTo":AK.push(" c ",K(Z.cp1x),",",K(Z.cp1y),",",K(Z.cp2x),",",K(Z.cp2y),",",K(Z.x),",",K(Z.y));break;case"at":case"wa":AK.push(" ",Z.type," ",K(Z.x-this.arcScaleX_*Z.radius),",",K(Z.y-this.arcScaleY_*Z.radius)," ",K(Z.x+this.arcScaleX_*Z.radius),",",K(Z.y+this.arcScaleY_*Z.radius)," ",K(Z.xStart),",",K(Z.yStart)," ",K(Z.xEnd),",",K(Z.yEnd));break}if(Z){if(AG.x==null||Z.x<AG.x){AG.x=Z.x}if(AL.x==null||Z.x>AL.x){AL.x=Z.x}if(AG.y==null||Z.y<AG.y){AG.y=Z.y}if(AL.y==null||Z.y>AL.y){AL.y=Z.y}}}AK.push(' ">');if(!AM){R(this,AK)}else{a(this,AK,AG,AL)}AK.push("</g_vml_:shape>");this.element_.insertAdjacentHTML("beforeEnd",AK.join(""))}};function R(j,AE){var i=Y(j.strokeStyle);var m=i.color;var p=i.alpha*j.globalAlpha;var Z=j.lineScale_*j.lineWidth;if(Z<1){p*=Z}AE.push("<g_vml_:stroke",' opacity="',p,'"',' joinstyle="',j.lineJoin,'"',' miterlimit="',j.miterLimit,'"',' endcap="',t(j.lineCap),'"',' weight="',Z,'px"',' color="',m,'" />')}function a(AO,AG,Ah,AP){var AH=AO.fillStyle;var AY=AO.arcScaleX_;var AX=AO.arcScaleY_;var Z=AP.x-Ah.x;var m=AP.y-Ah.y;if(AH instanceof v){var AL=0;var Ac={x:0,y:0};var AU=0;var AK=1;if(AH.type_=="gradient"){var AJ=AH.x0_/AY;var j=AH.y0_/AX;var AI=AH.x1_/AY;var Aj=AH.y1_/AX;var Ag=AO.getCoords_(AJ,j);var Af=AO.getCoords_(AI,Aj);var AE=Af.x-Ag.x;var p=Af.y-Ag.y;AL=Math.atan2(AE,p)*180/Math.PI;if(AL<0){AL+=360}if(AL<0.000001){AL=0}}else{var Ag=AO.getCoords_(AH.x0_,AH.y0_);Ac={x:(Ag.x-Ah.x)/Z,y:(Ag.y-Ah.y)/m};Z/=AY*D;m/=AX*D;var Aa=z.max(Z,m);AU=2*AH.r0_/Aa;AK=2*AH.r1_/Aa-AU}var AS=AH.colors_;AS.sort(function(Ak,i){return Ak.offset-i.offset});var AN=AS.length;var AR=AS[0].color;var AQ=AS[AN-1].color;var AW=AS[0].alpha*AO.globalAlpha;var AV=AS[AN-1].alpha*AO.globalAlpha;var Ab=[];for(var Ae=0;Ae<AN;Ae++){var AM=AS[Ae];Ab.push(AM.offset*AK+AU+" "+AM.color)}AG.push('<g_vml_:fill type="',AH.type_,'"',' method="none" focus="100%"',' color="',AR,'"',' color2="',AQ,'"',' colors="',Ab.join(","),'"',' opacity="',AV,'"',' g_o_:opacity2="',AW,'"',' angle="',AL,'"',' focusposition="',Ac.x,",",Ac.y,'" />')}else{if(AH instanceof u){if(Z&&m){var AF=-Ah.x;var AZ=-Ah.y;AG.push("<g_vml_:fill",' position="',AF/Z*AY*AY,",",AZ/m*AX*AX,'"',' type="tile"',' src="',AH.src_,'" />')}}else{var Ai=Y(AO.fillStyle);var AT=Ai.color;var Ad=Ai.alpha*AO.globalAlpha;AG.push('<g_vml_:fill color="',AT,'" opacity="',Ad,'" />')}}}M.fill=function(){this.stroke(true)};M.closePath=function(){this.currentPath_.push({type:"close"})};M.getCoords_=function(j,i){var Z=this.m_;return{x:D*(j*Z[0][0]+i*Z[1][0]+Z[2][0])-F,y:D*(j*Z[0][1]+i*Z[1][1]+Z[2][1])-F}};M.save=function(){var Z={};Q(this,Z);this.aStack_.push(Z);this.mStack_.push(this.m_);this.m_=d(V(),this.m_)};M.restore=function(){if(this.aStack_.length){Q(this.aStack_.pop(),this);this.m_=this.mStack_.pop()}};function H(Z){return isFinite(Z[0][0])&&isFinite(Z[0][1])&&isFinite(Z[1][0])&&isFinite(Z[1][1])&&isFinite(Z[2][0])&&isFinite(Z[2][1])}function y(i,Z,j){if(!H(Z)){return }i.m_=Z;if(j){var p=Z[0][0]*Z[1][1]-Z[0][1]*Z[1][0];i.lineScale_=k(b(p))}}M.translate=function(j,i){var Z=[[1,0,0],[0,1,0],[j,i,1]];y(this,d(Z,this.m_),false)};M.rotate=function(i){var m=U(i);var j=J(i);var Z=[[m,j,0],[-j,m,0],[0,0,1]];y(this,d(Z,this.m_),false)};M.scale=function(j,i){this.arcScaleX_*=j;this.arcScaleY_*=i;var Z=[[j,0,0],[0,i,0],[0,0,1]];y(this,d(Z,this.m_),true)};M.transform=function(p,m,AF,AE,i,Z){var j=[[p,m,0],[AF,AE,0],[i,Z,1]];y(this,d(j,this.m_),true)};M.setTransform=function(AE,p,AG,AF,j,i){var Z=[[AE,p,0],[AG,AF,0],[j,i,1]];y(this,Z,true)};M.drawText_=function(AK,AI,AH,AN,AG){var AM=this.m_,AQ=1000,i=0,AP=AQ,AF={x:0,y:0},AE=[];var Z=P(X(this.font),this.element_);var j=AA(Z);var AR=this.element_.currentStyle;var p=this.textAlign.toLowerCase();switch(p){case"left":case"center":case"right":break;case"end":p=AR.direction=="ltr"?"right":"left";break;case"start":p=AR.direction=="rtl"?"right":"left";break;default:p="left"}switch(this.textBaseline){case"hanging":case"top":AF.y=Z.size/1.75;break;case"middle":break;default:case null:case"alphabetic":case"ideographic":case"bottom":AF.y=-Z.size/2.25;break}switch(p){case"right":i=AQ;AP=0.05;break;case"center":i=AP=AQ/2;break}var AO=this.getCoords_(AI+AF.x,AH+AF.y);AE.push('<g_vml_:line from="',-i,' 0" to="',AP,' 0.05" ',' coordsize="100 100" coordorigin="0 0"',' filled="',!AG,'" stroked="',!!AG,'" style="position:absolute;width:1px;height:1px;">');if(AG){R(this,AE)}else{a(this,AE,{x:-i,y:0},{x:AP,y:Z.size})}var AL=AM[0][0].toFixed(3)+","+AM[1][0].toFixed(3)+","+AM[0][1].toFixed(3)+","+AM[1][1].toFixed(3)+",0,0";var AJ=K(AO.x/D)+","+K(AO.y/D);AE.push('<g_vml_:skew on="t" matrix="',AL,'" ',' offset="',AJ,'" origin="',i,' 0" />','<g_vml_:path textpathok="true" />','<g_vml_:textpath on="true" string="',AD(AK),'" style="v-text-align:',p,";font:",AD(j),'" /></g_vml_:line>');this.element_.insertAdjacentHTML("beforeEnd",AE.join(""))};M.fillText=function(j,Z,m,i){this.drawText_(j,Z,m,i,false)};M.strokeText=function(j,Z,m,i){this.drawText_(j,Z,m,i,true)};M.measureText=function(j){if(!this.textMeasureEl_){var Z='<span style="position:absolute;top:-20000px;left:0;padding:0;margin:0;border:none;white-space:pre;"></span>';this.element_.insertAdjacentHTML("beforeEnd",Z);this.textMeasureEl_=this.element_.lastChild}var i=this.element_.ownerDocument;this.textMeasureEl_.innerHTML="";this.textMeasureEl_.style.font=this.font;this.textMeasureEl_.appendChild(i.createTextNode(j));return{width:this.textMeasureEl_.offsetWidth}};M.clip=function(){};M.arcTo=function(){};M.createPattern=function(i,Z){return new u(i,Z)};function v(Z){this.type_=Z;this.x0_=0;this.y0_=0;this.r0_=0;this.x1_=0;this.y1_=0;this.r1_=0;this.colors_=[]}v.prototype.addColorStop=function(i,Z){Z=Y(Z);this.colors_.push({offset:i,color:Z.color,alpha:Z.alpha})};function u(i,Z){q(i);switch(Z){case"repeat":case null:case"":this.repetition_="repeat";break;case"repeat-x":case"repeat-y":case"no-repeat":this.repetition_=Z;break;default:n("SYNTAX_ERR")}this.src_=i.src;this.width_=i.width;this.height_=i.height}function n(Z){throw new o(Z)}function q(Z){if(!Z||Z.nodeType!=1||Z.tagName!="IMG"){n("TYPE_MISMATCH_ERR")}if(Z.readyState!="complete"){n("INVALID_STATE_ERR")}}function o(Z){this.code=this[Z];this.message=Z+": DOM Exception "+this.code}var x=o.prototype=new Error;x.INDEX_SIZE_ERR=1;x.DOMSTRING_SIZE_ERR=2;x.HIERARCHY_REQUEST_ERR=3;x.WRONG_DOCUMENT_ERR=4;x.INVALID_CHARACTER_ERR=5;x.NO_DATA_ALLOWED_ERR=6;x.NO_MODIFICATION_ALLOWED_ERR=7;x.NOT_FOUND_ERR=8;x.NOT_SUPPORTED_ERR=9;x.INUSE_ATTRIBUTE_ERR=10;x.INVALID_STATE_ERR=11;x.SYNTAX_ERR=12;x.INVALID_MODIFICATION_ERR=13;x.NAMESPACE_ERR=14;x.INVALID_ACCESS_ERR=15;x.VALIDATION_ERR=16;x.TYPE_MISMATCH_ERR=17;G_vmlCanvasManager=E;CanvasRenderingContext2D=W;CanvasGradient=v;CanvasPattern=u;DOMException=o})()};
</script>

<script type="text/javascript">
/*
 * jQuery.flot.dashes
 * 
 * options = {
 *   series: {
 *     dashes: {
 *       
 *       // show
 *       // default: false
 *       // Whether to show dashes for the series.
 *       show: <boolean>,
 *       
 *       // lineWidth
 *       // default: 2
 *       // The width of the dashed line in pixels.
 *       lineWidth: <number>,
 *       
 *       // dashLength
 *       // default: 10
 *       // Controls the length of the individual dashes and the amount of 
 *       // space between them.
 *       // If this is a number, the dashes and spaces will have that length.
 *       // If this is an array, it is read as [ dashLength, spaceLength ]
 *       dashLength: <number> or <array[2]>
 *     }
 *   }
 * }
 */
(function($){
  
  function init(plot) {
    
    plot.hooks.processDatapoints.push(function(plot, series, datapoints) {
      
      if (!series.dashes.show) return;
      
      plot.hooks.draw.push(function(plot, ctx) {
        
        var plotOffset = plot.getPlotOffset(), 
            axisx = series.xaxis,
            axisy = series.yaxis;
        
        function plotDashes(xoffset, yoffset) {
          
          var points = datapoints.points,
              ps = datapoints.pointsize,
              prevx = null, 
              prevy = null,
              dashRemainder = 0, 
              dashOn = true,
              dashOnLength,
              dashOffLength;
          
          if (series.dashes.dashLength[0]) {
            dashOnLength = series.dashes.dashLength[0];
            if (series.dashes.dashLength[1]) {
              dashOffLength = series.dashes.dashLength[1];
            } else {
              dashOffLength = dashOnLength;
            }
          } else {
            dashOffLength = dashOnLength = series.dashes.dashLength;
          }
          
          ctx.beginPath();
          
          for (var i = ps; i < points.length; i += ps) {
            
            var x1 = points[i - ps], 
                y1 = points[i - ps + 1],
                x2 = points[i], 
                y2 = points[i + 1];
            
            if (x1 == null || x2 == null) continue;

            // clip with ymin
            if (y1 <= y2 && y1 < axisy.min) {
              if (y2 < axisy.min) continue;   // line segment is outside
              // compute new intersection point
              x1 = (axisy.min - y1) / (y2 - y1) * (x2 - x1) + x1;
              y1 = axisy.min;
            } else if (y2 <= y1 && y2 < axisy.min) {
              if (y1 < axisy.min) continue;
              x2 = (axisy.min - y1) / (y2 - y1) * (x2 - x1) + x1;
              y2 = axisy.min;
            }

            // clip with ymax
            if (y1 >= y2 && y1 > axisy.max) {
              if (y2 > axisy.max) continue;
              x1 = (axisy.max - y1) / (y2 - y1) * (x2 - x1) + x1;
              y1 = axisy.max;
            } else if (y2 >= y1 && y2 > axisy.max) {
              if (y1 > axisy.max) continue;
              x2 = (axisy.max - y1) / (y2 - y1) * (x2 - x1) + x1;
              y2 = axisy.max;
            }

            // clip with xmin
            if (x1 <= x2 && x1 < axisx.min) {
              if (x2 < axisx.min) continue;
              y1 = (axisx.min - x1) / (x2 - x1) * (y2 - y1) + y1;
              x1 = axisx.min;
            } else if (x2 <= x1 && x2 < axisx.min) {
              if (x1 < axisx.min) continue;
              y2 = (axisx.min - x1) / (x2 - x1) * (y2 - y1) + y1;
              x2 = axisx.min;
            }

            // clip with xmax
            if (x1 >= x2 && x1 > axisx.max) {
              if (x2 > axisx.max) continue;
              y1 = (axisx.max - x1) / (x2 - x1) * (y2 - y1) + y1;
              x1 = axisx.max;
            } else if (x2 >= x1 && x2 > axisx.max) {
              if (x1 > axisx.max) continue;
              y2 = (axisx.max - x1) / (x2 - x1) * (y2 - y1) + y1;
              x2 = axisx.max;
            }

            if (x1 != prevx || y1 != prevy) {
              ctx.moveTo(axisx.p2c(x1) + xoffset, axisy.p2c(y1) + yoffset);
            }
            
            var ax1 = axisx.p2c(x1) + xoffset,
                ay1 = axisy.p2c(y1) + yoffset,
                ax2 = axisx.p2c(x2) + xoffset,
                ay2 = axisy.p2c(y2) + yoffset,
                dashOffset;
            
            function lineSegmentOffset(segmentLength) {
              
              var c = Math.sqrt(Math.pow(ax2 - ax1, 2) + Math.pow(ay2 - ay1, 2));
              
              if (c <= segmentLength) {
                return {
                  deltaX: ax2 - ax1,
                  deltaY: ay2 - ay1,
                  distance: c,
                  remainder: segmentLength - c
                }
              } else {
                var xsign = ax2 > ax1 ? 1 : -1,
                    ysign = ay2 > ay1 ? 1 : -1;
                return {
                  deltaX: xsign * Math.sqrt(Math.pow(segmentLength, 2) / (1 + Math.pow((ay2 - ay1)/(ax2 - ax1), 2))),
                  deltaY: ysign * Math.sqrt(Math.pow(segmentLength, 2) - Math.pow(segmentLength, 2) / (1 + Math.pow((ay2 - ay1)/(ax2 - ax1), 2))),
                  distance: segmentLength,
                  remainder: 0
                };
              }
            }
            //-end lineSegmentOffset
            
            do {
              
              dashOffset = lineSegmentOffset(
                  dashRemainder > 0 ? dashRemainder : 
                    dashOn ? dashOnLength : dashOffLength);
              
              if (dashOffset.deltaX != 0 || dashOffset.deltaY != 0) {
                if (dashOn) {
                  ctx.lineTo(ax1 + dashOffset.deltaX, ay1 + dashOffset.deltaY);
                } else {
                  ctx.moveTo(ax1 + dashOffset.deltaX, ay1 + dashOffset.deltaY);
                }
              }
              
              dashOn = !dashOn;
              dashRemainder = dashOffset.remainder;
              ax1 += dashOffset.deltaX;
              ay1 += dashOffset.deltaY;
              
            } while (dashOffset.distance > 0);
            
            prevx = x2;
            prevy = y2;
          }
          
          ctx.stroke();
        }
        //-end plotDashes
        
        ctx.save();
        ctx.translate(plotOffset.left, plotOffset.top);
        ctx.lineJoin = 'round';
        
        var lw = series.dashes.lineWidth,
            sw = series.shadowSize;
        
        // FIXME: consider another form of shadow when filling is turned on
        if (lw > 0 && sw > 0) {
          // draw shadow as a thick and thin line with transparency
          ctx.lineWidth = sw;
          ctx.strokeStyle = "rgba(0,0,0,0.1)";
          // position shadow at angle from the mid of line
          var angle = Math.PI/18;
          plotDashes(Math.sin(angle) * (lw/2 + sw/2), Math.cos(angle) * (lw/2 + sw/2));
          ctx.lineWidth = sw/2;
          plotDashes(Math.sin(angle) * (lw/2 + sw/4), Math.cos(angle) * (lw/2 + sw/4));
        }

        ctx.lineWidth = lw;
        ctx.strokeStyle = series.color;

        if (lw > 0) {
          plotDashes(0, 0);
        }
        
        ctx.restore();
        
      });
      //-end draw hook
      
    });
    //-end processDatapoints hook
    
  }
  //-end init

  $.plot.plugins.push({
    init: init,
    options: {
      series: {
        dashes: {
          show: false,
          lineWidth: 2,
          dashLength: 10
        }
      }
    },
    name: 'dashes',
    version: '0.1'
  });

})(jQuery)


</script>

<script type="text/javascript">

var cumulative = {
    isInitialized: false,

    maxX: 0,
    maxY: 0,
    maxYTick: 0,
    series: null,
    showWithData: null,
    colors: [],

    draw: function(name, title, colors, filenames, listsOfLengths, refLengths, tickX,
                   placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {

        $(scalePlaceholder).empty();

        if (!this.isInitialized) {
            //    div.html(
            //        "<span class='plot-header'>" + addTooltipIfDefinitionExists(glossary, title) + "</span>" +
            //        "<div class='plot-placeholder' id='cumulative-plot-placeholder'></div>"
            //    );
            cumulative.series = [];
            var plotsN = filenames.length;
            var refLength = 0;
            if (refLengths) {
                for (var i = 0, size = refLengths.length; i < size; i++)
                    refLength += refLengths[i];
            }
            if (refLength) {
                cumulative.maxY = refLength;
            }

            cumulative.colors = colors;

            function addCumulativeLenData(label, index, color, lengths, isRef) {
                if (!(lengths instanceof Array))
                    lengths = [lengths];
                var size = lengths.length;
                var points = {
                    data: new Array(size + 1),
                    label: label,
                    number: index,
                    color: color
                };

                points.data[0] = [0, 0];

                var y = 0;
                for (var j = 0; j < size; j++) {
                    y += lengths[j];
                    points.data[j+1] = [j+1, y];
                    if (y > cumulative.maxY) {
                        cumulative.maxY = y;
                    }
                }

                if (size > cumulative.maxX) {
                    cumulative.maxX = size;
                }
                if (isRef){
                    points.isReference = true;
                    points.dashes = {show: true, lineWidth: 1};
                    if (size < cumulative.maxX) {
                        points.data[size + 1] = [cumulative.maxX, y];
                    }
                }
                return points;
            }

            for (var i = 0; i < plotsN; i++) {
                var lengths = listsOfLengths[order[i]];
                var asm_name = filenames[order[i]];
                var color = colors[order[i]];
                cumulative.series[i] = addCumulativeLenData(asm_name, i, color, lengths);
            }

//            var lineColors = [];
//
//            for (i = 0; i < colors.length; i++) {
//                lineColors.push(changeColor(colors[i], 0.9, false));
//            }

            for (i = 0; i < plotsN; i++) {
                if (typeof broken_scaffolds_labels !== undefined && $.inArray(filenames[order[i]], broken_scaffolds_labels) != -1) {
                    cumulative.series[i].dashes = {
                        show: true,
                        lineWidth: 1
                    };
                }
                else {
                    cumulative.series[i].lines = {
                        show: true,
                        lineWidth: 1
                    };
                }
            }

            for (i = 0; i < plotsN; i++) {
                cumulative.colors.push(cumulative.series[i].color);
            }

            //cumulative.maxYTick = getMaxDecimalTick(cumulative.maxY);

            if (refLengths) {
                size = refLengths.length;
                var ref_label = 'reference,&nbsp;' + toPrettyString(refLength, 'bp');
                cumulative.series.push(addCumulativeLenData(ref_label, i, '#000000', refLengths, true));
                cumulative.colors.push('#000000');
            }


            //    if (referenceLength) {
            //        yaxes.push({
            //            ticks: [referenceLength],
            //            min: 0,
            //            max: maxYTick,
            //            position: 'right',
            ////            labelWidth: 50,
            //            reserveSpace: true,
            //            tickFormatter: function (val, axis) {
            //                return '<div style="">' + toPrettyStringWithDimension(referenceLength, 'bp') +
            //                    ' <span style="margin-left: -0.2em;">(reference)</span></div>';
            //            },
            //            minTickSize: 1,
            //        });
            //    }
            var yaxis = {
                min: 0,
                //max: cumulative.maxYTick,
                labelWidth: 120,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000000',
                tickFormatter: getBpTickFormatter(cumulative.maxY),
                minTickSize: 1
            };
            var yaxes = [yaxis];

            cumulative.showWithData = function(series, colors) {
                var plot = $.plot(placeholder, series, {
                    shadowSize: 0,
                    colors: cumulative.colors,
                    legend: {
                        container: $('useless-invisible-element-that-does-not-even-exist'),
                    },
                    //            legend: {
                    //                container: legendPlaceholder,
                    //                position: 'se',
                    //                labelBoxBorderColor: '#FFF',
                    //                labelFormatter: labelFormatter,
                    //            },
                    grid: {
                        borderWidth: 1,
                        hoverable: true,
                        autoHighlight: false,
                        mouseActiveRadius: 1000
                    },
                    yaxes: yaxes,
                    xaxis: {
                        min: 0,
                        max: cumulative.maxX,
                        lineWidth: 0.5,
                        color: '#000',
                        tickFormatter: getContigNumberTickFormatter(cumulative.maxX, tickX),
                        minTickSize: tickX
                    }
                });

                bindTip(placeholder, series, plot, ordinalNumberToPrettyString, tickX, 'contig', 'bottom right');
            };

            cumulative.isInitialized = true;
        }

        addLegendClickEvents(cumulative, filenames.length, showPlotWithInfo, refLengths);

        showPlotWithInfo(cumulative);

        $('#contigs_are_ordered').show();
    },
};

</script>

<script type="text/javascript">

var nx = {
    nx: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },

    nax: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },

    ngx: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },

    ngax: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },

    draw: function (name, title, colors, filenames, data, refPlotValue, tickX,
                    placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {

        $(scalePlaceholder).empty();

        var coordX = data.coord_x;
        var coordY = data.coord_y;

        var cur_filenames = data.filenames;
        var info = nx[name];

        if (!info.isInitialized) {
            var plotsN = cur_filenames.length;
            info.series = new Array(plotsN);

            for (var i = 0; i < plotsN; i++) {
                var index = $.inArray(cur_filenames[order[i]], filenames);
                var plot_coordX = coordX[order[i]];
                var plot_coordY = coordY[order[i]];
                var size = plot_coordX.length;

                info.series[i] = {
                    data: [],
                    label: filenames[index],
                    number: index,
                    color: colors[index]
                };
                info.series[i].data.push([0.0, plot_coordY[0]]);
                var currentLen = 0;
                var x = 0.0;

                for (var k = 0; k < size; k++) {
                    info.series[i].data.push([plot_coordX[k], plot_coordY[k]]);
                }

                if (info.series[i].data[0][1] > info.maxY) {
                    info.maxY = info.series[i].data[0][1];
                }

                var lastPt = info.series[i].data[info.series[i].data.length-1];
                info.series[i].data.push([lastPt[0], 0]);
            }

            for (i = 0; i < plotsN; i++) {
                if (typeof broken_scaffolds_labels !== undefined && $.inArray(filenames[order[i]], broken_scaffolds_labels) != -1) {
                    info.series[i].dashes = {
                        show: true,
                        lineWidth: 1
                    };
                }
                else {
                    info.series[i].lines = {
                        show: true,
                        lineWidth: 1
                    };
                }
            }

            // for (i = 0; i < plotsN; i++) {
            //     plotsData[i].points = {
            //         show: true,
            //         radius: 1,
            //         fill: 1,
            //         fillColor: false,
            //     }
            // }

            info.showWithData = function(series, colors) {
                var plot = $.plot(placeholder, series, {
                        shadowSize: 0,
                        colors: colors,
                        legend: {
                            container: $('useless-invisible-element-that-does-not-even-exist'),
                        },
                        grid: {
                            borderWidth: 1,
                            hoverable: true,
                            autoHighlight: false,
                            mouseActiveRadius: 1000
                        },
                        yaxis: {
                            min: 0,
//                        max: info.maxY,
                            labelWidth: 120,
                            reserveSpace: true,
                            lineWidth: 0.5,
                            color: '#000',
                            tickFormatter: getBpTickFormatter(info.maxY),
                            minTickSize: 1
                        },
                        xaxis: {
                            min: 0,
                            max: 100,
                            lineWidth: 0.5,
                            color: '#000',
                            tickFormatter: function (val, axis) {
                                if (val == 100) {
                                    return '&nbsp;x<span class="rhs">&nbsp;</span>=<span class="rhs">&nbsp;</span>100%'
                                } else {
                                    return val;
                                }
                            }
                        },
                        minTickSize: tickX
                    }
                );

                var firstLabel = $('.yAxis .tickLabel').last();
                firstLabel.prepend(title + '<span class="rhs">&nbsp;</span>=<span class="rhs">&nbsp;</span>');

                bindTip(placeholder, series, plot, toPrettyString, 1, '%', 'top right');

            };

            info.isInitialized = true;
        }

        addLegendClickEvents(info, filenames.length, showPlotWithInfo);
        showPlotWithInfo(info);
    }
};



</script>

<script type="text/javascript">

var normal_scale_span =
    "<span class='selected-switch'>" +
        'Normal' +
        "</span>";
var normal_scale_a =
    "<a class='dotted-link' onClick='setNormalScale()'>" +
        'Normal' +
        "</a>";
var log_scale_span =
    "<span class='selected-switch'>" +
        'logarithmic' +
        "</span>";
var log_scale_a =
    "<a class='dotted-link' onClick='setLogScale()'>" +
        'logarithmic' +
        "</a>";

var show_all_span =
    "<span class='selected-switch gc'>" +
        'Back to overview' +
        "</span>";
var show_all_a =
    "&nbsp;&nbsp;&nbsp;<a class='dotted-link' onClick='showAll()'>" +
        'Back to overview' +
        "</a>";

var gc = {
    isInitialized: false,

    maxY: 0,
    plot: null,
    series: null,
    showWithData: null,
    minPow: 0,
    ticks: null,
    placeholder: null,
    legendPlaceholder: null,
    colors: null,
    yAxisLabeled: false,

    show_all_el: show_all_span,
    reference: false,

    normal_scale_el: null,
    log_scale_el: null,

    draw: function(name, title, colors, filenames, gcInfos, reflen, tickX,
                   placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {
        gc.normal_scale_el = normal_scale_span;
        gc.log_scale_el = log_scale_a;
        $(scalePlaceholder).html(
            "<div id='change-scale' style='margin-right: 3px; visibility: hidden;'>" +
                "<span id='normal_scale_label'>" +
                gc.normal_scale_el +
                "</span>&nbsp;/&nbsp;" +
                "<span id='log_scale_label'>" +
                gc.log_scale_el +
                "</span> scale" +
                "</div>"
        );

        var refIndex = gcInfos.reference_index;
        if (!gc.isInitialized) {
            gc.legendPlaceholder = legendPlaceholder;
            gc.placeholder = placeholder;
            gc.colors = colors;
            gc.filenames = filenames;

            var bin_size = 1.0;
            var plotsN = filenames.length;
            gc.series = new Array(plotsN + 1);
            gc.series[0] = new Array(plotsN);

            gc.maxY = 0;
            var minY = Number.MAX_VALUE;

            function updateMinY(y) {
                if (y < minY && y != 0) {
                    minY = y;
                }
            }
            function updateMaxY(y) {
                if (y > gc.maxY) {
                    gc.maxY = y;
                }
            }

            for (var i = 0; i < plotsN; i++) {
                gc.series[0][i] = {
                    data: [],
                    label: filenames[order[i]],
                    number: order[i],
                    color: colors[order[i]]
                };
            }

            function makeSeriesFromDistributions(distributionsXandY, series_i, plot_i) {
                var distributionsX = distributionsXandY[0];
                var distributionsY = distributionsXandY[1];

                for (var j = 0; j < distributionsX.length; j++) {
                    var x = distributionsX[j];
                    var y = distributionsY[j];
                    gc.series[series_i][plot_i].data.push([x, y]);
                    updateMinY(y);
                    updateMaxY(y);
                }
            }

            function makeSeries(listsOfGCInfo, listOfGcDistributions, seriesIdx) {
                for (var i = 0; i < plotsN; i++) {
                    if (listsOfGCInfo) {
                        makeSeriesFromInfo(listsOfGCInfo[order[i]], seriesIdx, i);
                    } else {
                        makeSeriesFromDistributions(listOfGcDistributions[order[i]], seriesIdx, i);
                    }
                }
            }

            var listsOfGCInfo = gcInfos.lists_of_gc_info;
            var listOfGcDistributions = gcInfos.list_of_GC_distributions;
            makeSeries(listsOfGCInfo, listOfGcDistributions, 0);

            function makeSeriesFromInfo(GC_info, series_i, i) {
                var cur_bin = 0.0;

                var x = cur_bin;
                var y = filterAndSumGcInfo(GC_info, function(GC_percent) {
                    return GC_percent == cur_bin;
                });
                gc.series[series_i][i].data.push([x, y]);

                updateMinY(y);
                updateMaxY(y);

                while (cur_bin < 100.0 - bin_size) {
                    cur_bin += bin_size;

                    x = cur_bin;
                    y = filterAndSumGcInfo(GC_info, function(GC_percent) {
                        return GC_percent > (cur_bin - bin_size) && GC_percent <= cur_bin;
                    });
                    gc.series[series_i][i].data.push([x, y]);

                    updateMinY(y);
                    updateMaxY(y);
                }

                x = 100.0;
                y = filterAndSumGcInfo(GC_info, function(GC_percent) {
                    return GC_percent > cur_bin && GC_percent <= 100.0;
                });

                gc.series[series_i][i].data.push([x, y]);

                updateMinY(y);
                updateMaxY(y);
            }

            for (i = 0; i < plotsN; i++) {
                if (typeof broken_scaffolds_labels !== undefined && $.inArray(filenames[order[i]], broken_scaffolds_labels) != -1) {
                    gc.series[0][i].dashes = {
                        show: true,
                        lineWidth: 1
                    };
                }
                else {
                    gc.series[0][i].lines = {
                        show: true,
                        lineWidth: 1
                    };
                }
            }

            if (refIndex) {
                gc.reference = true;
                gc.series[0].push({
                    data: [],
                    label: 'reference',
                    isReference: true,
                    number: filenames.length,
                    lines: {},
                    dashes: {
                        show: true,
                        lineWidth: 1
                    },
                    color: '#000000'
                });
                if (listsOfGCInfo) {
                    makeSeriesFromInfo(listsOfGCInfo[refIndex], 0, refIndex);
                } else {
                    makeSeriesFromDistributions(listOfGcDistributions[refIndex], 0, refIndex);
                }
                gc.colors.push('#000000')
            }

            if (gcInfos.list_of_GC_contigs_distributions) {
                listOfGcDistributions = gcInfos.list_of_GC_contigs_distributions;
                var maxY = 0;
                for (var file_n = 0; file_n < filenames.length; file_n++) {
                    gc.series[file_n + 1] = new Array(1);
                    gc.series[file_n + 1][0] = {
                        data: [],
                        label: filenames[order[file_n]],
                        number: order[file_n],
                        color: colors[order[file_n]],
                        bars: {
                            show: true,
                            lineWidth: 0.6,
                            fill: 0.6,
                            barWidth: 5
                        }
                    };

                    var distributionsX = listOfGcDistributions[file_n][0];
                    var distributionsY = listOfGcDistributions[file_n][1];

                    for (var j = 0; j < distributionsX.length; j++) {
                        var x = distributionsX[j];
                        var y = distributionsY[j];
                        gc.series[file_n + 1][0].data.push([x, y]);
                        maxY = Math.max(maxY, y);
                    }
                }
                for (var file_n = 0; file_n < filenames.length; file_n++) {
                    gc.series[file_n + 1][0].maxY = maxY;
                }
            }
            // Calculate the minimum possible non-zero Y to clip useless bottoms
            // of logarithmic plots.
            var maxYTick = getMaxDecimalTick(gc.maxY);
            gc.minPow = Math.round(Math.log(minY) / Math.log(10));
            gc.ticks = [];
            for (var pow = gc.minPow; Math.pow(10, pow) < maxYTick; pow++) {
                gc.ticks.push(Math.pow(10, pow));
            }
            gc.ticks.push(Math.pow(10, pow));

            gc.isInitialized = true;
        }

        gc.showWithData = showInNormalScaleWithData;
        if (gcInfos.list_of_GC_contigs_distributions) {
            createLegend(gc.filenames, gc.colors, 0, gc.reference);
        }
        addLegendClickEvents(gc, filenames.length, showPlotWithInfo, false, 0);

        showPlotWithInfo(gc, 0);

        $('#change-scale').css('visibility', 'visible');
        $('#gc_info').show();
    }
};

function showAll() {
    $('#change-scale').show();
    $('#gc_info').show();
    $('#gc_contigs_info').hide();
    createLegend(gc.filenames, gc.colors, 0, gc.reference);

    gc.show_all_el = show_all_span;
    gc.showWithData = gc.log_scale_el == log_scale_a ? showInNormalScaleWithData : showInLogarithmicScaleWithData;

    $('#show_all_label').html(gc.show_all_el);
    showPlotWithInfo(gc, 0);
}

function showPlot(index) {
    $('#change-scale').hide();
    $('#gc_info').hide();
    $('#gc_contigs_info').show();
    createLegend([gc.filenames[index - 1]], [gc.colors[index - 1]], index);

    gc.show_all_el = show_all_a;
    gc.showWithData = showOneAssembly;

    $('#show_all_label').html(gc.show_all_el);
    showPlotWithInfo(gc, index);
}

function showOneAssembly(series, colors) {
    if (series == null) {
        return;
    }

    gc.plot = $.plot(gc.placeholder, series, {
            shadowSize: 0,
            colors: colors,
            legend: {
                container: $('useless-invisible-element-that-does-not-even-exist')
            },
            grid: {
                hoverable: true,
                borderWidth: 1,
                autoHighlight: false,
                mouseActiveRadius: 1000
            },
            yaxis: {
                min: 0,
                max: series[0].maxY * 1.1,
                labelWidth: 120,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: getJustNumberTickFormatter(gc.maxY),
                minTickSize: 1
            },
            xaxis: {
                min: 0,
                max: 100,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: function (val, axis) {
                    if (val == 100) {
                        return '&nbsp;100% GC'
                    } else {
                        return val;
                    }
                }
            },
            minTickSize: 1
        }
    );

    var firstLabel = $('.yAxis .tickLabel').last();
    firstLabel.append(' contigs');
    unBindTips(gc.placeholder);
    bindTip(gc.placeholder, series, gc.plot, getIntervalToPrettyString(5), 1, '%<span class="rhs">&nbsp;</span>GC', 'top right');
}

function showInNormalScaleWithData(series, colors) {
    if (series == null || gc.maxY == null)
        return;

    gc.yAxisLabeled = false;

    gc.plot = $.plot(gc.placeholder, series, {
            shadowSize: 0,
            colors: colors,
            legend: {
                container: $('useless-invisible-element-that-does-not-even-exist'),
            },
            grid: {
                borderWidth: 1,
                hoverable: true,
                autoHighlight: false,
                mouseActiveRadius: 1000,
            },
            yaxis: {
                min: 0,
//                max: gc.maxY + 0.1 * gc.maxY,
                labelWidth: 120,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: windowsTickFormatter,
                minTickSize: 1
            },
            xaxis: {
                min: 0,
                max: 100,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: function (val, axis) {
                    if (val == 100) {
                        return '&nbsp;100% GC'
                    } else {
                        return val;
                    }
                }
            },
            minTickSize: 1
        }
    );

    var firstLabel = $('.yAxis .tickLabel').last();
    firstLabel.append(' windows');
    bindTip(gc.placeholder, series, gc.plot, toPrettyString, 1, '%<span class="rhs">&nbsp;</span>GC', 'top right');
}


function showInLogarithmicScaleWithData(series, colors) {
    if (series == null || gc.maxY == null || gc.minPow == null) {
        return;
    }

    gc.yAxisLabeled = false;

    gc.plot = $.plot(gc.placeholder, series, {
            shadowSize: 0,
            colors: colors,
            legend: {
                container: $('useless-invisible-element-that-does-not-even-exist'),
            },
            grid: {
                hoverable: true,
                borderWidth: 1,
                autoHighlight: false,
                mouseActiveRadius: 1000
            },
            yaxis: {
                min: Math.pow(10, gc.minPow),
//                max: gc.maxY,
                labelWidth: 120,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: windowsTickFormatter,
                minTickSize: 1,
                ticks: gc.ticks,

                transform:  function(v) {
                    return Math.log(v + 0.0001)/*move away from zero*/ / Math.log(10);
                },
                inverseTransform: function(v) {
                    return Math.pow(v, 10);
                },
                tickDecimals: 3
            },
            xaxis: {
                min: 0,
                max: 100,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: function (val, axis) {
                    if (val == 100) {
                        return '&nbsp;100%<span class="rhs">&nbsp;</span>GC'
                    } else {
                        return val;
                    }
                }
            },
            minTickSize: 1
        }
    );

    var firstLabel = $('.yAxis .tickLabel').last();
    firstLabel.append(' windows');
    bindTip(gc.placeholder, series, gc.plot, toPrettyString, 1, '% GC', 'top right');
}


function setLogScale() {
    gc.normal_scale_el = normal_scale_a;
    gc.log_scale_el = log_scale_span;
    gc.showWithData = showInLogarithmicScaleWithData;

    $('#normal_scale_label').html(gc.normal_scale_el);
    $('#log_scale_label').html(gc.log_scale_el);
    showPlotWithInfo(gc, 0);
}


function setNormalScale() {
    gc.normal_scale_el = normal_scale_span;
    gc.log_scale_el = log_scale_a;
    gc.showWithData = showInNormalScaleWithData;

    $('#normal_scale_label').html(gc.normal_scale_el);
    $('#log_scale_label').html(gc.log_scale_el);
    showPlotWithInfo(gc, 0);
}



function filterAndSumGcInfo(GC_info, condition) {
    var contigs_lengths_cur_bin = [];
    for (var j = 0; j < GC_info.length; j++) {
        var GC = GC_info[j];
        var contig_length = GC[0];
        var GC_percent = GC[1];

        if (condition(GC_percent) == true) {
            contigs_lengths_cur_bin.push(contig_length);
        }
    }
    var val_bp = 0;
    for (var j = 0; j < contigs_lengths_cur_bin.length; j++) {
        val_bp += contigs_lengths_cur_bin[j];
    }
    return val_bp;
}

function createLegend(labels, colors, index, reference) {
    var selectedAssemblies = getSelectedAssemblies(labels);
    $('#legend-placeholder').empty();
    var selectors = "";

    labels.forEach(function(label, i) {
        var link = index ? '' : '<span id="' + labels[i] + '-switch"' + "class='plot-gc-type-switch dotted-link'>by contigs<br></span><br>";
        var assemblyIdx = gc.filenames.indexOf(label);
        addLabelToLegend(assemblyIdx, label, selectedAssemblies, colors, link);
    });
    if (reference) {
        isChecked = (selectedAssemblies.length > 0 && selectedAssemblies.indexOf(gc.filenames.length.toString())) != -1 ? 'checked="checked"' : "";
        $('#legend-placeholder').append(
            '<div id="reference-label">' +
                '<label for="reference" style="color: #000000;">' +
                '<input type="checkbox" name="' + gc.filenames.length +
                '" checked="' + isChecked + '" id="reference">&nbsp;' + 'reference' +
                '</label>' +
                '</div>'
        );
    }
    if (index > 0) {
        for (var filenames_n = 0; filenames_n < gc.filenames.length; filenames_n++){
            selectors += '<br><span id="' + gc.filenames[filenames_n] + '-switch" ' +
                "class='plot-switch dotted-link'>" +
                gc.filenames[filenames_n] + "</span>";
        }
        $('#legend-placeholder').append(
            "<br><br><div id='change-assembly' style='margin-right: 3px;'>" +
                "<span id='show_all_label'>" +
                gc.show_all_el +
                "</span><br>" + selectors +
                "</div>"
        );
    }
    addLinksToSwitches(index - 1);
    addLegendClickEvents(gc, gc.series.length, showPlotWithInfo, false, index);
}

function addLinksToSwitches(index) {
    var filenames = gc.filenames;
    for (filenames_n = 0; filenames_n < filenames.length; filenames_n++){
        var switchSpan = document.getElementById(filenames[filenames_n] + "-switch");
        $(switchSpan).click(getToggleSwitchFunction(filenames_n + 1));
        if (filenames_n == index) {
            switchSpan.className = 'plot-switch selected-switch gc';
        }
    }
}

function getToggleSwitchFunction(index) {
    return function() {
        if (index > 0) {
            showPlot(index);
        }
        else {
            showAll();
        }
    };
}
</script>

<script type="text/javascript">

var frc = {
    genes: {
        isInitialized: false,

        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null,

        yAxisLabeled: false
    },

    operons: {
        isInitialized: false,

        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null,

        yAxisLabeled: false
    },

    misassemblies: {
        isInitialized: false,

        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null,

        yAxisLabeled: false
    },

    draw: function (name, title, colors, filenames, data, refGenesNumber, tickX,
                    placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {
//    div.html(
//        "<span class='plot-header'>" + kind[0].toUpperCase() + kind.slice(1) + "s covered</span>" +
//        "<div class='plot-placeholder' id='" + kind + "s-plot-placeholder'></div>"
//    );
        $(scalePlaceholder).empty();

        var info = frc[name];
        var coordX = data.coord_x;
        var coordY = data.coord_y;

        info.yAxisLabeled = false;
        var cur_filenames = data.filenames;
        if (!info.isInitialized) {
            var plotsN = cur_filenames.length;
            info.series = new Array(plotsN);

            info.maxY = 0;
            info.maxX = 0;

            for (var i = 0; i < plotsN; i++) {
                var index = $.inArray(cur_filenames[order[i]], filenames);
                var plot_coordX = coordX[order[i]];
                var plot_coordY = coordY[order[i]];
                var featureSpace = plot_coordX[plot_coordX.length - 1];
                var maxY = plot_coordY[plot_coordY.length - 1];

                info.series[i] = {
                    data: [],
                    label: filenames[index],
                    number: index,
                    color: colors[index]
                };

                info.series[i].data.push([0.0, plot_coordY[0]]);

                if (featureSpace > info.maxX) {
                    info.maxX = featureSpace;
                }
                for (var k = 0; k < plot_coordX.length; k++) {
                    info.series[i].data.push([plot_coordX[k], plot_coordY[k]]);
                }

                if (maxY > info.maxY) {
                    info.maxY = maxY;
                }
            }

            for (i = 0; i < plotsN; i++) {
                if (typeof broken_scaffolds_labels !== undefined && $.inArray(filenames[order[i]], broken_scaffolds_labels) != -1) {
                    info.series[i].dashes = {
                        show: true,
                        lineWidth: 1
                    };
                }
                else {
                    info.series[i].lines = {
                        show: true,
                        lineWidth: 1
                    };
                }
            }

            info.showWithData = function(series, colors) {
                var plot = $.plot(placeholder, series, {
                    shadowSize: 0,
                    colors: colors,
                    legend: {
                        container: $('useless-invisible-element-that-does-not-even-exist')
                    },
                    grid: {
                        borderWidth: 1,
                        hoverable: true,
                        autoHighlight: false,
                        mouseActiveRadius: 1000,
                    },
                    yaxis: {
                        min: 0,
                        max: Math.max(100, info.maxY),
                        labelWidth: 145,
                        reserveSpace: true,
                        lineWidth: 0.5,
                        color: '#000',
                        tickFormatter: getPercentTickFormatter(Math.max(100, info.maxY)),
                        minTickSize: 1
                    },
                    xaxis: {
                        min: 0,
                        max: info.maxX,
                        lineWidth: 0.5,
                        color: '#000',
                        tickFormatter: getJustNumberTickFormatter(info.maxX, ' ' + name),
                        minTickSize: tickX
                    }
                });

                var firstLabel = $('.yAxis .tickLabel').last();
                firstLabel.prepend('Genome coverage<span class="rhs">&nbsp;</span>=<span class="rhs">&nbsp;</span>');

                bindTip(placeholder, series, plot, frcNumberToPrettyString, tickX, name, 'bottom right', false, '%');
            };

            info.isInitialized = true;
        }

        addLegendClickEvents(info, filenames.length, showPlotWithInfo, refGenesNumber);
        showPlotWithInfo(info);
        $('#frc_info').show();
        $('.frc_plot_name').html(name);
    }
};



</script>

<script type="text/javascript">

/**********/
/* COLORS */

// var colors = ["#FF5900", "#008FFF", "#168A16", "#7C00FF", "#00B7FF", "#FF0080", "#7AE01B", "#782400", "#E01B6A"];
var standard_colors = [
    '#FF0000', //red
    '#0000FF', //blue
    '#008000', //green
    '#FFA500', //orange
    '#FF00FF', //fushua
    '#CCCC00', //yellow
    '#800000', //maroon
    '#00CCCC', //aqua
    '#808080', //gray
    '#800080', //purple
    '#808000', //olive
    '#000080', //navy
    '#008080', //team
    '#00FF00', //lime
];

function distinctColors(count) {
    var colors = [];
    for(var hue = 0; hue < 360; hue += 360 / count) {
        var color = hsvToRgb(hue, 100, 100);
        var colorStr = '#' + color[0].toString(16) + color[1].toString(16) + color[2].toString(16);
        colors.push();
    }
    return colors;
}

/**************/
/* FORMATTING */
function isIntegral(num) {
    return num % 1 === 0;
}

function isFractional(num) {
    return !isIntegral(num);
}

function getIntervalToPrettyString(interval) {
    return function(num, unit) {
        return intervalToPrettyString(interval, num, unit);
    }
}

function intervalToPrettyString(interval, num, unit) {
    if (typeof num === 'number') {
        var str = toPrettyString(num);
        str += '-' + toPrettyString(num + interval);
        str += (unit ? '<span class="rhs">&nbsp;</span>' + unit : '');
        return str;
    } else {
        return num;
    }
}

function toPrettyString(num, unit) {
    if (typeof num === 'number') {
        var str;
        if (num <= 9999) {
            if (isFractional(num)) {
                if (isIntegral(num * 10)) {
                    str = num.toFixed(1);
                } else if (isIntegral(num * 100) || num >= 100) {
                    str = num.toFixed(2);
                } else {
                    str = num.toFixed(3);
                    if (str.slice(-1) == '0')
                        str = str.slice(0, -1);
                }
            } else {
                str = num.toFixed(0);
            }
        } else {
            str = num.toFixed(0).replace(/(\d)(?=(\d\d\d)+(?!\d))/g,'$1<span class="hs"></span>');
        }
        str += (unit ? '<span class="rhs">&nbsp;</span>' + unit : '');
        return str;
    } else {
        return num;
    }
}

        //elif isinstance(value, float):
        //    if value == 0.0:
        //        return '0'
        //    if human_readable:
        //        if unit == '%':
        //            value *= 100
        //        precision = 2
        //        for i in range(10, 1, -1):
        //            if abs(value) < 1./(10**i):
        //                precision = i + 1
        //                break
        //        return '{value:.{precision}f}{unit_str}'.format(**locals())
        //    else:
        //        return str(value)


function refToPrettyString(num, refs) {
    return refs[Math.round(num)-1];
}

function ordinalNumberToPrettyString(num, unit, tickX) {
    num = num * tickX;
    var numStr = num.toString();
    var lastDigit = numStr[numStr.length-1];
    var beforeLastDigit = numStr[numStr.length-2];

    var res = toPrettyString(num);

    if (lastDigit == '1' && beforeLastDigit != '1') {
        res += "st";
    } else if (lastDigit == '2' && beforeLastDigit != '1') {
        res += "nd";
    } else if (lastDigit == '3' && beforeLastDigit != '1') {
        res += "rd";
    } else {
        res += 'th';
    }

    res += (unit ? '<span class="rhs">&nbsp;</span>' + unit : '');

    return res;
}

function frcNumberToPrettyString(num, unit, tickX, index) {
    if (index % 2 == 0 && num > 0) num--;
    return toPrettyString(num) + ' ' + unit;
}

function getMaxDecimalTick(maxY) {
    var maxYTick = maxY;
    if (maxY <= 100000000000) {
        maxYTick = Math.ceil((maxY+1)/10000000000)*10000000000;
    } if (maxY <= 10000000000) {
        maxYTick = Math.ceil((maxY+1)/1000000000)*1000000000;
    } if (maxY <= 1000000000) {
        maxYTick = Math.ceil((maxY+1)/100000000)*100000000;
    } if (maxY <= 100000000) {
        maxYTick = Math.ceil((maxY+1)/10000000)*10000000;
    } if (maxY <= 10000000) {
        maxYTick = Math.ceil((maxY+1)/1000000)*1000000;
    } if (maxY <= 1000000) {
        maxYTick = Math.ceil((maxY+1)/100000)*100000;
    } if (maxY <= 100000) {
        maxYTick = Math.ceil((maxY+1)/10000)*10000;
    } if (maxY <= 10000) {
        maxYTick = Math.ceil((maxY+1)/1000)*1000;
    } if (maxY <= 1000) {
        maxYTick = Math.ceil((maxY+1)/100)*100.
    } if (maxY <= 100) {
        maxYTick = Math.ceil((maxY+1)/10)*10.
    }
    return maxYTick;
}

function getBpTickFormatter(maxY, additionalText) {
    additionalText = additionalText || '';

    return function(val, axis) {
        var res;
        if (val == 0) {
            res = 0;

        } else if (axis.max >= 1000000) {
            res = val / 1000000;

            if (val > axis.max - 1 || val + axis.tickSize >= 1000000000) {
                res = additionalText + toPrettyString(res, 'Mbp');
            } else {
                res = toPrettyString(res);
            }
        } else if (axis.max >= 1000) {
            res = val / 1000;

            if (val > axis.max - 1 || val + axis.tickSize >= 1000000) {
                res = additionalText + toPrettyString(res, 'kbp');
            } else {
                res = toPrettyString(res);
            }
        } else if (axis.max >= 1) {
            res = val;

            if (val > axis.max - 1 || val + axis.tickSize >= 1000) {
                res = additionalText + toPrettyString(res, 'bp');
            } else {
                res = toPrettyString(res);
            }
        }
        return res;
    }
}

function windowsTickFormatter(v, axis) {
    return toPrettyString(v);
//    var val = v.toFixed(0);
//    if (!gc.yAxisLabeled && val > gc.maxY) {
//        gc.yAxisLabeled = true;
//        var res = val + ' window';
//        if (val > 1) {
//            res += 's'
//        }
//        return res;
//    } else {
//        return val;
//    }
}

function getBpLogTickFormatter(maxY) {
    return getBpTickFormatter(maxY);
}

function getContigNumberTickFormatter(maxX, tickX) {
    return function (val, axis) {
        if (typeof axis.tickSize == 'number' && val > maxX - axis.tickSize) {
            return "&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;" + ordinalNumberToPrettyString(val, 'contig', tickX);
        } else {
            return val * tickX;
        }
    }
}

function getJustNumberTickFormatter(maxY, additionalText) {
    return function (val, axis) {
        additionalText = additionalText || '';
        if (val > axis.max - axis.tickSize) {
            res = toPrettyString(val) + additionalText;
        } else {
            res = toPrettyString(val);
        }
        return res;
    }
}

function getPercentTickFormatter(maxY, additionalText) {
    return function (val, axis) {
        additionalText = additionalText || '';
        if (val > maxY + 1 || val == 100) {
            res = additionalText + toPrettyString(val, '%');
        } else {
            res = toPrettyString(val);
        }
        return res;
    }
}

function trim(str) {
    return str.replace(/^\s+/g, '');
}

function initial_spaces_to_nbsp(str, metricName) {
    if (metricName.length > 0 && metricName[0] == ' ') {
        str = '&nbsp;&nbsp;&nbsp;' + str;
    }
    return str;
}

function containsObject(obj, list) {
    var i;
    for (i = 0; i < list.length; i++) {
        if (list[i] === obj) {
            return true;
        }
    }

    return false;
}

function addLegendClickEvents(info, numLegendItems, showPlotWithInfo, showReference, index) {
    if (showReference) numLegendItems++;
    for (var i = 0; i < numLegendItems; i++) {
        $('#legend-placeholder input[name=' + i + ']').click(function() {
            showPlotWithInfo(info, index);
        });
    }
}

/*********************/
/* GLOSSARY TOOLTIPS */
function addTooltipIfDefinitionExists(glossary, metricName) {
    metricName = trim(metricName);

    if (containsObject(metricName, Object.keys(glossary))) {
        return '<a class="tooltip-link" rel="tooltip" title="' +
            metricName + ' ' + glossary[metricName] + '">' + metricName + '</a>';
    } else {
        return metricName;
    }
}

/*************/
/* PLOT TIPS */
function bindTip(placeholder, series, plot, xToPrettyStringFunction, tickX, xUnit, position, summaryPlots, unitY) {
    var prevPoint = null;
    var prevIndex = null;

    $(placeholder).bind("plothover", function(event, pos, item) {
        if (dragTable && dragTable.isDragging)
            return;

        if (item) {
            if (prevPoint != item.dataIndex || (summaryPlots && item.seriesIndex != prevIndex)) {
                prevPoint = item.dataIndex;
                prevIndex = item.seriesIndex;
                var x = item.datapoint[0];

                showTip(item.pageX, item.pageY, plot.offset(),
                    plot.width(), plot.height(),
                    series, item.seriesIndex, x, item.dataIndex,
                    xToPrettyStringFunction(x, xUnit, tickX, item.dataIndex) + ':',
                    position, summaryPlots, unitY);
            }
        } else {
            $('#plot_tip').hide();
            $('#plot_tip_vertical_rule').hide();
            $('#plot_tip_horizontal_rule').hide();
            prevPoint = null;
        }
    });
}

function unBindTips(placeholder) {
    $(placeholder).unbind("plothover");
}

var tipElementExists = false;
function showTip(pageX, pageY, offset, plotWidth, plotHeight,
                 series, centralSeriesIndex, xPos, xIndex, xStr, position, summaryPlots, unitY) {
    const LINE_HEIGHT = 16; // pixels

    position = ((position != null) ? position : 'bottom right');
//    pageY -= LINE_HEIGHT * (centralSeriesIndex + 1.5);

    var directions = position.split(' ');

    if (!tipElementExists) {
        $('<div id="plot_tip" class="white_stroked"></div>').appendTo('body');

        $('<div id="plot_tip_vertical_rule"></div>').css({
            height: plotHeight,
        }).appendTo('body');

        $('<div id="plot_tip_horizontal_rule"></div>').css({
            width: plotWidth,
        }).appendTo('body');

        tipElementExists = true;
    }

    $('#plot_tip').html('').css({
        top: pageY + 5 - ((directions[0] == 'top') ? LINE_HEIGHT * (series.length + 2) : 0),
        left: pageX + 10,
    }).show();

    $('#plot_tip_vertical_rule').html('').css({
        top: offset.top,
        left: pageX,
    }).show();

    $('#plot_tip_horizontal_rule').html('').css({
        top: pageY,
        left: offset.left,
    }).show();

    $('<div>' + xStr + '</div>').css({
        height: LINE_HEIGHT,
    }).appendTo('#plot_tip');

    var sortedYsAndColors = [];
    for (var i = 0; i < series.length; i++) {
        if (!summaryPlots || (summaryPlots && series[i].data[xIndex] != undefined && series[i].data[xIndex][1] != null)) {
            sortedYsAndColors.push({
                y: summaryPlots ? series[i].data[xIndex][1] : (i == centralSeriesIndex ? (series[i].data[xIndex] || series[i].data[series[i].data.length - 1])[1] :
                    findNearestPoint(series[i].data, xPos)),
                color: series[i].color,
                label: (series[i].isReference ? 'Reference' : series[i].label),
                isCurrent: i == centralSeriesIndex,
            });
        }
    }
    sortedYsAndColors.sort(function(a, b) { return a.y < b.y;});

    for (i = 0; i < sortedYsAndColors.length; i++) {
        var item = sortedYsAndColors[i];

        $('<div id="tip_line' + i + '">' + toPrettyString(item.y) + (unitY ? unitY : '') +
            ', <span style="color: ' + item.color + ';">' + item.label + '</span></div>').css({
            height: LINE_HEIGHT,
            "font-weight": item.isCurrent ? "bold" : "normal",
        }).appendTo('#plot_tip');
    }
}

function findNearestPoint(points, x) {
    for (var i = 0; i < points.length; i++) {
        if (points[i][0] >= x) return points[i][1];
    }
    return points[points.length-1][1]
}

function addLabelToLegend(idx, label, selectedLabels, colors, link) {
    var isChecked = (selectedLabels.length > 0 && selectedLabels.indexOf(idx.toString())) != -1 ? 'checked="checked"' : "";
    $('#legend-placeholder').append('<div>' +
        '<label for="' + label + '" style="color: ' + colors[idx] + '">' +
        '<input type="checkbox" name="' + idx + '"' + isChecked + ' id="' + label + '">&nbsp;' + label + '</label>' +
        (link ? '<br>' + link : '') + '</div>');
}

function getSelectedAssemblies(labels) {
    var selectedAssemblies = [];
    var labelsMatch = false;
    var legendLabels = [];
    $('#legend-placeholder input[type="checkbox"]').each(function() {
        legendLabels.push($(this).attr('id'));
    });
    if (labels.every(function(label, i) { return ($.inArray(label, legendLabels) != -1 || label == 'reference')}) ) {
        labelsMatch = true;
    }
    if (labelsMatch) {
        $('#legend-placeholder input:checked[type="checkbox"]').each(function() {
            selectedAssemblies.push($(this).attr('name'));
        });
    }
    else {
        selectedAssemblies = Array.apply(null, {length: labels}).map(Number.call, Number);
    }
    return selectedAssemblies;
}

// Cookie functions based on http://www.quirksmode.org/js/cookies.html
// Cookies won't work for local files.

var createCookie = function(name, value, days) {
    var expires = '';
    if (days) {
        var date = new Date();
        date.setTime(date.getTime() + (days * 24 * 60 * 60 * 1000));
        expires = '; expires=' + date.toGMTString();
    }
    var path = document.location.pathname;
    document.cookie = name + '=' + value + expires + '; path=' + path;
};

var readCookie = function(name) {
    var nameEQ = name + '=';
    var ca = document.cookie.split(';');
    for(var i = 0; i < ca.length; i++) {
        var c = ca[i];
        while (c.charAt(0) == ' ') {
            c = c.substring(1, c.length);
        }
        if (c.indexOf(nameEQ) == 0) {
            return c.substring(nameEQ.length, c.length);
        }
    }
    return null;
};

var eraseCookie = function(name) {
    createCookie(name, '', -1);
};


// Dean's forEach: http://dean.edwards.name/base/forEach.js
/*forEach, version 1.0
 Copyright 2006, Dean Edwards
 License: http://www.opensource.org/licenses/mit-license.php */

// array-like enumeration
if (!Array.forEach) { // mozilla already supports this
    Array.forEach = function(array, block, context) {
        for (var i = 0; i < array.length; i++) {
            block.call(context, array[i], i, array);
        }
    };
}

// generic enumeration
Function.prototype.forEach = function(object, block, context) {
    for (var key in object) {
        if (typeof this.prototype[key] == "undefined") {
            block.call(context, object[key], key, object);
        }
    }
};

// character enumeration
String.forEach = function(string, block, context) {
    Array.forEach(string.split(""), function(chr, index) {
        block.call(context, chr, index, string);
    });
};

// globally resolve forEach enumeration
var forEach = function(object, block, context) {
    if (object) {
        if (object instanceof Function) {                 // functions have a "length" property
            Function.forEach(object, block, context);

        } else if (object.each instanceof Function) {     // jQuery
            object.each(function(i, elt) {
                block(elt);
            }, context);

        } else if (object.forEach instanceof Function) {  // the object implements a custom forEach method
            object.forEach(block, context);

        } else if (typeof object == "string") {           // a string
            String.forEach(object, block, context);

        } else if (typeof object.length == "number") {    // array-like object
            Array.forEach(object, block, context);

        } else {
            Object.forEach(object, block, context);
        }
    }
};


jQuery.fn.exists = function(){
    return jQuery(this).length > 0;
};


function Range(from, to) {
    var r  = [];
    for (var i = from; i < to; i++) {
        r.push(i);
    }
    return r;
}



</script>

<script type="text/javascript">
/**
 * HSV to RGB color conversion
 *
 * H runs from 0 to 360 degrees
 * S and V run from 0 to 100
 *
 * Ported from the excellent java algorithm by Eugene Vishnevsky at:
 * http://www.cs.rit.edu/~ncs/color/t_convert.html
 */
function hsvToRgb(h, s, v) {
    var r, g, b;
    var i;
    var f, p, q, t;

    // Make sure our arguments stay in-range
    h = Math.max(0, Math.min(360, h));
    s = Math.max(0, Math.min(100, s));
    v = Math.max(0, Math.min(100, v));

    // We accept saturation and value arguments from 0 to 100 because that's
    // how Photoshop represents those values. Internally, however, the
    // saturation and value are calculated from a range of 0 to 1. We make
    // That conversion here.
    s /= 100;
    v /= 100;

    if(s == 0) {
        // Achromatic (grey)
        r = g = b = v;
        return [Math.round(r * 255), Math.round(g * 255), Math.round(b * 255)];
    }

    h /= 60; // sector 0 to 5
    i = Math.floor(h);
    f = h - i; // factorial part of h
    p = v * (1 - s);
    q = v * (1 - s * f);
    t = v * (1 - s * (1 - f));

    switch(i) {
        case 0:
            r = v;
            g = t;
            b = p;
            break;

        case 1:
            r = q;
            g = v;
            b = p;
            break;

        case 2:
            r = p;
            g = v;
            b = t;
            break;

        case 3:
            r = p;
            g = q;
            b = v;
            break;

        case 4:
            r = t;
            g = p;
            b = v;
            break;

        default: // case 5:
            r = v;
            g = p;
            b = q;
    }

    return [Math.round(r * 255), Math.round(g * 255), Math.round(b * 255)];
}

function hslToRgb(h, s, l){
    var r, g, b;

    if(s == 0) {
        r = g = b = l; // achromatic
    } else {
        function hue2rgb(p, q, t){
            if(t < 0) t += 1;
            if(t > 1) t -= 1;
            if(t < 1/6) return p + (q - p) * 6 * t;
            if(t < 1/2) return q;
            if(t < 2/3) return p + (q - p) * (2/3 - t) * 6;
            return p;
        }

        var q = l < 0.5 ? l * (1 + s) : l + s - l * s;
        var p = 2 * l - q;
        r = hue2rgb(p, q, h + 1/3);
        g = hue2rgb(p, q, h);
        b = hue2rgb(p, q, h - 1/3);
    }

    return [Math.round(r * 255), Math.round(g * 255), Math.round(b * 255)];
}
</script>

<script type="text/javascript">

var gns = {
    features: {
        isInitialized: false,

        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null,

        yAxisLabeled: false
    },

    operons: {
        isInitialized: false,

        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null,

        yAxisLabeled: false
    },

    draw: function (name, title, colors, filenames, data, refFeaturesNumber, tickX,
                    placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {
//    div.html(
//        "<span class='plot-header'>" + kind[0].toUpperCase() + kind.slice(1) + "s covered</span>" +
//        "<div class='plot-placeholder' id='" + kind + "s-plot-placeholder'></div>"
//    );
        $(scalePlaceholder).empty();

        var info = gns[name];

        info.yAxisLabeled = false;
        var cur_filenames = data.filenames;
        if (!info.isInitialized) {
            var filesFeatureInContigs = data.filesFeatureInContigs;
            var kind = data.kind;

            var plotsN = cur_filenames.length;
            info.series = new Array(plotsN);

            info.maxY = 0;
            info.maxX = 0;

            if (refFeaturesNumber) {
                info.maxY = refFeaturesNumber;
            }

            for (var fi = 0; fi < plotsN; fi++) {
                var index = $.inArray(cur_filenames[order[fi]], filenames);
                var filename = filenames[index];
                var featureInContigs = filesFeatureInContigs[filename];

                info.series[fi] = {
                    data: [[0, 0]],
                    label: filenames[index],
                    number: index,
                    color: colors[index]
                };

                var contigNo = 0;
                var totalFull = 0;

                for (var k = 0; k < featureInContigs.length; k++) {
                    contigNo += 1;
                    totalFull += featureInContigs[k];

                    info.series[fi].data.push([contigNo, totalFull]);

                    if (info.series[fi].data[k][1] > info.maxY) {
                        info.maxY = info.series[fi].data[k][1];
                    }
                }

                if (featureInContigs.length > info.maxX) {
                    info.maxX = featureInContigs.length;
                }
            }

            for (i = 0; i < plotsN; i++) {
                if (typeof broken_scaffolds_labels !== undefined && $.inArray(filenames[order[i]], broken_scaffolds_labels) != -1) {
                    info.series[i].dashes = {
                        show: true,
                        lineWidth: 1
                    };
                }
                else {
                    info.series[i].lines = {
                        show: true,
                        lineWidth: 1
                    };
                }
            }

            //    for (i = 0; i < plotsN; i++) {
            //        plotsData[i].points = {
            //            show: true,
            //            radius: 1,
            //            fill: 1,
            //            fillColor: false,
            //        }
            //    }

            if (refFeaturesNumber) {
                info.series.push({
                    data: [[0, refFeaturesNumber], [info.maxX, refFeaturesNumber]],
                    label: 'reference,&nbsp;' + toPrettyString(refFeaturesNumber, 'features'),
                    isReference: true,
                    dashes: {
                        show: true,
                        lineWidth: 1
                    },
                    yaxis: 1,
                    number: filenames.length,
                    color: '#000000'
                });

                colors.push('#000000');
            }

            info.showWithData = function(series, colors) {
                var plot = $.plot(placeholder, series, {
                    shadowSize: 0,
                    colors: colors,
                    legend: {
                        container: $('useless-invisible-element-that-does-not-even-exist'),
                    },
                    grid: {
                        borderWidth: 1,
                        hoverable: true,
                        autoHighlight: false,
                        mouseActiveRadius: 1000,
                    },
                    yaxis: {
                        min: 0,
//                        max: info.maxY,
                        labelWidth: 120,
                        reserveSpace: true,
                        lineWidth: 0.5,
                        color: '#000',
                        tickFormatter: function (val, axis) {
//                            if (!info.yAxisLabeled && val > info.maxY) {
//                                info.yAxisLabeled = true;
//                                var res = val + ' ' + kind;
//                                if (val > 1) {
//                                    res += 's'
//                                }
//                                return res;
//                            } else {
                            return val;
//                            }
                        },
                        minTickSize: 1
                    },
                    xaxis: {
                        min: 0,
                        max: info.maxX,
                        lineWidth: 0.5,
                        color: '#000',
                        tickFormatter: getContigNumberTickFormatter(info.maxX, tickX),
                        minTickSize: tickX
                    },
                });

                var firstLabel = $('.yAxis .tickLabel').last();
                firstLabel.append(' ' + name);

                bindTip(placeholder, series, plot, ordinalNumberToPrettyString, tickX, 'contig', 'bottom right');
            };

            info.isInitialized = true;
        }

        addLegendClickEvents(info, filenames.length, showPlotWithInfo, refFeaturesNumber);

        showPlotWithInfo(info);

        $('#contigs_are_ordered').show();
    }
};



</script>

<script type="text/javascript">
/*
 DT v1.0
 June 26, 2008
 Dan Vanderkam, http://danvk.org/DT/
 http://code.google.com/p/DT/

 This is code was based on:
 - Stuart Langridge's SortTable (kryogenix.org/code/browser/sorttable)
 - Mike Hall's draggable class (http://www.brainjar.com/dhtml/drag/)
 - A discussion of permuting table columns on comp.lang.javascript

 Licensed under the MIT license.
 */

// Here's the notice from Mike Hall's draggable script:
//*****************************************************************************
// Do not remove this notice.
//
// Copyright 2001 by Mike Hall.
// See http://www.brainjar.com for terms of use.
//*****************************************************************************

var classes = {
    cell_of_emptySpace: 'cell_of_empty_space',
    cell_of_movingTable: 'cell_of_moving_table',
    newTable: 'moving_table',
    handle: 'drag_handle',
    hoveredCell: 'hovered_column_td',
    assemblyName: 'assembly_name',

    bottomCell_of_emptySpace: 'bottom_cell_of_empty_space',
    topCell_of_emptySpace: 'top_cell_of_empty_space',
    leftToCell_of_emptySpace: 'left_to_cell_of_empty_space',
    rightToCell_of_emptySpace: 'right_to_cell_of_empty_space',

    mainReportTable_id: 'main_report_table',
    totalReportJson_id: 'total-report-json',
};

var dragTable = {
    isDragging: false,

    // How far should the mouse move before it's considered a drag, not a click?
    dragRadius2: 0, // 100,
    setMinDragDistance: function(x) {
        DT.dragRadius2 = x * x;
    },

    // How long should cookies persist? (in days)
    cookieDays: 365,
    setCookieDays: function(x) {
        DT.cookieDays = x;
    },

    // Determine browser and version.
    // TODO: eliminate browser sniffing except where it's really necessary.
    Browser: function() {
        var ua, s, i;

        this.isIE    = false;
        this.isNS    = false;
        this.version = null;
        ua = navigator.userAgent;

        s = "MSIE";
        if ((i = ua.indexOf(s)) >= 0) {
            this.isIE = true;
            this.version = parseFloat(ua.substr(i + s.length));
            return;
        }

        s = "Netscape6/";
        if ((i = ua.indexOf(s)) >= 0) {
            this.isNS = true;
            this.version = parseFloat(ua.substr(i + s.length));
            return;
        }

        // Treat any other "Gecko" browser as NS 6.1.
        s = "Gecko";
        if ((i = ua.indexOf(s)) >= 0) {
            this.isNS = true;
            this.version = 6.1;
            return;
        }
    },
    browser: null,

    // Detect all draggable tables and attach handlers to their headers.
    init: function() {
        // Don't initialize twice
        if (arguments.callee.done)
            return;
        else
            arguments.callee.done = true;

//    if (_dgtimer)
//      clearInterval(_dgtimer);

        if (!document.createElement || !document.getElementsByTagName)
            return;

        DT.dragObj.zIndex = 0;
        DT.browser = new DT.Browser();
        forEach(document.getElementsByTagName('table'), function(table) {
            if ($(table).hasClass('draggable')) {
                DT.makeDraggable(table);
            }
        });
    },

    // The thead business is taken straight from sorttable.
    makeDraggable: function(table) {
        DT.dragObj.table = table;

        if (table.getElementsByTagName('thead').length == 0) {
            thead = document.createElement('thead');
            thead.appendChild(table.rows[0]);
            table.insertBefore(thead, table.firstChild);
        }

        // Safari doesn't support table.tHead, sigh
        if (table.tHead == null) {
            table.tHead = table.getElementsByTagName('thead')[0];
        }

        var handles = $('.' + classes.handle) || table.tHead.rows[0].cells;

        forEach(handles, function(handle) {
            handle.onmousedown = DT.dragStart;
        });
    },

    // Global object to hold drag information.
    dragObj: {},

    getEventPosition: function(event) {
        var x, y;
        if (DT.browser.isIE) {
            x = window.event.clientX + document.documentElement.scrollLeft
                + document.body.scrollLeft;
            y = window.event.clientY + document.documentElement.scrollTop
                + document.body.scrollTop;
            return {x: x, y: y};
        }
        return {x: event.pageX, y: event.pageY};
    },

    // Determine the position of this element on the page. Many thanks to Magnus
    // Kristiansen for help making this work with "position: fixed" elements.
    getPositionOnThePage: function(elt, stopAtRelative) {
        var ex = 0, ey = 0;
        do {
            var curStyle = DT.browser.isIE ? elt.currentStyle
                : window.getComputedStyle(elt, '');
            var supportFixed = !(DT.browser.isIE &&
                DT.browser.version < 7);
            if (stopAtRelative && curStyle.position == 'relative') {
                break;
            } else if (supportFixed && curStyle.position == 'fixed') {
                // Get the fixed el's offset
                ex += parseInt(curStyle.left, 10);
                ey += parseInt(curStyle.top, 10);
                // Compensate for scrolling
                ex += document.body.scrollLeft;
                ey += document.body.scrollTop;
                // End the loop
                break;
            } else {
                ex += elt.offsetLeft;
                ey += elt.offsetTop;
            }
        } while (elt = elt.offsetParent);
        return {x: ex, y: ey};
    },

    // MouseDown handler -- sets up the appropriate mousemove/mouseup handlers
    // and fills in the global DT.dragObj object.
    dragStart: function(event, id) {
        DT.isDragging = true;

        var drag_obj = DT.dragObj;

        var browser = DT.browser;
        var clicked_node = browser.isIE ? window.event.srcElement : event.target;
        var pos = DT.getEventPosition(event);

        var table = $(clicked_node).closest('table')[0];
        var top_cell = $(clicked_node).closest('td, th')[0];
        drag_obj.topCell = top_cell;
        drag_obj.table = table;

        var col_index = top_cell.cellIndex;
        drag_obj.movingColIndex = drag_obj.startIndex = col_index;
        drag_obj.movingColWidth = $(top_cell).width();

        // Since a column header can't be dragged directly, duplicate its contents
        // in a div and drag that instead.
        var new_table = DT.fullCopy(table, false);

        // Copy the entire column
        var forEachSection = function(table, func) {
            if (table.tHead) {
                func(table.tHead);
            }
            forEach(table.tBodies, function(tbody) {
                func(tbody);
            });
            if (table.tFoot) {
                func(table.tFoot);
            }
        };

        forEachSection(table, function(sec) {
            var new_sec = DT.fullCopy(sec, false);

            forEach(sec.rows, function(row) {
                var cell = row.cells[col_index];
                var new_tr = DT.fullCopy(row, false);
                if (row.offsetHeight) new_tr.style.height = row.offsetHeight + "px";
                var new_td = DT.fullCopy(cell, true);
                if (cell.offsetWidth) new_td.style.width = cell.offsetWidth + "px !important";
                new_tr.appendChild(new_td);
                new_sec.appendChild(new_tr);
            });

            new_table.appendChild(new_sec);
        });

        var obj_pos = DT.getPositionOnThePage(top_cell, true);
        new_table.style.position = "absolute";
        new_table.style.left = obj_pos.x + "px";
        new_table.style.top = obj_pos.y + "px";
        // new_table.style.width = dragObj.origNode.offsetWidth + "px";
        // new_table.style.height = dragObj.origNode.offsetHeight + "px";

        // Hold off adding the element until this is clearly a drag.
        drag_obj.addedNode = false;

        drag_obj.tableContainer = drag_obj.table.parentNode || document.body;
        drag_obj.newTable = new_table;

        // Save starting positions of cursor and element.
        drag_obj.startPos = pos;
        drag_obj.prevPos = pos;
        drag_obj.elStartLeft = parseInt(drag_obj.newTable.style.left, 10);
        drag_obj.elStartTop = parseInt(drag_obj.newTable.style.top,  10);

        if (isNaN(drag_obj.elStartLeft)) drag_obj.elStartLeft = 0;
        if (isNaN(drag_obj.elStartTop))  drag_obj.elStartTop  = 0;

        // Update element's z-index.
        drag_obj.newTable.style.zIndex = ++drag_obj.zIndex;

        // Capture mousemove and mouseup events on the page.
        if (browser.isIE) {
            document.attachEvent("onmousemove", DT.dragMove);
            document.attachEvent("onmouseup",   DT.dragEnd);
            window.event.cancelBubble = true;
            window.event.returnValue = false;
        } else {
            document.addEventListener("mousemove", DT.dragMove, true);
            document.addEventListener("mouseup",   DT.dragEnd, true);
            event.preventDefault();
        }
    },

    // Move the floating column header with the mouse
    // TODO: Reorder columns as the mouse moves for a more interactive feel.
    dragMove: function(event) {
        var x, y;
        var drag_obj = DT.dragObj;
        var col_index = drag_obj.movingColIndex;

        // Get cursor position with respect to the page.
        var pos = DT.getEventPosition(event);

        var full_dx = drag_obj.startPos.x - pos.x;
        var full_dy = drag_obj.startPos.y - pos.y;

        if (!drag_obj.addedNode && full_dx * full_dx + full_dy * full_dy > DT.dragRadius2) {
            // Real dragging has began, the following block performs once

            // Attach a moving table
            drag_obj.tableContainer.insertBefore(drag_obj.newTable, drag_obj.table);
            drag_obj.addedNode = true;

            // Add classes for the dragged column (original and the clone)
            $(drag_obj.newTable).addClass(classes.newTable);
            $(DT.getColumn(drag_obj.newTable)).addClass(classes.cell_of_movingTable);

            $(DT.getColumn(drag_obj.table, col_index)).addClass(classes.cell_of_emptySpace);
            $(DT.getColumn(drag_obj.table, col_index)[0]).addClass(classes.topCell_of_emptySpace);
            var visibleCells = $(DT.getColumn(drag_obj.table, col_index)).filter(':visible');
            $(visibleCells[visibleCells.length - 1]).addClass(classes.bottomCell_of_emptySpace);

            DT.addStylesForAdjacentColumns(col_index);
        }

        // Move drag element by the same amount the cursor has moved.
        var style = drag_obj.newTable.style;
        style.left = (drag_obj.elStartLeft + pos.x - drag_obj.startPos.x) + "px";
        style.top  = (drag_obj.elStartTop  + pos.y - drag_obj.startPos.y) + "px";

        if (DT.browser.isIE) {
            window.event.cancelBubble = true;
            window.event.returnValue = false;
        } else {
            event.preventDefault();
        }

        // If moving over an adjacent column, do swapping, swapping with the adjacent column
        var column_pos_x = DT.getPositionOnThePage(DT.getColumn(drag_obj.newTable)[0]).x;
        var column_center_x = column_pos_x + drag_obj.movingColWidth / 2;
        var hovered_col_index = DT.getColumnAtPosition(drag_obj.table, column_center_x);

        var dx = drag_obj.prevPos.x - pos.x;

        if (DT.isSuitableColumnToSwap(hovered_col_index, col_index)) {
            if (dx < 0 && hovered_col_index > col_index ||  // moving right
                dx > 0 && hovered_col_index < col_index) {  // moving left
                DT.removeStylesForAdjacentColumns(col_index);
                DT.moveColumn(drag_obj.table, col_index, hovered_col_index);
                DT.addStylesForAdjacentColumns(hovered_col_index);
                drag_obj.movingColIndex = hovered_col_index;
            }
        }
        drag_obj.prevPos = pos;
    },

    removeStylesForAdjacentColumns: function(oldIndex) {
        var table = DT.dragObj.table;
        var rows = table.rows;
        var oldColumn = DT.getColumn(table, oldIndex);
        $(DT.getColumn(table, oldIndex - 1)).removeClass(classes.leftToCell_of_emptySpace);
        $(DT.getColumn(table, oldIndex + 1)).removeClass(classes.rightToCell_of_emptySpace);
    },

    addStylesForAdjacentColumns: function(index) {
        var table = DT.dragObj.table;
        var rows = table.rows;
        var column = DT.getColumn(table, index);
        $(DT.getColumn(table, index - 1)).addClass(classes.leftToCell_of_emptySpace);
        $(DT.getColumn(table, index + 1)).addClass(classes.rightToCell_of_emptySpace);
    },

    // Stop capturing mousemove and mouseup events.
    // Determine which (if any) column we're over and shuffle the table.
    dragEnd: function(event) {
        if (DT.browser.isIE) {
            document.detachEvent("onmousemove", DT.dragMove);
            document.detachEvent("onmouseup", DT.dragEnd);
        } else {
            document.removeEventListener("mousemove", DT.dragMove, true);
            document.removeEventListener("mouseup", DT.dragEnd, true);
        }

        var drag_obj = DT.dragObj;
        var col_index = drag_obj.movingColIndex;

        // If the floating header wasn't added, the mouse didn't move far enough.
        if (drag_obj.addedNode) {
            // TODO: Move with animation
            var final_pos = DT.getPositionOnThePage(DT.getColumn(drag_obj.table, col_index)[0]);

            $(drag_obj.newTable).animate({
                left: final_pos.x,
                top: final_pos.y,
            }, 'fast', function() {
                $(drag_obj.newTable).remove();
                $(DT.getColumn(drag_obj.table, col_index)).removeClass(classes.cell_of_emptySpace);
                $(DT.getColumn(drag_obj.table, col_index)[0]).removeClass(classes.topCell_of_emptySpace);
                var visibleCells = $(DT.getColumn(drag_obj.table, col_index)).filter(':visible');
                $(visibleCells[visibleCells.length - 1]).removeClass(classes.bottomCell_of_emptySpace);

                DT.removeStylesForAdjacentColumns(col_index);
            });

            DT.storeDrag(drag_obj.table);
        }
        DT.isDragging = false;
    },

    moveColumn: function(table, index, nextIndex) {
        forEach(table.rows, function(row) {
            var startCell = $(row.cells[index]);
            var finishCell = $(row.cells[nextIndex]);

            // startCell.animate({
            // left: finishCell.offset().left,
            // }, 'slow', function() {
            if (nextIndex > index) {
                startCell.before(finishCell);
            } else {
                startCell.after(finishCell);
            }
            // });
        });
    },

    // clone an element, copying its style and class.
    fullCopy: function(elt, deep) {
        if (!elt) {
            return null;
        }
        var new_elt = elt.cloneNode(deep);
        new_elt.className = elt.className;
        $(new_elt).addClass('clone');
        forEach(elt.style, function(value, key, object) {
            if (value == null) return;
            if (typeof(value) == "string" && value.length == 0) return;

            new_elt.style[key] = elt.style[key];
        });
        return new_elt;
    },

    getColumn: function(table, index) {
        index = typeof index !== 'undefined' ? index : 0;

        if (index < 0 || index >= table.rows[0].cells.length) {
            return [];

        } else {
            var column = [];
            forEach(table.rows, function(row) {
                column.push(row.cells[index]);
            });
            return column;
        }
    },

    isSuitableColumnToSwap: function(targetIndex, movingColIndex) {
        if (targetIndex != -1) {
            if (typeof movingColIndex === 'undefined' || targetIndex != movingColIndex) {  // check if equals the moving one
                var top_cell = $(DT.getColumn(DT.dragObj.table, targetIndex)[0]);
                if (top_cell.find('.' + classes.handle).addBack('.' + classes.handle).exists()) {
                    return true;
                }
            }
        }
        return false;
    },

    // Which column does the x value fall inside of? x should include scrollLeft.
    getColumnAtPosition: function(table, x) {
        var header = table.tHead.rows[0].cells;
        for (var i = 0; i < header.length; i++) {
            //var left = header[i].offsetLeft;
            var pos = DT.getPositionOnThePage(header[i]);
            //if (left <= x && x <= left + header[i].offsetWidth) {
            var px = pos.x;
            var ow = header[i].offsetWidth;
            var xpow = pos.x + header[i].offsetWidth;
            if (pos.x <= x && x <= pos.x + header[i].offsetWidth) {
                return i;
            }
        }
        return -1;
    },

    // Store a column swap in a cookie for posterity.
    storeDrag: function(table) {
        var order = [];
        forEach(table.rows[0].cells, function(cell) {
            var pos = cell.getAttribute('position');
            if (pos)
                order.push(pos);
        });

        var order_string = order[0];
        for (var i = 1; i < order.length; i++)
            order_string += ' ' + order[i];

        DT.storeInCookies(order_string);
        DT.sendOrderToServer(order_string);
    },

    storeInCookies: function(string) {
        if (navigator.cookieEnabled) {
            var cookieName = 'order';
            createCookie(cookieName, string, DT.cookieDays);
        }
    },

    sendOrderToServer: function(string) {
        if (document.reportId) {
            $.ajax({
                type: 'GET',
                url: '/reorder-report-columns',
                dataType: 'json',
                data: {
                    reportId: document.reportId,
                    order: string,
                },
            });
        }
    },
};

var DT = dragTable;

/* ******************************************************************
 Supporting functions: bundled here to avoid depending on a library
 ****************************************************************** */


//// Dean Edwards/Matthias Miller/John Resig
//// has a hook for DT.init already been added? (see below)
//var dgListenOnLoad = false;
//
///* for Mozilla/Opera9 */
//if (document.addEventListener) {
//  dgListenOnLoad = true;
//  document.addEventListener("DOMContentLoaded", DT.init, false);
//}
//
///* for Internet Explorer */
///*@cc_on @*/
///*@if (@_win32)
//  dgListenOnLoad = true;
//  document.write("<script id=__dt_onload defer src=//0)><\/script>");
//  var script = document.getElementById("__dt_onload");
//  script.onreadystatechange = function() {
//    if (this.readyState == "complete") {
//      DT.init(); // call the onload handler
//    }
//  };
///*@end @*/
//
///* for Safari */
//if (/WebKit/i.test(navigator.userAgent)) { // sniff
//  dgListenOnLoad = true;
//  var _dgtimer = setInterval(function() {
//    if (/loaded|complete/.test(document.readyState)) {
//      DT.init(); // call the onload handler
//    }
//  }, 10);
//}
//
///* for other browsers */
///* Avoid this unless it's absolutely necessary (it breaks sorttable) */
//if (!dgListenOnLoad) {
//  window.onload = DT.init;
//}

</script>

<script type="text/javascript">
/*! HTML5 Shiv vpre3.6 | @afarkas @jdalton @jon_neal @rem | MIT/GPL2 Licensed
  Uncompressed source: https://github.com/aFarkas/html5shiv  */
(function(a,b){function h(a,b){var c=a.createElement("p"),d=a.getElementsByTagName("head")[0]||a.documentElement;return c.innerHTML="x<style>"+b+"</style>",d.insertBefore(c.lastChild,d.firstChild)}function i(){var a=l.elements;return typeof a=="string"?a.split(" "):a}function j(a){var b={},c=a.createElement,f=a.createDocumentFragment,g=f();a.createElement=function(a){if(!l.shivMethods)return c(a);var f;return b[a]?f=b[a].cloneNode():e.test(a)?f=(b[a]=c(a)).cloneNode():f=c(a),f.canHaveChildren&&!d.test(a)?g.appendChild(f):f},a.createDocumentFragment=Function("h,f","return function(){var n=f.cloneNode(),c=n.createElement;h.shivMethods&&("+i().join().replace(/\w+/g,function(a){return c(a),g.createElement(a),'c("'+a+'")'})+");return n}")(l,g)}function k(a){var b;return a.documentShived?a:(l.shivCSS&&!f&&(b=!!h(a,"article,aside,details,figcaption,figure,footer,header,hgroup,nav,section{display:block}audio{display:none}canvas,video{display:inline-block;*display:inline;*zoom:1}[hidden]{display:none}audio[controls]{display:inline-block;*display:inline;*zoom:1}mark{background:#FF0;color:#000}")),g||(b=!j(a)),b&&(a.documentShived=b),a)}var c=a.html5||{},d=/^<|^(?:button|form|map|select|textarea|object|iframe|option|optgroup)$/i,e=/^<|^(?:a|b|button|code|div|fieldset|form|h1|h2|h3|h4|h5|h6|i|iframe|img|input|label|li|link|ol|option|p|param|q|script|select|span|strong|style|table|tbody|td|textarea|tfoot|th|thead|tr|ul)$/i,f,g;(function(){var c=b.createElement("a");c.innerHTML="<xyz></xyz>",f="hidden"in c,f&&typeof injectElementWithStyles=="function"&&injectElementWithStyles("#modernizr{}",function(b){b.hidden=!0,f=(a.getComputedStyle?getComputedStyle(b,null):b.currentStyle).display=="none"}),g=c.childNodes.length==1||function(){try{b.createElement("a")}catch(a){return!0}var c=b.createDocumentFragment();return typeof c.cloneNode=="undefined"||typeof c.createDocumentFragment=="undefined"||typeof c.createElement=="undefined"}()})();var l={elements:c.elements||"abbr article aside audio bdi canvas data datalist details figcaption figure footer header hgroup mark meter nav output progress section summary time video",shivCSS:c.shivCSS!==!1,shivMethods:c.shivMethods!==!1,type:"default",shivDocument:k};a.html5=l,k(b)})(this,document)
</script>

<script type="text/javascript">
/**
* Bootstrap.js by @fat & @mdo
* plugins: bootstrap-tooltip.js
* Copyright 2012 Twitter, Inc.
* http://www.apache.org/licenses/LICENSE-2.0.txt
*/
!function(a){var b=function(a,b){this.init("tooltip",a,b)};b.prototype={constructor:b,init:function(b,c,d){var e,f;this.type=b,this.$element=a(c),this.options=this.getOptions(d),this.enabled=!0,this.options.trigger!="manual"&&(e=this.options.trigger=="hover"?"mouseenter":"focus",f=this.options.trigger=="hover"?"mouseleave":"blur",this.$element.on(e,this.options.selector,a.proxy(this.enter,this)),this.$element.on(f,this.options.selector,a.proxy(this.leave,this))),this.options.selector?this._options=a.extend({},this.options,{trigger:"manual",selector:""}):this.fixTitle()},getOptions:function(b){return b=a.extend({},a.fn[this.type].defaults,b,this.$element.data()),b.delay&&typeof b.delay=="number"&&(b.delay={show:b.delay,hide:b.delay}),b},enter:function(b){var c=a(b.currentTarget)[this.type](this._options).data(this.type);if(!c.options.delay||!c.options.delay.show)return c.show();clearTimeout(this.timeout),c.hoverState="in",this.timeout=setTimeout(function(){c.hoverState=="in"&&c.show()},c.options.delay.show)},leave:function(b){var c=a(b.currentTarget)[this.type](this._options).data(this.type);this.timeout&&clearTimeout(this.timeout);if(!c.options.delay||!c.options.delay.hide)return c.hide();c.hoverState="out",this.timeout=setTimeout(function(){c.hoverState=="out"&&c.hide()},c.options.delay.hide)},show:function(){var a,b,c,d,e,f,g;if(this.hasContent()&&this.enabled){a=this.tip(),this.setContent(),this.options.animation&&a.addClass("fade"),f=typeof this.options.placement=="function"?this.options.placement.call(this,a[0],this.$element[0]):this.options.placement,b=/in/.test(f),a.remove().css({top:0,left:0,display:"block"}).appendTo(b?this.$element:document.body),c=this.getPosition(b),d=a[0].offsetWidth,e=a[0].offsetHeight;switch(b?f.split(" ")[1]:f){case"bottom":g={top:c.top+c.height,left:c.left+c.width/2-d/2};break;case"top":g={top:c.top-e+4,left:c.left+c.width/2-d/2};break;case"left":g={top:c.top+c.height/2-e/2,left:c.left-d};break;case"right":g={top:c.top+c.height/2-e/2,left:c.left+c.width}}a.css(g).addClass(f).addClass("in")}},isHTML:function(a){return typeof a!="string"||a.charAt(0)==="<"&&a.charAt(a.length-1)===">"&&a.length>=3||/^(?:[^<]*<[\w\W]+>[^>]*$)/.exec(a)},setContent:function(){var a=this.tip(),b=this.getTitle();a.find(".tooltip-inner")[this.isHTML(b)?"html":"text"](b),a.removeClass("fade in top bottom left right")},hide:function(){function d(){var b=setTimeout(function(){c.off(a.support.transition.end).remove()},500);c.one(a.support.transition.end,function(){clearTimeout(b),c.remove()})}var b=this,c=this.tip();c.removeClass("in"),a.support.transition&&this.$tip.hasClass("fade")?d():c.remove()},fixTitle:function(){var a=this.$element;(a.attr("title")||typeof a.attr("data-original-title")!="string")&&a.attr("data-original-title",a.attr("title")||"").removeAttr("title")},hasContent:function(){return this.getTitle()},getPosition:function(b){return a.extend({},b?{top:0,left:0}:this.$element.offset(),{width:this.$element[0].offsetWidth,height:this.$element[0].offsetHeight})},getTitle:function(){var a,b=this.$element,c=this.options;return a=b.attr("data-original-title")||(typeof c.title=="function"?c.title.call(b[0]):c.title),a},tip:function(){return this.$tip=this.$tip||a(this.options.template)},validate:function(){this.$element[0].parentNode||(this.hide(),this.$element=null,this.options=null)},enable:function(){this.enabled=!0},disable:function(){this.enabled=!1},toggleEnabled:function(){this.enabled=!this.enabled},toggle:function(){this[this.tip().hasClass("in")?"hide":"show"]()}},a.fn.tooltip=function(c){return this.each(function(){var d=a(this),e=d.data("tooltip"),f=typeof c=="object"&&c;e||d.data("tooltip",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.tooltip.Constructor=b,a.fn.tooltip.defaults={animation:!0,placement:"top",selector:!1,template:'<div class="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>',trigger:"hover",title:"",delay:0}}(window.jQuery)
</script>

<script type="text/javascript">
/*!
* Bootstrap.js by @fat & @mdo
* Copyright 2012 Twitter, Inc.
* http://www.apache.org/licenses/LICENSE-2.0.txt
*/
!function(a){a(function(){"use strict",a.support.transition=function(){var a=function(){var a=document.createElement("bootstrap"),b={WebkitTransition:"webkitTransitionEnd",MozTransition:"transitionend",OTransition:"oTransitionEnd",msTransition:"MSTransitionEnd",transition:"transitionend"},c;for(c in b)if(a.style[c]!==undefined)return b[c]}();return a&&{end:a}}()})}(window.jQuery),!function(a){"use strict";var b='[data-dismiss="alert"]',c=function(c){a(c).on("click",b,this.close)};c.prototype.close=function(b){function f(){e.trigger("closed").remove()}var c=a(this),d=c.attr("data-target"),e;d||(d=c.attr("href"),d=d&&d.replace(/.*(?=#[^\s]*$)/,"")),e=a(d),b&&b.preventDefault(),e.length||(e=c.hasClass("alert")?c:c.parent()),e.trigger(b=a.Event("close"));if(b.isDefaultPrevented())return;e.removeClass("in"),a.support.transition&&e.hasClass("fade")?e.on(a.support.transition.end,f):f()},a.fn.alert=function(b){return this.each(function(){var d=a(this),e=d.data("alert");e||d.data("alert",e=new c(this)),typeof b=="string"&&e[b].call(d)})},a.fn.alert.Constructor=c,a(function(){a("body").on("click.alert.data-api",b,c.prototype.close)})}(window.jQuery),!function(a){"use strict";var b=function(b,c){this.$element=a(b),this.options=a.extend({},a.fn.button.defaults,c)};b.prototype.setState=function(a){var b="disabled",c=this.$element,d=c.data(),e=c.is("input")?"val":"html";a+="Text",d.resetText||c.data("resetText",c[e]()),c[e](d[a]||this.options[a]),setTimeout(function(){a=="loadingText"?c.addClass(b).attr(b,b):c.removeClass(b).removeAttr(b)},0)},b.prototype.toggle=function(){var a=this.$element.parent('[data-toggle="buttons-radio"]');a&&a.find(".active").removeClass("active"),this.$element.toggleClass("active")},a.fn.button=function(c){return this.each(function(){var d=a(this),e=d.data("button"),f=typeof c=="object"&&c;e||d.data("button",e=new b(this,f)),c=="toggle"?e.toggle():c&&e.setState(c)})},a.fn.button.defaults={loadingText:"loading..."},a.fn.button.Constructor=b,a(function(){a("body").on("click.button.data-api","[data-toggle^=button]",function(b){var c=a(b.target);c.hasClass("btn")||(c=c.closest(".btn")),c.button("toggle")})})}(window.jQuery),!function(a){"use strict";var b=function(b,c){this.$element=a(b),this.options=c,this.options.slide&&this.slide(this.options.slide),this.options.pause=="hover"&&this.$element.on("mouseenter",a.proxy(this.pause,this)).on("mouseleave",a.proxy(this.cycle,this))};b.prototype={cycle:function(b){return b||(this.paused=!1),this.options.interval&&!this.paused&&(this.interval=setInterval(a.proxy(this.next,this),this.options.interval)),this},to:function(b){var c=this.$element.find(".active"),d=c.parent().children(),e=d.index(c),f=this;if(b>d.length-1||b<0)return;return this.sliding?this.$element.one("slid",function(){f.to(b)}):e==b?this.pause().cycle():this.slide(b>e?"next":"prev",a(d[b]))},pause:function(a){return a||(this.paused=!0),clearInterval(this.interval),this.interval=null,this},next:function(){if(this.sliding)return;return this.slide("next")},prev:function(){if(this.sliding)return;return this.slide("prev")},slide:function(b,c){var d=this.$element.find(".active"),e=c||d[b](),f=this.interval,g=b=="next"?"left":"right",h=b=="next"?"first":"last",i=this,j=a.Event("slide");this.sliding=!0,f&&this.pause(),e=e.length?e:this.$element.find(".item")[h]();if(e.hasClass("active"))return;if(a.support.transition&&this.$element.hasClass("slide")){this.$element.trigger(j);if(j.isDefaultPrevented())return;e.addClass(b),e[0].offsetWidth,d.addClass(g),e.addClass(g),this.$element.one(a.support.transition.end,function(){e.removeClass([b,g].join(" ")).addClass("active"),d.removeClass(["active",g].join(" ")),i.sliding=!1,setTimeout(function(){i.$element.trigger("slid")},0)})}else{this.$element.trigger(j);if(j.isDefaultPrevented())return;d.removeClass("active"),e.addClass("active"),this.sliding=!1,this.$element.trigger("slid")}return f&&this.cycle(),this}},a.fn.carousel=function(c){return this.each(function(){var d=a(this),e=d.data("carousel"),f=a.extend({},a.fn.carousel.defaults,typeof c=="object"&&c);e||d.data("carousel",e=new b(this,f)),typeof c=="number"?e.to(c):typeof c=="string"||(c=f.slide)?e[c]():f.interval&&e.cycle()})},a.fn.carousel.defaults={interval:5e3,pause:"hover"},a.fn.carousel.Constructor=b,a(function(){a("body").on("click.carousel.data-api","[data-slide]",function(b){var c=a(this),d,e=a(c.attr("data-target")||(d=c.attr("href"))&&d.replace(/.*(?=#[^\s]+$)/,"")),f=!e.data("modal")&&a.extend({},e.data(),c.data());e.carousel(f),b.preventDefault()})})}(window.jQuery),!function(a){"use strict";var b=function(b,c){this.$element=a(b),this.options=a.extend({},a.fn.collapse.defaults,c),this.options.parent&&(this.$parent=a(this.options.parent)),this.options.toggle&&this.toggle()};b.prototype={constructor:b,dimension:function(){var a=this.$element.hasClass("width");return a?"width":"height"},show:function(){var b,c,d,e;if(this.transitioning)return;b=this.dimension(),c=a.camelCase(["scroll",b].join("-")),d=this.$parent&&this.$parent.find("> .accordion-group > .in");if(d&&d.length){e=d.data("collapse");if(e&&e.transitioning)return;d.collapse("hide"),e||d.data("collapse",null)}this.$element[b](0),this.transition("addClass",a.Event("show"),"shown"),this.$element[b](this.$element[0][c])},hide:function(){var b;if(this.transitioning)return;b=this.dimension(),this.reset(this.$element[b]()),this.transition("removeClass",a.Event("hide"),"hidden"),this.$element[b](0)},reset:function(a){var b=this.dimension();return this.$element.removeClass("collapse")[b](a||"auto")[0].offsetWidth,this.$element[a!==null?"addClass":"removeClass"]("collapse"),this},transition:function(b,c,d){var e=this,f=function(){c.type=="show"&&e.reset(),e.transitioning=0,e.$element.trigger(d)};this.$element.trigger(c);if(c.isDefaultPrevented())return;this.transitioning=1,this.$element[b]("in"),a.support.transition&&this.$element.hasClass("collapse")?this.$element.one(a.support.transition.end,f):f()},toggle:function(){this[this.$element.hasClass("in")?"hide":"show"]()}},a.fn.collapse=function(c){return this.each(function(){var d=a(this),e=d.data("collapse"),f=typeof c=="object"&&c;e||d.data("collapse",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.collapse.defaults={toggle:!0},a.fn.collapse.Constructor=b,a(function(){a("body").on("click.collapse.data-api","[data-toggle=collapse]",function(b){var c=a(this),d,e=c.attr("data-target")||b.preventDefault()||(d=c.attr("href"))&&d.replace(/.*(?=#[^\s]+$)/,""),f=a(e).data("collapse")?"toggle":c.data();a(e).collapse(f)})})}(window.jQuery),!function(a){function d(){a(b).parent().removeClass("open")}"use strict";var b='[data-toggle="dropdown"]',c=function(b){var c=a(b).on("click.dropdown.data-api",this.toggle);a("html").on("click.dropdown.data-api",function(){c.parent().removeClass("open")})};c.prototype={constructor:c,toggle:function(b){var c=a(this),e,f,g;if(c.is(".disabled, :disabled"))return;return f=c.attr("data-target"),f||(f=c.attr("href"),f=f&&f.replace(/.*(?=#[^\s]*$)/,"")),e=a(f),e.length||(e=c.parent()),g=e.hasClass("open"),d(),g||e.toggleClass("open"),!1}},a.fn.dropdown=function(b){return this.each(function(){var d=a(this),e=d.data("dropdown");e||d.data("dropdown",e=new c(this)),typeof b=="string"&&e[b].call(d)})},a.fn.dropdown.Constructor=c,a(function(){a("html").on("click.dropdown.data-api",d),a("body").on("click.dropdown",".dropdown form",function(a){a.stopPropagation()}).on("click.dropdown.data-api",b,c.prototype.toggle)})}(window.jQuery),!function(a){function c(){var b=this,c=setTimeout(function(){b.$element.off(a.support.transition.end),d.call(b)},500);this.$element.one(a.support.transition.end,function(){clearTimeout(c),d.call(b)})}function d(a){this.$element.hide().trigger("hidden"),e.call(this)}function e(b){var c=this,d=this.$element.hasClass("fade")?"fade":"";if(this.isShown&&this.options.backdrop){var e=a.support.transition&&d;this.$backdrop=a('<div class="modal-backdrop '+d+'" />').appendTo(document.body),this.options.backdrop!="static"&&this.$backdrop.click(a.proxy(this.hide,this)),e&&this.$backdrop[0].offsetWidth,this.$backdrop.addClass("in"),e?this.$backdrop.one(a.support.transition.end,b):b()}else!this.isShown&&this.$backdrop?(this.$backdrop.removeClass("in"),a.support.transition&&this.$element.hasClass("fade")?this.$backdrop.one(a.support.transition.end,a.proxy(f,this)):f.call(this)):b&&b()}function f(){this.$backdrop.remove(),this.$backdrop=null}function g(){var b=this;this.isShown&&this.options.keyboard?a(document).on("keyup.dismiss.modal",function(a){a.which==27&&b.hide()}):this.isShown||a(document).off("keyup.dismiss.modal")}"use strict";var b=function(b,c){this.options=c,this.$element=a(b).delegate('[data-dismiss="modal"]',"click.dismiss.modal",a.proxy(this.hide,this))};b.prototype={constructor:b,toggle:function(){return this[this.isShown?"hide":"show"]()},show:function(){var b=this,c=a.Event("show");this.$element.trigger(c);if(this.isShown||c.isDefaultPrevented())return;a("body").addClass("modal-open"),this.isShown=!0,g.call(this),e.call(this,function(){var c=a.support.transition&&b.$element.hasClass("fade");b.$element.parent().length||b.$element.appendTo(document.body),b.$element.show(),c&&b.$element[0].offsetWidth,b.$element.addClass("in"),c?b.$element.one(a.support.transition.end,function(){b.$element.trigger("shown")}):b.$element.trigger("shown")})},hide:function(b){b&&b.preventDefault();var e=this;b=a.Event("hide"),this.$element.trigger(b);if(!this.isShown||b.isDefaultPrevented())return;this.isShown=!1,a("body").removeClass("modal-open"),g.call(this),this.$element.removeClass("in"),a.support.transition&&this.$element.hasClass("fade")?c.call(this):d.call(this)}},a.fn.modal=function(c){return this.each(function(){var d=a(this),e=d.data("modal"),f=a.extend({},a.fn.modal.defaults,d.data(),typeof c=="object"&&c);e||d.data("modal",e=new b(this,f)),typeof c=="string"?e[c]():f.show&&e.show()})},a.fn.modal.defaults={backdrop:!0,keyboard:!0,show:!0},a.fn.modal.Constructor=b,a(function(){a("body").on("click.modal.data-api",'[data-toggle="modal"]',function(b){var c=a(this),d,e=a(c.attr("data-target")||(d=c.attr("href"))&&d.replace(/.*(?=#[^\s]+$)/,"")),f=e.data("modal")?"toggle":a.extend({},e.data(),c.data());b.preventDefault(),e.modal(f)})})}(window.jQuery),!function(a){"use strict";var b=function(a,b){this.init("tooltip",a,b)};b.prototype={constructor:b,init:function(b,c,d){var e,f;this.type=b,this.$element=a(c),this.options=this.getOptions(d),this.enabled=!0,this.options.trigger!="manual"&&(e=this.options.trigger=="hover"?"mouseenter":"focus",f=this.options.trigger=="hover"?"mouseleave":"blur",this.$element.on(e,this.options.selector,a.proxy(this.enter,this)),this.$element.on(f,this.options.selector,a.proxy(this.leave,this))),this.options.selector?this._options=a.extend({},this.options,{trigger:"manual",selector:""}):this.fixTitle()},getOptions:function(b){return b=a.extend({},a.fn[this.type].defaults,b,this.$element.data()),b.delay&&typeof b.delay=="number"&&(b.delay={show:b.delay,hide:b.delay}),b},enter:function(b){var c=a(b.currentTarget)[this.type](this._options).data(this.type);if(!c.options.delay||!c.options.delay.show)return c.show();clearTimeout(this.timeout),c.hoverState="in",this.timeout=setTimeout(function(){c.hoverState=="in"&&c.show()},c.options.delay.show)},leave:function(b){var c=a(b.currentTarget)[this.type](this._options).data(this.type);this.timeout&&clearTimeout(this.timeout);if(!c.options.delay||!c.options.delay.hide)return c.hide();c.hoverState="out",this.timeout=setTimeout(function(){c.hoverState=="out"&&c.hide()},c.options.delay.hide)},show:function(){var a,b,c,d,e,f,g;if(this.hasContent()&&this.enabled){a=this.tip(),this.setContent(),this.options.animation&&a.addClass("fade"),f=typeof this.options.placement=="function"?this.options.placement.call(this,a[0],this.$element[0]):this.options.placement,b=/in/.test(f),a.remove().css({top:0,left:0,display:"block"}).appendTo(b?this.$element:document.body),c=this.getPosition(b),d=a[0].offsetWidth,e=a[0].offsetHeight;switch(b?f.split(" ")[1]:f){case"bottom":g={top:c.top+c.height,left:c.left+c.width/2-d/2};break;case"top":g={top:c.top-e,left:c.left+c.width/2-d/2};break;case"left":g={top:c.top+c.height/2-e/2,left:c.left-d};break;case"right":g={top:c.top+c.height/2-e/2,left:c.left+c.width}}a.css(g).addClass(f).addClass("in")}},isHTML:function(a){return typeof a!="string"||a.charAt(0)==="<"&&a.charAt(a.length-1)===">"&&a.length>=3||/^(?:[^<]*<[\w\W]+>[^>]*$)/.exec(a)},setContent:function(){var a=this.tip(),b=this.getTitle();a.find(".tooltip-inner")[this.isHTML(b)?"html":"text"](b),a.removeClass("fade in top bottom left right")},hide:function(){function d(){var b=setTimeout(function(){c.off(a.support.transition.end).remove()},500);c.one(a.support.transition.end,function(){clearTimeout(b),c.remove()})}var b=this,c=this.tip();c.removeClass("in"),a.support.transition&&this.$tip.hasClass("fade")?d():c.remove()},fixTitle:function(){var a=this.$element;(a.attr("title")||typeof a.attr("data-original-title")!="string")&&a.attr("data-original-title",a.attr("title")||"").removeAttr("title")},hasContent:function(){return this.getTitle()},getPosition:function(b){return a.extend({},b?{top:0,left:0}:this.$element.offset(),{width:this.$element[0].offsetWidth,height:this.$element[0].offsetHeight})},getTitle:function(){var a,b=this.$element,c=this.options;return a=b.attr("data-original-title")||(typeof c.title=="function"?c.title.call(b[0]):c.title),a},tip:function(){return this.$tip=this.$tip||a(this.options.template)},validate:function(){this.$element[0].parentNode||(this.hide(),this.$element=null,this.options=null)},enable:function(){this.enabled=!0},disable:function(){this.enabled=!1},toggleEnabled:function(){this.enabled=!this.enabled},toggle:function(){this[this.tip().hasClass("in")?"hide":"show"]()}},a.fn.tooltip=function(c){return this.each(function(){var d=a(this),e=d.data("tooltip"),f=typeof c=="object"&&c;e||d.data("tooltip",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.tooltip.Constructor=b,a.fn.tooltip.defaults={animation:!0,placement:"top",selector:!1,template:'<div class="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>',trigger:"hover",title:"",delay:0}}(window.jQuery),!function(a){"use strict";var b=function(a,b){this.init("popover",a,b)};b.prototype=a.extend({},a.fn.tooltip.Constructor.prototype,{constructor:b,setContent:function(){var a=this.tip(),b=this.getTitle(),c=this.getContent();a.find(".popover-title")[this.isHTML(b)?"html":"text"](b),a.find(".popover-content > *")[this.isHTML(c)?"html":"text"](c),a.removeClass("fade top bottom left right in")},hasContent:function(){return this.getTitle()||this.getContent()},getContent:function(){var a,b=this.$element,c=this.options;return a=b.attr("data-content")||(typeof c.content=="function"?c.content.call(b[0]):c.content),a},tip:function(){return this.$tip||(this.$tip=a(this.options.template)),this.$tip}}),a.fn.popover=function(c){return this.each(function(){var d=a(this),e=d.data("popover"),f=typeof c=="object"&&c;e||d.data("popover",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.popover.Constructor=b,a.fn.popover.defaults=a.extend({},a.fn.tooltip.defaults,{placement:"right",content:"",template:'<div class="popover"><div class="arrow"></div><div class="popover-inner"><h3 class="popover-title"></h3><div class="popover-content"><p></p></div></div></div>'})}(window.jQuery),!function(a){function b(b,c){var d=a.proxy(this.process,this),e=a(b).is("body")?a(window):a(b),f;this.options=a.extend({},a.fn.scrollspy.defaults,c),this.$scrollElement=e.on("scroll.scroll.data-api",d),this.selector=(this.options.target||(f=a(b).attr("href"))&&f.replace(/.*(?=#[^\s]+$)/,"")||"")+" .nav li > a",this.$body=a("body"),this.refresh(),this.process()}"use strict",b.prototype={constructor:b,refresh:function(){var b=this,c;this.offsets=a([]),this.targets=a([]),c=this.$body.find(this.selector).map(function(){var b=a(this),c=b.data("target")||b.attr("href"),d=/^#\w/.test(c)&&a(c);return d&&c.length&&[[d.position().top,c]]||null}).sort(function(a,b){return a[0]-b[0]}).each(function(){b.offsets.push(this[0]),b.targets.push(this[1])})},process:function(){var a=this.$scrollElement.scrollTop()+this.options.offset,b=this.$scrollElement[0].scrollHeight||this.$body[0].scrollHeight,c=b-this.$scrollElement.height(),d=this.offsets,e=this.targets,f=this.activeTarget,g;if(a>=c)return f!=(g=e.last()[0])&&this.activate(g);for(g=d.length;g--;)f!=e[g]&&a>=d[g]&&(!d[g+1]||a<=d[g+1])&&this.activate(e[g])},activate:function(b){var c,d;this.activeTarget=b,a(this.selector).parent(".active").removeClass("active"),d=this.selector+'[data-target="'+b+'"],'+this.selector+'[href="'+b+'"]',c=a(d).parent("li").addClass("active"),c.parent(".dropdown-menu")&&(c=c.closest("li.dropdown").addClass("active")),c.trigger("activate")}},a.fn.scrollspy=function(c){return this.each(function(){var d=a(this),e=d.data("scrollspy"),f=typeof c=="object"&&c;e||d.data("scrollspy",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.scrollspy.Constructor=b,a.fn.scrollspy.defaults={offset:10},a(function(){a('[data-spy="scroll"]').each(function(){var b=a(this);b.scrollspy(b.data())})})}(window.jQuery),!function(a){"use strict";var b=function(b){this.element=a(b)};b.prototype={constructor:b,show:function(){var b=this.element,c=b.closest("ul:not(.dropdown-menu)"),d=b.attr("data-target"),e,f,g;d||(d=b.attr("href"),d=d&&d.replace(/.*(?=#[^\s]*$)/,""));if(b.parent("li").hasClass("active"))return;e=c.find(".active a").last()[0],g=a.Event("show",{relatedTarget:e}),b.trigger(g);if(g.isDefaultPrevented())return;f=a(d),this.activate(b.parent("li"),c),this.activate(f,f.parent(),function(){b.trigger({type:"shown",relatedTarget:e})})},activate:function(b,c,d){function g(){e.removeClass("active").find("> .dropdown-menu > .active").removeClass("active"),b.addClass("active"),f?(b[0].offsetWidth,b.addClass("in")):b.removeClass("fade"),b.parent(".dropdown-menu")&&b.closest("li.dropdown").addClass("active"),d&&d()}var e=c.find("> .active"),f=d&&a.support.transition&&e.hasClass("fade");f?e.one(a.support.transition.end,g):g(),e.removeClass("in")}},a.fn.tab=function(c){return this.each(function(){var d=a(this),e=d.data("tab");e||d.data("tab",e=new b(this)),typeof c=="string"&&e[c]()})},a.fn.tab.Constructor=b,a(function(){a("body").on("click.tab.data-api",'[data-toggle="tab"], [data-toggle="pill"]',function(b){b.preventDefault(),a(this).tab("show")})})}(window.jQuery),!function(a){"use strict";var b=function(b,c){this.$element=a(b),this.options=a.extend({},a.fn.typeahead.defaults,c),this.matcher=this.options.matcher||this.matcher,this.sorter=this.options.sorter||this.sorter,this.highlighter=this.options.highlighter||this.highlighter,this.updater=this.options.updater||this.updater,this.$menu=a(this.options.menu).appendTo("body"),this.source=this.options.source,this.shown=!1,this.listen()};b.prototype={constructor:b,select:function(){var a=this.$menu.find(".active").attr("data-value");return this.$element.val(this.updater(a)).change(),this.hide()},updater:function(a){return a},show:function(){var b=a.extend({},this.$element.offset(),{height:this.$element[0].offsetHeight});return this.$menu.css({top:b.top+b.height,left:b.left}),this.$menu.show(),this.shown=!0,this},hide:function(){return this.$menu.hide(),this.shown=!1,this},lookup:function(b){var c=this,d,e;return this.query=this.$element.val(),this.query?(d=a.grep(this.source,function(a){return c.matcher(a)}),d=this.sorter(d),d.length?this.render(d.slice(0,this.options.items)).show():this.shown?this.hide():this):this.shown?this.hide():this},matcher:function(a){return~a.toLowerCase().indexOf(this.query.toLowerCase())},sorter:function(a){var b=[],c=[],d=[],e;while(e=a.shift())e.toLowerCase().indexOf(this.query.toLowerCase())?~e.indexOf(this.query)?c.push(e):d.push(e):b.push(e);return b.concat(c,d)},highlighter:function(a){var b=this.query.replace(/[\-\[\]{}()*+?.,\\\^$|#\s]/g,"\\$&");return a.replace(new RegExp("("+b+")","ig"),function(a,b){return"<strong>"+b+"</strong>"})},render:function(b){var c=this;return b=a(b).map(function(b,d){return b=a(c.options.item).attr("data-value",d),b.find("a").html(c.highlighter(d)),b[0]}),b.first().addClass("active"),this.$menu.html(b),this},next:function(b){var c=this.$menu.find(".active").removeClass("active"),d=c.next();d.length||(d=a(this.$menu.find("li")[0])),d.addClass("active")},prev:function(a){var b=this.$menu.find(".active").removeClass("active"),c=b.prev();c.length||(c=this.$menu.find("li").last()),c.addClass("active")},listen:function(){this.$element.on("blur",a.proxy(this.blur,this)).on("keypress",a.proxy(this.keypress,this)).on("keyup",a.proxy(this.keyup,this)),(a.browser.webkit||a.browser.msie)&&this.$element.on("keydown",a.proxy(this.keypress,this)),this.$menu.on("click",a.proxy(this.click,this)).on("mouseenter","li",a.proxy(this.mouseenter,this))},keyup:function(a){switch(a.keyCode){case 40:case 38:break;case 9:case 13:if(!this.shown)return;this.select();break;case 27:if(!this.shown)return;this.hide();break;default:this.lookup()}a.stopPropagation(),a.preventDefault()},keypress:function(a){if(!this.shown)return;switch(a.keyCode){case 9:case 13:case 27:a.preventDefault();break;case 38:if(a.type!="keydown")break;a.preventDefault(),this.prev();break;case 40:if(a.type!="keydown")break;a.preventDefault(),this.next()}a.stopPropagation()},blur:function(a){var b=this;setTimeout(function(){b.hide()},150)},click:function(a){a.stopPropagation(),a.preventDefault(),this.select()},mouseenter:function(b){this.$menu.find(".active").removeClass("active"),a(b.currentTarget).addClass("active")}},a.fn.typeahead=function(c){return this.each(function(){var d=a(this),e=d.data("typeahead"),f=typeof c=="object"&&c;e||d.data("typeahead",e=new b(this,f)),typeof c=="string"&&e[c]()})},a.fn.typeahead.defaults={source:[],items:8,menu:'<ul class="typeahead dropdown-menu"></ul>',item:'<li><a href="#"></a></li>'},a.fn.typeahead.Constructor=b,a(function(){a("body").on("focus.typeahead.data-api",'[data-provide="typeahead"]',function(b){var c=a(this);if(c.data("typeahead"))return;b.preventDefault(),c.typeahead(c.data())})})}(window.jQuery);
</script>

<script type="text/javascript">
/* ===========================================================
 * bootstrap-tooltip.js v2.0.4
 * http://twitter.github.com/bootstrap/javascript.html#tooltips
 * Inspired by the original jQuery.tipsy by Jason Frame
 * ===========================================================
 * Copyright 2012 Twitter, Inc.
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 * ========================================================== */


!function ($) {

  "use strict"; // jshint ;_;


 /* TOOLTIP PUBLIC CLASS DEFINITION
  * =============================== */

  var Tooltip = function (element, options) {
    this.init('tooltip', element, options)
  };

  Tooltip.prototype = {

    constructor: Tooltip

  , init: function (type, element, options) {
      var eventIn
        , eventOut;

      this.type = type;
      this.$element = $(element);
      this.options = this.getOptions(options);
      this.enabled = true;

      if (this.options.trigger != 'manual') {
        eventIn  = this.options.trigger == 'hover' ? 'mouseenter' : 'focus';
        eventOut = this.options.trigger == 'hover' ? 'mouseleave' : 'blur';
        this.$element.on(eventIn, this.options.selector, $.proxy(this.enter, this));
        this.$element.on(eventOut, this.options.selector, $.proxy(this.leave, this));
      }

      this.options.selector ?
        (this._options = $.extend({}, this.options, { trigger: 'manual', selector: '' })) :
        this.fixTitle();
    }

  , getOptions: function (options) {
      options = $.extend({}, $.fn[this.type].defaults, options, this.$element.data())

      if (options.delay && typeof options.delay == 'number') {
        options.delay = {
          show: options.delay
        , hide: options.delay
        };
      }

      return options;
    }

  , enter: function (e) {
      if (dragTable && dragTable.isDragging)
          return;

      var self = $(e.currentTarget)[this.type](this._options).data(this.type);

      if (!self.options.delay || !self.options.delay.show) self.show();

      clearTimeout(this.timeout);
      self.hoverState = 'in';
      this.timeout = setTimeout(
          function() { if (self.hoverState == 'in') self.show(); },
          self.options.delay.show
      );
    }

  , leave: function (e) {
      var self = $(e.currentTarget)[this.type](this._options).data(this.type);

      if (this.timeout) clearTimeout(this.timeout);
      if (!self.options.delay || !self.options.delay.hide) self.hide();

      self.hoverState = 'out';
      this.timeout = setTimeout(
          function() { if (self.hoverState == 'out') self.hide() },
          self.options.delay.hide
      );
    }

  , show: function () {
      var $tip
        , inside
        , pos
        , actualWidth
        , actualHeight
        , placement
        , tp;

      if (this.hasContent() && this.enabled) {
        $tip = this.tip();
        this.setContent();

        if (this.options.animation) {
          $tip.addClass('fade');
        }

        placement = typeof this.options.placement == 'function' ?
          this.options.placement.call(this, $tip[0], this.$element[0]) :
          this.options.placement;

        inside = /in/.test(placement);

        $tip
          .remove()
          .css({ top: 0, left: 0, display: 'block' })
          .appendTo(inside ? this.$element : document.body);

        pos = this.getPosition(inside);

        actualWidth = $tip[0].offsetWidth;
        actualHeight = $tip[0].offsetHeight;

        switch (inside ? placement.split(' ')[1] : placement) {
          case 'bottom':
            tp = {top: pos.top + pos.height, left: pos.left + pos.width / 2 - actualWidth / 2};
            break;
          case 'top': // modificated by Vlad Saveliev: +4px to top position
            tp = {top: pos.top - actualHeight + 4, left: pos.left + pos.width / 2 - actualWidth / 2};
            break;
          case 'left':
            tp = {top: pos.top + pos.height / 2 - actualHeight / 2, left: pos.left - actualWidth};
            break;
          case 'right':
            tp = {top: pos.top + pos.height / 2 - actualHeight / 2, left: pos.left + pos.width};
            break;
        }

        $tip
          .css(tp)
          .addClass(placement)
          .addClass('in');
      }
    }

  , isHTML: function(text) {
      // html string detection logic adapted from jQuery
      return typeof text != 'string'
        || ( text.charAt(0) === "<"
          && text.charAt( text.length - 1 ) === ">"
          && text.length >= 3
        ) || /^(?:[^<]*<[\w\W]+>[^>]*$)/.exec(text);
    }

  , setContent: function () {
      var $tip = this.tip()
        , title = this.getTitle();

      $tip.find('.tooltip-inner')[this.isHTML(title) ? 'html' : 'text'](title);
      $tip.removeClass('fade in top bottom left right');
    }

  , hide: function () {
      var that = this
        , $tip = this.tip();

      $tip.removeClass('in');

      function removeWithAnimation() {
        var timeout = setTimeout(function () {
          $tip.off($.support.transition.end).remove()
        }, 500);

        $tip.one($.support.transition.end, function () {
          clearTimeout(timeout);
          $tip.remove();
        })
      }

      $.support.transition && this.$tip.hasClass('fade') ?
        removeWithAnimation() :
        $tip.remove()
    }

  , fixTitle: function () {
      var $e = this.$element;
      if ($e.attr('title') || typeof($e.attr('data-original-title')) != 'string') {
        $e.attr('data-original-title', $e.attr('title') || '').removeAttr('title');
      }
    }

  , hasContent: function () {
      return this.getTitle();
    }

  , getPosition: function (inside) {
      return $.extend({}, (inside ? {top: 0, left: 0} : this.$element.offset()), {
        width: this.$element[0].offsetWidth
      , height: this.$element[0].offsetHeight
      })
    }

  , getTitle: function () {
      var title
        , $e = this.$element
        , o = this.options;

      title = $e.attr('data-original-title')
        || (typeof o.title == 'function' ? o.title.call($e[0]) :  o.title);

      return title;
    }

  , tip: function () {
      return this.$tip = this.$tip || $(this.options.template);
    }

  , validate: function () {
      if (!this.$element[0].parentNode) {
        this.hide();
        this.$element = null;
        this.options = null;
      }
    }

  , enable: function () {
      this.enabled = true;
    }

  , disable: function () {
      this.enabled = false;
    }

  , toggleEnabled: function () {
      this.enabled = !this.enabled;
    }

  , toggle: function () {
      this[this.tip().hasClass('in') ? 'hide' : 'show']();
    }

  };


 /* TOOLTIP PLUGIN DEFINITION
  * ========================= */

  $.fn.tooltip = function ( option ) {
    return this.each(function () {
      var $this = $(this)
        , data = $this.data('tooltip')
        , options = typeof option == 'object' && option;
      if (!data) $this.data('tooltip', (data = new Tooltip(this, options)));
      if (typeof option == 'string') data[option]();
    })
  };

  $.fn.tooltip.Constructor = Tooltip;

  $.fn.tooltip.defaults = {
    animation: true
  , placement: 'top'
  , selector: false
  , template: '<div class="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>'
  , trigger: 'hover'
  , title: ''
  , delay: 0
  };

}(window.jQuery);

</script>

<script type="text/javascript">
function showPlotWithInfo(info, index) {
    var newSeries = [];
    var newColors = [];
    var oldSeries = info.series;
    var usingSeries;
    var sortOrder;
    if ($("input[name=sortRefs]")[0])
        sortOrder = getSortOrder();
    if (index != undefined) {
        oldSeries = info.series[index];
    }
    if (sortOrder == 'alphabet') {
        usingSeries = [];
        sortedRefs = info.references.slice(0).sort();
        for(var i = 0; i < oldSeries.length; i++) {
            usingSeries.push($.extend(true, {}, oldSeries[i]));
            for(var j = 0; j < info.references.length; j++) {
                usingSeries[i].data[j][1] = oldSeries[i].data[info.references.indexOf(sortedRefs[j])][1];
            }
        }
    }
    else usingSeries = oldSeries;
    $('#legend-placeholder').find('input[type="checkbox"]:checked').each(function() {
        var number = $(this).attr('name');
        if (number && usingSeries && usingSeries.length > 0) {
            var i = 0;
            do {
                var series = usingSeries[i];
                i++;
            } while (i <= usingSeries.length && (series == null || series.number != number));
            //
            if (i <= usingSeries.length) {
                newSeries.push(series);
                newColors.push(series.color);
            } else {
                console.log('no series with number ' + number);
            }
        }
    });
    if (sortOrder) sortReferences(sortOrder, info);

    if (newSeries.length === 0) {
        newSeries.push({
            data: []
        });
        newColors.push('#FFF');
    }

    info.showWithData(newSeries, newColors);
}

function recoverOrderFromCookies() {
    if (!navigator.cookieEnabled)
        return null;

    var order_string = readCookie("order");
    if (!order_string)
        return null;

    var order = [];
    var fail = false;
    forEach(order_string.split(' '), function(val) {
        val = parseInt(val);
        if (isNaN(val))
            fail = true;
        else
            order.push(val);
    });

    if (fail)
        return null;

    return order;
}


function readJson(what) {
    var result;
    try {
        result = JSON.parse($('#' + what + '-json').html());
    } catch (e) {
        result = null;
    }
    return result;
}


function getToggleFunction(assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX) {
    return function() {
        this.parentNode.getElementsByClassName('selected-switch')[0].className = 'plot-switch dotted-link';
        this.className = 'plot-switch selected-switch';
        togglePlots(assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX)
    };
}

function togglePlots(assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX) {
    var plotPlaceholder = document.getElementById('plot-placeholder');
    var legendPlaceholder = document.getElementById('legend-placeholder');
    var scalePlaceholder = document.getElementById('scale-placeholder');

    var glossary = JSON.parse($('#glossary-json').html());

    if (name === 'cumulative') {
        $(plotPlaceholder).addClass('cumulative-plot-placeholder');
    } else {
        $(plotPlaceholder).removeClass('cumulative-plot-placeholder');
    }

    $('#contigs_are_ordered').hide();
    $('#gc_info').hide();
    $('#gc_contigs_info').hide();
    $('#frc_info').hide();

    var selectedAssemblies = getSelectedAssemblies(assembliesNames);
    var sortBtnClass;
    if ($("input[name=sortRefs]")[0]) {
        sortBtnClass = getSortRefsRule();
    }
    $('#legend-placeholder').empty();
    assembliesNames.forEach(function(filename, i) {
        addLabelToLegend(i, filename, selectedAssemblies, colors);
    });
    if (refPlotValue) {
        $('#legend-placeholder').append(
            '<div id="reference-label">' +
                '<label for="reference" style="color: #000000;">' +
                '<input type="checkbox" name="' + assembliesNames.length +
                '" checked="checked" id="reference">&nbsp;' + 'reference' +
                '</label>' +
                '</div>'
        );
    }
    if (sortBtnClass) {
        addSortRefsBtn(sortBtnClass);
    }

    drawPlot(name, title, colors, assembliesNames, data, refPlotValue, tickX,
        plotPlaceholder, legendPlaceholder, glossary, order, scalePlaceholder);
}

function makePlot(firstPlot, assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX) {
    var switchSpan = document.createElement('span');
    switchSpan.id = name + '-switch';
    switchSpan.innerHTML = title;
    var plotsSwitchesDiv = document.getElementById('plots-switches');
    plotsSwitchesDiv.appendChild(switchSpan);

    if (firstPlot) {
        switchSpan.className = 'plot-switch selected-switch';
        togglePlots(assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX);
    } else {
        switchSpan.className = 'plot-switch dotted-link';
    }

    $(switchSpan).click(getToggleFunction(assembliesNames, order, name, title, drawPlot, data, refPlotValue, tickX));
}
</script>

<script type="text/javascript">
String.prototype.trunc =
    function(n){
        return this.substr(0, n-1) + (this.length > n ? '&hellip;' : '');
    };


function getColor (hue, lightness) {
    lightness = lightness ? lightness : 92;
    var rgb = hslToRgb(hue / 360, 0.8, lightness / 100);
    return '#' + rgb[0].toString(16) + rgb[1].toString(16) + rgb[2].toString(16);
}

function getMedian (x) {
    if (x.length == 0) return null;
    if (x.length % 2 == 1) return x[(x.length - 1) / 2];
    else return (x[(x.length / 2) - 1] + x[(x.length / 2)]) / 2;
}

function toggleSecondary(event, caller) {
    event = event || window.event;
    if(event.target.nodeName == "IMG") return;
    if (!caller.hasClass('primary') || caller.hasClass('not_extend')) {
        return;
    }
    var nextRow = caller.next('.content-row');
    $(caller).find('.metric-name').toggleClass('collapsed').toggleClass('expanded');

    while (!nextRow.hasClass('primary') && (nextRow.length > 0)) {
        nextRow.toggleClass('secondary_hidden');
        nextRow.find('.left_column_td').css('background-color', '#E8E8E8');
        nextRow = nextRow.next('.content-row');
    }
}

function setUpHeatMap(table) {
    
    (function () {
        $(function () {
            $('tr.group_empty').removeClass('row_hidden');
        });
    })();

    $('#main_report').append(table);
    var rows = $('#main_report_table').find('.content-row');
    var showHeatmap = false;
    for (var rows_n = 0; rows_n < rows.length; rows_n++) {
        if ($(rows[rows_n]).find('td[number]').length > 1) {
            showHeatmap = true;
            break
        }
    }

    if (showHeatmap) {
        var canvas = document.getElementById('gradientHeatmap');
          var context = canvas.getContext('2d');
          context.rect(0, 0, canvas.width, canvas.height);

          var gradient = context.createLinearGradient(0, 0, canvas.width, canvas.height);
          gradient.addColorStop(0, getColor(0, 65));
          gradient.addColorStop(0.5, 'white');
          gradient.addColorStop(1, getColor(240, 65));
          canvas.style.border = "0px solid rgba(0, 0, 0, .1)";
          context.fillStyle = gradient;
          context.fill();
          $('#heatmaps_chbox').change(function(){
               if($(this).is(':checked')) toggleHeatMap('on');
               else toggleHeatMap('off');
            });
          toggleHeatMap('on');
          $('#heatmap_header').show();
    }
}

function toggleHeatMap(state){
    var rows = $('#main_report_table').find('.content-row');
    for (var rows_n = 0; rows_n < rows.length; rows_n++) {
        var cells = $(rows[rows_n]).find('td[number]');
        if (state == 'on') {
            var quality = $(rows[rows_n]).attr('quality');
            heatMapOneRow(cells, quality);
        }
        else cells.each(function (i) {
            $(this).css('background', 'white');
            $(this).css('color', 'black');
        });
    }
}

function heatMapOneRow (cells, quality) {
    if (quality == 'Equal')
        return;
    var BLUE_HUE = 240;
    var BLUE_OUTER_BRT = 55;
    var BLUE_INNER_BRT = 65;

    var RED_HUE = 0;
    var RED_OUTER_BRT = 50;
    var RED_INNER_BRT = 60;

    var MIN_NORMAL_BRT = 80;
    var MEDIAN_BRT = 100;

    var numbers = $.map(cells, function (cell) {
        return parseFloat($(cell).attr('number'));
    });
    
    var min = Math.min.apply(null, numbers);
    var max = Math.max.apply(null, numbers);

    var topHue = BLUE_HUE;
    var lowHue = RED_HUE;
    
    var innerTopBrt = BLUE_INNER_BRT;
    var outerTopBrt = BLUE_OUTER_BRT;
    var innerLowBrt = RED_INNER_BRT;
    var outerLowBrt = RED_OUTER_BRT;

    if (quality == 'Less is better') {
        topHue = RED_HUE;
        lowHue = BLUE_HUE;

        innerTopBrt = RED_INNER_BRT;
        outerTopBrt = RED_OUTER_BRT;
        innerLowBrt = BLUE_INNER_BRT;
        outerLowBrt = BLUE_OUTER_BRT;
    }

    var twoCols = cells.length == 2;

    if (max == min) {
        $(cells).css('color', MEDIAN_BRT);
    } else {
        var sortedValues = numbers.slice().sort(function(a, b) {
          return a - b;
        });
        var median = getMedian(sortedValues);
        var l = numbers.length;
        var q1 = sortedValues[Math.floor((l - 1) / 4)];
        var q3 = sortedValues[Math.floor((l - 1) * 3 / 4)];

        var d = q3 - q1;
        var low_outer_fence = q1 - 3 * d;
        var low_inner_fence = q1 - 1.5 * d;
        var top_inner_fence = q3 + 1.5 * d;
        var top_outer_fence = q3 + 3 * d;
        cells.each(function (i) {
            var number = numbers[i];
            if (number < low_outer_fence) {
                $(this).css('background', getColor(lowHue, twoCols ? null : outerLowBrt));
                if (twoCols != true) $(this).css('color', 'white');
            }
            else if (number < low_inner_fence) {
                $(this).css('background', getColor(lowHue, twoCols ? null : innerLowBrt));
            }
            else if (number < median) {
                var k = (MEDIAN_BRT - MIN_NORMAL_BRT) / (median - low_inner_fence);
                var brt = Math.round(MEDIAN_BRT - (median - number) * k);
                $(this).css('background', getColor(lowHue, twoCols ? null : brt));
            }
            else if (number > top_inner_fence) {
                $(this).css('background', getColor(topHue, twoCols ? null : innerTopBrt));
            }
            else if (number > top_outer_fence) {
                $(this).css('background', getColor(topHue, twoCols ? null : outerTopBrt));
                if (twoCols != true) $(this).css('color', 'white');
            }
            else if (number > median) {
                var k = (MEDIAN_BRT - MIN_NORMAL_BRT) / (top_inner_fence - median);
                var brt = Math.round(MEDIAN_BRT - (number - median) * k);
                $(this).css('background', getColor(topHue, twoCols ? null : brt));
            }
        });
    }
}
function extendedClick() {
    $('.row_to_hide').toggleClass('row_hidden');

    var link = $('#extended_report_link');
    if (link.html() == 'Extended report') {
        link.html('Short report');
    } else {
        link.html('Extended report');
    }
}

function buildExtendedLinkClick() {
    return '<p id="extended_link"><a class="dotted-link" id="extended_report_link" onclick="extendedClick($(this))">Extended report</a></p>';
}

function appendIcarusLinks() {
    if (icarusLinks = readJson('icarus')) {
        if (icarusLinks.links != undefined) {
            var links = '';
            for (var link_n = 0; link_n < icarusLinks.links.length; link_n++) {
                //links += '<a href="' + icarusLinks.links[link_n] + '">' + icarusLinks.links_names[link_n] + '</a><br>'
                links += '<a class="btn btn-default btn-xs" role="button" href="' + icarusLinks.links[link_n] + '">' + icarusLinks.links_names[link_n] + '</a><br>'
            }
            $('#icarus').html(links);
        }
    }
}

</script>


    <script type="text/javascript">
        var colors = standard_colors;
        var broken_scaffolds_labels = [];
    </script>
    <script type="text/javascript">
function fillOneRow(metric, mainMetrics, group_n, order, glossary, is_primary, rowName,
                    report_n, assembliesNames, notAlignedContigs, notExtendedMetrics, isEmptyRows, metricsNotForCombinedReference) {
    (function(group_n) {
        var id_group = '#group_' + group_n;
        $(function() {
            $(id_group).removeClass('group_empty');
        });
    })(group_n);

    var table = '';
    var metricName = metric.metricName;
    var quality = metric.quality;
    var values = metric.values;

    var trClass = 'content-row';
    var iconPlots;
    if (metric.isMain || $.inArray(metricName, mainMetrics) != -1) {
        if ($.inArray(metricName, mainMetrics) != -1) {
            var numPlot = $.inArray(metricName, mainMetrics);
            iconPlots = '<img id="' + numPlot + '" class="icon_plot" style="vertical-align: bottom" onclick="setPlot($(this))"/>';
        }
        (function(group_n) {
            var id_group = '#group_' + group_n;
            $(function() {
                $(id_group).removeClass('row_hidden');
                $(id_group).removeClass('row_to_hide');
            });
        })(group_n);
    } else {
        trClass = 'content-row row_hidden row_to_hide';
    }
    var tdClass = '';
    if (!is_primary) {
        trClass += ' secondary_hidden';
        tdClass = 'secondary_td';
    }
    else {
        trClass += ' primary';
    }

    var not_extend = false;
    if ($.inArray(metricName, notExtendedMetrics) > -1 || isEmptyRows == true){
        not_extend = true;
        trClass += ' not_extend';
    }

    table +=
        '<tr class="' + trClass + '" quality="' + quality + '" onclick="toggleSecondary(event, $(this))">' +
        '<td class="left_column_td ' + tdClass + '">' +
        '<span class="metric-name' +
          (is_primary ? ' primary' : ' secondary') + (not_extend || !is_primary ? '' : ' expandable collapsed') + '">' +
           initial_spaces_to_nbsp(addTooltipIfDefinitionExists(glossary, rowName.trunc(55)), metricName) +
          (iconPlots && is_primary ? ("&nbsp" + iconPlots) : '') +
        '</span></td>';

    tooltipForGenomeStatistics = 'Metrics that depend on the reference length are not calculated for the combined reference.';
    tooltipForGCStatistics = 'GC content is not calculated for the combined reference.';
    if (is_primary && $.inArray(metricName, metricsNotForCombinedReference) != -1) {
        for (var val_n = 0; val_n < assembliesNames.length; val_n++) {
            table += '<td><a class="tooltip-link" rel="tooltip" title="' +
                (metricName.indexOf('GC') == -1 ? tooltipForGenomeStatistics : tooltipForGCStatistics)  + '"> ... </a></td>';
        }
        return table;
    }

    if (report_n > -1) {
        for (var not_aligned_n = 0; not_aligned_n < notAlignedContigs[report_n].length; not_aligned_n++) {
            values.splice(assembliesNames.indexOf(notAlignedContigs[report_n][not_aligned_n]), 0, '');
        }
    }

    for (var val_n = 0; val_n < values.length; val_n++) {
        var value = values[order[val_n]];

        if (value === null || value === '') {
            table += '<td><span>-</span></td>';
        } else {
            if (typeof value === 'number') {
                table +=
                    '<td number="' + value + '"><span>'
                        + toPrettyString(value) + '</span></td>';
            } else {
                var result = /([0-9\.]+)(.*)/.exec(value);
                var num = parseFloat(result[1]);
                var rest = result[2];

                if (num !== null) {
                    table += '<td number="' + num + '"><span>' + toPrettyString(num) + rest + '</span></td>';
                } else {
                    table += '<td><span>' + value + '</span></td>';
                }
            }
        }
    }

    return table;
}


function buildGenomeTable(reports, group_n, numColumns) {
    var tableGenome = '';
    tableGenome += '<div class="report" id="ref_report">';
    tableGenome += '<table cellspacing="0" id="refgenome">';
    tableGenome += '<tr class="top_row_tr"><td class="left_column_td"><span>' + 'Reference' + '</span></td>';
    var colNames = ['Size, bp', 'Fragments', 'GC, %', 'Genomic features', 'Operons'];
    for (var col_n = 0; col_n < numColumns; col_n++) {
        var columnName = colNames[col_n];
        tableGenome += '<td class="second_through_last_col_headers_td">' +
            '<span class="assembly_name">' + columnName + '</span>' +
        '</td>';
    }

    var combined_reference_size = 0;
    var notAlignedReport;
    for (var report_n = 0; report_n < reports.length; report_n++ ) {
        var trClass = 'content-row';
        var refName = reports[report_n].name;
        if (refName == 'not_aligned') {
            notAlignedReport = reports[report_n];
            continue;
        }
        tableGenome +=
            '<tr class="' + trClass + '">' +
            '<td class="left_column_td">' +
                '<span class="metric-name">' +
                    '<a href="runs_per_reference/' + refName + '/report.html">' + refName + '</a>' +
                '</span>' +
            '</td>';
        var metrics = reports[report_n].report[group_n][1];
        var referenceMetrics = ['Reference length', 'Reference fragments', 'Reference GC (%)',
                                'Reference genomic features', 'Reference operons'];
        for (var metric_n = 0; metric_n < metrics.length; metric_n++) {
            var metric = metrics[metric_n];
            if (referenceMetrics.indexOf(metric.metricName) === -1) continue;

            var value = metric.values[0];

            if (metric.metricName == 'Reference length')
                combined_reference_size += value;

            if (value === null || value === '') {
                tableGenome += '<td><span>-</span></td>';
            } else {
                if (typeof value === 'number') {
                    tableGenome +=
                        '<td number="' + value + '" class="number"><span>'
                        + toPrettyString(value) + '</span></td>';
                } else {
                    var result = /([0-9\.]+)(.*)/.exec(value);
                    var num = parseFloat(result[1]);
                    var rest = result[2];
//                        alert('value = ' + value + ' result = ' + result);

//                        var num = parseFloat(value);

                    if (num !== null) {
                        tableGenome += '<td number="' + num + '" class="number"><span>' + toPrettyString(num) + rest + '</span></td>';
                    } else {
                        tableGenome += '<td><span>' + value + '</span></td>';
                    }
                }
                //if (metric.metricName == 'Reference size'
            }
        }
        tableGenome += '</tr>';

    }

    tableGenome += '<tr><td></td><td class="second_through_last_col_headers_td last_row">' + toPrettyString(combined_reference_size) + '</td>';
    for (metric_n = 1; metric_n < metrics.length; metric_n++) {
        tableGenome += '<td class="second_through_last_col_headers_td"></td>';
    }
    tableGenome += '</tr>';

    //tableGenome += '<hr>';

    tableGenome += '</table>';
    tableGenome += '<br>';
    tableGenome += '<br>';
    tableGenome +=
        '<span class="metric-name">' +
            '<a href="combined_reference/report.html">' + 'Combined reference' + '</a>' +
        '</span><br>';
    if (notAlignedReport) {
        tableGenome +=
            '<span class="metric-name">' +
                '<a href="not_aligned/report.html">' + 'Not aligned contigs' + '</a>' +
            '</span>';
    }

    tableGenome += '</div>';
    return tableGenome;
}


function buildTotalReport(assembliesNames, report, order, date, minContig, glossary,
                          qualities, mainMetrics, reports) {
    $('#report_date').html('<p>' + date + '</p>');
    var extraInfo = '<p>All statistics are based on contigs of size &ge; ' + minContig +
        '<span class="rhs">&nbsp;</span>bp, unless otherwise noted (e.g., "# contigs (>= 0 bp)" and "Total length (>= 0 bp)" include all contigs).</p>';
    $('#extrainfo').html(extraInfo);
    $('#plot-caption').show();
    $('#per_ref_msg').html('<p>Rows show values for the whole assembly (column name) vs. the combined reference (concatenation of all provided references).<br>' +
        'Clicking on a row with <span style="color: #CCC">+</span> sign will expand values for contigs aligned to each of the references separately.<br>' +
        'Note that some metrics (e.g. # contigs) may not sum up, because one contig may be aligned to several references and thus, counted several times.<br>' +
        'All metrics that depend on the reference length (such as NG50, LG50, etc), plus the GC % are not calculated for the combined reference.<br>' +
        'The combined reference is just a concatenation of all available reference genomes of the species, presumably represented in the metagenomic dataset, ' +
        'but not necessarily the real content.<br>So it might miss many correctly assembled species, ' +
        'and therefore it does not make sense to apply the size and the GC content of the combined reference for assembly evaluation.</p>');
    $('#quast_name').html('MetaQUAST');
    $('#report_name').html('summary report');
    if (kronaPaths = readJson('krona')) {
        if (kronaPaths.paths != undefined) {
            $('#krona').html('Krona charts: ');
            for (var assembly_n = 0; assembly_n < assembliesNames.length; assembly_n++ ) {
                var assemblyName = assembliesNames[assembly_n];
                $('#krona').append(
                    '&nbsp&nbsp<span class="metric-name">' +
                    '<a href="' + kronaPaths.paths[assembly_n] + '">' + assemblyName + '</a>' +
                    '</span>&nbsp&nbsp');
            }
            if (assembliesNames.length > 1)  $('#krona').append(
                    '&nbsp&nbsp&nbsp&nbsp<span class="metric-name">' +
                    '<a href="' + kronaPaths.paths[assembliesNames.length] + '">Summary</a>' +
                    '</span>&nbsp');
        }
    }

    var table = '';
    table += '<table cellspacing="0" class="report_table draggable" id="main_report_table">';
    var refNames = [];
    for (var report_n = 0; report_n < reports.length; report_n++) {
        var _refName = reports[report_n].referenceName;
        if (!_refName) _refName = 'not_aligned';
        refNames.push(_refName);
    }
    reports = refNames.map(function (name, report_n) {
    return {
        name: name,
        report: this[report_n].report,
        asmNames: this[report_n].assembliesNames
        };
    }, reports);
    notAlignedContigs = {};
    for(report_n = 0; report_n < reports.length; report_n++ ) {
        notAlignedContigs[report_n] = [];
        for (var assembly_n = 0; assembly_n < assembliesNames.length; assembly_n++) {
            var assemblyName = assembliesNames[assembly_n];
            if (reports[report_n].asmNames.indexOf(assemblyName) == -1) {
                notAlignedContigs[report_n].push(assemblyName);
            }
        }
    }
    var notExtendedMetrics = [];
    if (minContig > 0)
        notExtendedMetrics = [ '# contigs (&gt;= 0 bp)', 'Total length (&gt;= 0 bp)', 'Fully unaligned length', '# fully unaligned contigs'];
    for (var group_n = 0; group_n < report.length; group_n++) {
        var group = report[group_n];
        var groupName = group[0];
        var metrics = group[1];
        var metricsNames = [];
        for (var metric_n = 0; metric_n < metrics.length; metric_n++)
            metricsNames.push(metrics[metric_n].metricName);
        var metricsNotForCombinedReference = ['GC (%)', 'NG50', 'NGA50', 'NG75', 'NGA75', 'LG50', 'LGA50', 'LG75', 'LGA75',
                'Avg contig read support'];
        for (var report_n = 0; report_n < reports.length; report_n++) {
            var metrics_by_refs = reports[report_n].report[group_n][1];
            for (var metric_n = 0; metric_n < metrics_by_refs.length; metric_n++) {
                var metric_by_refs = metrics_by_refs[metric_n].metricName;
                if ($.inArray(metric_by_refs, metricsNotForCombinedReference) != -1) {
                    if ($.inArray(metric_by_refs, metricsNames) == -1) {
                        metrics.push(metrics_by_refs[metric_n]);
                        metricsNames.push(metric_by_refs);
                    }
                }
            }
        }

        var width = assembliesNames.length + 1;

        if (groupName == 'Reference statistics') {
            var referenceValues = {};
            for (var metric_n = 0; metric_n < metrics.length; metric_n++) {
                var metric = metrics[metric_n];
                var metricName = metric.metricName;
                var value = metric.values[0];
                referenceValues[metricName] = value;
            }
            var refLen = referenceValues['Reference length'];
            var refFragments = referenceValues['Reference fragments'];
            var refGC = referenceValues['Reference GC (%)'];
            var refFeatures = referenceValues['Reference genomic features'];
            var refOperons = referenceValues['Reference operons'];
            var totalReads = referenceValues['# total reads'];
            var refMappedReads = referenceValues['Reference mapped (%)'];
            var refPairedReads = referenceValues['Reference properly paired (%)'];

            var numColumns = 1; // no GC in combined reference

            $('#combined_reference_name').show();
            if (refLen) {
                $('#reference_length').show().find('.val').html(toPrettyString(refLen));
                numColumns++;
            }
            if (refFragments) {
                $('#reference_fragments').show().find('.val').html(toPrettyString(refFragments));
                if (refFragments > 1)
                    $('#reference_fragments').find('.plural_ending').show();
                numColumns++;
            }
            var refFiles = 0;
            for (var report_n = 0; report_n < reports.length; report_n++ ) {
                var refName = reports[report_n].name;
                if (refName != 'not_aligned') {
                    refFiles += 1;
                }
            }
            if (refFiles) {
                $('#combined_reference_files').show().find('.val').html(toPrettyString(refFiles));
                if (refFiles > 1)
                    $('#combined_reference_files').find('.plural_ending').show();
            }
            if (refGC) {
                $('#reference_gc').show().find('.val').html(toPrettyString(refGC));
            }
            if (refFeatures) {
                $('#reference_features').show().find('.val').html(toPrettyString(refFeatures));
                numColumns++;
            }
            if (refOperons) {
                $('#reference_operons').show().find('.val').html(toPrettyString(refOperons));
                numColumns++;
            }

            if (totalReads)
                $('#total_reads').show().find('.val').html(toPrettyString(totalReads));
            if (refMappedReads !== undefined)
                $('#reference_mapped_reads').show().find('.val').html(toPrettyString(refMappedReads));
            if (refPairedReads !== undefined)
                $('#reference_paired_reads').show().find('.val').html(toPrettyString(refPairedReads));
            $('#main_ref_genome').html(buildGenomeTable(reports, group_n, numColumns));
            continue;
        }

        if (group_n == 0) {
            table += '<tr class="top_row_tr"><td id="top_left_td" class="left_column_td"><span>' + groupName + '</span></td>';

            for (var assembly_n = 0; assembly_n < assembliesNames.length; assembly_n++) {
                var assemblyName = assembliesNames[order[assembly_n]];
                if (assemblyName.length > 30) {
                    assemblyName =
                        '<span class="tooltip-link" rel="tooltip" title="' + assemblyName + '">' +
                            assemblyName.trunc(30) +
                            '</span>'
                }

                table += '<td class="second_through_last_col_headers_td" position="' + order[assembly_n] + '">' +
                    '<span class="drag_handle"><span class="drag_image"></span></span>' +
                    '<span class="assembly_name">' + assemblyName + '</span>' +
                    '</td>';
            }

        } else {
            table +=
                '<tr class="group_header row_to_hide row_hidden group_empty" id="group_' + group_n + '">' +
                    '<td class="left_column_td"><span>' + groupName + '</span></td>'; //colspan="' + width + '"
            for (var i = 1; i < width; i++) {
                table += '<td></td>';
            }
            table += '</tr>';
        }
        for (metric_n = 0; metric_n < metrics.length; metric_n++) {
            var metric = metrics[metric_n];
            var isEmptyRows = true;
            for(report_n = 0; report_n < reports.length; report_n++ ) {  //  add information for each reference
                var metrics_ref = reports[report_n].report[group_n][1];
                for (var metric_ext_n = 0; metric_ext_n < metrics_ref.length; metric_ext_n++){
                    if (metrics_ref[metric_ext_n].metricName == metrics[metric_n].metricName) {
                        isEmptyRows = false;
                        break;
                    }
                }
            }
            table += fillOneRow(metric, mainMetrics, group_n, order, glossary, true, metric.metricName, -1, assembliesNames,
                notAlignedContigs, notExtendedMetrics, isEmptyRows, metricsNotForCombinedReference);
            for(report_n = 0; report_n < reports.length; report_n++ ) {  //  add information for each reference
                var metrics_ref = reports[report_n].report[group_n][1];
                for (var metric_ext_n = 0; metric_ext_n < metrics_ref.length; metric_ext_n++){
                    if (metrics_ref[metric_ext_n].metricName == metrics[metric_n].metricName) {
                        table += fillOneRow(metrics_ref[metric_ext_n], mainMetrics, group_n, order, glossary, false,
                            reports[report_n].name, report_n, assembliesNames, notAlignedContigs, notExtendedMetrics);
                        break;
                    }
                }
            }
        }
        table += '</tr>';
    }
    table += '</table>';

    //table += '<p id="extended_link"><a class="dotted-link" id="extended_report_link" onclick="extendedLinkClick($(this))">Extended report</a></p>';
    table += buildExtendedLinkClick();

    setUpHeatMap(table);
}

function setPlot(icon) {
    num = icon.attr('id');
    names = ['contigs', 'largest', 'totallen', 'misassemblies', 'misassembled', 'mismatches', 'indels',
            'ns', 'genome', 'duplication', 'nga50'];
    switchSpan = names[num] + '-switch';
    document.getElementById(switchSpan).click();
}
</script>

    <script type="text/javascript">
/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this file,
 * You can obtain one at http://mozilla.org/MPL/2.0/. */

/*
 * flot-tickrotor: flot plugin to display angled X-axis tick labels.
 *
 * Requires flot 0.7 or higher and a browser supporting <canvas>.
 *
 * To activate, just set xaxis.rotateTicks to an angle in degrees.  Labels
 * are rotated clockwise, so if you want the labels to angle up and to the
 * right (/) you need to provide an angle > 90.  The text will be flipped so
 * that it is still right-side-up.
 * Angles greater than or equal to 180 are ignored.
 */
(function ($) {
    var options = { };

    function init(plot) {
        // Taken from flot-axislabels.
        // This is kind of a hack. There are no hooks in Flot between
        // the creation and measuring of the ticks (setTicks, measureTickLabels
        // in setupGrid() ) and the drawing of the ticks and plot box
        // (insertAxisLabels in setupGrid() ).
        //
        // Therefore, we use a trick where we run the draw routine twice:
        // the first time to get the tick measurements, so that we can change
        // them, and then have it draw it again.

        var ticks = [];  // preserve between draw() calls.
        var font;
        var secondPass = false;
        var rotateTicks, rotateTicksRads, radsAboveHoriz;

        plot.hooks.draw.push(function (plot, ctx) {
            var xaxis;  // for convenience
            if (!secondPass) {
                var opts = plot.getAxes().xaxis.options;
                if (opts.rotateTicks === undefined) {
                    return;
                }

                rotateTicks = parseInt(opts.rotateTicks, 10);
                if (rotateTicks.toString() != opts.rotateTicks || rotateTicks == 0 || rotateTicks >= 180) {
                    return;
                }

                rotateTicksRads = rotateTicks * Math.PI/180;
                if (rotateTicks > 90) {
                    radsAboveHoriz = Math.PI - rotateTicksRads;
                } else {
                    radsAboveHoriz = Math.PI/2 - rotateTicksRads;
                }

                font = opts.rotateTicksFont;
                if (!font) {
                    font = 'smaller sans-serif';
                }

                var elem, maxLabelWidth = 0, maxLabelHeight = 0, minX = 0, maxX = 0;

                // We have to clear the ticks option so that flot core
                // doesn't draw ticks superimposed with ours, but we preserve
                // the tick data as xaxis.rotatedTicks so that external code
                // can still get to it.

                // FIXME: It would obviously be better to just interrupt
                // the drawing of the ticks and preserve the 'ticks'
                // property.  That probably requires another hook.

                xaxis = plot.getAxes().xaxis;
                ticks = plot.getAxes().xaxis.ticks;
                xaxis.rotatedTicks = ticks;
                opts.ticks = [];  // we'll make our own

                var x;
                for (var i = 0; i < ticks.length; i++) {
                  elem = $('<span style="font:' + font + '">' + ticks[i].label + '</span>');
                  plot.getPlaceholder().append(elem);
                  ticks[i].height = elem.outerHeight(true);
                  ticks[i].width = elem.outerWidth(true) + 10;
                  elem.remove();
                  if (ticks[i].height > maxLabelHeight) {
                      maxLabelHeight = ticks[i].height;
                  }
                  if (ticks[i].width > maxLabelWidth) {
                      maxLabelWidth = ticks[i].width;
                  }
                  var tick = ticks[i];
                  // See second-draw code below for explanation of offsets.
                  if (rotateTicks > 90) {
                      // See if any labels are too long and require increased left
                      // padding.
                      x = Math.round(plot.getPlotOffset().left + xaxis.p2c(tick.v))
                          - Math.ceil(Math.cos(radsAboveHoriz) * tick.height)
                          - Math.ceil(Math.cos(radsAboveHoriz) * tick.width);
                      if (x < minX) {
                          minX = x;
                      }
                  } else {
                      // See if any labels are too long and require increased right
                      // padding.
                      x = Math.round(plot.getPlotOffset().left + xaxis.p2c(tick.v))
                          + Math.ceil(Math.cos(radsAboveHoriz) * tick.height)
                          + Math.ceil(Math.cos(radsAboveHoriz) * tick.width);
                      if (x > maxX) {
                          maxX = x;
                      }
                  }
                }

                // Calculate maximum label height after rotating.
                if (rotateTicks > 90) {
                    var acuteRads = rotateTicksRads - Math.PI/2;
                    opts.labelHeight = Math.ceil(Math.sin(acuteRads) * maxLabelWidth)
                                       + Math.ceil(Math.sin(acuteRads) * maxLabelHeight);
                } else {
                    var acuteRads = Math.PI/2 - rotateTicksRads;
                    // Center such that the top of the label is at the center of the tick.
                    opts.labelHeight = Math.ceil(Math.sin(rotateTicksRads) * maxLabelWidth)
                                       + Math.ceil(Math.sin(acuteRads) * maxLabelHeight);
                }

                if (minX < 0) {
                  plot.getAxes().yaxis.options.labelWidth = -1 * minX;
                }

                // Doesn't seem to work if there are no values using the
                // second y axis.
                //if (maxX > xaxis.box.left + xaxis.box.width) {
                //  plot.getAxes().y2axis.options.labelWidth = maxX - xaxis.box.left - xaxis.box.width;
                //}

                // re-draw with new label widths and heights
                secondPass = true;
                plot.setupGrid();
                plot.draw();
            } else {
                if (ticks.length == 0) {
                    return;
                }
                xaxis = plot.getAxes().xaxis;
                var box = xaxis.box;
                var tick, label, xoffset, yoffset;
                for (var i = 0; i < ticks.length; i++) {
                    tick = ticks[i];
                    if (!tick.label) {
                        continue;
                    }
                    ctx.save();
                    ctx.font = font;
                    if (rotateTicks < 90) {
                        // Center such that the top of the label is at the center of the tick.
                        xoffset = -Math.ceil(Math.cos(radsAboveHoriz) * tick.height);
                        yoffset = Math.ceil(Math.sin(radsAboveHoriz) * tick.height);
                        ctx.translate(Math.round(plot.getPlotOffset().left + xaxis.p2c(tick.v)) + xoffset,
                                      box.top + box.padding + plot.getOptions().grid.labelMargin + yoffset);
                        ctx.rotate(rotateTicksRads);
                    } else {
                        // We want the text to facing up, so we have to
                        // rotate counterclockwise, which means the label
                        // has to *end* at the center of the tick.
                        xoffset = Math.ceil(Math.cos(radsAboveHoriz) * tick.height)
                                  - Math.ceil(Math.cos(rotateTicksRads) * tick.width);
                        yoffset = Math.ceil(Math.sin(rotateTicksRads) * tick.width)
                                  + Math.ceil(Math.sin(radsAboveHoriz) * tick.height);
                        ctx.translate(Math.round(plot.getPlotOffset().left + xaxis.p2c(tick.v) + xoffset),
                                      box.top - box.padding + plot.getOptions().grid.labelMargin + yoffset);
                        ctx.rotate(-rotateTicksRads);
                    }
                    ctx.fillText(tick.label, 0, 0);
                    ctx.restore();
                }
            }
        });
    }

    $.plot.plugins.push({
        init: init,
        options: options,
        name: 'tickRotor',
        version: '1.0'
    });
})(jQuery);
</script>

<script type="text/javascript">
/*
Flot plugin for stacking data sets, i.e. putting them on top of each
other, for accumulative graphs.

The plugin assumes the data is sorted on x (or y if stacking
horizontally). For line charts, it is assumed that if a line has an
undefined gap (from a null point), then the line above it should have
the same gap - insert zeros instead of "null" if you want another
behaviour. This also holds for the start and end of the chart. Note
that stacking a mix of positive and negative values in most instances
doesn't make sense (so it looks weird).

Two or more series are stacked when their "stack" attribute is set to
the same key (which can be any number or string or just "true"). To
specify the default stack, you can set

  series: {
    stack: null or true or key (number/string)
  }

or specify it for a specific series

  $.plot($("#placeholder"), [{ data: [ ... ], stack: true }])
  
The stacking order is determined by the order of the data series in
the array (later series end up on top of the previous).

Internally, the plugin modifies the datapoints in each series, adding
an offset to the y value. For line series, extra data points are
inserted through interpolation. If there's a second y value, it's also
adjusted (e.g for bar charts or filled areas).
*/

(function ($) {
    var options = {
        series: { stack: null } // or number/string
    };
    
    function init(plot) {
        function findMatchingSeries(s, allseries) {
            var res = null
            for (var i = 0; i < allseries.length; ++i) {
                if (s == allseries[i])
                    break;
                
                if (allseries[i].stack == s.stack && allseries[i].datapoints.points.length != 0 )
                    res = allseries[i];
            }
			
            return res;
        }
        
        function stackData(plot, s, datapoints) {
            if (s.stack == null || datapoints.points.length == 0)
                return;
		
            var other = findMatchingSeries(s, plot.getData());

            if (!other)
                return;
            
            // less points ? add points to match the previous serie
            if ( other.datapoints.points.length > datapoints.points.length ) {
            	var completedpoints = [];
            	var w = 0, x = 0, y = 0, z = -1,
            		ods = other.datapoints.points.length,
            		ops = other.datapoints.pointsize,
            		ds = datapoints.points.length,
            		dps = datapoints.pointsize;
            	for( x = 0; x < ods; x += ops ) {
            		z = -1;
            		for( y = 0; y < ds ; y += dps ) {
            			if ( other.datapoints.points[x] == datapoints.points[y] ) {
            				z = y;
            				break;
            			}
            		}
            		if ( z > -1 ) {
            			for( w = 0; w < dps; ++w )
            				completedpoints.push( datapoints.points[z + w] );
            		}
            		else {
            			completedpoints.push( other.datapoints.points[x] );
						for( w = 1; w < ops; ++w )
							completedpoints.push( 0 );
            		}
            	}
            	s.datapoints.points = completedpoints;
            	datapoints.points = completedpoints;
            }
	
            var ps = datapoints.pointsize,
                points = datapoints.points,
                otherps = other.datapoints.pointsize,
                otherpoints = other.datapoints.points,
                newpoints = [],
                px, py, intery, qx, qy, bottom,
                withlines = s.lines.show,
                horizontal = s.bars.horizontal,
                withbottom = ps > 2 && (horizontal ? datapoints.format[2].x : datapoints.format[2].y),
                withsteps = withlines && s.lines.steps,
                fromgap = true,
                keyOffset = horizontal ? 1 : 0,
                accumulateOffset = horizontal ? 0 : 1,
                i = 0, j = 0, l;

            while (true) {
                if (i >= points.length)
                    break;

                l = newpoints.length;

                if (points[i] == null) {
                    // copy gaps
                    for (m = 0; m < ps; ++m)
                        newpoints.push(points[i + m]);
                    i += ps;
                }
                else if (j >= otherpoints.length) {
					// for lines, we can't use the rest of the points
                    if (!withlines) {
                        for (m = 0; m < ps; ++m)
                            newpoints.push(points[i + m]);
                    }
                    i += ps;
                }
                else if (otherpoints[j] == null) {
                    // oops, got a gap
                    for (m = 0; m < ps; ++m)
                        newpoints.push(null);
                    fromgap = true;
                    j += otherps;
                }
                else {
                    // cases where we actually got two points
                    px = points[i + keyOffset];
                    py = points[i + accumulateOffset];
                    qx = otherpoints[j + keyOffset];
                    qy = otherpoints[j + accumulateOffset];
                    bottom = 0;

                    if (px == qx) {
                        for (m = 0; m < ps; ++m)
                            newpoints.push(points[i + m]);

                        newpoints[l + accumulateOffset] += qy;
                        bottom = qy;
                        
                        i += ps;
                        j += otherps;
                    }
                    else if (px > qx) {
                        // we got past point below, might need to
                        // insert interpolated extra point
                        if (withlines && i > 0 && points[i - ps] != null) {
                            intery = py + (points[i - ps + accumulateOffset] - py) * (qx - px) / (points[i - ps + keyOffset] - px);
                            newpoints.push(qx);
                            newpoints.push(intery + qy);
                            for (m = 2; m < ps; ++m)
                                newpoints.push(points[i + m]);
                            bottom = qy; 
                        }

                        j += otherps;
                    }
                    else { // px < qx	
                        if (fromgap && withlines) {
                            // if we come from a gap, we just skip this point
                            i += ps;
                            continue;
                        }
                            
                        for (m = 0; m < ps; ++m)
                            newpoints.push(points[i + m]);
                        
                        // we might be able to interpolate a point below,
                        // this can give us a better y
                        if (withlines && j > 0 && otherpoints[j - otherps] != null)
                            bottom = qy + (otherpoints[j - otherps + accumulateOffset] - qy) * (px - qx) / (otherpoints[j - otherps + keyOffset] - qx);

                        newpoints[l + accumulateOffset] += bottom;
                        
                        i += ps;
                    }

                    fromgap = false;
                    
                    if (l != newpoints.length && withbottom)
                        newpoints[l + 2] += bottom;
                }

                // maintain the line steps invariant
                if (withsteps && l != newpoints.length && l > 0
                    && newpoints[l] != null
                    && newpoints[l] != newpoints[l - ps]
                    && newpoints[l + 1] != newpoints[l - ps + 1]) {
                    for (m = 0; m < ps; ++m)
                        newpoints[l + ps + m] = newpoints[l + m];
                    newpoints[l + 1] = newpoints[l - ps + 1];
                }
            }

            datapoints.points = newpoints;
        }
        
        plot.hooks.processDatapoints.push(stackData);
    }
    
    $.plot.plugins.push({
        init: init,
        options: options,
        name: 'stack',
        version: '1.2'
    });
})(jQuery);
</script>

<script type="text/javascript">
var refNames;
var refTicks;

var plotHeight = 0;
var sortRefsBtnHeight = 85;

var summary = {
    largest: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    contigs: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },nga50: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    misassemblies: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    totallen: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    mismatches: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    ns: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    indels: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    misassembled: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },
    genome: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },duplication: {
        isInitialized: false,
        maxY: 0,
        maxYTick: 0,
        series: null,
        showWithData: null
    },

    draw: function (name, title, colors, filenames, data, refPlotValue, tickX,
                    placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {
        var sortBtnClass = getSortRefsRule();

        var selectedAssemblies = getSelectedAssemblies(filenames);
        $('#legend-placeholder').empty();

        filenames.forEach(function(filename, i) {
            addLabelToLegend(i, filename, selectedAssemblies, colors)
        });

        $(scalePlaceholder).empty();
        addSortRefsBtn(sortBtnClass);

        var coordX = data.coord_x;
        var coordY = data.coord_y;

        refNames = data.refnames;
        var cur_filenames = data.filenames;
        var info = summary[name];

        if (!info.isInitialized) {
            var plotsN = cur_filenames.length;
            info.series = new Array(plotsN);
            info.references = refNames;

            for (var i = 0; i < plotsN; i++) {
                var index = $.inArray(cur_filenames[order[i]], filenames);
                var plot_coordX = coordX[order[i]];
                var plot_coordY = coordY[order[i]];
                var size = plot_coordX.length;

                info.series[i] = {
                    data: [],
                    label: filenames[index],
                    number: index,
                    color: colors[index],
                    points: {
                        show: true,
                        fillColor: colors[index]
                    }
                };

                var x = 0.0;

                for (var k = 0; k < size; k++) {
                    info.series[i].data.push([plot_coordX[k], plot_coordY[k]]);
                    info.maxY = Math.max(info.maxY, plot_coordY[k]);
                }
            }

            for (i = 0; i < plotsN; i++) {
                info.series[i].lines = {show: true, lineWidth: 0.1}
            }
            var yFormatter = getBpTickFormatter;
            if ($.inArray(name, ['contigs', 'misassemblies', 'mismatches', 'indels', 'ns', 'duplication']) > -1)
                yFormatter = getJustNumberTickFormatter;
            else if (name == 'genome')
                yFormatter = getPercentTickFormatter;
            refTicks = [];
            for (var i = 0; i < refNames.length; i++)
            {
                refTicks.push([i + 1, refNames[i]]);
            }
            info.showWithData = function(series, colors) {
                var plot = $.plot(placeholder, series, {
                        shadowSize: 0,
                        colors: colors,
                        legend: {
                            container: $('useless-invisible-element-that-does-not-even-exist')
                        },
                        grid: {
                            borderWidth: 1,
                            hoverable: true,
                            autoHighlight: false,
                            mouseActiveRadius: 1000
                        },
                        yaxis: {
                            min: 0,
                            max: name == 'genome' ? Math.min(100, info.maxY + 10) : null,
                            labelWidth: 125,
                            reserveSpace: true,
                            lineWidth: 0.5,
                            color: '#000',
                            tickFormatter: yFormatter(info.maxY),
                            minTickSize: 1
                        },
                        xaxis: {
                            min: 0,
                            max: refNames.length + 1,
                            lineWidth: 1,
                            rotateTicks: 90,
                            color: '#000',
                            ticks: refTicks
                        },
                        minTickSize: 1
                    }
                );
                plotHeight = plot.height();

                var firstLabel = $('.yAxis .tickLabel').last();
                firstLabel.prepend(title + '<span class="rhs">&nbsp;</span>=<span class="rhs">&nbsp;</span>');

                bindTip(placeholder, series, plot, refToPrettyString, 1, refNames, 'top right', true);
            };

            info.isInitialized = true;
        }

        addLegendClickEvents(info, filenames.length, showPlotWithInfo);

        showPlotWithInfo(info);
        moveSortRefsBtns();
        setSortRefsBtns(info);
    }
};

function getSortRefsRule() {
    if ($("input[name=sortRefs]")[0])
        return $("input[name=sortRefs]:checked").val();
}

function moveSortRefsBtns() {
    var spaceAfterHeight = Math.max(50, plotHeight - $('#legend-placeholder').height() + 5);
    $('#sortRefsBtnSpace').height(spaceAfterHeight);
}

function addSortRefsBtn(sortBtnClass) {
    $('#legend-placeholder').append('<div id="sortRefsBtnSpace"></div>');
    $('#legend-placeholder').append('<span>Sort references by: <br></span>');
    $('#legend-placeholder').append(
        '<label><input type="radio" id="sortByName" name="sortRefs" value="name">&nbspname</label><br>'
    );
    $('#legend-placeholder').append(
        '<label><input type="radio" id="sortByValue" name="sortRefs" value="value" checked="checked">' +
        '&nbspthe best average value<br>among all assemblies</label>'
    );
    if (sortBtnClass)
       $("input[name=sortRefs][value=" + sortBtnClass + "]").prop('checked', true);
}

function setSortRefsBtns(info, index) {
    $("#legend-placeholder input[name='sortRefs']").click(function(){
        showPlotWithInfo(info, index);
    });
}

function getSortOrder() {
    var sortOrder = getSortRefsRule();
    if (!sortOrder)
        return;
    if (sortOrder == 'name') {
        sortOrder = 'alphabet';
    }
    else if (sortOrder) {
        sortOrder = 'value';
    }
    return sortOrder;
}

function sortReferences(sortOrder, info) {
    if (sortOrder == 'alphabet') {
        refNames = sortedRefs;
    }
    else if (sortOrder == 'value') {
        refNames = info.references;
    }
    refTicks = [];
    for (var i = 0; i < refNames.length; i++)
    {
        refTicks.push([i + 1, refNames[i]]);
    }
}

</script>

<script type="text/javascript">

var show_all_span =
    "<span class='selected-switch misassemblies'>" +
        'Back to overview' +
        "</span>";
var show_all_a =
    "&nbsp;&nbsp;&nbsp;<a class='dotted-link' onClick='showAll()'>" +
        'Back to overview' +
        "</a>";


var misassemblies = {
    isInitialized: false,

    maxY: 0,
    plot: null,
    series: null,
    showWithData: null,
    minPow: 0,
    ticks: null,
    placeholder: null,
    legendPlaceholder: null,
    colors: null,
    yAxisLabeled: false,

    show_all_el: show_all_span,

    draw: function(name, title, colors, filenames, data, reflen, tickX,
                   placeholder, legendPlaceholder, glossary, order, scalePlaceholder) {

        var cur_filenames = data.filenames;
        var main_coordX = data.main_coord_x;
        var main_coordY = data.main_coord_y;
        var refs = data.refnames;
        var main_refnames = data.main_refnames;

        misassemblies.refs = main_refnames;
        misassemblies.colors = colors;
        misassemblies.filenames = filenames;
        misassemblies.placeholder = placeholder;

        if (!misassemblies.isInitialized) {
            var plotsN = cur_filenames.length;
            misassemblies.series = new Array(plotsN + 1);
            misassemblies.series[0] = new Array(plotsN);
            for (var i = 0; i < plotsN; i++) {
                var index = $.inArray(cur_filenames[order[i]], filenames);
                var plot_coordX = main_coordX[order[i]];
                var plot_coordY = main_coordY[order[i]];
                var size = plot_coordX.length;
                misassemblies.series[0][i] = {
                    data: [],
                    label: filenames[index],
                    number: index,
                    color: colors[index],
                    points: {
                        show: true,
                        fillColor: colors[index]
                    },
                    lines: {
                        show: true,
                        lineWidth: 0.1
                    }
                };
                for (var k = 0; k < size; k++) {
                    misassemblies.series[0][i].data.push([plot_coordX[k], plot_coordY[k]]);
                }
                if (misassemblies.series[0][i].data[0][1] > misassemblies.maxY) {
                    misassemblies.maxY = misassemblies.series[0][i].data[0][1];
                }
            }
            var misassembl_colors = ['#E31A1C', '#1F78B4', '#33A02C'];
            var misassembl_labels = ['relocations', 'translocations', 'inversions'];

            refTicks = [];
            for (var ref_n = 0; ref_n < main_refnames.length; ref_n++) {
                    refTicks.push([ref_n + 1, main_refnames[ref_n]]);
            }
            misassemblies.references = main_refnames;
            refs = refs.filter( function(el) {
              return main_refnames.indexOf(el) > -1;
            } );
            for (var file_n = 0; file_n < filenames.length; file_n++) {
                var coordX = data.coord_x[file_n];
                var coordY = data.coord_y[file_n];
                plotsN = 3;
                var sortedRefs = [];
                for (var refs_n = 0; refs_n < refs.length; refs_n++){
                    var curRefN = $.inArray(main_refnames[refs_n], refs);
                    for (i = 0; i < plotsN; i++) {
                        sortedRefs.push([refs_n+1, coordY[plotsN*curRefN+i]]);
                    }
                }

                misassemblies.series[file_n + 1] = new Array(plotsN);
                for (i = 0; i < plotsN; i++) {

                    size = coordX.length/plotsN;
                    misassemblies.series[file_n + 1][i] = {
                        data: [],
                        label: misassembl_labels[i],
                        number: i,
                        color: misassembl_colors[i],
                        stack: 0,
                        lines: {
                            show: false,
                            steps: false
                        },
                        bars: {
                            show: true,
                            lineWidth: 0.6,
                            fill: 0.6,
                            barWidth: 0.9,
                            align: 'center'
                        }
                    };

                    for (k = 0; k < size; k++) {
                        misassemblies.series[file_n + 1][i].data.push(sortedRefs[k*plotsN+i]);
                    }
                    if (misassemblies.series[file_n + 1][i].data[0][1] > misassemblies.maxY) {
                        misassemblies.maxY = misassemblies.series[file_n + 1][i].data[0][1];
                    }
                }
            }

            misassemblies.showWithData = showAllAssemblies;
            misassemblies.isInitialized = true;
        }
        createLegend(misassemblies.filenames, misassemblies.colors, 0);
        showPlotWithInfo(misassemblies, 0);

        $('#change-assembly').css('visibility', 'visible');

        $('#misassemblies_info').hide();
    }
};


function showAllAssemblies(series, colors) {
    if (series == null)
        return;

    misassemblies.plot = $.plot(misassemblies.placeholder, series, {
            shadowSize: 0,
            colors: colors,
            legend: {
                container: $('useless-invisible-element-that-does-not-even-exist')
            },
            grid: {
                borderWidth: 1,
                hoverable: true,
                autoHighlight: false,
                mouseActiveRadius: 1000
            },
            yaxis: {
                min: 0,
                labelWidth: 125,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: getJustNumberTickFormatter(misassemblies.maxY),
                minTickSize: 1
            },
            xaxis: {
                min: 0,
                max: refNames.length + 1,
                lineWidth: 1,
                rotateTicks: 90,
                color: '#000',
                ticks: refTicks
            },
            minTickSize: 1
        }
    );
    var firstLabel = $('.yAxis .tickLabel').last();
    firstLabel.prepend('Misassemblies<span class="rhs">&nbsp;</span>=<span class="rhs">&nbsp;</span>');
    unBindTips(misassemblies.placeholder);
    bindTip(misassemblies.placeholder, series, misassemblies.plot, refToPrettyString, 1, refNames, 'top right', true);
}


function showOneAssembly(series, colors) {
    if (series == null) {
        return;
    }

    misassemblies.plot = $.plot(misassemblies.placeholder, series, {
            shadowSize: 0,
            colors: colors,
            legend: {
                container: $('useless-invisible-element-that-does-not-even-exist')
            },
            grid: {
                hoverable: true,
                borderWidth: 1,
                autoHighlight: false,
                mouseActiveRadius: 1000
            },
            yaxis: {
                min: 0,
                labelWidth: 125,
                reserveSpace: true,
                lineWidth: 0.5,
                color: '#000',
                tickFormatter: getJustNumberTickFormatter(misassemblies.maxY),
                minTickSize: 1
            },
            xaxis: {
                min: 0,
                max: refNames.length + 1,
                lineWidth: 1,
                rotateTicks: 90,
                color: '#000',
                ticks: refTicks
            },
            minTickSize: 1
        }
    );

    var firstLabel = $('.yAxis .tickLabel').last();
    firstLabel.append(' misassemblies');
    unBindTips(misassemblies.placeholder);
    bindTip(misassemblies.placeholder, series, misassemblies.plot, refToPrettyString, 1, refNames,  'top right');
}

function showAll() {
    createLegend(misassemblies.filenames, misassemblies.colors, 0);
    misassemblies.show_all_el = show_all_span;
    misassemblies.showWithData = showAllAssemblies;

    $('#show_all_label').html(misassemblies.show_all_el);
    showPlotWithInfo(misassemblies, 0);
}


function showPlot(index) {
    var misassembl_labels = ['relocations', 'translocations', 'inversions'];
    var misassembl_colors = ['#E31A1C', '#1F78B4', '#33A02C'];
    createLegend(misassembl_labels, misassembl_colors, index);

    misassemblies.show_all_el = show_all_a;
    misassemblies.showWithData = showOneAssembly;

    $('#show_all_label').html(misassemblies.show_all_el);
    showPlotWithInfo(misassemblies, index);
}

function createLegend(labels, colors, index){
    var sortBtnClass = getSortRefsRule();

    var filenames = misassemblies.filenames;
    var selectedLabels = getSelectedAssemblies(labels);
    $('#legend-placeholder').empty();
    var selectors = "";

    labels.forEach(function(label, i) {
        var link = index == 0 ? '<span id="' + labels[i] + '-switch"' + " class='plot-mis-type-switch dotted-link'>by type<br></span><br>" : '';
        addLabelToLegend(i, label, selectedLabels, colors, link);
    });
    if (index > 0) {
        for (var filenames_n = 0; filenames_n < filenames.length; filenames_n++){
            selectors += '<br><span id="' + filenames[filenames_n] + '-switch" ' +
                "class='plot-switch dotted-link'>" +
                filenames[filenames_n] + "</span>";
        }
        $('#legend-placeholder').append(
            "<br><br><div id='change-assembly' style='margin-right: 3px;'>" +
                "<span id='show_all_label'>" +
                misassemblies.show_all_el +
                "</span><br>" + selectors +
                "</div>"
        );
        addSortRefsBtn(sortBtnClass);
    }
    else addSortRefsBtn(sortBtnClass);
    addLinksToSwitches(index - 1);
    addLegendClickEvents(misassemblies, misassemblies.series.length, showPlotWithInfo, false, index);
    moveSortRefsBtns();
    setSortRefsBtns(misassemblies, index);
}

function addLinksToSwitches(index) {
    var filenames = misassemblies.filenames;
    for (filenames_n = 0; filenames_n < filenames.length; filenames_n++){
        var switchSpan = document.getElementById(filenames[filenames_n] + "-switch");
        $(switchSpan).click(getToggleSwitchFunction(filenames_n+1));
        if (filenames_n == index) {
            switchSpan.className = 'plot-switch selected-switch misassemblies';
        }
    }
}

function getToggleSwitchFunction(index) {
    return function() {
        if (index > 0) showPlot(index);
        else showAll();
    };
}
</script>

    <script type="text/javascript">
function buildReport() {
    var assembliesNames;
    var order;

    var totalReport = null;
    var qualities = null;
    var mainMetrics = null;
    var contigs = null;

    var glossary = JSON.parse($('#glossary-json').html());

    /****************/
    /* Total report */

    if (!(totalReports = readJson('total-report'))) {
        console.log("Error: cannot read #total-report-json");
        return 1;
    }

    totalReport = totalReports[0];

    assembliesNames = totalReport.assembliesNames;

    order = recoverOrderFromCookies() || totalReport.order || Range(0, assembliesNames.length);

    mainMetrics = ['# contigs', 'Largest alignment', 'Total aligned length', '# misassemblies', 'Misassembled contigs length',
        '# mismatches per 100 kbp', '# indels per 100 kbp', "# N's per 100 kbp", 'Genome fraction (%)', 'Duplication ratio', 'NGA50'];
    buildTotalReport(assembliesNames, totalReport.report, order, totalReport.date,
        totalReport.minContig, glossary, qualities, mainMetrics, totalReports.slice(1));

    var plotsSwitchesDiv = document.getElementById('plots-switches');
    $(plotsSwitchesDiv).html('<b>Plots:</b>');
    
    var misassembl_coordX = null;
    var misassembl_coordY = null;
    var misassembl_refNames = null;
    var summaryReports = null;
    var firstPlot = true;

    if (summaryReports = readJson('summary')){
        name_reports = ['contigs', 'largest', 'totallen', 'misassemblies', 'misassembled', 'mismatches',
            'indels', 'ns', 'genome', 'duplication', 'nga50'];
        title_reports = ['Contigs', 'Largest alignment', 'Total aligned len', 'Misassemblies', 'Mis. len', 'Mismatches',
            'Indels', "N's per 100 kbp", 'Genome frac.', 'Dup. ratio', 'NGA50'];
        for (var i = 0; i < summaryReports.length; i++) {
            if (summaryReports[i].refnames != undefined && summaryReports[i].refnames.length > 0) {
                if (name_reports[i] == 'misassemblies') {
                    misassembl_coordX = summaryReports[i].coord_x;
                    misassembl_coordY = summaryReports[i].coord_y;
                    misassembl_refNames = summaryReports[i].refnames;
                    var misassembliesReports = null;
                    if (misassembliesReports = readJson('misassemblies')) {
                        if (misassembliesReports.refnames  != undefined) {
                            makePlot(firstPlot, assembliesNames, order, 'misassemblies', 'Misassemblies', misassemblies.draw, {
                                    main_coord_x: misassembl_coordX,
                                    main_coord_y: misassembl_coordY,
                                    main_refnames: misassembl_refNames,
                                    coord_x: misassembliesReports.coord_x,
                                    coord_y: misassembliesReports.coord_y,
                                    filenames: misassembliesReports.filenames,
                                    refnames: misassembliesReports.refnames
                                },
                                null, null
                            );
                        }
                    }
                }
                else {
                    makePlot(firstPlot, assembliesNames, order, name_reports[i], title_reports[i], summary.draw, {
                            coord_x: summaryReports[i].coord_x,
                            coord_y: summaryReports[i].coord_y,
                            filenames: summaryReports[i].filenames,
                            refnames: summaryReports[i].refnames
                        },
                        null, null
                    );
                }
                firstPlot = false;
            }
        }
    }

    if (firstPlot) $('.plots').hide();
    $('#contigs_are_ordered').hide();
    appendIcarusLinks();
    return 0;
}

</script>


    <script type="text/javascript">
        $(function() {

            buildReport();

            dragTable.init();

            $("[rel=tooltip]").tooltip({ animation: false, });
        });
    </script>
</head>

<body>
<div class="top-panel">
    <table class="layout-table">
        <tr>
        <td class='layout-table-td'>
            <div class="page_title">
                <h1>
                    QUAST
                </h1>
                <p class="page_subtitle">
                    <b>Quality Assessment Tool for Genome Assemblies</b>
                    by <a href="http://cab.spbu.ru" class='dark_bg_link'>CAB</a>
                </p>
            </div>
        </td>
        </tr>
    </table>
</div>
<div class="content">
    <div id='header'>
        <p id='report_date'></p>

        <p id='icarus'></p>
        <p id='extrainfo'>Loading...</p>
        <p id='per_ref_msg'></p>
        <p id='krona'></p>

        <p id="data_set_p">
            <span style="display: none;" id="reference_name">Aligned to
                "<span class='val'></span>"
            </span>

            <span style="display: none;" id="combined_reference_name">
                Combined reference
            </span>

            <span style="display: none;" id="est_reference_length">Estimated reference size:
                <span class='val'></span><span class='rhs'>&nbsp;</span>bp
            </span>

            <span style="display: none;" id="reference_length">|
                <span class='val'></span><span class='rhs'>&nbsp;</span>bp
            </span>

            <span style="display: none;" id="combined_reference_files">|
                <span class='val'></span><span class='rhs'>&nbsp;</span>reference<span style="display: none;" class='plural_ending'>s</span>
            </span>

            <span style="display: none;" id="reference_fragments">|
                <span class='val'></span><span class='rhs'>&nbsp;</span>fragment<span style="display: none;" class='plural_ending'>s</span>
            </span>

            <span style="display: none;" id="reference_gc">|
                <span class='val'></span><span class='rhs'>&nbsp;</span>% G+C
            </span>

            <span style="display: none;" id="reference_chr">|
                <span class='val'></span> chromosome<span style="display: none;" class='plural_ending'>s</span>
            </span>

            <span style="display: none;" id="total_reads"><br>
                <span class='val'></span> reads
            </span>

            <span style="display: none;" id="reference_mapped_reads">|
                <span class='val'></span>% mapped
            </span>

            <span style="display: none;" id="reference_paired_reads">|
                <span class='val'></span>% properly paired
            </span>

            <span style="display: none;" id="reference_features"><br>
                <span class='val'></span> genomic features
            </span>

            <span style="display: none;" id="reference_operons">|
                <span class='val'></span> operons
            </span>
        </p>
    </div>

    <noscript>
        <div style="margin-left: 0;">
            Unfortunately, <span style="font-weight: bold">JavaScript</span> in your
            browser <span style="font-weight: bold">is disabled</span> or is not supported.</br>
            We need JavaScript to build report and plots.
        </div>
    </noscript>

    <div class='report-and-plots'>
        <div class='report' id='main_report'>
            <div id="heatmap_header" style='display: none; float: left;'>
                <div style="padding-right: 10px; display: inline-block; vertical-align:top;">
                    <canvas id="gradientHeatmap" width="150" height="6"></canvas>
                    <div style="margin-left: 0; text-align: justify;">
                        <span class="heatmap_legend">Worst</span>
                        <span style="margin-left: -5px;" class="heatmap_legend">Median</span>
                        <span class="heatmap_legend">Best</span>
                        <span class="heatmap_legend" style='width: 100%; height: 0;'></span>
                    </div>
                </div>
                <label for="heatmaps_chbox"><input type="checkbox" checked="checked" id="heatmaps_chbox">Show heatmap</label>
            </div>
            <div style="clear: both;"></div>
        </div>
        <div class='plots'>
            <table style='margin: -3px;'>
                <tr id='all-switches'>
                    <td>
                        <div id='plots-switches'></div>
                    </td>
                    <td style='text-align: right; padding-right: 22px;'>
                        <span id='scale-placeholder'></span>
                    </td>
                </tr>
                <tr>
                    <td colspan="2">
                        <div id='plot-placeholder' style=''></div>
                    </td>
                    <td>
                        <div id='legend-placeholder'></div>
                    </td>
                </tr>
                <tr>
                    <td colspan="3" id="plot-caption" style="display: none;">
                        <p id="contigs_are_ordered">Contigs are ordered from largest (contig #1) to smallest.</p>
                        <p id="frc_info">FRCurve:
                            Y is the total number of aligned bases divided by the reference length,
                            in the contigs having the total number of <span class="frc_plot_name"></span> at most X.</p>
                        <p id="gc_info">Contigs are broken into nonoverlapping 100 bp windows.
                            Plot shows number of windows for each GC percentage.</p>
                        <p id="gc_contigs_info">Plot shows number of contigs with GC percentage in a certain range.</p>
                    </td>
                </tr>
            </table>
        </div>

        <p>
            <span id="main_ref_genome"></span>
        </p>

        <div style="clear: left"></div>
        <div style="clear: right"></div>

        <div class='json-code'>
            <div id='total-report-json'>
                [{"date":"17 September 2025, Wednesday, 15:07:41","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"combined_reference","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.008","0.005","0.042","0.042","0.016","0.030","0.113","0.170"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.008","1.002","1.009","1.006","1.008","1.011","1.016","1.016"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5844,4846,7689,4742,5864,6026,11154,18654],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[48841,76030,515327,635026,223349,275232,2546456,3503238],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[674,786,822,724,899,840,717,751],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[561,610,595,582,644,612,555,561],"isMain":false},{"metricName":"LA50","quality":"Less is better","values":[16,26,193,289,74,104,1042,1385],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[37,54,382,540,149,201,2083,2783],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,6,19,36,15,34,415,658],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,7,12,2,2,92,128],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,3,10,3,14,192,300],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,6,9,14,10,18,129,228],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,6,19,34,14,31,390,603],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[7808,5030,16281,26162,14283,31900,324336,446528],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,6,5,2,2,36,67],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,1,0,1,6],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,2,7,1,2,4,22],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,1314,5261,510,1007,3203,16011],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,2272],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[24855,13030,105290,86389,51477,118631,327629,536342],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2074,781,7114,4593,4529,9015,22299,33926],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[5473,1487,16473,9215,9451,21131,50900,77274],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1550.06","1312.15","1312.34","1097.73","1733.72","2115.83","1534.59","1670.32"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["129.34","78.65","88.67","58.36","152.53","160.79","104.45","105.66"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1851,723,6627,4338,4220,8238,20662,31340],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[223,58,487,255,309,777,1637,2586],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[58,85,613,823,245,317,3185,4270],"isMain":true},{"metricName":"# contigs (>= 0 bp)","quality":"Equal","values":[570,1178,4524,17767,769,2840,18255,66988],"isMain":false},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[8,16,126,135,59,74,466,735],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,0,1,1,21,21],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,2,6],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,4847,7920,4743,5908,6478,11155,18654],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[49922,76847,518671,644704,225473,279142,2579933,3569774],"isMain":true},{"metricName":"Total length (>= 0 bp)","quality":"More is better","values":[197523,254915,1734801,3225255,418837,800739,7996424,17764779],"isMain":false},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[18990,30755,200161,196678,99443,116186,838921,1263219],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,0,7920,0,5908,6478,164571,195398],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,21298,92078],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[738,837,836,744,917,875,751,797],"isMain":false},{"metricName":"N75","quality":"More is better","values":[575,638,602,595,664,644,581,603],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[15,25,191,284,73,100,996,1326],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[35,52,376,528,146,192,1984,2632],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[18930192355,18930192355,18930192355,18930192355,18930192355,18930192355,18930192355,18930192355],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[768,768,768,768,768,768,768,768],"isMain":false}]]],"subreferences":["Alnus_glutinosa_Chromosome","Amborella_trichopoda_Chloroplast","Carex_acutiformis_Chromosome","Carex_acutiformis_Mitochondrion","Carex_echinata_Chloroplast","Carex_echinata_Chromosome","Carex_echinata_Mitochondrion","Carex_elata_Chromosome","Carex_elata_Mitochondrion","Carex_myosuroides_Chromosome","Carex_nigra_Chloroplast","Carex_nigra_Chromosome","Carex_nigra_Mitochondrion","Carex_rostrata_Chloroplast","Carex_rostrata_Chromosome","Carex_rostrata_Mitochondrion","Carex_sylvatica_Chloroplast","Carex_sylvatica_Chromosome","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Carica_papaya_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chloroplast","Dryas_octopetala_Chromosome","Dryas_octopetala_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Eriophorum_vaginatum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Ipomoea_triloba_Chromosome","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Chromosome","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chloroplast","Populus_trichocarpa_Chromosome","Salix_caprea_Chloroplast","Salix_caprea_Chromosome","Salix_caprea_Mitochondrion","Salix_reticulata_Chloroplast","Salix_reticulata_Chromosome","Salix_reticulata_Mitochondrion","Scirpus_sylvaticus_Chromosome","Scirpus_sylvaticus_Mitochondrion","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Chromosome","Triglochin_maritima_Mitochondrion","Vaccinium_myrtillus_Chromosome","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion"],"subreports":[[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,2,0,8,4],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,2,0,8,4],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,1,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,1,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,4,7],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,4,5],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,2,0,1,4,11],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,1,0,0,1,3],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0,1,3,7],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,1,0,2,3],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,1,0,2,3],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,5,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,5,2],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,0,0,1,7],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,0,0,0,1,7],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,2,0,2,1,10,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,0,0,0,3,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,0,2,1,7,2],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,2,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,1,0,1,11,29],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,1,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,6],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,1,0,1,9,22],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,4,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,4,1],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,1,0,3,8],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,0,1,0,3,8],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,6,3,2,3,7,7,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,6,3,2,3,7,7,1],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,5,17,5,12,203,355],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,4,7,1,1,47,68],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,1,6,3,9,126,204],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,2,1],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,4,1,2,28,82],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,1,2,0,0,15,39],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,3,7,0,0,30,27],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,2,0,0,7,6],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,4],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,2,5,0,0,22,16],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,2,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,1,1,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,1,1,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,0,0,0,0,1,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,0,0,0,1,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,1,0,1,1,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0,1,1,1],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,3,0,0,12,13],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,3,0,0,10,11],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,1,0,0,4,3],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0,0,4,3],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,0,1,0,1,2,17],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,1,0,1,2,15],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,4,1,1,5,6,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,4,1,1,5,6,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,2,8,1,9,97,175],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,3,0,0,14,21],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,1,1,0,3,24,30],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,1,4,1,6,59,124],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,1,1,0,0,8,12],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,2,1,0,0,9,15],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,1,2],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,2,1,0,0,7,13],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,3,0,1,1,3,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,3,0,1,1,3,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,3,3,2,8,103,176],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,0,1,1,16,22],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,1,2,0,1,36,39],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,1,1,1,6,51,115],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,2,2,2,2,8,12],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,2,0,0,10,14],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,3,6],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,2,0,0,7,8],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,2,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,1,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,1,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,1,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,0,1,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,2,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,8],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,7],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,1,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Unaligned",[]],["Mismatches",[]],["Statistics without reference",[]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]]],"minContig":500},{"date":"17 September 2025, Wednesday, 15:12:40","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Alnus_glutinosa_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.001","0.001","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.998","1.000","0.998","1.012","2.515"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[3393,732,3393,3393,235],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5244,2357,3993,6439,235],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[1851,null,3393,1851,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,null,1,2,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,1,1,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[1,0,1,1,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,0,1,1,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[7920,0,6478,7908,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,1278],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,2,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2676,2407,2485,3428,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[152,61,126,207,9],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[16,0,15,16,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[22,0,21,22,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2894.14","2588.04","3150.00","3210.80","3829.79"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["304.65","0.00","375.00","248.18","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[16,0,15,16,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,3,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,2,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,4764,6478,7908,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7920,4764,6478,9950,1869],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7920,4764,6478,9098,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7920,4764,6478,7908,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7920,4764,6478,7908,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.34","38.20","56.85","53.91","39.65"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[456722532,456722532,456722532,456722532,456722532],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[14,14,14,14,14],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["25.86","25.86","25.86","25.86","25.86"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:12:50","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_10mil_long"],"referenceName":"Amborella_trichopoda_Chloroplast","order":[0,1,2],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.119","2.396","1.063"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["3.912","1.248","1.002"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[102,3806,612],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[193,3908,1729],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,3806,530],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1,3],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,707,666],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[7,212,39],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,9,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,57,2],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3626.94","5438.69","2255.64"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","230.89","115.67"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,6,2],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,3,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,4764,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,5573,2399],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,4764,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,4764,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","38.02","43.23"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[162686,162686,162686],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["38.34","38.34","38.34"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:16:50","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_acutiformis_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.001","0.001","0.005","0.008"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.053","1.052","1.172","1.220","1.140"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1814,1028,825,2082,1779],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[7061,4965,2701,21043,31578],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[230,637,null,250,643],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,541,null,null,179],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[11,4,null,32,22],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,6,null,null,46],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,1,5,3],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,1,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,1,0,4,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,2,8,4],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,1,1,3,3],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,525,614,2427,3368],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,1,2,2],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,668],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[2,0,2,4,4],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[6276,0,3709,14821,3875],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[174,74,76,597,867],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[12,5,2,31,25],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[18,9,10,160,58],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2462.15","1488.33","2813.77","2908.93","2746.19"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["169.80","100.56","74.05","151.05","79.19"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[12,5,2,23,24],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,8,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[8,8,4,29,50],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,1,1,7,10],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,0,2,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,1,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,1038,4764,10143,1779],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[13719,5233,6875,39867,40532],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[10161,1038,4764,25845,12806],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,0,18051,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,10143,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7920,643,4764,2847,830],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,542,809,745,658],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,4,1,3,19],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,6,2,12,33],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["45.39","33.90","37.11","38.40","38.82"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[388193004,388193004,388193004,388193004,388193004],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[39,39,39,39,39],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["22.87","22.87","22.87","22.87","22.87"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:16:58","assembliesNames":["megahit_1mil_long","megahit_10mil_long"],"referenceName":"Carex_acutiformis_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.011","0.056"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.007","1.853"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[151,529],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[151,756],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,227],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4613,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,24],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2666.67","3174.60"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["666.67","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,882],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,1401],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,882],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,519],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","45.68"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1353240,1353240],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[5,5],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.18","41.18"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:17:12","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_echinata_Chloroplast","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.787","2.569","0.849","0.270","8.048","9.661"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.126","1.011","1.379","1.222","1.059","1.025"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1137,1806,825,477,1932,1773],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5098,4629,1521,477,14548,17347],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[563,944,null,477,703,764],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,852,null,477,478,686],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[5,2,null,1,9,9],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,3,null,1,16,15],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,1,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,1,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,1,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,810,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,2,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,0,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2115,0,3476,0,1993,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[51,41,48,18,287,269],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,3,0,1,20,17],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,4,0,6,105,53],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1021.84","891.30","3155.82","3726.71","1991.40","1554.82"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["20.04","65.22","0.00","207.04","138.77","98.26"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,3,0,0,13,14],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,1,7,3],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[8,5,2,1,18,21],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,1,1,0,3,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1806,4764,590,2847,1779],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7737,4650,5573,590,17261,17728],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[3379,1806,4764,0,6118,5286],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[984,945,4764,590,871,902],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,853,4764,590,703,700],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[3,2,1,1,6,8],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[5,3,1,1,12,14],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["33.05","33.66","38.02","34.24","33.86","36.36"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[179086,179086,179086,179086,179086,179086],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["34.20","34.20","34.20","34.20","34.20","34.20"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:21:06","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_echinata_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.000","0.002","0.004"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.225","1.004","1.455","1.231","1.238","1.225"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1137,1027,581,533,797,1101],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[2350,3358,1273,925,9120,15128],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[984,944,null,392,417,226],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[135,852,null,392,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,2,null,2,12,23],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[3,3,null,2,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,3,3],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,2,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,1,3],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,4,5],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,3,3],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,1927,2085],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,1,0,1,2],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1,3],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,631,2791],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,1,6],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,4301,0,1997,5155],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[22,63,32,28,141,330],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,3,3,3,8,17],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[3,7,3,9,20,80],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["936.57","1874.44","2511.77","3023.76","1556.64","2190.51"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["85.14","89.26","235.48","323.97","88.32","112.84"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,3,3,2,6,14],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,1,2,3],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,4,3,2,18,36],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,0,1,5],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1138,1038,4764,590,2847,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[2877,3375,6155,1140,13838,26406],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[1138,1038,4764,0,2847,6393],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[984,945,4764,590,737,706],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,853,4764,550,577,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,2,1,1,6,14],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,3,1,2,12,24],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["36.36","34.07","38.51","36.75","38.36","40.87"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[378389622,378389622,378389622,378389622,378389622,378389622],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[29,29,29,29,29,29],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["21.94","21.94","21.94","21.94","21.94","21.94"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:21:16","assembliesNames":["megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_echinata_Mitochondrion","order":[0,1,2],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.009","0.065","0.011"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.007","1.676","3.195"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[151,459,116],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[151,1057,185],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4613,561,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,36,8],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2666.67","3405.87","4324.32"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["666.67","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,3,1],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,932,591],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,2333,591],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,882,591],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,882,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,2,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2,1],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","42.01","50.93"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1637334,1637334,1637334],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.31","41.31","41.31"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:25:28","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_elata_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.001","0.000","0.004","0.011"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.145","1.088","1.165","1.429","1.266","1.142"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[694,1030,624,535,1339,1272],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[2202,3579,2296,1143,16296,46780],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[532,602,null,500,518,617],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[253,502,null,null,null,471],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[3,3,null,2,18,33],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[4,5,null,null,null,58],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,2,6],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,2,5],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,1,3,7],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,1,5],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,560,3210],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,1,2,0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,3405,0,2103,584],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[28,53,60,35,335,1065],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,1,0,4,17,54],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,1,0,7,42,141],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1271.57","1480.45","2613.24","3048.78","2055.72","2275.30"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","27.93","0.00","348.43","104.32","115.37"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,1,0,4,15,52],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,2,2],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[4,6,3,3,32,78],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,0,2,5],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,1038,4764,583,2847,1290],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[2521,3896,6080,1640,22729,54041],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,1038,4764,0,4186,6192],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[694,603,4764,550,610,673],"isMain":false},{"metricName":"N75","quality":"More is better","values":[539,525,4764,507,543,573],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,3,1,2,12,31],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,5,1,3,22,53],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["37.29","35.78","38.17","37.20","38.87","39.15"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[407598430,407598430,407598430,407598430,407598430,407598430],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[40,40,40,40,40,40],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.07","23.07","23.07","23.07","23.07","23.07"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:25:36","assembliesNames":["megahit_1mil_long","megahit_10mil_long"],"referenceName":"Carex_elata_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.011","0.057"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.007","1.853"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[151,529],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[151,756],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,227],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4613,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,23],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2666.67","3042.33"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["666.67","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,882],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,1401],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,882],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,519],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","45.68"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1324694,1324694],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[4,4],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.28","41.28"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:29:40","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_myosuroides_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.004","0.006"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.176","1.002","1.426","1.094","1.141"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1137,1806,820,1932,2230],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[3957,3886,1468,14413,23674],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[598,944,null,580,750],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,637,null,116,107],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[4,2,null,9,15],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,3,null,23,37],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,3,4],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,3,4],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,2,3],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,2,4],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,2530,3604],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,2,1,2],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,631,853],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,2,5],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2124,0,3479,2795,3092],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[31,39,45,268,535],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,2,0,19,33],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,2,0,124,106],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["804.99","1003.60","3065.40","1875.70","2253.58"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["25.97","51.47","0.00","132.98","139.01"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,2,0,10,27],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,9,6],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[6,4,2,19,36],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,1,1,6,7],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1806,4764,2847,2231],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[6654,3894,5573,19054,31022],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[3379,1806,4764,10451,9768],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[1138,945,4764,1012,851],"isMain":false},{"metricName":"N75","quality":"More is better","values":[938,643,4764,669,700],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,2,1,6,14],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[4,3,1,11,24],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["33.39","36.85","38.02","36.12","38.20"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[400001627,400001627,400001627,400001627,400001627],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[29,29,29,29,29],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.06","23.06","23.06","23.06","23.06"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:29:53","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_nigra_Chloroplast","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["5.562","6.211","0.971","14.786","22.924"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.045","1.008","1.200","1.042","1.019"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1137,1806,825,2407,5442],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[11608,12905,2005,30660,47684],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[601,756,null,739,1118],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[519,615,null,572,814],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[8,7,null,15,14],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[14,12,null,28,26],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,5,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,5,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,5,2],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,5,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,4025,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2124,0,3166,1680,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[83,79,57,379,461],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,7,0,34,55],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[22,30,0,185,203],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["722.93","616.13","2842.89","1241.65","974.16"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["17.42","54.59","0.00","111.39","116.22"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,4,0,24,47],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[1,3,0,10,8],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[17,17,2,36,44],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[4,1,1,10,18],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,1],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1806,4764,2847,5481],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[14117,12922,5573,33495,48209],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[5561,1806,4764,15853,29736],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,5481],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[938,756,4764,932,1242],"isMain":false},{"metricName":"N75","quality":"More is better","values":[598,616,4764,693,877],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[6,7,1,11,13],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[11,12,1,22,25],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["32.10","32.02","38.02","33.05","35.65"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[206436,206436,206436,206436,206436],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["34.15","34.15","34.15","34.15","34.15"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:34:12","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_nigra_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.001","0.000","0.000","0.004","0.012"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.242","1.067","1.375","1.014","1.180","1.120"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[539,1027,500,500,951,1301],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1580,4758,1092,500,15930,51208],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,560,null,500,506,605],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,548,null,500,null,477],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,4,null,1,18,38],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,6,null,1,null,65],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,3,5],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,3,5],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,0,0,1,7],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,2,5],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,1742,3769],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1,2],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,602,1170],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,0,1,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2106,0,4579,0,2053,1163],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[27,96,32,15,316,1326],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,5,0,0,20,72],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,78,0,0,58,163],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1708.86","2022.76","2930.40","3000.00","1983.43","2590.10"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","105.35","0.00","0.00","125.53","140.64"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,2,0,0,17,68],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,3,0,0,3,4],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[4,8,3,1,31,86],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,0,1,5],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1038,4764,507,2847,1337],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4068,5063,6080,507,21462,59689],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[2241,1038,4764,0,2847,6337],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2241,603,4764,507,606,673],"isMain":false},{"metricName":"N75","quality":"More is better","values":[539,559,4764,507,543,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,4,1,1,12,35],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,6,1,1,21,59],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["33.38","33.70","38.17","40.04","38.02","39.37"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[416690608,416690608,416690608,416690608,416690608,416690608],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[43,43,43,43,43,43],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.20","23.20","23.20","23.20","23.20","23.20"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:34:17","assembliesNames":["megahit_10mil_long"],"referenceName":"Carex_nigra_Mitochondrion","order":[0],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.052"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.848"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[531],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[758],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[227],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[18],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2374.67"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[882],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1401],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[882],"isMain":false},{"metricName":"N75","quality":"More is better","values":[519],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["45.68"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1469559],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.32"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:34:33","assembliesNames":["spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_rostrata_Chloroplast","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.356","8.997","9.376","1.838","1.028","25.914","39.951"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.026","1.009","1.044","1.189","1.043","1.027"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[786,1882,1806,825,974,3567,5475],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[786,20000,20813,4066,2277,57644,89882],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[786,668,691,310,552,858,1250],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[786,521,565,null,532,572,797],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,12,12,7,2,24,24],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,22,20,null,3,48,47],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,0,1,0,5,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,1,0,1,0,5,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,2,1,7,2],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,1,0,1,0,5,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,587,0,614,0,6527,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,2,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,0,1,0,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,1952,0,3070,0,4184,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[0,80,37,82,23,380,527],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,8,6,2,2,49,56],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,25,20,6,11,330,309],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["0.00","402.43","178.61","2019.70","1012.32","663.66","597.01"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","40.24","28.96","49.26","88.03","85.58","63.44"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,7,5,2,1,32,38],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,1,0,1,17,18],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,28,29,5,4,63,82],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,5,2,1,0,20,34],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,1],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[786,2241,1806,4764,975,4483,5481],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[786,22348,20899,7307,2702,63890,90620],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,7443,2846,4764,0,34531,57665],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,5481],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[786,738,692,4764,590,1082,1272],"isMain":false},{"metricName":"N75","quality":"More is better","values":[786,594,566,809,584,734,816],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,10,12,1,2,18,24],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,18,20,2,3,36,46],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["34.86","30.95","31.33","37.21","34.72","32.68","33.54"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[220951,220951,220951,220951,220951,220951,220951],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["34.04","34.04","34.04","34.04","34.04","34.04","34.04"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:38:27","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_rostrata_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.003","0.000","0.000","0.010","0.023"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.076","1.023","1.085","1.455","1.149","1.104"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[838,1028,552,255,1479,1782],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[7585,9749,1616,367,37556,85721],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[530,648,null,112,519,660],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,565,null,null,231,457],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[8,7,null,2,37,55],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,11,null,null,67,99],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,1,1,10,11],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,3,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,1,1,1,7,10],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,1,0,1,9,22],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,1,1,1,8,10],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,525,809,534,8845,8498],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1,5],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,591,3804],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,0,2,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2172,0,4400,0,4822,1094],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[61,139,30,17,558,1589],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[7,8,5,0,48,103],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[15,13,13,0,225,259],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["805.71","1424.76","1869.16","4632.15","1489.51","1856.13"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["92.46","82.00","311.53","0.00","128.13","120.32"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[7,8,5,0,43,92],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,5,11],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[14,15,3,1,65,133],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,0,7,18],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1038,4764,534,4483,1783],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[10322,9978,6141,534,48461,99439],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[2241,1038,4764,0,12791,22434],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[694,648,4764,534,631,768],"isMain":false},{"metricName":"N75","quality":"More is better","values":[546,566,4764,534,559,598],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,7,1,1,22,50],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[10,11,1,1,43,88],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["32.81","35.72","37.21","38.39","36.54","38.09"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[376029344,376029344,376029344,376029344,376029344,376029344],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[35,35,35,35,35,35],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["22.57","22.57","22.57","22.57","22.57","22.57"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:38:35","assembliesNames":["megahit_1mil_long","megahit_10mil_long"],"referenceName":"Carex_rostrata_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.009","0.048"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.007","1.853"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[151,529],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[151,756],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,227],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,2],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4613,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,21],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2666.67","2777.78"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["666.67","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,882],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,1401],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,882],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,519],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","45.68"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1581338,1581338],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[4,4],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.23","41.23"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:38:48","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_sylvatica_Chloroplast","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["4.307","4.958","1.287","14.344","18.050"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.053","1.007","1.146","1.043","1.025"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1137,1806,825,3172,2803],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[9372,10708,2769,30959,39580],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[601,795,null,768,928],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[374,566,null,572,694],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[7,5,null,14,14],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[13,10,null,27,26],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,5,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,5,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,4,1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,5,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,6664,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2124,0,2918,1927,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[64,87,71,420,432],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,4,0,33,39],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,14,0,190,225],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["690.77","815.60","2564.10","1361.03","1112.46"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["21.59","37.50","0.00","106.94","100.43"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,3,0,20,28],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,13,11],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[14,14,3,35,41],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[3,2,1,9,13],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1806,4764,3903,2803],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[11881,10737,6090,34120,39821],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4487,2861,4764,15987,19692],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[938,795,4764,932,987],"isMain":false},{"metricName":"N75","quality":"More is better","values":[601,566,4764,747,751],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,5,1,11,14],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[9,10,1,21,25],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["32.75","32.53","37.37","32.44","35.22"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[215139,215139,215139,215139,215139],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["33.13","33.13","33.13","33.13","33.13"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:42:32","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_sylvatica_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.001","0.005","0.006"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.163","1.067","1.458","1.158","1.242"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[984,1022,407,2211,1351],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[2943,3431,1864,16925,21354],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[132,944,null,513,516],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,494,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[7,2,null,17,23],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,4,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,1,0,5,3],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[1,1,0,5,3],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,1,3,8],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,1,0,4,3],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[527,525,0,6198,2488],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,3,2],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,1696,1066],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,2,1,4],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2124,0,4225,1750,3450],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[40,98,56,326,422],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,3,5,24,18],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,11,20,52,34],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1359.16","2860.48","3010.75","1926.60","1977.41"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","87.57","268.82","141.84","84.34"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,2,4,22,17],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,1,2,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[6,5,4,29,44],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,3,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,1038,4764,3903,1519],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[5547,3654,6937,23036,31020],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[2241,1038,4764,7766,5393],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[984,945,4764,604,673],"isMain":false},{"metricName":"N75","quality":"More is better","values":[527,525,809,553,560],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,2,1,9,17],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[4,4,2,18,30],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["33.15","35.99","36.46","35.86","38.61"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[362446279,362446279,362446279,362446279,362446279],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[29,29,29,29,29],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["22.74","22.74","22.74","22.74","22.74"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:42:41","assembliesNames":["megahit_10mil_long","spades_10mil_long"],"referenceName":"Carex_sylvatica_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.051","0.027"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["2.109","1.060"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[529,500],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[944,500],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,500],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,500],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,1],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[28,21],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,18],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2966.10","4200.00"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","200.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,1],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[882,530],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1991,530],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[590,530],"isMain":false},{"metricName":"N75","quality":"More is better","values":[519,530],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,1],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["48.32","46.23"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1867792,1867792],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[3,3],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["40.97","40.97"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:42:55","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","spades_10mil_long"],"referenceName":"Carica_papaya_Chloroplast","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.557","0.575","0.328","3.369","0.051","1.842"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.376","1.011","1.071","6.341","1.005"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,509,525,4392,82,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4094,921,525,5402,82,2949],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[2533,412,525,4392,null,530],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,525,807,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,2,1,1,null,5],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,null,1,2,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,1,0,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,606,0,1847],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[41,104,0,134,3,20],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,2,0,14,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,10,0,63,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1001.47","11292.07","0.00","2484.70","3658.54","678.20"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["48.85","217.16","0.00","259.60","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,1,0,10,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,4,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,2,1,3,1,6],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,0,0,1,0,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,755,531,4764,520,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4094,1267,531,6383,520,4811],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4094,0,0,4764,0,1918],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2533,755,531,4764,520,647],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,512,531,810,520,572],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,2,1,2,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.40","38.99","58.38","38.12","25.96","49.10"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[160100,160100,160100,160100,160100,160100],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.89","36.89","36.89","36.89","36.89","36.89"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:43:08","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Carica_papaya_Mitochondrion","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.333","0.346","0.748","0.265","0.233"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.006","1.006","1.336","1.348","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[923,1065,1884,645,809],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1587,1644,3571,1265,1113],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[923,1065,1687,508,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[664,579,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,2,2,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,2,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,2,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,1,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,4764,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,2,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,1,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,553,2637],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[23,30,177,93,30],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[3,4,7,2,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[3,6,24,12,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1450.19","1820.39","4962.15","7351.78","2695.42"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["189.16","242.72","196.24","158.10","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,4,5,1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,2,1,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,2,1,3,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[932,1068,4764,882,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1596,1658,4764,2258,3750],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,1068,4764,0,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[932,1068,4764,711,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[664,590,4764,665,1438],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,2,1,3,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["50.38","49.34","38.20","44.20","43.41"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[476890,476890,476890,476890,476890],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["45.12","45.12","45.12","45.12","45.12"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:43:21","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_10mil_long"],"referenceName":"Chamaenerion_angustifolium_Chloroplast","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.552",null,"0.474","2.493","1.471"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000",null,"1.000","1.222","1.002"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,null,761,3872,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4094,null,761,3993,2359],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[2533,null,761,3872,587],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,761,null,572],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,1,1,2],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,null,1,null,3],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,null,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,null,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,null,0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,null,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,null,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,1,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,755,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,null,0,1,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,null,0,688,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[57,null,9,152,24],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,null,2,7,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,null,2,22,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1392.28",null,"1182.65","3800.95","1017.38"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["48.85",null,"262.81","175.04","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,null,2,6,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,null,0,1,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,1,2,4],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,0,0,1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,755,761,4764,647],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4094,755,761,5573,2363],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4094,0,0,4764,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2533,755,761,4764,591],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,755,761,4764,572],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,1,3],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.40","39.34","49.54","38.02","54.25"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[160412,160412,160412,160412,160412],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["38.18","38.18","38.18","38.18","38.18"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:49:45","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Chamaenerion_angustifolium_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.001","0.001","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","0.998","0.999","0.999","1.006"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5211,3633,3968,5211,587],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5211,3633,3968,5211,691],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5211,3633,3968,5211,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,3633,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,1,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,1,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1,1,1,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2709,1131,2510,2697,1486],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[183,140,156,182,20],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[20,8,18,20,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[26,24,24,26,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3507.76","3846.15","3927.49","3488.59","2894.36"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["383.36","219.78","453.17","383.36","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[20,7,18,20,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,4764,6478,7908,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7920,4764,6478,7908,2181],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7920,4764,6478,7908,1590],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7920,4764,6478,7908,1590],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7920,4764,6478,7908,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.34","38.20","56.85","56.99","43.33"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[633522186,633522186,633522186,633522186,633522186],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[18,18,18,18,18],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["20.67","20.67","20.67","20.67","20.67"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:49:58","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Chamaenerion_angustifolium_Mitochondrion","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.112","0.141","0.024","0.381","0.584"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.001","1.000","1.264","1.002"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[555,697,121,810,809],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[555,697,121,1887,2893],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,697,null,309,601],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,697,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1,null,3,4],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,1,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[1,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[755,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,1,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,688,881,2116],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,4,5,36,44],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["720.72","573.89","4132.23","1907.79","1520.91"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,1,4,5],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0,0,1,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,698,809,1190,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1310,698,809,3267,5015],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0,0,1190,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,698,809,811,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[555,698,809,711,737],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,3,3],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["46.41","55.59","36.96","46.46","46.20"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[495176,495176,495176,495176,495176],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["46.43","46.43","46.43","46.43","46.43"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:50:12","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Citrus_sinensis_Chloroplast","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.557","0.064","0.332","2.807","0.168","1.830"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","7.330","1.000","1.083","1.000","1.001"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,103,531,4392,269,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4094,103,531,4495,269,2930],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[2533,null,531,4392,null,587],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,531,4392,null,530],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,1,1,null,3],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,null,1,1,null,5],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,1,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,706,921,665],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[45,5,2,119,8,48],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[4,0,0,16,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[4,0,0,76,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1099.17","4854.37","376.65","2647.39","2973.98","1638.23"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["97.70","0.00","0.00","355.95","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[4,0,0,12,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,4,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,1,2,1,5],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,0,0,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,755,531,4764,1190,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4094,755,531,5573,1190,3599],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4094,0,0,4764,1190,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2533,755,531,4764,1190,647],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,755,531,4764,1190,553],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.40","39.34","58.38","38.02","43.45","47.24"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[160129,160129,160129,160129,160129,160129],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["38.48","38.48","38.48","38.48","38.48","38.48"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:53:21","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Citrus_sinensis_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.002","0.000","0.001","0.002","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","0.999","1.000","0.999","1.059","1.850"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5351,5351,483,4103,5351,350],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5351,5351,1013,4103,5983,1173],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5351,5351,null,4103,5351,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,null,1,1,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[2,2,0,2,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,1,2,4],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2457,2569,3751,2375,3469,4958],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[141,141,37,119,160,28],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[16,16,0,15,16,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[22,22,0,21,22,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2633.55","2633.55","3652.52","2898.20","2672.90","2387.04"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["298.84","298.84","0.00","365.32","267.29","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[16,16,0,15,16,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,1,3,7],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,2,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,4764,6478,7908,1597],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,7920,4764,6478,9809,7128],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,4764,6478,9098,5422],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,4764,6478,7908,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,4764,6478,7908,1109],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","56.34","38.20","56.85","54.79","51.19"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[298977511,298977511,298977511,298977511,298977511,298977511],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[9,9,9,9,9,9],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["25.93","25.93","25.93","25.93","25.93","25.93"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 15:53:35","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Citrus_sinensis_Mitochondrion","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.243","0.160","0.083","0.377","0.200","0.413"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.001","1.644","1.000","0.998","1.220","1.002"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1561,923,531,1248,849,813],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1561,1026,531,2414,1279,2646],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[1561,923,531,null,430,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,531,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,null,2,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,null,1,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,2,1,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,3160,777,2777],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[8,18,2,58,154,69],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,2,0,6,2,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,2,0,23,13,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["512.82","1754.39","376.65","2399.67","12040.66","2607.71"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["64.10","194.93","0.00","248.24","156.37","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,2,0,5,1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,1,1,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,2,2,5],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,0,1,1,3],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1561,932,531,4764,1626,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1561,1687,531,5573,2337,5427],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[1561,0,0,4764,1626,4306],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[1561,932,531,4764,1626,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,755,531,4764,711,1278],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2,1,1,2,3],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.80","44.87","58.38","38.02","43.05","43.03"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[640906,640906,640906,640906,640906,640906],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["43.89","43.89","43.89","43.89","43.89","43.89"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:23:00","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Diphasiastrum_alpinum_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.000","1.000","1.000","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[421,421,421,421,87],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[919,919,824,919,87],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,1,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,1109],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[6889,7001,5654,6989,513],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[34,34,31,34,3],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3699.67","3699.67","3762.14","3699.67","3448.28"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,6478,7908,1109],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,7920,6478,7908,1709],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,6478,7908,1109],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,6478,7908,1109],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,6478,7908,600],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","56.34","56.85","56.99","52.95"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[3005336282,3005336282,3005336282,3005336282,3005336282],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[23,23,23,23,23],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["13.02","13.02","13.02","13.02","13.02"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:23:18","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Dryas_octopetala_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["7.729","8.420","53.449","44.169","46.060","42.435","71.547","76.820"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.040","1.195","1.022","1.012","1.024","1.032","1.044","1.007"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,1958,4468,3832,4389,1677,9143,18654],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[12231,13376,85849,70408,73665,67722,117415,122318],"isMain":true},{"metricName":"NG50","quality":"More is better","values":[null,null,616,null,null,null,5359,9587],"isMain":false},{"metricName":"NG75","quality":"More is better","values":[null,null,null,null,null,null,1497,698],"isMain":false},{"metricName":"NA50","quality":"More is better","values":[657,681,1137,930,903,838,3892,11621],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[533,311,731,690,653,658,1773,4621],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,558,null,null,null,2690,9587],"isMain":true},{"metricName":"NGA75","quality":"More is better","values":[null,null,null,null,null,null,null,620],"isMain":false},{"metricName":"LG50","quality":"Less is better","values":[null,null,63,null,null,null,11,6],"isMain":false},{"metricName":"LG75","quality":"Less is better","values":[null,null,null,null,null,null,27,23],"isMain":false},{"metricName":"LA50","quality":"Less is better","values":[5,9,25,25,28,32,10,4],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[10,16,49,47,53,56,23,8],"isMain":false},{"metricName":"LGA50","quality":"Less is better","values":[null,null,69,null,null,null,14,6],"isMain":true},{"metricName":"LGA75","quality":"Less is better","values":[null,null,null,null,null,null,null,24],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,6,3,2,3,7,7,1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,3,0,0,1,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,0,0,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,1,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,611,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,3,1,2,2,5,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,3287,949,1379,1206,13356,1666],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[145,333,218,86,365,447,389,36],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[9,9,10,8,26,13,20,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[25,23,15,17,118,51,82,2],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1186.00","2500.19","257.85","123.09","500.99","665.94","343.72","29.63"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["73.61","67.57","11.83","11.45","35.69","19.37","17.67","0.82"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[9,8,10,7,21,9,15,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,1,5,4,5,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[16,20,82,75,83,82,41,31],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[3,4,31,22,20,22,33,19],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,11,7],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,4],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,1960,4468,3834,4764,2076,9149,18654],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[12712,16523,89650,71653,75969,70463,131553,123996],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[5113,5416,53850,34780,31175,28314,126287,116073],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,81951,92155],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,63661],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[665,813,1290,958,917,869,6353,11621],"isMain":false},{"metricName":"N75","quality":"More is better","values":[538,644,799,696,688,708,2614,4622],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,8,23,24,28,30,9,4],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[10,13,46,46,51,53,17,8],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["42.86","38.93","32.28","31.69","33.32","33.21","34.11","34.73"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[158178,158178,158178,158178,158178,158178,158178,158178],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.48","36.48","36.48","36.48","36.48","36.48","36.48","36.48"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:26:00","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Dryas_octopetala_Chromosome","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.003","0.004","0.064","0.099","0.013","0.033","0.463","0.652"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.001","1.010","1.005","1.011","1.019","1.028","1.024","1.035"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5361,874,5361,4742,1412,4110,5361,6316],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[7109,8637,146017,226058,29644,75677,1062285,1495968],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5361,727,714,718,553,717,639,686],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,664,547,564,482,547,538,543],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,6,66,104,24,37,580,741],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,9,128,197,40,70,1050,1389],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,4,13,2,7,147,237],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,3,7,1,1,34,58],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[1,0,1,6,1,6,111,177],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,4,1,2,28,82],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,0,4,12,2,7,141,217],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[539,0,2763,8103,1049,4772,99930,164930],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[2,0,2,1,0,3,10,23],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,1,0,0,6],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[1,0,0,0,1,1,3,3],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[738,0,0,0,665,610,1996,1756],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,3,1,3,2,4,11],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2447,0,5230,1068,4079,3788,5198,9431],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[145,150,1135,2106,461,1225,11881,17073],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[12,5,79,138,16,95,1096,1689],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[18,7,342,368,49,259,4462,5992],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2037.95","1737.72","777.05","931.93","1554.75","1618.59","1118.95","1141.92"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["168.66","57.92","54.09","61.07","53.96","125.52","103.22","112.97"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[12,5,68,130,13,88,928,1482],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,11,8,3,7,168,207],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[5,12,190,298,47,99,1513,2020],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,33,43,3,16,169,313],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,0,0,1,1,2],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,895,7920,4743,4764,6478,7908,6316],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[10308,8719,152072,229576,34955,82209,1094226,1558491],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,0,52955,65213,7177,27342,244577,454148],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,0,7920,0,0,6478,7908,11416],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,728,771,735,680,814,681,740],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,666,574,577,553,592,564,587],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,6,60,102,16,32,551,699],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,9,119,191,31,62,997,1298],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["53.73","43.49","39.97","38.85","42.14","43.01","37.78","38.02"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[229255605,229255605,229255605,229255605,229255605,229255605,229255605,229255605],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[9,9,9,9,9,9,9,9],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["28.68","28.68","28.68","28.68","28.68","28.68","28.68","28.68"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:26:15","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Dryas_octopetala_Mitochondrion","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["8.845","14.430","0.144","0.191","67.970","69.979"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.041","1.031","1.000","0.995","1.036","1.028"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1165,1568,354,647,3647,3651],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[30676,50123,500,647,238634,244945],"isMain":true},{"metricName":"NG50","quality":"More is better","values":[null,null,null,null,775,850],"isMain":false},{"metricName":"NA50","quality":"More is better","values":[602,614,null,647,927,1022],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[525,541,null,647,677,694],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,727,774],"isMain":true},{"metricName":"LG50","quality":"Less is better","values":[null,null,null,null,143,127],"isMain":false},{"metricName":"LA50","quality":"Less is better","values":[22,34,null,1,90,85],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[36,56,null,1,169,161],"isMain":false},{"metricName":"LGA50","quality":"Less is better","values":[null,null,null,null,151,137],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,2,0,0,10,11],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,2,0,0,8,6],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,2,4],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[2,5,0,0,22,16],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,2,0,0,10,11],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,1327,0,0,11026,11820],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[2,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,2,0,4,6],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,5073,0,3277,4727],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[120,271,15,4,464,475],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[19,24,0,1,81,91],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[63,98,0,16,306,400],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["391.47","541.87","3000.00","603.32","196.97","195.85"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["61.98","47.99","0.00","150.83","34.38","37.52"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[15,19,0,0,59,65],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[4,5,0,1,22,26],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[50,78,2,1,261,251],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,3,1,0,82,100],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1166,1572,4764,660,3648,3653],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[31919,51560,5573,660,247427,254125],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[2314,3798,4764,0,120856,148438],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[616,657,4764,660,988,1088],"isMain":false},{"metricName":"N75","quality":"More is better","values":[536,557,4764,660,729,773],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[21,33,1,1,85,80],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[35,54,1,1,159,149],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["46.43","46.37","38.02","41.97","45.56","45.57"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[346574,346574,346574,346574,346574,346574],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[2,2,2,2,2,2],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["45.70","45.70","45.70","45.70","45.70","45.70"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:26:26","assembliesNames":["megahit_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Eleocharis_quinqueflora_Chloroplast","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.045","0.832","0.222","0.532"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["8.297","1.000","0.998","1.172"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[91,818,204,668],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[91,1679,448,1074],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,null,406],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,null,2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,2,3,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,3894,4141,610],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[2,68,15,91],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,1,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,1,13],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2197.80","4050.03","3340.76","8473.00"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","222.72","93.11"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,3,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,4764,2847,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,5573,4589,1869],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,4764,2847,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,4764,2847,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,932,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,2,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","38.02","34.39","39.65"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[201900,201900,201900,201900],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["32.26","32.26","32.26","32.26"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:31:32","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Eleocharis_quinqueflora_Chromosome","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.000","0.001","0.001","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.159","1.000","1.000","1.000","1.058","1.612"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[4080,4080,105,818,884,4080,668],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4080,4171,249,1058,2838,5244,1301],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[4080,null,null,null,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,null,null,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,1,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[4,4,0,0,0,4,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,1,1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,1,2],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,532,1843],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,2,1,4,4],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[3728,3840,789,4515,3640,8007,3621],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[166,169,10,49,102,209,35],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[10,10,0,0,1,14,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[16,16,0,0,1,21,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["4066.63","4049.84","4016.06","4631.38","3592.81","3983.23","2690.24"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["244.98","239.64","0.00","0.00","35.22","266.82","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[10,10,0,0,1,14,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,2,1,7,8],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1,2,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,1597],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,8675,1038,5573,6478,14089,7561],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,1038,4764,6478,10755,5229],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,1245],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,1038,4764,6478,2847,600],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,1,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,2,5],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","54.86","27.84","38.02","56.85","47.46","48.72"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[495377314,495377314,495377314,495377314,495377314,495377314,495377314],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[67,67,67,67,67,67,67],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["22.39","22.39","22.39","22.39","22.39","22.39","22.39"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:31:43","assembliesNames":["megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Eleocharis_quinqueflora_Mitochondrion","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":[null,null,"0.011","0.043"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":[null,null,"2.286","1.474"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[null,null,227,668],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[null,null,227,854],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[null,null,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[null,null,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[null,null,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[1,1,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[1020,1018,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[null,null,0,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[null,null,0,610],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[null,null,2,27],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":[null,null,"881.06","3161.59"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":[null,null,"0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[null,null,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,0,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1020,1018,519,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1020,1018,519,1869],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[1020,1018,0,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[1020,1018,519,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1020,1018,519,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["31.86","31.93","42.77","39.65"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[2000693,2000693,2000693,2000693],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[3,3,3,3],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.83","41.83","41.83","41.83"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:35:38","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Eriophorum_vaginatum_Chromosome","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.000","0.001","0.001","0.002"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","1.154","1.867","0.994","1.189","1.179","1.415"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1814,1814,555,331,572,1814,1779],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[3655,3820,555,496,2374,5360,6143],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,555,null,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,1,null,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,2],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,1,0,0,1,3,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,1,3],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,537,2757],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0,2,1,3,6],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4153,4265,0,5077,4187,7350,5060],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[117,122,17,14,79,172,103],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[6,8,1,3,1,9,6],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[10,12,1,5,1,13,6],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3199.34","3192.05","3057.55","2805.61","3326.32","3208.36","1676.71"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["164.07","209.31","179.86","601.20","42.11","167.88","97.67"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[6,8,1,3,1,9,6],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,2,2,7,19],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1,2,5],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,1779],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,8675,1038,5573,7012,14206,16508],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,1038,4764,6478,10755,7008],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,823],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,1038,4764,6478,2847,653],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,1,7],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,2,12],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","54.86","27.84","38.02","55.45","47.96","44.11"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[374393803,374393803,374393803,374393803,374393803,374393803,374393803],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[29,29,29,29,29,29,29],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["21.83","21.83","21.83","21.83","21.83","21.83","21.83"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:35:49","assembliesNames":["megahit_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Eriophorum_vaginatum_Mitochondrion","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.006","0.038","0.019","0.123"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["6.345","1.731","1.000","1.206"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[119,415,155,750],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[119,707,355,2276],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,null,669],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,null,null,661],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,null,2],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,null,null,3],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,2,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,4349,3302,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[3,31,13,35],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,0,3],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,0,34],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2521.01","4384.72","3661.97","1541.17"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","132.10"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,2],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,2,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,4764,2847,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,5573,3657,2738],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,4764,2847,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,4764,2847,751],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,2847,709],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,3],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","38.02","33.85","36.60"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1852193,1852193,1852193,1852193],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.28","41.28","41.28","41.28"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:42:18","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_1mil_long","megahit_10mil_long"],"referenceName":"Hydra_vulgaris_Chromosome","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.000","1.000","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[66,66,65,66],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[66,66,65,66],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[7742,7854,6413,7842],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[1,1,3,1],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1515.15","1515.15","4615.38","1515.15"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,1],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,6478,7908],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,7920,6478,7908],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,6478,7908],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,6478,7908],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,6478,7908],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,6478,7908],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","56.34","56.85","56.99"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[911675201,911675201,911675201,911675201],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[15,15,15,15],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["18.79","18.79","18.79","18.79"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:42:28","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_10mil_long"],"referenceName":"Ipomoea_triloba_Chloroplast","order":[0,1,2],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.172","2.234","1.115"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["2.716","1.001","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[105,3521,611],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[278,3618,1798],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,3521,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,2,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,1955,2514],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[9,134,43],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,5,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,17,5],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3237.41","3707.80","2384.91"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","138.35","55.46"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,4,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,4],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,4764,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,5573,4317],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,4764,3196],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,4764,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,3],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","38.02","42.18"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[161750,161750,161750],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["37.51","37.51","37.51"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:48:13","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Ipomoea_triloba_Chromosome","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.002","0.000","0.001","0.001","0.002","0.001"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","1.058","0.999","1.001","1.000","1.095","1.279"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5385,5385,584,2075,4138,5385,735],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[6946,6681,1133,3453,4138,7502,4063],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5385,5385,584,1248,4138,810,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,null,549,null,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,2,1,2,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,null,2,null,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,1,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,1,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,1,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,4764,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[2,2,0,0,2,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,0,0,0,2,2],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0,2,1,2,6],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2423,2535,0,2120,2340,3116,7230],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[206,219,14,152,165,259,92],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[19,22,3,5,16,19,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[23,29,9,19,20,23,5],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["2963.60","3277.46","1228.07","4407.07","3985.51","3450.11","2261.55"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["273.34","329.24","263.16","144.97","386.47","253.10","24.58"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[19,22,2,4,16,19,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,1,1,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,3,2,2,1,5,12],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,1,0,1,1,2,6],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,590,4764,6478,7908,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[9369,9607,1139,5573,6478,11339,12431],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[9369,7920,0,4764,6478,8935,8930],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,590,4764,6478,7908,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,549,4764,6478,1027,737],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,1,4],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,2,1,1,2,7],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.79","54.32","46.36","38.02","56.85","53.72","47.94"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[461827428,461827428,461827428,461827428,461827428,461827428,461827428],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[16,16,16,16,16,16,16],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["24.55","24.55","24.55","24.55","24.55","24.55","24.55"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:54:36","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Linum_lewisii_Chromosome","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.000","0.001","0.001","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.113","1.000","1.001","0.977","1.116","1.089","1.424"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5340,5340,697,638,4094,5340,711],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5459,6058,697,2326,4213,6133,3095],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5340,5340,697,null,4094,5340,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,null,697,null,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,null,1,1,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,null,1,null,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[2,2,0,0,2,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0,2],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,1,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,882,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,2,0,1,1,2,6],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2468,3172,0,2496,2384,3114,6867],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[195,206,8,78,159,205,73],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[12,13,0,1,11,12,4],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[14,15,0,5,13,14,7],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3570.77","3398.78","1147.78","3360.62","3773.14","3341.48","2450.49"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["219.74","214.49","0.00","43.08","261.03","195.60","134.27"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[12,13,0,1,11,12,4],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,3,1,1,2,5,11],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,0,1,1,1,5],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,698,4764,6478,7908,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[8546,9230,698,4764,7088,10675,11109],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,0,4764,6478,7908,7492],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,698,4764,6478,7908,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,698,4764,6478,882,617],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,1,4],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,2,7],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.73","54.93","55.59","38.20","56.55","54.40","48.84"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[610693565,610693565,610693565,610693565,610693565,610693565,610693565],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[9,9,9,9,9,9,9],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["21.56","21.56","21.56","21.56","21.56","21.56","21.56"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 16:54:46","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","spades_10mil_long"],"referenceName":"Matricaria_discoidea_Chloroplast","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.734","0.092","2.596","1.952"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.000","1.001","1.004"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,138,3760,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4094,138,3892,2913],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[2533,null,3760,572],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,null,528],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,1,3],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,null,null,5],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,2,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,617,1681,665],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[53,4,242,46],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,0,7,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,0,28,10],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1294.58","2898.55","6225.88","1573.73"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["48.85","0.00","180.09","68.42"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,0,6,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,1,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,2,5],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,0,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,755,4764,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4094,755,5573,3599],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4094,0,4764,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2533,755,4764,647],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,755,4764,553],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.40","39.34","38.02","47.24"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[149751,149751,149751,149751],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["37.46","37.46","37.46","37.46"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:15:27","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Matricaria_discoidea_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000","0.000","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.001","1.002","1.000","0.999","1.195","1.178"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5214,5214,2860,3946,5214,809],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5214,6275,2992,3946,5701,4358],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5214,5214,2860,3946,193,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,1,2,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,1590],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1,1,1,1,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,2,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,1,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,1190,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,2,2,1,1,4],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2594,3323,2581,2532,2694,3864],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[191,206,158,144,209,95],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[14,17,3,11,14,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[24,28,18,12,24,10],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3666.03","3283.92","5282.51","3645.57","3668.60","2174.91"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["268.71","271.00","100.30","278.48","245.74","45.79"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[13,16,2,11,13,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[1,1,1,0,1,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,3,2,1,4,10],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,2,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,4764,6478,7908,1597],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,9607,5573,6478,10691,9010],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,4764,6478,9098,5574],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,4764,6478,7908,1109],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,4764,6478,1190,600],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,4],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,2,6],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","54.32","38.02","56.85","54.18","51.03"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[2082820277,2082820277,2082820277,2082820277,2082820277,2082820277],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[9,9,9,9,9,9],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["15.63","15.63","15.63","15.63","15.63","15.63"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:15:37","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_10mil_long","spades_10mil_long"],"referenceName":"Matricaria_discoidea_Mitochondrion","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.414","0.983","0.134","0.401"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.008","1.005","1.000","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[923,1065,193,809],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[923,2193,299,897],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[923,579,null,809],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[923,549,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,2,null,1],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,3,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,1590],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,2,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1294,693],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[11,37,9,27],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[3,5,0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[4,15,0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1189.19","1684.88","3010.03","3010.03"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["324.32","227.69","0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,4,0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,3,2,1],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,0,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[932,1068,882,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[932,2207,1593,1590],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,1068,0,1590],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[932,590,882,1590],"isMain":false},{"metricName":"N75","quality":"More is better","values":[932,590,711,1590],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,2,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2,2,1],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["49.36","49.89","48.21","40.50"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[223480,223480,223480,223480],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["45.46","45.46","45.46","45.46"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:28:20","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Mentha_aquatica_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.001","1.011","1.000","1.001"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1561,643,3550,107,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1561,749,3550,107,3060],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[1561,106,3550,null,528],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,2,1,null,5],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,null,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,1,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,1190,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,1,1,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,649,1214,775,2123],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[17,13,104,2,52],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,1,14,0,3],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,1,47,0,7],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1089.05","1737.97","2962.12","1869.16","1697.68"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","133.69","398.75","0.00","97.94"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,1,12,0,3],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,2,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,2,6],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,1,1,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1561,755,4764,1190,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1561,1398,4764,2072,5189],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[1561,0,4764,1190,2868],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[1561,755,4764,1190,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,643,4764,882,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2,1,2,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.80","38.91","38.20","45.13","45.17"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1272369154,1272369154,1272369154,1272369154,1272369154],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[48,48,48,48,48],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.71","23.71","23.71","23.71","23.71"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:28:29","assembliesNames":["megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Mentha_aquatica_Mitochondrion","order":[0,1,2],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.138","0.062","0.292"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.012","3.782","1.521"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[424,189,721],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[424,189,889],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4340,0,2204],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[15,8,29],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[3,1,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[5,1,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3579.95","4255.32","3262.09"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["715.99","531.91","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,711,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,711,3556],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,711,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,711,1438],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","49.23","44.21"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[304461,304461,304461],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[2,2,2],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["43.85","43.85","43.85"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:28:42","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Myrica_gale_Chloroplast","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["2.872","0.983","1.219","3.190","0.686","1.883"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.193","1.001","1.159","1.083","1.001"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,563,786,3964,1099,698],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[4599,1575,1955,5122,1099,3015],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[2533,561,639,3964,1099,587],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,451,530,984,1099,553],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,2,2,1,1,3],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[2,3,3,2,1,4],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,2,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,1,0,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,635,0,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[48,45,26,153,48,19],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,2,3,11,0,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,6,8,81,0,1],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1043.71","2857.14","1331.29","2994.72","4367.61","629.97"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["43.49","126.98","153.61","215.31","0.00","33.16"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,2,3,8,0,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,3,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,3,3,3,1,5],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[2,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,755,786,4764,1190,698],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4599,1879,1955,6558,1190,3019],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4094,0,0,4764,1190,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2533,563,639,4764,1190,591],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,561,530,985,1190,553],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,2,2,1,1,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,3,3,2,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["53.49","39.81","45.12","38.17","43.45","50.15"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[160149,160149,160149,160149,160149,160149],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.07","36.07","36.07","36.07","36.07","36.07"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:35:08","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Myrica_gale_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000","0.000"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.003","1.006","1.002","1.001","1.101"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[555,695,670,555,601],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[770,695,1802,906,1248],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[215,695,null,null,179],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,695,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,1,null,null,4],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,1,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,591],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,852,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,2,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[540,0,2962,1550,1099],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[11,3,66,18,23],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[2,1,5,1,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[2,1,9,1,1],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1432.29","432.28","3668.70","1988.95","1844.43"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["260.42","144.09","277.93","110.50","80.19"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[2,1,5,1,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,1,4,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,698,4764,1190,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1310,698,4764,3308,2472],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0,4764,1190,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,698,4764,852,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[555,698,4764,711,603],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,3,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["46.41","55.59","38.20","45.89","43.89"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[643646905,643646905,643646905,643646905,643646905],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[24,24,24,24,24],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["29.24","29.24","29.24","29.24","29.24"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:35:21","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Myrica_gale_Mitochondrion","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.358","0.928","0.516","0.678","1.023"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.286","1.002","1.002","1.153","1.059"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[923,1065,1210,845,1133],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1738,4511,2513,3298,4984],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[555,724,null,652,601],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[70,697,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,3,null,3,5],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[5,4,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,2,0,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,2,1,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,3060,781,2637],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[20,68,62,67,97],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[3,6,2,1,4],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[7,21,5,1,8],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1147.45","1507.09","2472.09","2031.53","1948.57"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["172.12","132.98","79.74","30.32","80.35"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,4,2,1,4],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,2,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,6,2,5,8],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,1,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[932,1068,4764,1626,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[2242,4521,5573,4585,7911],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,1068,4764,1626,5285],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,724,4764,882,1133],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,698,4764,711,773],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,3,1,2,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,4,1,4,5],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["47.64","50.83","38.02","45.26","45.39"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[486427,486427,486427,486427,486427],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["45.39","45.39","45.39","45.39","45.39"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:35:40","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_euphratica_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["10.138","16.728","16.000","10.868","20.444","14.925","17.696","8.351"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.005","1.011","1.040","1.004","1.028","1.033","1.003","1.057"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,3411,3276,2563,4392,2550,8221,2546],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[15900,26209,25072,17088,32542,23611,27811,13836],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[955,1237,1517,786,1493,1017,2744,613],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[636,734,728,589,886,736,null,530],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[5,7,7,7,6,8,5,8],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[11,14,13,14,13,15,null,16],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,0,0,0,1,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,5,0,1,1,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,0,0,4,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,635,0,0,0,14579,2509],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[146,277,257,143,381,325,388,96],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[12,15,13,7,33,14,35,4],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[43,63,54,31,137,60,197,6],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["918.64","1056.28","1024.64","839.35","1188.77","1389.01","1398.60","733.33"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["75.50","57.20","51.83","41.09","102.96","59.83","126.16","30.56"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[9,8,9,5,23,12,20,4],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[3,7,4,2,10,2,15,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[17,24,22,21,24,23,9,19],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[4,9,8,4,11,10,9,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,4,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,3426,3276,2564,4764,2551,8810,2546],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[15973,26501,26713,17097,32936,24166,42393,16345],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7228,15998,16932,6411,23494,14881,42393,6830],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,30431,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[959,1238,1751,786,1493,1022,7919,857],"isMain":false},{"metricName":"N75","quality":"More is better","values":[637,736,812,590,887,772,3377,582],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,7,6,7,6,8,3,6],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[11,14,12,14,13,14,5,12],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["44.19","42.76","40.84","41.46","41.74","40.71","36.41","42.84"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[156766,156766,156766,156766,156766,156766,156766,156766],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.71","36.71","36.71","36.71","36.71","36.71","36.71","36.71"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:35:58","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_tremula_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["9.710","14.347","13.407","9.121","17.610","15.573","18.863","6.280"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.007","1.005","1.073","1.006","1.029","1.026","1.014","1.091"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,3411,3276,2563,4392,2550,8221,860],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[15211,22479,21030,14345,27954,24439,29912,10414],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[955,1237,1308,645,1493,1007,2509,582],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[636,734,655,582,776,707,null,501],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[5,6,6,7,5,9,6,10],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[10,12,12,13,11,17,null,16],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0,1,1,1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,5,0,1,1,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,0,1,2,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,1,0,1,4,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,1178,0,899,14603,2509],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[143,254,175,105,317,334,388,63],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[11,12,10,7,28,19,39,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[40,38,51,29,141,75,237,2],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["941.66","1132.01","834.61","736.07","1151.01","1371.38","1315.25","641.42"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["72.44","53.48","47.69","49.07","101.67","78.01","132.20","20.36"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[8,9,7,5,20,16,22,2],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[3,3,3,2,8,3,17,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[16,21,19,18,21,25,11,18],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[4,7,6,4,8,11,10,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,4,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,3426,3276,2564,4764,2551,8810,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[15289,22548,22494,15530,28348,25893,44518,13228],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7228,12826,13427,6800,19436,15910,43565,3196],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,30431,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[959,1238,1751,808,2143,1022,7919,647],"isMain":false},{"metricName":"N75","quality":"More is better","values":[637,736,812,610,777,738,3377,572],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,6,5,6,4,9,3,7],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[10,12,10,11,11,16,5,12],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["44.42","42.97","40.55","40.79","42.80","39.60","36.59","43.22"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[156393,156393,156393,156393,156393,156393,156393,156393],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.74","36.74","36.74","36.74","36.74","36.74","36.74","36.74"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:40:12","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_tremula_Chromosome","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.001","0.002","0.001","0.004","0.007"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.008","1.000","1.728","1.027","1.151","1.338","1.310"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[570,754,655,848,1697,778,1034,2566],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[570,1968,1706,2902,5950,3117,16463,29062],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[570,682,null,null,1072,507,null,165],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[570,532,null,null,608,473,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,2,null,null,3,3,null,46],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,3,null,null,5,5,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,2,0,3,7],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,1,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,1,0,3,7],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,3,0,0,10,11],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,1,0,3,7],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,4764,0,1804,4516],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,4,0,0,3,5],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,4,0,0,1,4],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,2762,0,0,828,3169],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,2,1,0,12,11],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1751,1773,1048,0,14156,9762],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[2,21,37,73,158,88,470,829],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,5,0,1,12,4,13,40],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,9,0,9,39,10,29,120],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["350.88","1070.88","2168.82","2523.33","2668.47","2823.23","2851.60","2850.17"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","254.97","0.00","34.57","202.67","128.33","78.87","137.52"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,5,0,0,10,4,12,35],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,1,2,0,1,5],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,3,3,12,4,6,44,65],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0,1,1,2,0,7,12],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[570,760,2241,1942,4764,780,2694,2597],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[570,1976,3457,9533,7131,3588,37032,51042],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0,2241,1942,5838,0,12054,16597],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[570,683,2241,741,4764,538,835,803],"isMain":false},{"metricName":"N75","quality":"More is better","values":[570,533,655,595,1074,511,605,598],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,2,1,5,1,3,15,23],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,3,2,8,2,5,27,42],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["35.44","36.64","31.44","36.23","37.16","37.90","38.27","38.19"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[394222270,394222270,394222270,394222270,394222270,394222270,394222270,394222270],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[19,19,19,19,19,19,19,19],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.23","23.23","23.23","23.23","23.23","23.23","23.23","23.23"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:40:27","assembliesNames":["spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_tremula_Mitochondrion","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.094","0.772","1.436","0.180","6.515","7.727"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.003","1.144","1.061","0.994","1.038","1.048"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[734,742,1397,1214,1568,1563],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[734,6017,11228,1398,51039,60626],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[734,null,643,null,695,677],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[734,null,516,null,529,541],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,8,null,31,35],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,null,13,null,55,62],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,3,8],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,0,3,8],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,1,0,4,3],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,1,0,3,8],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,667,0,3630,9600],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,3,0,3,4],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,0,2,4,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,7659,0,5389,3028,1294],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[8,63,118,27,342,573],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,9,20,2,59,102],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,32,89,9,269,341],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1089.92","1042.18","1049.26","1918.98","670.17","946.72"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","148.88","177.84","142.15","115.61","168.53"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,8,16,2,51,89],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,4,0,8,13],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,12,17,3,73,82],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,2,13,16],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,1,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[736,7920,1399,4764,1793,1941],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[736,14576,11937,6787,56017,64726],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,7920,1399,5978,16025,20884],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,7920,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[736,7920,698,4764,764,793],"isMain":false},{"metricName":"N75","quality":"More is better","values":[736,630,590,1214,598,598],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,7,1,27,29],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,6,12,2,48,53],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["44.84","51.71","45.71","39.19","45.40","45.25"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[783337,783337,783337,783337,783337,783337],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["44.74","44.74","44.74","44.74","44.74","44.74"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:40:44","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_trichocarpa_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["10.188","16.673","15.848","12.132","19.960","15.374","20.190","7.060"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.006","1.012","1.068","1.014","1.026","1.030","1.026","1.018"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,3411,3276,2563,4392,2550,8221,1088],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[16009,26209,25108,19100,31775,24308,32508,11290],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[946,1237,1517,789,1493,1007,2744,582],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[619,734,664,610,810,716,null,524],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[5,7,7,8,6,9,6,10],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[11,14,13,15,13,16,null,16],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,5,0,1,1,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,0,0,2,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,4,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,14333,2509],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[161,280,210,222,391,332,425,76],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[10,14,13,11,30,16,40,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[35,62,54,33,130,74,213,2],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1006.31","1069.44","843.85","1165.29","1247.45","1375.14","1340.48","685.49"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["62.50","53.47","52.24","57.74","95.71","66.27","126.16","18.04"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[7,7,10,9,20,13,23,2],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[3,7,3,2,10,3,17,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[18,24,22,23,24,24,11,18],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[3,9,8,5,10,10,11,3],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,4,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,3426,3276,2564,4764,2551,8810,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[16088,26501,26578,19318,32173,24864,46854,13799],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[6209,15998,16932,7767,22428,14881,46854,4284],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,30431,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[946,1238,1751,791,1493,1022,7919,735],"isMain":false},{"metricName":"N75","quality":"More is better","values":[620,736,812,610,811,738,3289,582],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[5,7,6,8,6,8,3,7],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[11,14,12,15,12,15,6,12],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["44.26","42.76","40.95","40.84","41.95","40.69","37.48","42.92"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[157033,157033,157033,157033,157033,157033,157033,157033],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.68","36.68","36.68","36.68","36.68","36.68","36.68","36.68"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:44:56","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Populus_trichocarpa_Chromosome","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.004","0.008","0.007","0.005","0.005","0.019","0.020"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.010","1.044","1.037","1.089","1.078","1.062","1.094","1.176"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[955,1744,6381,2563,4186,2550,8221,4989],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[8235,17254,31716,26883,21279,19565,74103,78616],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[636,974,917,697,1072,999,701,568],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[559,637,654,546,676,716,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[6,7,8,15,6,7,35,64],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[10,13,20,28,13,13,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,0,0,1,13],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,1,5],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,1,0,0,0,0,8],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,1,0,1,0,1,2,15],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,1,0,0,0,1,10],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,7920,0,0,0,613,8558],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,5,3,1,1,3,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,1,0,0,3,6],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,0,0,4,3],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,998,0,0,3406,2965],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,2,1,0,0,14,15],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1939,768,0,0,16196,14763],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[94,272,379,294,299,281,745,1176],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[6,8,28,20,14,11,51,84],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[27,47,76,89,43,61,228,250],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1141.89","1577.91","1203.33","1094.93","1467.41","1450.77","1010.44","1499.66"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["72.89","46.41","88.90","74.49","68.71","56.79","69.17","107.12"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[4,2,25,15,10,8,36,75],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[2,6,3,5,4,3,15,9],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[13,20,30,39,20,20,100,129],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,6,7,5,6,7,25,28],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,1,0,0,0,2,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[959,1747,7920,2564,4764,2551,8221,4990],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[8313,18000,34609,31011,21958,20568,100240,109947],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,8244,18926,7752,12243,11527,50203,42547],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,7920,0,0,0,13702,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[637,975,1517,741,1270,1219,1001,845],"isMain":false},{"metricName":"N75","quality":"More is better","values":[570,663,698,614,707,738,672,624],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[6,7,6,15,5,6,25,42],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[10,13,16,26,12,12,57,80],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["37.57","38.69","42.20","40.32","37.98","38.46","40.62","40.71"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[389204664,389204664,389204664,389204664,389204664,389204664,389204664,389204664],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[19,19,19,19,19,19,19,19],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["24.99","24.99","24.99","24.99","24.99","24.99","24.99","24.99"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:45:15","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_caprea_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["19.547","30.856","66.579","47.467","64.182","61.366","77.199","74.661"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.013","1.030","1.030","1.020","1.037","1.050","1.046","1.015"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,3411,4100,2563,4791,4175,11154,16369],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[30566,48031,105248,74938,102027,98229,125039,117949],"isMain":true},{"metricName":"NG50","quality":"More is better","values":[null,null,877,null,777,825,5554,4897],"isMain":false},{"metricName":"NG75","quality":"More is better","values":[null,null,null,null,null,null,2760,582],"isMain":false},{"metricName":"NA50","quality":"More is better","values":[793,846,1162,879,1139,1172,4938,6151],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[570,602,767,668,756,766,2744,2785],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,812,null,756,756,4212,4897],"isMain":true},{"metricName":"NGA75","quality":"More is better","values":[null,null,null,null,null,null,2010,530],"isMain":false},{"metricName":"LG50","quality":"Less is better","values":[null,null,51,null,54,54,10,10],"isMain":false},{"metricName":"LG75","quality":"Less is better","values":[null,null,null,null,null,null,19,36],"isMain":false},{"metricName":"LA50","quality":"Less is better","values":[12,17,29,28,27,29,9,7],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[24,34,58,52,55,57,19,13],"isMain":false},{"metricName":"LGA50","quality":"Less is better","values":[null,null,53,null,55,58,11,10],"isMain":true},{"metricName":"LGA75","quality":"Less is better","values":[null,null,null,null,null,null,24,39],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,4,1,1,5,6,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,0,0,2,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,2,0,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,0,2,5,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1499,0,0,2263,13292,2509],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[173,403,188,206,396,532,361,80],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[9,31,32,11,47,49,51,26],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[18,82,78,45,173,187,179,43],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["568.87","839.48","181.49","278.94","396.58","557.22","300.57","68.87"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["29.59","64.58","30.89","14.90","47.07","51.32","42.46","22.38"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[8,26,28,8,34,39,39,24],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[1,5,4,3,13,10,12,2],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[38,54,97,83,94,91,31,42],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[7,14,39,25,36,42,28,24],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,11,9],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,3426,4100,2564,4791,4187,11155,16369],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[30801,49424,108179,75361,103503,102490,138970,120458],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[11182,23118,66591,36011,62194,67787,136159,109030],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,87407,75346],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,21298,28417],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[793,921,1272,907,1154,1304,6687,6151],"isMain":false},{"metricName":"N75","quality":"More is better","values":[573,638,812,670,777,831,4064,2785],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[12,16,28,27,27,27,8,7],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[24,33,55,52,54,53,15,13],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.01","38.45","34.67","34.76","35.64","35.17","35.80","35.20"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[155582,155582,155582,155582,155582,155582,155582,155582],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.69","36.69","36.69","36.69","36.69","36.69","36.69","36.69"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:49:16","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_caprea_Chromosome","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.001","0.026","0.039","0.004","0.006","0.141","0.209"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.046","1.011","1.014","1.022","1.003","1.070","1.038","1.051"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[600,683,2247,2587,2718,2946,7057,5707],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1069,3056,93427,143453,15054,21328,514763,764411],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,698,674,595,519,627,687],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,null,548,564,null,302,523,532],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,51,73,9,17,286,362],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,null,93,133,null,33,526,704],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,3,5,0,5,101,136],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,1,0,0,28,42],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,2,4,0,5,72,94],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,1,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,1,4,1,6,59,124],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,3,4,0,5,90,124],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,2237,3039,0,3114,65807,88619],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,1,3,0,0,9,13],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,0,0,3,5],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[1,1,1,2,4,1,9,10],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[7808,4847,540,1249,2893,999,14229,6829],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,2,0,2,1,3,13],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,8274,0,5396,3532,2075,9866],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[6,39,769,1555,154,378,6491,11085],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[3,9,58,73,19,29,476,772],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[3,11,221,172,39,71,1571,2718],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["561.80","1276.60","822.61","1085.48","1023.32","1770.91","1261.60","1450.53"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["280.90","294.60","62.04","50.96","126.25","135.86","92.52","101.02"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,9,54,67,17,27,415,678],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,4,6,2,2,61,94],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,6,124,197,19,34,734,1020],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,22,28,4,2,92,154],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,0,1,1,2,2],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,4847,7920,2613,5908,6478,7908,5708],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[8925,7937,103560,147697,23387,27371,550598,819607],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,4847,36101,38266,13866,7480,142309,254701],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,0,7920,0,5908,6478,14966,11000],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,4847,813,701,2193,697,701,758],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,664,606,591,683,553,572,598],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,40,72,3,10,256,335],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,3,77,130,9,21,476,643],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["53.24","46.34","39.10","38.60","47.07","42.22","37.98","38.03"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[365398294,365398294,365398294,365398294,365398294,365398294,365398294,365398294],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[19,19,19,19,19,19,19,19],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["21.90","21.90","21.90","21.90","21.90","21.90","21.90","21.90"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:49:32","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_caprea_Mitochondrion","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["1.448","2.767","0.015","0.012","34.666","37.094"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.141","1.078","1.000","1.000","1.022","1.028"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[814,1397,93,74,2472,3338],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[8800,16872,93,74,210975,225819],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,590,null,null,721,749],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,512,null,null,559,574],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,12,null,null,118,113],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,20,null,null,208,202],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,0,0,8,12],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,3,7],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,1,0,0,5,5],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[2,1,0,0,7,13],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,1,0,0,8,12],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,667,0,0,7789,12263],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,0,3,1],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,1,3,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[7658,0,716,6404,9686,2366],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[76,152,3,0,328,501],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[7,17,0,0,65,117],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[33,117,0,0,277,445],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["866.10","906.49","3225.81","0.00","156.13","222.86"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["79.77","101.38","0.00","0.00","30.94","52.05"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[6,12,0,0,48,92],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[1,5,0,0,17,25],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[16,27,1,1,289,295],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,0,1,39,53],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,1399,809,6478,7908,3339],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[17667,18068,809,6478,224333,233363],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7920,1399,0,6478,55941,69475],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[767,667,809,6478,777,808],"isMain":false},{"metricName":"N75","quality":"More is better","values":[664,549,809,6478,596,615],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[3,12,1,1,104,106],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[9,19,1,1,187,189],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["50.61","45.71","36.96","56.85","45.29","44.70"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[606028,606028,606028,606028,606028,606028],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[2,2,2,2,2,2],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["44.99","44.99","44.99","44.99","44.99","44.99"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:49:50","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_reticulata_Chloroplast","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["20.678","31.470","65.647","47.545","66.195","62.427","80.113","77.493"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.013","1.033","1.035","1.017","1.032","1.045","1.031","1.011"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[2533,3411,4100,2563,4791,4175,11154,16369],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[32303,49052,104230,74766,104793,99879,127739,121880],"isMain":true},{"metricName":"NG50","quality":"More is better","values":[null,null,836,null,810,831,6687,4897],"isMain":false},{"metricName":"NG75","quality":"More is better","values":[null,null,null,null,null,null,3289,698],"isMain":false},{"metricName":"NA50","quality":"More is better","values":[674,845,1158,879,1139,1178,5554,6151],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[570,594,758,667,776,772,2760,2785],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,777,null,809,772,4938,4897],"isMain":true},{"metricName":"NGA75","quality":"More is better","values":[null,null,null,null,null,null,2215,591],"isMain":false},{"metricName":"LG50","quality":"Less is better","values":[null,null,52,null,53,53,9,10],"isMain":false},{"metricName":"LG75","quality":"Less is better","values":[null,null,null,null,null,null,18,30],"isMain":false},{"metricName":"LA50","quality":"Less is better","values":[13,17,29,28,28,29,9,7],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[27,36,57,52,56,57,18,14],"isMain":false},{"metricName":"LGA50","quality":"Less is better","values":[null,null,54,null,54,57,10,10],"isMain":true},{"metricName":"LGA75","quality":"Less is better","values":[null,null,null,null,null,null,22,33],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,3,0,1,1,3,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,0,0,1,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,2,0,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,0,2,5,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1499,0,0,2263,12859,2509],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[174,414,178,191,407,534,337,69],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[11,27,28,20,39,49,32,16],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[28,94,99,49,189,203,140,35],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["541.15","846.00","174.37","258.34","395.40","550.10","270.52","57.26"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["34.21","55.17","27.43","27.05","37.89","50.48","25.69","13.28"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[9,21,21,18,24,38,24,13],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[2,6,7,2,15,11,8,3],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[41,56,97,83,96,92,30,43],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[7,14,38,25,37,42,28,25],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0,11,9],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,2,2],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2533,3426,4100,2564,4791,4187,11155,16369],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[32578,50542,107161,75209,106266,103705,141236,124394],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[11182,23118,65288,36011,64161,68111,139338,112966],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0,90586,75346],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,21298,28417],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[675,904,1189,907,1164,1304,7353,6151],"isMain":false},{"metricName":"N75","quality":"More is better","values":[573,621,798,667,781,831,4064,2785],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[13,17,28,27,27,28,8,7],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[26,34,55,52,55,53,15,14],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.36","38.28","34.70","34.79","35.65","35.13","35.58","35.17"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[155500,155500,155500,155500,155500,155500,155500,155500],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.71","36.71","36.71","36.71","36.71","36.71","36.71","36.71"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:53:44","assembliesNames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_reticulata_Chromosome","order":[0,1,2,3,4,5,6,7],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.002","0.002","0.030","0.045","0.003","0.008","0.152","0.237"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.041","1.003","1.014","1.013","1.074","1.051","1.035","1.043"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[7214,2788,7528,2587,1000,5250,7226,5268],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[8595,5337,105457,157532,9883,29035,534376,832152],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[7214,2788,709,697,519,614,636,680],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[7214,663,579,581,null,511,523,532],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,48,76,11,12,277,403],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,3,90,139,null,25,520,775],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,3,6,0,7,91,120],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1,2,0,3,27,36],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[1,0,2,4,0,4,62,84],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,2,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,0,1,1,1,6,51,115],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,0,3,6,0,6,89,113],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[7808,0,2279,5098,0,9622,77381,83828],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0,1],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,1,3,0,0,5,18],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,2,1,2,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,0,0,1,7],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,543,0,0,520,4485],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,1,0,7,17],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,1695,0,4213,0,4761,11978],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[35,47,705,1527,188,480,6598,11488],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[5,5,57,84,14,42,461,853],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[27,5,167,275,49,107,1518,2536],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["408.12","880.81","668.72","970.44","1900.91","1654.77","1235.04","1380.99"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["58.30","93.70","54.07","53.38","141.56","144.79","86.29","102.54"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[3,5,51,75,12,37,406,753],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[2,0,6,9,2,5,55,100],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,5,134,208,16,35,749,1131],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,19,33,2,3,84,159],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,0,0,1,2,1],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,2790,7920,2613,4764,6478,7908,5292],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[8925,5350,108585,159877,14836,30485,558082,884116],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,2790,32755,45952,5765,10453,138500,254567],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,0,7920,0,0,6478,14966,5292],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,2790,762,720,842,759,678,743],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,664,600,596,604,571,566,588],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,45,74,4,9,259,381],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,3,85,136,10,21,486,719],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["53.24","41.83","38.09","38.16","41.10","43.12","37.35","37.43"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[351751266,351751266,351751266,351751266,351751266,351751266,351751266,351751266],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[19,19,19,19,19,19,19,19],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["23.08","23.08","23.08","23.08","23.08","23.08","23.08","23.08"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:53:59","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Salix_reticulata_Mitochondrion","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["1.262","2.827","0.014","0.012","32.709","35.502"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.152","1.072","1.000","1.000","1.020","1.024"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[814,1397,93,74,2472,3338],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[8140,18289,93,74,211068,229142],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,590,null,null,716,749],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,515,null,null,559,575],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,13,null,null,118,113],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,21,null,null,208,204],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,1,0,0,9,13],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,1,0,0,9,13],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,2,0,0,7,8],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,1,0,0,9,13],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,667,0,0,8519,12882],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,1,0,0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,1,1,3,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[7658,0,716,6404,9686,2979],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[69,177,3,0,298,500],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[7,19,0,0,61,106],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[33,124,0,0,261,428],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["850.28","973.38","3225.81","0.00","141.66","218.98"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["86.26","104.49","0.00","0.00","29.00","46.42"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[6,13,0,0,44,78],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[1,6,0,0,17,28],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[15,29,1,1,288,297],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,0,1,38,55],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,1399,809,6478,7908,3339],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[17003,19486,809,6478,224308,236875],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7920,1399,0,6478,56174,73035],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[816,698,809,6478,772,811],"isMain":false},{"metricName":"N75","quality":"More is better","values":[675,563,809,6478,596,623],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,12,1,1,103,106],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[8,20,1,1,186,189],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["50.57","45.80","36.96","56.85","45.28","44.67"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[643145,643145,643145,643145,643145,643145],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["44.95","44.95","44.95","44.95","44.95","44.95"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:57:27","assembliesNames":["megahit_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Scirpus_sylvaticus_Chromosome","order":[0,1,2,3,4,5,6],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.000","0.000","0.001","0.002","0.001"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","1.206","1.614","1.554","1.159","1.197","1.259"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1814,1814,336,597,572,1814,1101],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[2988,3094,643,1254,2064,5120,4758],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,307,null,null,null,198],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,2,null,null,null,7],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1,0,0,1,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,1,0,0,1,1],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,1,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,1,0,0,1,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,1038,0,0,810,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,1,0,1,2,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0,598],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0,1,1,2,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4820,4942,0,3626,4620,7460,1710],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[100,102,30,45,63,163,116],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[6,6,0,1,2,11,3],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[10,10,0,1,2,16,3],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3344.48","3294.57","4665.63","3591.38","3052.33","3182.35","2438.51"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["200.67","193.80","0.00","79.81","96.90","214.76","63.06"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[6,6,0,1,2,11,3],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,2,2,6,10],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1,2,3],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,8675,1038,5573,7012,13591,8295],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,1038,4764,6478,10755,3558],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,1038,4764,6478,7908,823],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,1038,4764,6478,2847,619],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,1,4],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,2,7],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","54.86","27.84","38.02","55.45","47.67","39.40"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[320572876,320572876,320572876,320572876,320572876,320572876,320572876],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[31,31,31,31,31,31,31],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["25.87","25.87","25.87","25.87","25.87","25.87","25.87"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:57:37","assembliesNames":["megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Scirpus_sylvaticus_Mitochondrion","order":[0,1,2],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.029","0.021","0.042"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","2.123","1.446"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[419,440,662],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[607,440,884],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,null,662],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,null,1],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,1],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,1],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,1278],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,0,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[4157,0,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[28,13,18],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,1,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,1,27],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["4612.85","2961.28","2036.20"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","227.79","113.12"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,1,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[4764,932,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[4764,932,1278],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[4764,0,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[4764,932,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[4764,932,1278],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["38.20","36.48","34.43"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[2128005,2128005,2128005],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[5,5,5],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["41.08","41.08","41.08"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:57:48","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_1mil_long","spades_10mil_long"],"referenceName":"Solanum_tuberosum_Chloroplast","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.286","2.482","0.062","1.931"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.869","1.263","5.417","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[109,3754,95,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[444,3858,95,2991],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,3754,null,528],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1,null,5],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[1,1,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,2,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2166,705,0,2520],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[15,120,3,47],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,5,1,3],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,14,1,9],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3378.38","3113.65","3125.00","1567.71"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","129.74","1041.67","100.07"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,5,1,3],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,2,1,6],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[2241,4764,520,1918],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[2996,5573,520,5517],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[2241,4764,0,3196],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[2241,4764,520,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,520,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["31.31","38.02","25.96","45.02"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[155296,155296,155296,155296],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["37.88","37.88","37.88","37.88"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:57:59","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_10mil_long","spades_10mil_long"],"referenceName":"Trifolium_pratense_Mitochondrion","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.184","0.415","0.490","0.499"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","0.994","1.451","1.001"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[555,697,652,809],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[555,1246,1480,1507],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[555,697,555,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[555,549,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,2,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,2,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,0,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,0,2122],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,4,42,34],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,3,0,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,10,0,2],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["720.72","318.98","2837.84","2256.14"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","239.23","0.00","132.71"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,2,0,2],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,3,3],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0,0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[555,698,882,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[555,1247,2148,3631],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0,0,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[555,698,711,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[555,549,555,1438],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,2,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2,3,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.04","53.81","50.23","45.22"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[301823,301823,301823,301823],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["45.20","45.20","45.20","45.20"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 17:58:10","assembliesNames":["megahit_10mil_short","megahit_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Triglochin_maritima_Chloroplast","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.064","2.406","0.451","1.951"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["7.550","1.270","1.224","1.167"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[100,3751,470,613],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[100,3751,704,3044],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[null,3751,null,503],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,3751,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[null,1,null,4],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,1,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,2,2],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,2,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,1],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,2613,665],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[4,178,0,45],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,10,0,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,55,0,1],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["4000.00","4744.14","0.00","1478.80"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","266.52","0.00","32.86"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,7,0,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,3,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,2,6],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,1,1],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,4764,2847,1278],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,4764,3475,4217],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,4764,2847,1278],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,4764,2847,645],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,4764,2847,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","38.20","32.49","37.56"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[155938,155938,155938,155938],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["36.48","36.48","36.48","36.48"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:08:57","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Triglochin_maritima_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.000","0.000","0.000","0.000","0.001","0.003"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","0.980","1.003","1.012","1.007","1.045"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[726,576,506,508,1251,1500],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1922,576,1813,508,12146,26965],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[609,576,null,508,573,623],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[587,576,null,508,545,502],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,1,null,1,8,19],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[3,1,null,1,14,33],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,5],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,4],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,5],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,3380],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,1,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,1590],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,0,1,0,0,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,0,3455,0,0,2319],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[32,10,60,7,240,604],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[1,2,0,0,13,47],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[1,19,0,0,57,127],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1664.07","1680.67","3309.43","1377.95","1975.47","2238.70"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["52.00","336.13","0.00","0.00","107.00","174.20"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[1,1,0,0,12,44],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,1,0,0,1,3],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[3,1,2,1,19,40],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,0,1,0,1,8],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[726,583,4764,514,1251,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1922,583,5273,514,12233,32093],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,0,4764,0,1251,10655],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[609,583,4764,514,574,758],"isMain":false},{"metricName":"N75","quality":"More is better","values":[587,583,4764,514,545,619],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[2,1,1,1,8,15],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[3,1,1,1,14,26],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["40.01","39.28","39.84","55.45","44.46","43.94"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[1055183918,1055183918,1055183918,1055183918,1055183918,1055183918],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[24,24,24,24,24,24],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["17.46","17.46","17.46","17.46","17.46","17.46"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:09:05","assembliesNames":["megahit_10mil_short","spades_10mil_long"],"referenceName":"Triglochin_maritima_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.043","0.278"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["4.364","1.003"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[98,819],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[173,1110],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[1,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,1918],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[7,48],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,2],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,5],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["4046.24","4336.04"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","180.67"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,2],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[755,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[755,3028],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[755,1590],"isMain":false},{"metricName":"N75","quality":"More is better","values":[755,1438],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["39.34","42.87"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[398819,398819],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["48.37","48.37"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:14:04","assembliesNames":["megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Vaccinium_myrtillus_Chromosome","order":[0,1,2,3,4],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.000","0.001","0.001","0.001"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["0.999","1.000","0.999","1.161","1.068"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[3393,548,3393,3393,660],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5485,982,4091,5869,3523],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[1537,null,3393,140,282],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[2,null,1,4,7],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[1,0,1,1,2],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[1,0,1,1,2],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[1,0,1,1,2],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[7920,0,6478,7908,1102],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1,0,1,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,1,0,1,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,1,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,1190,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2990,3782,2387,2978,2225],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[177,37,150,184,88],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[21,0,20,21,1],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[25,0,24,25,1],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3224.04","3767.82","3663.00","3132.45","2497.87"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["382.51","0.00","488.40","357.51","28.38"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[21,0,20,21,1],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,1,1,5,7],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,2,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7920,4764,6478,7908,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[8475,4764,6478,10988,5986],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7920,4764,6478,9098,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7920,4764,6478,7908,1438],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7920,4764,6478,1190,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,2],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,2,5],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.32","38.20","56.85","53.93","46.12"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[464157312,464157312,464157312,464157312,464157312],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[12,12,12,12,12],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["29.11","29.11","29.11","29.11","29.11"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:14:15","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","spades_10mil_long"],"referenceName":"Veronica_serpyllifolia_Chloroplast","order":[0,1,2,3],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["1.034","0.093","2.495","0.824"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.000","1.265","0.996"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[1561,140,3768,647],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[1561,140,3768,1234],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[1561,null,3768,647],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[1561,null,3768,587],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,null,1,1],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[1,null,1,2],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0,2,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[0,1,0,0],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[0,615,0,0],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[20,7,149,18],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0,7,3],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0,15,11],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["1281.23","5000.00","3957.50","1448.11"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00","185.92","241.35"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0,7,3],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,1,1,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,0,1,0],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[1561,755,4764,647],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1561,755,4764,1238],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[1561,0,4764,0],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[1561,755,4764,647],"isMain":false},{"metricName":"N75","quality":"More is better","values":[1561,755,4764,591],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["55.80","39.34","38.20","53.63"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[150926,150926,150926,150926],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1,1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["38.11","38.11","38.11","38.11"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:18:07","assembliesNames":["megahit_1mil_short","megahit_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"Veronica_serpyllifolia_Chromosome","order":[0,1,2,3,4,5],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.001","0.001","0.001","0.001","0.001","0.001"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["1.000","1.124","1.310","1.000","1.094","1.005"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[5223,5223,3640,3976,5223,1000],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[5223,5320,3640,3976,5426,3119],"isMain":true},{"metricName":"NA50","quality":"More is better","values":[5223,5223,3640,3976,5223,null],"isMain":false},{"metricName":"NA75","quality":"More is better","values":[null,null,3640,null,null,null],"isMain":false},{"metricName":"NGA50","quality":"More is better","values":[null,null,null,null,null,null],"isMain":true},{"metricName":"LA50","quality":"Less is better","values":[1,1,1,1,1,null],"isMain":false},{"metricName":"LA75","quality":"Less is better","values":[null,null,1,null,null,null],"isMain":false}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[1,1,3,1,1,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,1,0,1,1,3],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[2585,2697,0,2502,2685,3838],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[164,168,144,134,174,79],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[15,15,6,15,15,6],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[18,18,12,18,18,55],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["3139.96","3157.89","3959.31","3370.22","3206.78","2526.38"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["287.19","281.95","164.97","377.26","276.45","191.88"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[15,15,6,15,15,5],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0,0,0,0,1],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[1,2,1,1,2,7],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[1,1,1,1,1,4],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[1,1,0,1,1,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[7808,7920,4764,6478,7908,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[7808,8675,4764,6478,8619,6980],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[7808,7920,4764,6478,7908,5311],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[7808,7920,0,6478,7908,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[7808,7920,4764,6478,7908,1278],"isMain":false},{"metricName":"N75","quality":"More is better","values":[7808,7920,4764,6478,7908,1005],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1,1,1,1,3],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[1,1,1,1,1,4],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["56.99","54.86","38.20","56.85","56.35","40.01"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[367244687,367244687,367244687,367244687,367244687,367244687],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[7,7,7,7,7,7],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["21.11","21.11","21.11","21.11","21.11","21.11"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:18:15","assembliesNames":["megahit_10mil_long","spades_10mil_long"],"referenceName":"Veronica_serpyllifolia_Mitochondrion","order":[0,1],"report":[["Genome statistics",[{"metricName":"Genome fraction (%)","quality":"More is better","values":["0.118","0.244"],"isMain":true},{"metricName":"Duplication ratio","quality":"Less is better","values":["2.289","1.000"],"isMain":true},{"metricName":"Largest alignment","quality":"More is better","values":[203,711],"isMain":true},{"metricName":"Total aligned length","quality":"More is better","values":[394,814],"isMain":true},{"metricName":"NGA50","quality":"More is better","values":[null,null],"isMain":true}]],["Reads mapping",[]],["Misassemblies",[{"metricName":"# misassemblies","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"    # relocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # inversions","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # interspecies translocations","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Misassembled contigs length","quality":"Less is better","values":[0,0],"isMain":true},{"metricName":"# possibly misassembled contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # possible misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# local misassemblies","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap ext. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# scaffold gap loc. mis.","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# unaligned mis. contigs","quality":"Less is better","values":[0,0],"isMain":false}]],["Unaligned",[{"metricName":"# fully unaligned contigs","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Fully unaligned length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# partially unaligned contigs","quality":"Less is better","values":[1,2],"isMain":false},{"metricName":"Partially unaligned length","quality":"Less is better","values":[691,2214],"isMain":false}]],["Mismatches",[{"metricName":"# mismatches","quality":"Less is better","values":[18,22],"isMain":false},{"metricName":"# indels","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"Indels length","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# mismatches per 100 kbp","quality":"Less is better","values":["4568.53","2702.70"],"isMain":true},{"metricName":"# indels per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true},{"metricName":"    # indels (<= 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"    # indels (> 5 bp)","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's","quality":"Less is better","values":[0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,2],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,2],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[882,1590],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1593,3028],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,3028],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[882,1590],"isMain":false},{"metricName":"N75","quality":"More is better","values":[711,1438],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,1],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,2],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["48.21","42.87"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[{"metricName":"# similar correct contigs","quality":"Equal","values":[0,0],"isMain":false},{"metricName":"# similar misassembled blocks","quality":"Equal","values":[0,0],"isMain":false}]],["Reference statistics",[{"metricName":"Reference length","quality":"Equal","values":[334109,334109],"isMain":false},{"metricName":"Reference fragments","quality":"Equal","values":[1,1],"isMain":false},{"metricName":"Reference GC (%)","quality":"Equal","values":["44.34","44.34"],"isMain":false}]]],"subreferences":[],"subreports":[],"minContig":500},{"date":"17 September 2025, Wednesday, 18:18:21","assembliesNames":["megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"referenceName":"","order":[0,1,2,3,4,5],"report":[["Genome statistics",[]],["Reads mapping",[]],["Misassemblies",[]],["Unaligned",[]],["Mismatches",[{"metricName":"# N's","quality":"Less is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# N's per 100 kbp","quality":"Less is better","values":["0.00","0.00","0.00","0.00","0.00","0.00"],"isMain":true}]],["Statistics without reference",[{"metricName":"# contigs","quality":"Equal","values":[2,7,1,2,4,22],"isMain":true},{"metricName":"# contigs (>= 1000 bp)","quality":"Equal","values":[0,1,0,0,1,3],"isMain":false},{"metricName":"# contigs (>= 5000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 10000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 25000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"# contigs (>= 50000 bp)","quality":"Equal","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Largest contig","quality":"More is better","values":[762,1271,510,506,1254,1055],"isMain":true},{"metricName":"Total length","quality":"More is better","values":[1314,5261,510,1007,3203,16011],"isMain":true},{"metricName":"Total length (>= 1000 bp)","quality":"More is better","values":[0,1271,0,0,1254,3077],"isMain":true},{"metricName":"Total length (>= 5000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 10000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"Total length (>= 25000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":false},{"metricName":"Total length (>= 50000 bp)","quality":"More is better","values":[0,0,0,0,0,0],"isMain":true},{"metricName":"N50","quality":"More is better","values":[762,788,510,506,927,717],"isMain":false},{"metricName":"N75","quality":"More is better","values":[552,540,510,501,516,606],"isMain":false},{"metricName":"L50","quality":"Less is better","values":[1,3,1,1,2,9],"isMain":false},{"metricName":"L75","quality":"Less is better","values":[2,5,1,2,3,15],"isMain":false},{"metricName":"GC (%)","quality":"Equal","values":["57.00","48.51","37.65","44.49","52.70","44.98"],"isMain":false}]],["K-mer-based statistics",[]],["Predicted genes",[]],["Similarity statistics",[]],["Reference statistics",[]]],"subreferences":[],"subreports":[],"minContig":500}]
            </div>
            <div id='qualities-json'>
                {}
            </div>
            <div id='main-metrics-json'>
                {}
            </div>

            <div id="contigs-lengths-json">
                {}
            </div>
            <div id='assemblies-lengths-json'>
                {}
            </div>
            <div id='reference-length-json'>
                {}
            </div>
            <div id='tick-x-json'>
                {}
            </div>
            <div id='coord-nx-json'>
                {}
            </div>
            <div id='coord-ngx-json'>
                {}
            </div>
            <div id='coord-nax-json'>
                {}
            </div>
            <div id='coord-ngax-json'>
                {}
            </div>
            <div id='coord-misassemblies-json'>
                {}
            </div>
            <div id='features-in-contigs-json'>
                {}
            </div>
            <div id='operons-in-contigs-json'>
                {}
            </div>
            <div id='summary-json'>
                [{"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755,76.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825,76.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895,76.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965,76.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035,77.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105,77.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175,77.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245,77.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,1,0,0,1,0,0,0,0,0,0,2,2,0,0,0,0,2,1,0,1,1,1,2,0,3,1,1,0,1,2,2,1,0,0,0,0,0,0,0,0,0,0,0,1,16,17,18,0,0,0,13,16,38,41,0,0,0,3,3,5],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,21,24,24,1,1,0,20,20,54,56,0,0,0,6,5,12],[0,0,0,0,0,0,0,0,0,0,1,0,1,0,0,0,0,1,1,1,1,1,1,1,1,1,1,1,1,1,2,2,2,1,2,2,2,1,2,2,2,3,3,2,3,2,3,3,2,2,8,3,3,6,6,8,14,17,4,4,3,19,22,22,12,28,14,30,82,97,97,16,15,50,124,134,190],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,3,0,0,0,2,1,0,0,2,1,0,1,0,1,1,0,0,1,0,3,0,1,6,1,1,2,1,7,5,4,1,4,5,8,14,17,6,8,12,18,21,23,17,29,15,39,75,83,83,27,29,78,197,208,298],[0,0,0,0,0,0,1,1,1,1,0,0,0,0,1,1,1,1,2,1,0,2,0,1,2,2,0,2,2,1,1,1,2,2,1,1,2,1,1,3,1,3,2,2,2,2,1,2,2,1,2,3,2,2,4,4,3,2,3,3,4,21,24,24,3,5,3,20,83,94,96,1,1,2,19,16,47],[0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,1,1,0,0,1,0,0,0,0,0,0,0,0,1,0,0,0,0,1,1,1,1,0,1,1,0,2,2,1,2,2,1,2,1,0,0,0,0,0,3,1,6,25,23,24,0,4,1,20,82,91,92,1,1,1,34,35,99],[0,0,0,0,0,2,2,2,2,1,0,3,1,2,3,1,1,0,0,1,1,0,2,3,3,2,3,0,0,2,3,4,0,1,2,4,2,3,2,0,5,1,4,7,5,6,5,5,7,4,18,18,19,19,29,29,35,36,32,31,44,11,9,11,73,63,65,100,41,31,30,289,288,261,734,749,1513],[0,0,0,0,0,0,0,0,0,1,2,1,0,2,1,2,3,2,3,2,2,4,1,2,2,3,3,4,5,6,3,3,6,5,6,5,5,7,7,6,7,5,10,8,8,10,11,12,19,22,21,36,40,36,44,50,41,44,78,86,65,18,19,18,82,82,133,129,31,42,43,295,297,251,1020,1131,2020]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Mitochondrion","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Carex_nigra_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_elata_Mitochondrion","Carex_rostrata_Mitochondrion","Scirpus_sylvaticus_Mitochondrion","Triglochin_maritima_Mitochondrion","Carex_sylvatica_Mitochondrion","Hydra_vulgaris_Chromosome","Veronica_serpyllifolia_Mitochondrion","Carex_echinata_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Mentha_aquatica_Mitochondrion","Veronica_serpyllifolia_Chloroplast","Amborella_trichopoda_Chloroplast","Chamaenerion_angustifolium_Chromosome","Diphasiastrum_alpinum_Chromosome","Ipomoea_triloba_Chloroplast","Matricaria_discoidea_Mitochondrion","Alnus_glutinosa_Chromosome","Eleocharis_quinqueflora_Chloroplast","Eriophorum_vaginatum_Mitochondrion","Trifolium_pratense_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Matricaria_discoidea_Chloroplast","Triglochin_maritima_Chloroplast","Carica_papaya_Mitochondrion","Myrica_gale_Chromosome","Solanum_tuberosum_Chloroplast","Citrus_sinensis_Chloroplast","Mentha_aquatica_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Mitochondrion","Citrus_sinensis_Chromosome","Veronica_serpyllifolia_Chromosome","Carica_papaya_Chloroplast","Vaccinium_myrtillus_Chromosome","Myrica_gale_Chloroplast","Matricaria_discoidea_Chromosome","Eleocharis_quinqueflora_Chromosome","Myrica_gale_Mitochondrion","Scirpus_sylvaticus_Chromosome","Linum_lewisii_Chromosome","Ipomoea_triloba_Chromosome","Eriophorum_vaginatum_Chromosome","not_aligned","Carex_echinata_Chloroplast","Carex_echinata_Chromosome","Triglochin_maritima_Chromosome","Carex_myosuroides_Chromosome","Carex_sylvatica_Chromosome","Carex_acutiformis_Chromosome","Carex_sylvatica_Chloroplast","Carex_nigra_Chloroplast","Carex_elata_Chromosome","Carex_nigra_Chromosome","Populus_tremula_Chromosome","Populus_tremula_Chloroplast","Populus_euphratica_Chloroplast","Populus_trichocarpa_Chloroplast","Populus_tremula_Mitochondrion","Carex_rostrata_Chloroplast","Carex_rostrata_Chromosome","Populus_trichocarpa_Chromosome","Dryas_octopetala_Chloroplast","Salix_caprea_Chloroplast","Salix_reticulata_Chloroplast","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Dryas_octopetala_Mitochondrion","Salix_caprea_Chromosome","Salix_reticulata_Chromosome","Dryas_octopetala_Chromosome"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[2533,2533,2533,7214,5361,955,2533,2533,2533,600,5223,5385,5214,5340,5351,0,0,4080,0,0,0,0,0,2533,570,2533,1814,2533,0,0,1814,0,0,2533,0,0,2533,1561,1561,0,0,1561,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,421,0,0,0,0,0,0,0,0,0,0,0,0,66,0,0,0,0,0],[3411,3411,1958,2788,874,1744,3411,3411,3411,683,0,0,0,0,0,0,786,0,0,0,0,0,0,0,754,0,0,0,0,0,0,0,0,0,0,734,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[4100,4100,4468,7528,5361,6381,3276,3276,3276,2247,5223,5385,5214,5340,5351,5211,1882,4080,1137,3393,3393,1165,1137,563,655,509,1814,103,814,814,1814,1137,1137,0,1814,742,138,643,140,984,838,923,694,923,1137,923,726,100,539,109,102,105,555,555,923,555,91,421,0,119,0,0,98,0,0,0,0,0,0,0,66,0,0,0,0,0],[2563,2563,3832,2587,4742,2563,2563,2563,2563,2587,0,584,0,697,0,0,1806,105,1806,0,0,1568,1806,786,848,525,555,531,1397,1397,336,1806,1806,761,1028,1397,0,0,0,1022,1028,531,1030,1065,1027,1065,576,0,1027,0,0,0,695,697,1065,697,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[4791,4791,4389,1000,1412,4186,4392,4392,4392,2718,3640,2075,2860,638,483,3633,825,818,825,548,732,354,825,3964,1697,4392,331,4392,93,93,597,825,820,3872,825,1214,3760,3550,3768,407,552,1248,624,1884,581,1210,506,3751,500,3754,3806,3521,670,121,0,0,818,0,419,415,424,0,0,0,0,151,151,151,151,0,0,0,0,0,0,0],[4175,4175,1677,5250,4110,2550,2550,2550,2550,2946,3976,4138,3946,4094,4103,3968,974,884,0,3393,3393,647,0,0,778,82,572,0,74,74,572,477,0,0,0,0,0,0,0,0,255,0,535,0,533,0,508,0,500,95,0,0,0,0,0,0,0,421,0,0,0,0,0,0,0,0,0,0,0,0,65,0,0,0,0,0],[11154,11154,9143,7226,5361,8221,8221,8221,8221,7057,5223,5385,5214,5340,5351,5211,3567,4080,2407,3393,3393,3647,3172,1099,1034,0,1814,269,2472,2472,1814,1932,1932,0,2082,1568,0,107,0,2211,1479,849,1339,645,797,845,1251,470,951,0,0,0,555,810,193,652,204,421,440,155,189,529,0,203,227,459,529,529,529,531,66,0,0,0,0,0],[16369,16369,18654,5268,6316,4989,2546,1088,860,5707,1000,735,809,711,350,587,5475,668,5442,660,235,3651,2803,698,2566,647,1779,647,3338,3338,1101,1773,2230,647,1779,1563,647,647,647,1351,1782,813,1272,809,1101,1133,1500,613,1301,647,612,611,601,809,809,809,668,87,662,750,721,500,819,711,668,116,0,0,0,0,0,0,0,0,0,0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Salix_caprea_Chloroplast","Salix_reticulata_Chloroplast","Dryas_octopetala_Chloroplast","Salix_reticulata_Chromosome","Dryas_octopetala_Chromosome","Populus_trichocarpa_Chromosome","Populus_euphratica_Chloroplast","Populus_trichocarpa_Chloroplast","Populus_tremula_Chloroplast","Salix_caprea_Chromosome","Veronica_serpyllifolia_Chromosome","Ipomoea_triloba_Chromosome","Matricaria_discoidea_Chromosome","Linum_lewisii_Chromosome","Citrus_sinensis_Chromosome","Chamaenerion_angustifolium_Chromosome","Carex_rostrata_Chloroplast","Eleocharis_quinqueflora_Chromosome","Carex_nigra_Chloroplast","Vaccinium_myrtillus_Chromosome","Alnus_glutinosa_Chromosome","Dryas_octopetala_Mitochondrion","Carex_sylvatica_Chloroplast","Myrica_gale_Chloroplast","Populus_tremula_Chromosome","Carica_papaya_Chloroplast","Eriophorum_vaginatum_Chromosome","Citrus_sinensis_Chloroplast","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Scirpus_sylvaticus_Chromosome","Carex_echinata_Chloroplast","Carex_myosuroides_Chromosome","Chamaenerion_angustifolium_Chloroplast","Carex_acutiformis_Chromosome","Populus_tremula_Mitochondrion","Matricaria_discoidea_Chloroplast","Mentha_aquatica_Chromosome","Veronica_serpyllifolia_Chloroplast","Carex_sylvatica_Chromosome","Carex_rostrata_Chromosome","Citrus_sinensis_Mitochondrion","Carex_elata_Chromosome","Carica_papaya_Mitochondrion","Carex_echinata_Chromosome","Myrica_gale_Mitochondrion","Triglochin_maritima_Chromosome","Triglochin_maritima_Chloroplast","Carex_nigra_Chromosome","Solanum_tuberosum_Chloroplast","Amborella_trichopoda_Chloroplast","Ipomoea_triloba_Chloroplast","Myrica_gale_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Matricaria_discoidea_Mitochondrion","Trifolium_pratense_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Diphasiastrum_alpinum_Chromosome","Scirpus_sylvaticus_Mitochondrion","Eriophorum_vaginatum_Mitochondrion","Mentha_aquatica_Mitochondrion","Carex_sylvatica_Mitochondrion","Triglochin_maritima_Mitochondrion","Veronica_serpyllifolia_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Carex_echinata_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_elata_Mitochondrion","Carex_rostrata_Mitochondrion","Carex_nigra_Mitochondrion","Hydra_vulgaris_Chromosome","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Mitochondrion","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[7109,8595,1069,32303,30566,0,12231,0,0,8235,0,16009,15900,15211,0,0,0,0,0,0,0,570,0,0,0,0,6946,0,5214,5459,5223,5351,3655,0,2988,4080,0,0,4599,0,4094,4094,4094,4094,1561,0,1561,0,0,1561,0,0,0,0,0,0,919,0,0,0,0,0,0,0,0,0,0,0,0,0,66,0,0,0,0,0],[8637,5337,3056,49052,48031,0,13376,0,0,17254,786,26209,26209,22479,0,734,0,0,0,0,0,1968,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[146017,105457,93427,104230,105248,30676,85849,8140,8800,31716,20000,25108,25072,21030,7585,6017,11608,9372,1580,2202,7061,1706,3957,2943,1922,5098,6681,2350,6275,6058,5320,5351,3820,5485,3094,4171,5211,5244,1575,1738,921,103,0,138,1026,1587,749,100,444,140,555,193,278,770,555,923,919,119,91,0,0,0,0,173,0,0,0,0,0,0,66,0,0,0,0,0],[226058,157532,143453,74766,74938,50123,70408,18289,16872,26883,20813,19100,17088,14345,9749,11228,12905,10708,4758,3579,4965,2902,3886,3431,576,4629,1133,3358,0,697,0,0,555,0,643,249,0,0,1955,4511,525,531,761,0,531,1644,0,0,0,0,697,0,0,695,1246,2193,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[29644,9883,15054,104793,102027,500,73665,93,93,21279,4066,31775,32542,27954,1616,1398,2005,2769,1092,2296,2701,5950,1468,1864,1813,1521,3453,1273,2992,2326,3640,1013,496,982,1254,1058,3633,2357,5122,2513,5402,4495,3993,3892,2414,3571,3550,3751,3858,3768,121,3908,3618,1802,0,0,0,707,1679,607,424,0,151,0,0,0,151,151,151,0,0,0,0,0,0,0],[75677,29035,21328,99879,98229,647,67722,74,74,19565,2277,24308,23611,24439,367,0,0,0,500,1143,0,3117,0,0,508,477,4138,925,3946,4213,3976,4103,2374,4091,2064,2838,3968,3993,0,0,82,0,0,0,0,0,0,0,95,0,0,0,0,0,0,0,824,0,0,0,0,0,0,0,0,0,0,0,0,0,65,0,0,0,0,0],[1062285,534376,514763,127739,125039,238634,117415,211068,210975,74103,57644,32508,27811,29912,37556,51039,30660,30959,15930,16296,21043,16463,14413,16925,12146,14548,7502,9120,5701,6133,5426,5983,5360,5869,5120,5244,5211,6439,1099,3298,0,269,0,0,1279,1265,107,704,0,0,1887,0,0,906,1480,299,919,355,448,440,189,944,1057,0,394,227,756,756,756,758,66,0,0,0,0,0],[1495968,832152,764411,121880,117949,244945,122318,229142,225819,78616,89882,11290,13836,10414,85721,60626,47684,39580,51208,46780,31578,29062,23674,21354,26965,17347,4063,15128,4358,3095,3119,1173,6143,3523,4758,1301,691,235,3015,4984,2949,2930,2359,2913,2646,1113,3060,3044,2991,1234,2893,1729,1798,1248,1507,897,87,2276,1074,884,889,500,185,1110,814,854,0,0,0,0,0,0,0,0,0,0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Dryas_octopetala_Chromosome","Salix_reticulata_Chromosome","Salix_caprea_Chromosome","Salix_reticulata_Chloroplast","Salix_caprea_Chloroplast","Dryas_octopetala_Mitochondrion","Dryas_octopetala_Chloroplast","Salix_reticulata_Mitochondrion","Salix_caprea_Mitochondrion","Populus_trichocarpa_Chromosome","Carex_rostrata_Chloroplast","Populus_trichocarpa_Chloroplast","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Carex_rostrata_Chromosome","Populus_tremula_Mitochondrion","Carex_nigra_Chloroplast","Carex_sylvatica_Chloroplast","Carex_nigra_Chromosome","Carex_elata_Chromosome","Carex_acutiformis_Chromosome","Populus_tremula_Chromosome","Carex_myosuroides_Chromosome","Carex_sylvatica_Chromosome","Triglochin_maritima_Chromosome","Carex_echinata_Chloroplast","Ipomoea_triloba_Chromosome","Carex_echinata_Chromosome","Matricaria_discoidea_Chromosome","Linum_lewisii_Chromosome","Veronica_serpyllifolia_Chromosome","Citrus_sinensis_Chromosome","Eriophorum_vaginatum_Chromosome","Vaccinium_myrtillus_Chromosome","Scirpus_sylvaticus_Chromosome","Eleocharis_quinqueflora_Chromosome","Chamaenerion_angustifolium_Chromosome","Alnus_glutinosa_Chromosome","Myrica_gale_Chloroplast","Myrica_gale_Mitochondrion","Carica_papaya_Chloroplast","Citrus_sinensis_Chloroplast","Chamaenerion_angustifolium_Chloroplast","Matricaria_discoidea_Chloroplast","Citrus_sinensis_Mitochondrion","Carica_papaya_Mitochondrion","Mentha_aquatica_Chromosome","Triglochin_maritima_Chloroplast","Solanum_tuberosum_Chloroplast","Veronica_serpyllifolia_Chloroplast","Chamaenerion_angustifolium_Mitochondrion","Amborella_trichopoda_Chloroplast","Ipomoea_triloba_Chloroplast","Myrica_gale_Chromosome","Trifolium_pratense_Mitochondrion","Matricaria_discoidea_Mitochondrion","Diphasiastrum_alpinum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Scirpus_sylvaticus_Mitochondrion","Mentha_aquatica_Mitochondrion","Carex_sylvatica_Mitochondrion","Carex_echinata_Mitochondrion","Triglochin_maritima_Mitochondrion","Veronica_serpyllifolia_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_elata_Mitochondrion","Carex_rostrata_Mitochondrion","Carex_nigra_Mitochondrion","Hydra_vulgaris_Chromosome","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Mitochondrion","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,1],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,0,0,0,1,0,1,0,0,1,0,0,0,0,3,3,4],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,0,0,0,0,0,0,0,1,1,0,1,0,1,2,1,1,6,5,13],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,1,0,2,0,0,0,0,0,1,0,0,2],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,0,1,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,7,5,7],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,1,1,0,1,3,5,5,3,2,3,5,5,5,3,3,1,8,10,9,10,91,101,147],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1,0,1,1,1,1,1,2,0,2,5,2,3,1,1,4,6,5,1,3,3,7,8,13,12,11,13,11,120,136,237]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Amborella_trichopoda_Chloroplast","Carex_acutiformis_Mitochondrion","Carex_echinata_Mitochondrion","Carex_elata_Mitochondrion","Carex_nigra_Mitochondrion","Carex_rostrata_Mitochondrion","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chloroplast","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_trichocarpa_Chloroplast","Salix_caprea_Chloroplast","Salix_reticulata_Chloroplast","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Mitochondrion","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion","Carica_papaya_Mitochondrion","Eriophorum_vaginatum_Mitochondrion","Ipomoea_triloba_Chromosome","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Myrica_gale_Chromosome","Scirpus_sylvaticus_Mitochondrion","Carex_echinata_Chloroplast","Eriophorum_vaginatum_Chromosome","Alnus_glutinosa_Chromosome","Scirpus_sylvaticus_Chromosome","Triglochin_maritima_Chromosome","Vaccinium_myrtillus_Chromosome","Carex_echinata_Chromosome","Carex_nigra_Chloroplast","Carex_sylvatica_Chloroplast","Carex_myosuroides_Chromosome","Carex_elata_Chromosome","Carex_nigra_Chromosome","Carex_rostrata_Chloroplast","Carex_acutiformis_Chromosome","Carex_sylvatica_Chromosome","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chromosome","Salix_caprea_Mitochondrion","Dryas_octopetala_Mitochondrion","Salix_reticulata_Mitochondrion","Carex_rostrata_Chromosome","Salix_reticulata_Chromosome","Salix_caprea_Chromosome","Dryas_octopetala_Chromosome"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,7808,539],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,587,527,0,0,7920,0,0,0,7920,7920,0,2237,2279,2763],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1038,0,0,0,0,0,0,0,0,525,0,0,525,0,667,0,525,667,667,0,0,1327,3039,5098,8103],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,4764,4764,0,0,0,614,0,614,0,4764,0,0,809,0,0,0,0,0,0,0,1049],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,534,0,0,6478,6478,0,3114,9622,4772],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,810,810,0,560,1927,0,0,4025,1742,2530,2427,6664,6527,6198,1804,3630,613,8845,7789,8519,7908,7908,11026,65807,77381,99930],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,591,1278,1278,1278,1590,1590,1278,1278,3380,3210,2085,0,0,1278,3769,3604,3368,1278,1278,2488,4516,9600,8558,8498,12263,12882,0,1102,11820,88619,83828,164930]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Amborella_trichopoda_Chloroplast","Carex_acutiformis_Mitochondrion","Carex_echinata_Mitochondrion","Carex_elata_Mitochondrion","Carex_nigra_Mitochondrion","Carex_rostrata_Mitochondrion","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chloroplast","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_trichocarpa_Chloroplast","Salix_caprea_Chloroplast","Salix_reticulata_Chloroplast","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Mitochondrion","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion","Myrica_gale_Chromosome","Eriophorum_vaginatum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Scirpus_sylvaticus_Mitochondrion","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Carex_echinata_Chloroplast","Scirpus_sylvaticus_Chromosome","Triglochin_maritima_Chromosome","Carex_elata_Chromosome","Carex_echinata_Chromosome","Carica_papaya_Mitochondrion","Ipomoea_triloba_Chromosome","Carex_nigra_Chloroplast","Carex_nigra_Chromosome","Carex_myosuroides_Chromosome","Carex_acutiformis_Chromosome","Carex_sylvatica_Chloroplast","Carex_rostrata_Chloroplast","Carex_sylvatica_Chromosome","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chromosome","Carex_rostrata_Chromosome","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Alnus_glutinosa_Chromosome","Vaccinium_myrtillus_Chromosome","Dryas_octopetala_Mitochondrion","Salix_caprea_Chromosome","Salix_reticulata_Chromosome","Dryas_octopetala_Chromosome"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,541.15,568.87,0,0,0,0,0,0,1186.0,0,0,0,0,0,0,0,0,1392.28,941.66,0,918.64,1006.31,0,0,0,408.12,0,1515.15,561.8,0,1089.05,0,0,0,0,1141.89,2037.95,0,0,0,0,1281.23,1294.58,0,0,0,0,0,0,0,1043.71,1099.17,0,0,0,2633.55,350.88,0,0,0,3699.67,1001.47,3139.96,512.82,3199.34,3570.77,2963.6,3666.03,3344.48,4066.63],[0,0,0,0,0,0,846.0,839.48,0,0.0,0,0,0,0,2500.19,0,0,0,0,0,1089.92,0,0,0,1132.01,0,1056.28,1069.44,0,0,0,880.81,0,0,1276.6,0,0,0,0,0,0,1577.91,1737.72,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1070.88,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,174.37,181.49,0,402.43,391.47,866.1,850.28,0,257.85,0,0,720.72,722.93,690.77,1042.18,0,0,0,834.61,4046.24,1024.64,843.85,720.72,1189.19,804.99,668.72,1147.45,1515.15,822.61,3237.41,1737.97,1432.29,0,4000.0,0,1203.33,777.05,0,1359.16,3378.38,3626.94,5000.0,2898.55,805.71,936.57,2521.01,1664.07,1021.84,2462.15,1271.57,2857.14,4854.37,1708.86,2894.14,3224.04,2633.55,2168.82,3507.76,2197.8,1450.19,3699.67,11292.07,3157.89,1754.39,3192.05,3398.78,3277.46,3283.92,3294.57,4049.84],[0,0,0,0,0,0,258.34,278.94,0,178.61,541.87,906.49,973.38,0,123.09,0,0,318.98,616.13,815.6,1049.26,0,0,1182.65,736.07,0,839.35,1165.29,573.89,1684.88,1003.6,970.44,1507.09,0,1085.48,0,0,432.28,0,0,0,1094.93,931.93,0,2860.48,0,0,0,0,1424.76,1874.44,0,1680.67,891.3,1488.33,1480.45,1331.29,376.65,2022.76,0,0,0,2523.33,0,0,1820.39,0,0.0,0,376.65,3057.55,1147.78,1228.07,0,4665.63,4016.06],[0,0,0,0,0,0,395.4,396.58,0,2019.7,3000.0,3225.81,3225.81,2666.67,500.99,2666.67,2666.67,0,2842.89,2564.1,1918.98,0,0,3800.95,1151.01,0,1188.77,1247.45,4132.23,0,3065.4,1900.91,2472.09,0,1023.32,3707.8,2962.12,3668.7,4612.85,4744.14,2666.67,1467.41,1554.75,3579.95,3010.75,3113.65,5438.69,3957.5,6225.88,1869.16,2511.77,4384.72,3309.43,3155.82,2813.77,2613.24,2994.72,2647.39,2930.4,2588.04,3767.82,3652.52,2668.47,3846.15,4050.03,4962.15,0,2484.7,3959.31,2399.67,2805.61,3360.62,4407.07,5282.51,3591.38,4631.38],[0,0,0,0,0,0,550.1,557.22,0,1012.32,603.32,0.0,0.0,0,665.94,0,0,0,0,0,0,0,0,0,1371.38,0,1389.01,1375.14,0,0,0,1654.77,0,4615.38,1770.91,0,0,0,0,0,0,1450.77,1618.59,0,0,3125.0,0,0,0,4632.15,3023.76,0,1377.95,3726.71,0,3048.78,0,0,3000.0,3150.0,3663.0,2898.2,2823.23,3927.49,0,0,3762.14,3658.54,3370.22,0,3326.32,3773.14,3985.51,3645.57,3052.33,3592.81],[0,0,0,0,0,2374.67,270.52,300.57,881.06,663.66,196.97,156.13,141.66,2777.78,343.72,3042.33,3174.6,2837.84,1241.65,1361.03,670.17,2966.1,4568.53,0,1315.25,0,1398.6,1340.48,1907.79,3010.03,1875.7,1235.04,2031.53,1515.15,1261.6,0,1869.16,1988.95,2961.28,0.0,3405.87,1010.44,1118.95,4255.32,1926.6,0,0,0,0,1489.51,1556.64,3661.97,1975.47,1991.4,2908.93,2055.72,4367.61,2973.98,1983.43,3210.8,3132.45,2672.9,2851.6,3488.59,3340.76,7351.78,3699.67,0,3206.78,12040.66,3208.36,3341.48,3450.11,3668.6,3182.35,3983.23],[0,0,0,0,0,0,57.26,68.87,3161.59,597.01,195.85,222.86,218.98,0,29.63,0,0,2256.14,974.16,1112.46,946.72,4200.0,2702.7,1017.38,641.42,4336.04,733.33,685.49,1520.91,3010.03,2253.58,1380.99,1948.57,0,1450.53,2384.91,1697.68,1844.43,2036.2,1478.8,4324.32,1499.66,1141.92,3262.09,1977.41,1567.71,2255.64,1448.11,1573.73,1856.13,2190.51,1541.17,2238.7,1554.82,2746.19,2275.3,629.97,1638.23,2590.1,3829.79,2497.87,2387.04,2850.17,2894.36,8473.0,2695.42,3448.28,678.2,2526.38,2607.71,1676.71,2450.49,2261.55,2174.91,2438.51,2690.24]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Mitochondrion","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Carex_nigra_Mitochondrion","Salix_reticulata_Chloroplast","Salix_caprea_Chloroplast","Eleocharis_quinqueflora_Mitochondrion","Carex_rostrata_Chloroplast","Dryas_octopetala_Mitochondrion","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Carex_rostrata_Mitochondrion","Dryas_octopetala_Chloroplast","Carex_elata_Mitochondrion","Carex_acutiformis_Mitochondrion","Trifolium_pratense_Mitochondrion","Carex_nigra_Chloroplast","Carex_sylvatica_Chloroplast","Populus_tremula_Mitochondrion","Carex_sylvatica_Mitochondrion","Veronica_serpyllifolia_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Populus_tremula_Chloroplast","Triglochin_maritima_Mitochondrion","Populus_euphratica_Chloroplast","Populus_trichocarpa_Chloroplast","Chamaenerion_angustifolium_Mitochondrion","Matricaria_discoidea_Mitochondrion","Carex_myosuroides_Chromosome","Salix_reticulata_Chromosome","Myrica_gale_Mitochondrion","Hydra_vulgaris_Chromosome","Salix_caprea_Chromosome","Ipomoea_triloba_Chloroplast","Mentha_aquatica_Chromosome","Myrica_gale_Chromosome","Scirpus_sylvaticus_Mitochondrion","Triglochin_maritima_Chloroplast","Carex_echinata_Mitochondrion","Populus_trichocarpa_Chromosome","Dryas_octopetala_Chromosome","Mentha_aquatica_Mitochondrion","Carex_sylvatica_Chromosome","Solanum_tuberosum_Chloroplast","Amborella_trichopoda_Chloroplast","Veronica_serpyllifolia_Chloroplast","Matricaria_discoidea_Chloroplast","Carex_rostrata_Chromosome","Carex_echinata_Chromosome","Eriophorum_vaginatum_Mitochondrion","Triglochin_maritima_Chromosome","Carex_echinata_Chloroplast","Carex_acutiformis_Chromosome","Carex_elata_Chromosome","Myrica_gale_Chloroplast","Citrus_sinensis_Chloroplast","Carex_nigra_Chromosome","Alnus_glutinosa_Chromosome","Vaccinium_myrtillus_Chromosome","Citrus_sinensis_Chromosome","Populus_tremula_Chromosome","Chamaenerion_angustifolium_Chromosome","Eleocharis_quinqueflora_Chloroplast","Carica_papaya_Mitochondrion","Diphasiastrum_alpinum_Chromosome","Carica_papaya_Chloroplast","Veronica_serpyllifolia_Chromosome","Citrus_sinensis_Mitochondrion","Eriophorum_vaginatum_Chromosome","Linum_lewisii_Chromosome","Ipomoea_triloba_Chromosome","Matricaria_discoidea_Chromosome","Scirpus_sylvaticus_Chromosome","Eleocharis_quinqueflora_Chromosome"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0.0,0,0,0.0,0,0,0,0,0,0,0,0,73.61,0,0,0,34.21,48.85,0,0,29.59,0,0,0,0,0,0,0,0,0.0,97.7,48.85,0,48.85,0,62.5,75.5,0,72.44,43.49,0,0,72.89,0,0.0,64.1,0,0,0,0,0,0,58.3,168.66,0,0,244.98,0.0,200.67,0,0,0,219.74,280.9,268.71,298.84,0,0,0,0,164.07,287.19,273.34],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,67.57,0,0,0,55.17,0,0,0,64.58,0,0,0,0,0,0.0,0,0,0,0,0,0,0,0,53.47,57.2,0,53.48,0,0,0,46.41,0,0,0,0,0,0,0,0,0,93.7,57.92,0.0,0,0,254.97,0,0,0,0,0,294.6,0,0,0,0,0,0,0,0,0],[0,0.0,0,0.0,0,0,0.0,0,0,0,0,0.0,0.0,0.0,0,11.83,79.77,86.26,21.59,27.43,0.0,0.0,17.42,30.89,0.0,61.98,0,0.0,25.97,40.24,0.0,0.0,0.0,0.0,0,172.12,217.16,20.04,52.24,51.83,324.32,47.69,126.98,169.8,0.0,88.9,0.0,133.69,194.93,0,0,0,0,52.0,92.46,54.07,54.09,148.88,189.16,239.64,0.0,193.8,260.42,304.65,85.14,214.49,62.04,271.0,298.84,0,382.51,0.0,383.36,209.31,281.95,329.24],[0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,11.45,101.38,104.49,37.5,27.05,0,0,54.59,14.9,0,47.99,0,0,51.47,28.96,105.35,239.23,0,0.0,262.81,132.98,0.0,65.22,57.74,41.09,227.69,49.07,153.61,100.56,87.57,74.49,27.93,0,0.0,0,0,0,0,336.13,82.0,53.38,61.07,177.84,242.72,0.0,34.57,0.0,144.09,0,89.26,0.0,50.96,0,0,0,0,0,0,179.86,0,263.16],[0,0.0,0,0,0,0,0,0,0,0,0,0.0,0,138.35,0,35.69,0.0,0.0,0.0,37.89,180.09,266.52,0.0,47.07,0.0,0.0,0.0,230.89,0.0,49.26,0.0,0,185.92,355.95,175.04,79.74,259.6,0.0,95.71,102.96,0,101.67,215.31,74.05,268.82,68.71,0.0,398.75,248.24,666.67,666.67,666.67,666.67,0.0,311.53,141.56,53.96,142.15,196.24,0.0,202.67,79.81,277.93,0.0,235.48,43.08,126.25,100.3,0.0,715.99,0.0,129.74,219.78,601.2,164.97,144.97],[0,0,0,0.0,0,0,0.0,0,0,0,0,0,0,0,0,19.37,0.0,0.0,0,50.48,0,0,0,51.32,0,150.83,0,0,0,88.03,0.0,0,0,0,0,0,0.0,207.04,66.27,59.83,0,78.01,0,0,0,56.79,348.43,0,0,0,0,0,0,0.0,0.0,144.79,125.52,0,0,35.22,128.33,96.9,0,375.0,323.97,261.03,135.86,278.48,365.32,0,488.4,1041.67,453.17,42.11,377.26,386.47],[0.0,0.0,0,0.0,0,0.0,0.0,0,0,0,0.0,0.0,0,0,0.0,17.67,30.94,29.0,106.94,25.69,0,0.0,111.39,42.46,222.72,34.38,227.79,0,132.98,85.58,125.53,0.0,0,0.0,0,30.32,0,138.77,126.16,126.16,0.0,132.2,0.0,151.05,141.84,69.17,104.32,0.0,156.37,0.0,0.0,0.0,0.0,107.0,128.13,86.29,103.22,115.61,158.1,266.82,78.87,214.76,110.5,248.18,88.32,195.6,92.52,245.74,267.29,531.91,357.51,0,383.36,167.88,276.45,253.1],[0,0.0,0,0.0,0,0.0,0,0,0,0,0.0,132.1,180.67,55.46,200.0,0.82,52.05,46.42,100.43,13.28,68.42,32.86,116.22,22.38,93.11,37.52,113.12,115.67,139.01,63.44,140.64,132.71,241.35,0.0,0.0,80.35,0.0,98.26,18.04,30.56,0.0,20.36,33.16,79.19,84.34,107.12,115.37,97.94,0.0,0,0.0,0,0,174.2,120.32,102.54,112.97,168.53,0.0,0.0,137.52,63.06,80.19,0.0,112.84,134.27,101.02,45.79,0.0,0.0,28.38,100.07,0.0,97.67,191.88,24.58]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Carex_nigra_Mitochondrion","Chamaenerion_angustifolium_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Hydra_vulgaris_Chromosome","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Veronica_serpyllifolia_Mitochondrion","Eriophorum_vaginatum_Mitochondrion","Triglochin_maritima_Mitochondrion","Ipomoea_triloba_Chloroplast","Carex_sylvatica_Mitochondrion","Dryas_octopetala_Chloroplast","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Carex_sylvatica_Chloroplast","Salix_reticulata_Chloroplast","Matricaria_discoidea_Chloroplast","Triglochin_maritima_Chloroplast","Carex_nigra_Chloroplast","Salix_caprea_Chloroplast","Eleocharis_quinqueflora_Chloroplast","Dryas_octopetala_Mitochondrion","Scirpus_sylvaticus_Mitochondrion","Amborella_trichopoda_Chloroplast","Carex_myosuroides_Chromosome","Carex_rostrata_Chloroplast","Carex_nigra_Chromosome","Trifolium_pratense_Mitochondrion","Veronica_serpyllifolia_Chloroplast","Citrus_sinensis_Chloroplast","Chamaenerion_angustifolium_Chloroplast","Myrica_gale_Mitochondrion","Carica_papaya_Chloroplast","Carex_echinata_Chloroplast","Populus_trichocarpa_Chloroplast","Populus_euphratica_Chloroplast","Matricaria_discoidea_Mitochondrion","Populus_tremula_Chloroplast","Myrica_gale_Chloroplast","Carex_acutiformis_Chromosome","Carex_sylvatica_Chromosome","Populus_trichocarpa_Chromosome","Carex_elata_Chromosome","Mentha_aquatica_Chromosome","Citrus_sinensis_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_echinata_Mitochondrion","Carex_elata_Mitochondrion","Carex_rostrata_Mitochondrion","Triglochin_maritima_Chromosome","Carex_rostrata_Chromosome","Salix_reticulata_Chromosome","Dryas_octopetala_Chromosome","Populus_tremula_Mitochondrion","Carica_papaya_Mitochondrion","Eleocharis_quinqueflora_Chromosome","Populus_tremula_Chromosome","Scirpus_sylvaticus_Chromosome","Myrica_gale_Chromosome","Alnus_glutinosa_Chromosome","Carex_echinata_Chromosome","Linum_lewisii_Chromosome","Salix_caprea_Chromosome","Matricaria_discoidea_Chromosome","Citrus_sinensis_Chromosome","Mentha_aquatica_Mitochondrion","Vaccinium_myrtillus_Chromosome","Solanum_tuberosum_Chloroplast","Chamaenerion_angustifolium_Chromosome","Eriophorum_vaginatum_Chromosome","Veronica_serpyllifolia_Chromosome","Ipomoea_triloba_Chromosome"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0,0,0.0,0,0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0,0,0.0,0,0.0,0,0.0,0,0.0,0,0.0,0.0,0.0,0,0.0,0,0.0,0,0,0,0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0,0,0,0,0,0,0,0.0,0.0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0,0,0,0,0,0,0,0,0,0,0],[0.0,0.0,0.0,0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0],[0,0,0.0,0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0,0,0,0.0,0.0,0.0,0,0.0,0,0.0,0,0,0,0.0,0,0.0,0,0,0.0,0,0,0.0,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0,0.0,0,0.0,0,0,0,0,0],[0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0],[0.0,0,0,0,0.0,0.0,0,0.0,0,0,0,0.0,0,0.0,0.0,0,0,0,0,0.0,0,0,0.0,0,0,0.0,0,0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0.0,0,0.0,0,0.0,0,0.0,0,0.0,0,0,0,0,0,0,0,0,0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0,0.0,0,0.0,0,0.0,0],[0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0,0.0,0,0.0,0.0],[0.0,0.0,0.0,0,0.0,0.0,0.0,0.0,0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0.0,0.0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0,0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Alnus_glutinosa_Chromosome","Amborella_trichopoda_Chloroplast","Carex_acutiformis_Chromosome","Carex_acutiformis_Mitochondrion","Carex_echinata_Chloroplast","Carex_echinata_Chromosome","Carex_echinata_Mitochondrion","Carex_elata_Chromosome","Carex_elata_Mitochondrion","Carex_myosuroides_Chromosome","Carex_nigra_Chloroplast","Carex_nigra_Chromosome","Carex_nigra_Mitochondrion","Carex_rostrata_Chloroplast","Carex_rostrata_Chromosome","Carex_rostrata_Mitochondrion","Carex_sylvatica_Chloroplast","Carex_sylvatica_Chromosome","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Carica_papaya_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chloroplast","Dryas_octopetala_Chromosome","Dryas_octopetala_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Eriophorum_vaginatum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Ipomoea_triloba_Chromosome","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Chromosome","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chloroplast","Populus_trichocarpa_Chromosome","Salix_caprea_Chloroplast","Salix_caprea_Chromosome","Salix_caprea_Mitochondrion","Salix_reticulata_Chloroplast","Salix_reticulata_Chromosome","Salix_reticulata_Mitochondrion","Scirpus_sylvaticus_Chromosome","Scirpus_sylvaticus_Mitochondrion","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Chromosome","Triglochin_maritima_Mitochondrion","Vaccinium_myrtillus_Chromosome","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[20.678,19.547,7.729,0,10.188,10.138,9.71,0,0,0,0,0,0,0,2.872,2.557,2.557,2.734,2.552,0,0,1.034,0,0,0,0,0,0,0,0.243,0.003,0,0,0.002,0.0,0,0,0,0,0,0,0.002,0,0,0,0,0,0,0,0,0,0.0,0,0,0.002,0,0.002,0.001,0.001,0.001,0,0,0.001,0.001,0,0,0,0.0,0,0.0,0,0.0,0.0,0,0,0],[31.47,30.856,8.42,0,16.673,16.728,14.347,0.356,0,0,0,0,0,0.094,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0.004,0,0,0.002,0.001,0,0,0,0,0,0,0.004,0,0,0,0,0,0,0,0,0,0.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[65.647,66.579,53.449,8.845,15.848,16.0,13.407,8.997,1.448,1.262,5.562,4.307,2.787,0.772,0.983,0.575,0.064,0.092,0,0.064,0.286,0.093,0.119,0.172,0.358,0.414,0.333,0.045,0.184,0.16,0.064,0.112,0,0.03,0.026,0,0.043,0.006,0,0,0,0.008,0,0,0,0,0,0.002,0.001,0.002,0.0,0.0,0.001,0.001,0.002,0.001,0.002,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.0,0.001,0,0.0,0,0.0,0,0.0,0.0,0.0,0,0],[47.545,47.467,44.169,14.43,12.132,10.868,9.121,9.376,2.767,2.827,6.211,4.958,2.569,1.436,1.219,0.328,0.332,0,0.474,0,0,0,0,0,0.928,0.983,0.346,0,0.415,0.083,0.099,0.141,0,0.045,0.039,0,0,0,0,0,0,0.007,0,0,0,0,0,0.003,0.001,0.001,0.001,0.001,0.001,0.001,0.0,0.001,0,0.0,0.0,0,0,0,0.0,0.0,0.0,0,0,0,0,0,0,0,0,0.0,0,0],[66.195,64.182,46.06,0.144,19.96,20.444,17.61,1.838,0.015,0.014,0.971,1.287,0.849,0.18,3.19,3.369,2.807,2.596,2.493,2.406,2.482,2.495,2.396,2.234,0.516,0,0.748,0.832,0,0.377,0.013,0.024,0.138,0.003,0.004,0,0,0.038,0.029,0.009,0,0.005,0.011,0.011,0.009,0,0,0.0,0.001,0.001,0.0,0.002,0.001,0.0,0.001,0.0,0.0,0.0,0.0,0.001,0.001,0.001,0.0,0.0,0.0,0.0,0,0,0,0,0,0.0,0.0,0.0,0,0],[62.427,61.366,42.435,0.191,15.374,14.925,15.573,1.028,0.012,0.012,0,0,0.27,0,0,0.051,0,0,0,0,0.062,0,0,0,0,0,0,0,0,0,0.033,0,0,0.008,0.006,0,0,0,0,0,0,0.005,0,0,0,0,0,0.0,0.0,0,0.0,0.001,0,0,0.001,0.0,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0.0,0.001,0,0.0,0,0.0,0,0.0,0,0,0,0],[80.113,77.199,71.547,67.97,20.19,17.696,18.863,25.914,34.666,32.709,14.786,14.344,8.048,6.515,0.686,0,0.168,0,0,0.451,0,0,0,0,0.678,0.134,0.265,0.222,0.49,0.2,0.463,0.381,0.062,0.152,0.141,0.118,0,0.019,0.021,0.065,0.051,0.019,0.057,0.056,0.048,0.011,0.052,0.01,0.004,0.005,0.004,0.004,0.005,0.004,0.002,0.002,0.002,0.001,0.002,0.001,0.001,0.001,0.001,0.001,0.001,0.001,0,0.0,0,0.0,0,0.0,0.0,0.0,0,0],[77.493,74.661,76.82,69.979,7.06,8.351,6.28,39.951,37.094,35.502,22.924,18.05,9.661,7.727,1.883,1.842,1.83,1.952,1.471,1.951,1.931,0.824,1.063,1.115,1.023,0.401,0.233,0.532,0.499,0.413,0.652,0.584,0.292,0.237,0.209,0.244,0.278,0.123,0.042,0.011,0.027,0.02,0,0,0,0.043,0,0.023,0.011,0.008,0.012,0.007,0.006,0.006,0.001,0.004,0.0,0.002,0.001,0.001,0.0,0.0,0.0,0.0,0.003,0.001,0,0.0,0,0,0,0.0,0.0,0.0,0,0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Salix_reticulata_Chloroplast","Salix_caprea_Chloroplast","Dryas_octopetala_Chloroplast","Dryas_octopetala_Mitochondrion","Populus_trichocarpa_Chloroplast","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Carex_rostrata_Chloroplast","Salix_caprea_Mitochondrion","Salix_reticulata_Mitochondrion","Carex_nigra_Chloroplast","Carex_sylvatica_Chloroplast","Carex_echinata_Chloroplast","Populus_tremula_Mitochondrion","Myrica_gale_Chloroplast","Carica_papaya_Chloroplast","Citrus_sinensis_Chloroplast","Matricaria_discoidea_Chloroplast","Chamaenerion_angustifolium_Chloroplast","Triglochin_maritima_Chloroplast","Solanum_tuberosum_Chloroplast","Veronica_serpyllifolia_Chloroplast","Amborella_trichopoda_Chloroplast","Ipomoea_triloba_Chloroplast","Myrica_gale_Mitochondrion","Matricaria_discoidea_Mitochondrion","Carica_papaya_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Trifolium_pratense_Mitochondrion","Citrus_sinensis_Mitochondrion","Dryas_octopetala_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Mentha_aquatica_Mitochondrion","Salix_reticulata_Chromosome","Salix_caprea_Chromosome","Veronica_serpyllifolia_Mitochondrion","Triglochin_maritima_Mitochondrion","Eriophorum_vaginatum_Mitochondrion","Scirpus_sylvaticus_Mitochondrion","Carex_echinata_Mitochondrion","Carex_sylvatica_Mitochondrion","Populus_trichocarpa_Chromosome","Carex_elata_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_rostrata_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Carex_nigra_Mitochondrion","Carex_rostrata_Chromosome","Carex_elata_Chromosome","Carex_acutiformis_Chromosome","Carex_nigra_Chromosome","Populus_tremula_Chromosome","Carex_sylvatica_Chromosome","Carex_myosuroides_Chromosome","Ipomoea_triloba_Chromosome","Carex_echinata_Chromosome","Citrus_sinensis_Chromosome","Eriophorum_vaginatum_Chromosome","Scirpus_sylvaticus_Chromosome","Veronica_serpyllifolia_Chromosome","Alnus_glutinosa_Chromosome","Chamaenerion_angustifolium_Chromosome","Eleocharis_quinqueflora_Chromosome","Linum_lewisii_Chromosome","Triglochin_maritima_Chromosome","Vaccinium_myrtillus_Chromosome","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Hydra_vulgaris_Chromosome","Limulus_polyphemus_Mitochondrion","Matricaria_discoidea_Chromosome","Mentha_aquatica_Chromosome","Myrica_gale_Chromosome","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,1.0,1.0,0,1.0,1.0,0,0,0,1.0,0,1.0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.001,1.0,0,1.0,0,1.001,0,0.999,0,0,0,0,0.999,1.113,1.0,1.001,1.005,1.006,1.013,1.013,1.007,1.046,1.041,1.04,1.01,0.999,0.999,0,1.0,0,0,0,1.0,1.0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.003,0,0,0,0,0,0,0,0,0,0,0,1.0,0,0,0,0,1.01,1.011,1.012,1.033,1.03,1.005,1.011,1.003,1.195,1.044,0,0,0,1.008,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,1.0,1.0,1.008,0,1.0,1.0,0,2.716,1.0,0.999,1.001,1.003,0.999,1.0,1.053,1.045,4.364,1.286,1.053,1.006,1.176,0,0.999,1.163,1.041,3.912,1.152,1.141,1.144,0,1.002,1.193,0.998,1.124,1.126,1.644,1.076,0.999,1.242,1.145,1.026,1.225,1.058,1.0,1.159,1.005,1.04,1.068,1.035,1.03,1.073,1.014,1.014,1.022,1.037,1.154,1.206,1.869,1.0,6.345,7.55,8.297,1.376,7.33],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,1.005,1.0,0,0.994,0,0,0,0,0,1.006,0,1.001,1.007,1.008,0,1.002,1.052,1.006,1.002,0,0.98,1.067,1.031,0,1.072,1.078,1.061,0,0,1.001,0,0,1.011,1.0,1.023,0,1.067,1.088,1.009,1.004,0.999,1.001,1.0,1.011,1.004,1.014,1.017,1.02,1.006,1.022,1.013,1.012,1.089,1.867,1.614,0,1.728,0,0,0,1.011,1.0],[0,0,0,0,0,0,1.007,1.007,1.007,0,0,0,0,1.001,0,1.222,1.265,0,1.0,1.001,0,0.998,1.011,1.002,1.0,1.0,1.146,1.2,0,1.002,1.172,1.336,1.426,1.007,1.003,1.458,1.0,1.248,1.0,1.0,0.994,1.012,1.0,1.159,1.0,1.31,1.379,0.998,1.085,1.0,1.375,1.165,1.044,1.455,1.001,0.977,1.0,1.019,1.028,1.026,1.032,1.037,1.029,1.003,1.074,1.024,1.078,0.994,1.554,1.263,1.027,1.731,1.27,1.0,1.071,1.083],[0,0,0,0,0,0,0,0,0,0,0,0,1.0,0,0,0,0,0,0,0,1.0,0.999,0,0,0.999,0,0,0,0,0,0,0,0,0,1.012,0,0.995,0,1.0,1.0,0,0,0.999,0,0.998,1.0,1.222,0,1.455,0.999,1.014,1.429,1.189,1.231,1.0,1.116,1.0,1.028,1.033,1.03,1.045,1.05,1.026,1.07,1.051,1.032,1.062,1.189,1.159,5.417,1.151,0,0,0,6.341,0],[0,0,0,0,0,1.848,1.853,1.853,1.853,2.109,2.289,2.286,1.0,0,1.0,0,0,1.451,2.123,0,1.0,0.999,1.0,1.001,1.161,1.264,1.043,1.042,0,1.153,1.22,1.348,1.094,1.676,1.007,1.158,1.036,0,1.02,1.022,1.038,3.782,1.195,1.083,1.012,1.094,1.059,1.22,1.149,1.059,1.18,1.266,1.043,1.238,1.095,1.089,1.058,1.024,1.003,1.026,1.031,1.046,1.014,1.038,1.035,1.044,1.094,1.179,1.197,0,1.338,1.0,1.224,0.998,0,1.0],[0,0,0,0,0,0,0,0,0,1.06,1.0,1.474,0,1.004,1.0,1.002,0.996,1.001,1.446,1.0,1.0,1.006,1.001,1.101,1.068,1.002,1.025,1.019,1.003,1.059,1.14,1.0,1.141,3.195,1.045,1.242,1.028,1.002,1.024,1.028,1.048,1.521,1.178,1.001,2.515,1.005,1.025,1.002,1.104,1.85,1.12,1.142,1.027,1.225,1.279,1.424,1.612,1.035,1.057,1.018,1.011,1.015,1.091,1.051,1.043,1.007,1.176,1.415,1.259,1.0,1.31,1.206,1.167,1.172,1.005,1.001]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Mitochondrion","Limulus_polyphemus_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Carex_nigra_Mitochondrion","Carex_acutiformis_Mitochondrion","Carex_elata_Mitochondrion","Carex_rostrata_Mitochondrion","Carex_sylvatica_Mitochondrion","Veronica_serpyllifolia_Mitochondrion","Eleocharis_quinqueflora_Mitochondrion","Hydra_vulgaris_Chromosome","Matricaria_discoidea_Chloroplast","Matricaria_discoidea_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Veronica_serpyllifolia_Chloroplast","Trifolium_pratense_Mitochondrion","Scirpus_sylvaticus_Mitochondrion","Ipomoea_triloba_Chloroplast","Diphasiastrum_alpinum_Chromosome","Chamaenerion_angustifolium_Chromosome","Mentha_aquatica_Chromosome","Myrica_gale_Chromosome","Vaccinium_myrtillus_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Carex_sylvatica_Chloroplast","Carex_nigra_Chloroplast","Triglochin_maritima_Mitochondrion","Myrica_gale_Mitochondrion","Carex_acutiformis_Chromosome","Carica_papaya_Mitochondrion","Carex_myosuroides_Chromosome","Carex_echinata_Mitochondrion","Triglochin_maritima_Chromosome","Carex_sylvatica_Chromosome","Dryas_octopetala_Mitochondrion","Amborella_trichopoda_Chloroplast","Salix_reticulata_Mitochondrion","Salix_caprea_Mitochondrion","Populus_tremula_Mitochondrion","Mentha_aquatica_Mitochondrion","Matricaria_discoidea_Chromosome","Myrica_gale_Chloroplast","Alnus_glutinosa_Chromosome","Veronica_serpyllifolia_Chromosome","Carex_echinata_Chloroplast","Citrus_sinensis_Mitochondrion","Carex_rostrata_Chromosome","Citrus_sinensis_Chromosome","Carex_nigra_Chromosome","Carex_elata_Chromosome","Carex_rostrata_Chloroplast","Carex_echinata_Chromosome","Ipomoea_triloba_Chromosome","Linum_lewisii_Chromosome","Eleocharis_quinqueflora_Chromosome","Dryas_octopetala_Chromosome","Populus_euphratica_Chloroplast","Populus_trichocarpa_Chloroplast","Salix_reticulata_Chloroplast","Salix_caprea_Chloroplast","Populus_tremula_Chloroplast","Salix_caprea_Chromosome","Salix_reticulata_Chromosome","Dryas_octopetala_Chloroplast","Populus_trichocarpa_Chromosome","Eriophorum_vaginatum_Chromosome","Scirpus_sylvaticus_Chromosome","Solanum_tuberosum_Chloroplast","Populus_tremula_Chromosome","Eriophorum_vaginatum_Mitochondrion","Triglochin_maritima_Chloroplast","Eleocharis_quinqueflora_Chloroplast","Carica_papaya_Chloroplast","Citrus_sinensis_Chloroplast"]},
                {"coord_x":[[0.755,1.755,2.755,3.755,4.755,5.755,6.755,7.755,8.755,9.755,10.755,11.755,12.755,13.755,14.755,15.755,16.755,17.755,18.755,19.755,20.755,21.755,22.755,23.755,24.755,25.755,26.755,27.755,28.755,29.755,30.755,31.755,32.755,33.755,34.755,35.755,36.755,37.755,38.755,39.755,40.755,41.755,42.755,43.755,44.755,45.755,46.755,47.755,48.755,49.755,50.755,51.755,52.755,53.755,54.755,55.755,56.755,57.755,58.755,59.755,60.755,61.755,62.755,63.755,64.755,65.755,66.755,67.755,68.755,69.755,70.755,71.755,72.755,73.755,74.755,75.755],[0.825,1.825,2.825,3.825,4.825,5.825,6.825,7.825,8.825,9.825,10.825,11.825,12.825,13.825,14.825,15.825,16.825,17.825,18.825,19.825,20.825,21.825,22.825,23.825,24.825,25.825,26.825,27.825,28.825,29.825,30.825,31.825,32.825,33.825,34.825,35.825,36.825,37.825,38.825,39.825,40.825,41.825,42.825,43.825,44.825,45.825,46.825,47.825,48.825,49.825,50.825,51.825,52.825,53.825,54.825,55.825,56.825,57.825,58.825,59.825,60.825,61.825,62.825,63.825,64.825,65.825,66.825,67.825,68.825,69.825,70.825,71.825,72.825,73.825,74.825,75.825],[0.895,1.895,2.895,3.895,4.895,5.895,6.895,7.895,8.895,9.895,10.895,11.895,12.895,13.895,14.895,15.895,16.895,17.895,18.895,19.895,20.895,21.895,22.895,23.895,24.895,25.895,26.895,27.895,28.895,29.895,30.895,31.895,32.895,33.895,34.895,35.895,36.895,37.895,38.895,39.895,40.895,41.895,42.895,43.895,44.895,45.895,46.895,47.895,48.895,49.895,50.895,51.895,52.895,53.895,54.895,55.895,56.895,57.895,58.895,59.895,60.895,61.895,62.895,63.895,64.895,65.895,66.895,67.895,68.895,69.895,70.895,71.895,72.895,73.895,74.895,75.895],[0.965,1.965,2.965,3.965,4.965,5.965,6.965,7.965,8.965,9.965,10.965,11.965,12.965,13.965,14.965,15.965,16.965,17.965,18.965,19.965,20.965,21.965,22.965,23.965,24.965,25.965,26.965,27.965,28.965,29.965,30.965,31.965,32.965,33.965,34.965,35.965,36.965,37.965,38.965,39.965,40.965,41.965,42.965,43.965,44.965,45.965,46.965,47.965,48.965,49.965,50.965,51.965,52.965,53.965,54.965,55.965,56.965,57.965,58.965,59.965,60.965,61.965,62.965,63.965,64.965,65.965,66.965,67.965,68.965,69.965,70.965,71.965,72.965,73.965,74.965,75.965],[1.035,2.035,3.035,4.035,5.035,6.035,7.035,8.035,9.035,10.035,11.035,12.035,13.035,14.035,15.035,16.035,17.035,18.035,19.035,20.035,21.035,22.035,23.035,24.035,25.035,26.035,27.035,28.035,29.035,30.035,31.035,32.035,33.035,34.035,35.035,36.035,37.035,38.035,39.035,40.035,41.035,42.035,43.035,44.035,45.035,46.035,47.035,48.035,49.035,50.035,51.035,52.035,53.035,54.035,55.035,56.035,57.035,58.035,59.035,60.035,61.035,62.035,63.035,64.035,65.035,66.035,67.035,68.035,69.035,70.035,71.035,72.035,73.035,74.035,75.035,76.035],[1.105,2.105,3.105,4.105,5.105,6.105,7.105,8.105,9.105,10.105,11.105,12.105,13.105,14.105,15.105,16.105,17.105,18.105,19.105,20.105,21.105,22.105,23.105,24.105,25.105,26.105,27.105,28.105,29.105,30.105,31.105,32.105,33.105,34.105,35.105,36.105,37.105,38.105,39.105,40.105,41.105,42.105,43.105,44.105,45.105,46.105,47.105,48.105,49.105,50.105,51.105,52.105,53.105,54.105,55.105,56.105,57.105,58.105,59.105,60.105,61.105,62.105,63.105,64.105,65.105,66.105,67.105,68.105,69.105,70.105,71.105,72.105,73.105,74.105,75.105,76.105],[1.175,2.175,3.175,4.175,5.175,6.175,7.175,8.175,9.175,10.175,11.175,12.175,13.175,14.175,15.175,16.175,17.175,18.175,19.175,20.175,21.175,22.175,23.175,24.175,25.175,26.175,27.175,28.175,29.175,30.175,31.175,32.175,33.175,34.175,35.175,36.175,37.175,38.175,39.175,40.175,41.175,42.175,43.175,44.175,45.175,46.175,47.175,48.175,49.175,50.175,51.175,52.175,53.175,54.175,55.175,56.175,57.175,58.175,59.175,60.175,61.175,62.175,63.175,64.175,65.175,66.175,67.175,68.175,69.175,70.175,71.175,72.175,73.175,74.175,75.175,76.175],[1.245,2.245,3.245,4.245,5.245,6.245,7.245,8.245,9.245,10.245,11.245,12.245,13.245,14.245,15.245,16.245,17.245,18.245,19.245,20.245,21.245,22.245,23.245,24.245,25.245,26.245,27.245,28.245,29.245,30.245,31.245,32.245,33.245,34.245,35.245,36.245,37.245,38.245,39.245,40.245,41.245,42.245,43.245,44.245,45.245,46.245,47.245,48.245,49.245,50.245,51.245,52.245,53.245,54.245,55.245,56.245,57.245,58.245,59.245,60.245,61.245,62.245,63.245,64.245,65.245,66.245,67.245,68.245,69.245,70.245,71.245,72.245,73.245,74.245,75.245,76.245]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[558,777,812,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,809,756,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,772,756,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[2690,4938,4212,727,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[9587,4897,4897,774,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Dryas_octopetala_Chloroplast","Salix_reticulata_Chloroplast","Salix_caprea_Chloroplast","Dryas_octopetala_Mitochondrion","Alnus_glutinosa_Chromosome","Amborella_trichopoda_Chloroplast","Carex_acutiformis_Chromosome","Carex_acutiformis_Mitochondrion","Carex_echinata_Chloroplast","Carex_echinata_Chromosome","Carex_echinata_Mitochondrion","Carex_elata_Chromosome","Carex_elata_Mitochondrion","Carex_myosuroides_Chromosome","Carex_nigra_Chloroplast","Carex_nigra_Chromosome","Carex_nigra_Mitochondrion","Carex_rostrata_Chloroplast","Carex_rostrata_Chromosome","Carex_rostrata_Mitochondrion","Carex_sylvatica_Chloroplast","Carex_sylvatica_Chromosome","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Carica_papaya_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chromosome","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Eriophorum_vaginatum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Ipomoea_triloba_Chromosome","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Chromosome","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chloroplast","Populus_trichocarpa_Chromosome","Salix_caprea_Chromosome","Salix_caprea_Mitochondrion","Salix_reticulata_Chromosome","Salix_reticulata_Mitochondrion","Scirpus_sylvaticus_Chromosome","Scirpus_sylvaticus_Mitochondrion","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Chromosome","Triglochin_maritima_Mitochondrion","Vaccinium_myrtillus_Chromosome","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion"]}]
            </div>
            <div id='misassemblies-json'>
                {"coord_x":[[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76],[1,1,1,2,2,2,3,3,3,4,4,4,5,5,5,6,6,6,7,7,7,8,8,8,9,9,9,10,10,10,11,11,11,12,12,12,13,13,13,14,14,14,15,15,15,16,16,16,17,17,17,18,18,18,19,19,19,20,20,20,21,21,21,22,22,22,23,23,23,24,24,24,25,25,25,26,26,26,27,27,27,28,28,28,29,29,29,30,30,30,31,31,31,32,32,32,33,33,33,34,34,34,35,35,35,36,36,36,37,37,37,38,38,38,39,39,39,40,40,40,41,41,41,42,42,42,43,43,43,44,44,44,45,45,45,46,46,46,47,47,47,48,48,48,49,49,49,50,50,50,51,51,51,52,52,52,53,53,53,54,54,54,55,55,55,56,56,56,57,57,57,58,58,58,59,59,59,60,60,60,61,61,61,62,62,62,63,63,63,64,64,64,65,65,65,66,66,66,67,67,67,68,68,68,69,69,69,70,70,70,71,71,71,72,72,72,73,73,73,74,74,74,75,75,75,76,76,76]],"coord_y":[[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]],"filenames":["megahit_1mil_short","spades_1mil_short","megahit_10mil_short","spades_10mil_short","megahit_1mil_long","spades_1mil_long","megahit_10mil_long","spades_10mil_long"],"refnames":["Alnus_glutinosa_Chromosome","Amborella_trichopoda_Chloroplast","Carex_acutiformis_Chromosome","Carex_acutiformis_Mitochondrion","Carex_echinata_Chloroplast","Carex_echinata_Chromosome","Carex_echinata_Mitochondrion","Carex_elata_Chromosome","Carex_elata_Mitochondrion","Carex_myosuroides_Chromosome","Carex_nigra_Chloroplast","Carex_nigra_Chromosome","Carex_nigra_Mitochondrion","Carex_rostrata_Chloroplast","Carex_rostrata_Chromosome","Carex_rostrata_Mitochondrion","Carex_sylvatica_Chloroplast","Carex_sylvatica_Chromosome","Carex_sylvatica_Mitochondrion","Carica_papaya_Chloroplast","Carica_papaya_Mitochondrion","Chamaenerion_angustifolium_Chloroplast","Chamaenerion_angustifolium_Chromosome","Chamaenerion_angustifolium_Mitochondrion","Citrus_sinensis_Chloroplast","Citrus_sinensis_Chromosome","Citrus_sinensis_Mitochondrion","Diphasiastrum_alpinum_Chloroplast","Diphasiastrum_alpinum_Chromosome","Diphasiastrum_alpinum_Mitochondrion","Dryas_octopetala_Chloroplast","Dryas_octopetala_Chromosome","Dryas_octopetala_Mitochondrion","Eleocharis_quinqueflora_Chloroplast","Eleocharis_quinqueflora_Chromosome","Eleocharis_quinqueflora_Mitochondrion","Eriophorum_vaginatum_Chromosome","Eriophorum_vaginatum_Mitochondrion","Hydra_vulgaris_Chromosome","Ipomoea_triloba_Chloroplast","Ipomoea_triloba_Chromosome","Limulus_polyphemus_Mitochondrion","Linum_lewisii_Chromosome","Matricaria_discoidea_Chloroplast","Matricaria_discoidea_Chromosome","Matricaria_discoidea_Mitochondrion","Mentha_aquatica_Chromosome","Mentha_aquatica_Mitochondrion","Myrica_gale_Chloroplast","Myrica_gale_Chromosome","Myrica_gale_Mitochondrion","Nilaparvata_lugens_Chromosome","Nilaparvata_lugens_Mitochondrion","Populus_euphratica_Chloroplast","Populus_tremula_Chloroplast","Populus_tremula_Chromosome","Populus_tremula_Mitochondrion","Populus_trichocarpa_Chloroplast","Populus_trichocarpa_Chromosome","Salix_caprea_Chloroplast","Salix_caprea_Chromosome","Salix_caprea_Mitochondrion","Salix_reticulata_Chloroplast","Salix_reticulata_Chromosome","Salix_reticulata_Mitochondrion","Scirpus_sylvaticus_Chromosome","Scirpus_sylvaticus_Mitochondrion","Solanum_tuberosum_Chloroplast","Trifolium_pratense_Mitochondrion","Triglochin_maritima_Chloroplast","Triglochin_maritima_Chromosome","Triglochin_maritima_Mitochondrion","Vaccinium_myrtillus_Chromosome","Veronica_serpyllifolia_Chloroplast","Veronica_serpyllifolia_Chromosome","Veronica_serpyllifolia_Mitochondrion"]}
            </div>
            <div id='krona-json'>
                {}
            </div>
            <div id='gc-json'>
                {}
            </div>
            <div id='icarus-json'>
                {"links":["icarus.html"],"links_names":["View in Icarus contig browser"]}
            </div>
        </div>
    </div>

    <div class='json-code'>
        <div id='glossary-json'>
            {
    "# contigs"                     : "<span>is the total number of contigs in the assembly.</span>",
    "Largest contig"                : "<span>is the length of the longest contig in the assembly.</span>",
    "Total length"                  : "<span>is the total number of bases in the assembly.</span>",
    "Reference length"              : "<span>is the total number of bases in the reference.</span>",

    "# contigs (>= 0 bp)"           : "<span>is the total number of contigs in the assembly that have size greater than or equal to 0 bp.</span>",
    "Total length (>= 0 bp)"        : "<span>is the total number of bases in the contigs having size greater than or equal to 0 bp.</span>",

    "N50"                           : "<span>is the contig length such that using longer or equal length contigs produces half (50%) of the bases of the assembly. Usually there is no value that produces exactly 50%, so the technical definition is the maximum length x such that using contigs of length at least x accounts for at least 50% of the total assembly length.</span>",
    "NG50"                          : "<span>is the contig length such that using longer or equal length contigs produces half (50%) of the bases of the reference genome. This metric is computed only if a reference genome is provided.</span>",
    "N75"                           : "<span>is the contig length such that using longer or equal length contigs produces 75% of the bases of the assembly. Usually there is no value that produces exactly 75%, so the technical definition is the maximum length x such that using contigs of length at least x accounts for at least 75% of the total assembly length.</span>",
    "NG75"                          : "<span>is the contig length such that using longer or equal length contigs produces 75% of the bases of the reference genome. This metric is computed only if a reference genome is provided.</span>",

    "L50"                           : "<span>is the minimum number of contigs that produce half (50%) of the bases of the assembly. In other words, it's the number of contigs of length at least N50.</span>",
    "LG50"                          : "<span>is the minimum number of contigs that produce half (50%) of the bases of the reference genome. In other words, it's the number of contigs of length at least NG50. This metric is computed only if a reference genome is provided.</span>",
    "L75"                           : "<span>is the minimum number of contigs that produce 75% of the bases of the assembly. In other words, it's the number of contigs of length at least N75.</span>",
    "LG75"                          : "<span>is the minimum number of contigs that produce 75% of the bases of the reference genome. In other words, it's the number of contigs of length at least NG75. This metric is computed only if a reference genome is provided.</span>",

    "NA50"                          : "<span>is N50 where the lengths of aligned blocks are counted instead of contig lengths. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces. This metric is computed only if a reference genome is provided.</span>",
    "NGA50"                         : "<span>is NG50 where the lengths of aligned blocks are counted instead of contig lengths. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces. This metric is computed only if a reference genome is provided.</span>",
    "NA75"                          : "<span>is N75 where the lengths of aligned blocks are counted instead of contig lengths. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces. This metric is computed only if a reference genome is provided.</span>",
    "NGA75"                         : "<span>is NG75 where the lengths of aligned blocks are counted instead of contig lengths. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces. This metric is computed only if a reference genome is provided.</span>",

    "LA50"                          : "<span>is L50 where aligned blocks are counted instead of contigs. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces.</span>",
    "LGA50"                         : "<span>is LG50 where aligned blocks are counted instead of contigs. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces.</span>",
    "LA75"                          : "<span>is L75 where aligned blocks are counted instead of contigs. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces.</span>",
    "LGA75"                         : "<span>is LG75 where aligned blocks are counted instead of contigs. I.e., if a contig has a misassembly with respect to the reference, the contig is broken into smaller pieces.</span>",

    "Average %IDY"                  : "<span>is the average of alignment identity percent (Nucmer measure of alignment accuracy) among all contigs.</span>",
    "# misassemblies"               : "<span>is the number of positions in the assembled contigs where the left flanking sequence aligns over 1 kbp away from the right flanking sequence on the reference (<i>relocation</i>) or they overlap on more than 1 kbp (<i>relocation</i>) or flanking sequences align on different strands (<i>inversion</i>) or different chromosomes (<i>translocation</i>).</span>",
    "# large block misassemblies"   : "<span>is the number of misassemblies between alignments with length greater than or equal to 3 kbp and with the misassembly threshold equal to 5 kbp (instead of default 1 kbp for regular misassemblies).</span>",
    "# misassembled contigs"        : "<span>is the number of contigs that contain misassembly events.</span>",
    "Misassembled contigs length"   : "<span>is the number of total bases contained in all contigs that have one or more misassemblies.</span>",

    "# relocations"                 : "<span>is the number of relocation events among all misassembly events. Relocation is a misassembly where the left flanking sequence aligns over 1 kbp away from the right flanking sequence on the reference, or they overlap by more than 1 kbp and both flanking sequences align on the same chromosome.</span>",
    "# translocations"              : "<span>is the number of translocation events among all misassembly events. Translocation is a misassembly where the flanking sequences align on different chromosomes.</span>",
    "# interspecies translocations" : "<span>is the number of interspecies translocation events among all misassembly events. Interspecies translocation is a misassembly where the flanking sequences align on different references (based on alignments to the combined reference).</span>",
    "# inversions"                  : "<span>is the number of inversion events among all misassembly events. Inversion is a misassembly where it is not a <i>relocation</i> and the flanking sequences align on opposite strands of the same chromosome.</span>",
    "# large relocations"           : "<span>is the number of relocation events among all large block misassemblies. Relocation is a misassembly where the left flanking sequence aligns over 5 kbp away from the right flanking sequence on the reference, or they overlap by more than 5 kbp and both flanking sequences align on the same chromosome.</span>",
    "# large translocations"        : "<span>is the number of translocation events among all large block misassemblies. Translocation is a misassembly where the flanking sequences align on different chromosomes.</span>",
    "# large i/s translocations"    : "<span>is the number of interspecies translocation events among all large block misassemblies. Interspecies translocation is a misassembly where the flanking sequences align on different references (based on alignments to the combined reference).</span>",
    "# large inversions"            : "<span>is the number of inversion events among all large block misassemblies. Inversion is a misassembly where it is not a <i>relocation</i> and the flanking sequences align on opposite strands of the same chromosome.</span>",
    "# local misassemblies"         : "<span>is the number of local misassemblies. We define a local misassembly breakpoint as a breakpoint that satisfies these conditions: <ol> <li>Two or more distinct alignments cover the breakpoint.</li> <li>The gap between left and right flanking sequences is less than the misassembly threshold (1 kbp by default).</li> <li>The left and right flanking sequences both are on the same strand of the same chromosome of the reference genome.</li> </ol> </span>",
    "# scaffold gap ext. mis."      : "<span>is the number of scaffold gap size extensive misassemblies. We define a scaffold gap size misassembly as a breakpoint where the flanking sequences are combined in a scaffold on the wrong distance. These misassemblies are not included in the total number of misassemblies. </span>",
    "# scaffold gap loc. mis."      : "<span>is the number of scaffold gap size local misassemblies. Such breakpoints satisfy the local misassembly conditions but they occur inside scaffolds. These misassemblies are not included in the total number of local misassemblies. </span>",
    "# possibly misassembled contigs": "<span>is the number of contigs that contain large unaligned fragment (default min length is 500 bp) and thus could possibly contain interspecies translocation with unknown reference.</span>",
    "# possible misassemblies"      : "<span>is the number of putative interspecies translocations in possibly misassembled contigs if each large unaligned fragment is supposed to be a fragment of unknown reference.</span>",
    "# intergenomic misassemblies"  : "<span>is the number of all found and putative (possible) interspecies translocations.</span>",
    "# structural variations"       : "<span>is the number of misassemblies matched with structural variations.</span>",
    "# possible TEs"                : "<span>is the number of misassemblies possibly caused by transposable elements (TEs). We define a possible TE as an event that satisfies these conditions: <ol> <li>There are two misassembly breakpoints of the same type around a short alignment (less than 7 kbp by default)</li> <li>The gap between two long flanking sequences on the sides of the short alignment is less than 7 kbp.</li> <li> The long flanking sequences both are on the same strand of the same chromosome of the reference genome.</li> </ol> </span>",
    "# unaligned mis. contigs"      : "<span>is the number of contigs that have the number of unaligned bases more than 50% of contig length and a misassembly event in their aligned fragment. Note that such misassemblies are not counted in <i># misassemblies</i> and other <i>misassemblies</i> statistics.</span>",

    "# fully unaligned contigs"     : "<span>is the number of contigs that have no alignment to the reference sequence.</span>",
    "Fully unaligned length"        : "<span>is the total number of bases contained in all fully unaligned contigs. Uncalled bases (N's) are not counted.</span>",
    "# partially unaligned contigs" : "<span>is the number of contigs that have at least one alignment to the reference sequence but also have at least one unaligned fragment of length &ge; <i>unaligned-part-size threshold</i>.</span>",
    "Partially unaligned length"    : "<span>is the total number of unaligned bases in all partially unaligned contigs. Uncalled bases (N's) are not counted.</span>",

    "# ambiguous contigs"           : "<span>is the number of contigs that have reference alignments of equal quality in multiple locations on the reference.</span>",
    "Ambiguous contigs length"      : "<span>is the total number of bases contained in all ambiguous contigs.</span>",

    "Genome fraction (%)"           : "<span>is the total number of aligned bases in the reference, divided by the genome size. A base in the reference genome is counted as aligned if there is at least one contig with at least one alignment to this base. Contigs from repeat regions may map to multiple places, and thus may be counted multiple times in this quantity.</span>",
    "GC (%)"                        : "<span>is the total number of G and C nucleotides in the assembly, divided by the total length of the assembly.</span>",
    "Reference GC (%)"              : "<span>is the total number of G and C nucleotides in the reference, divided by the total length of the reference.</span>",

    "# mismatches per 100 kbp"      : "<span>is the average number of mismatches per 100000 aligned bases.</span>",
    "# mismatches"                  : "<span>is the number of mismatches in all aligned bases.</span>",
    "# indels per 100 kbp"          : "<span>is the average number of indels per 100000 aligned bases.</span>",
    "# indels"                      : "<span>is the number of indels in all aligned bases</span>",
    "# indels (<= 5 bp)"            : "<span>is the number of indels of length less than or equal to 5 bp</span>",
    "# indels (> 5 bp)"             : "<span>is the number of indels of length greater than 5 bp</span>",
    "Indels length"                 : "<span>is the number of total bases contained in all indels</span>",

    "# genomic features"            : "<span>is the number of genomic features (genes, transcripts, CDS) in the assembly (complete and partial), based on a user-provided annotated list of gene positions in the reference genome. A feature counts as 'partially covered' if the assembly contains at least 100 bp of this feature but not the whole feature.</span>",
    "# operons"                     : "<span>is the number of operons in the assembly (complete and partial), based on a user-provided annotated list of operon positions in the reference genome. An operon counts as 'partially covered' if the assembly contains at least 100 bp of this operon but not the whole operon.</span>",
    "# predicted genes (unique)"    : "<span>is the number of unique genes in the assembly found by a gene prediction tool.</span>",
    "# predicted genes (>= 0 bp)"   : "<span>is the number of found genes having length greater than or equal to 0 bp.</span>",

    "Complete BUSCO (%)"            : "<span>is the percent of BUSCO (Universal Single-Copy Ortholog) genes found in the assembly in a complete form.</span>",
    "Partial BUSCO (%)"             : "<span>is the percent of BUSCO (Universal Single-Copy Ortholog) genes found in the assembly in a partial form.</span>",

    "Cumulative length"             : "<span>plot shows the growth of assembly contig lengths. On the x-axis, contigs are ordered from largest (contig #1) to smallest. The y-axis gives the size of the x largest contigs in the assembly.</span>",
    "Nx"                            : "<span>plot shows the Nx metric value as x varies from 0 to 100.</span>  <span>Nx is the minimum contig length <b>y</b> such that using contigs of length at least <b>y</b> accounts for at least x% of the total assembly length.</span>",
    "NGx"                           : "<span>plot shows the NGx metric value as x varies from 0 to 100.</span> <span>NGx is the minimum contig length <b>y</b> such that using contigs of length at least <b>y</b> accounts for at least x% of the bases of the reference genome. This metric is computed only if a reference genome is provided.</span>",
    "NAx"                           : "<span>plot shows the NAx metric value as x varies from 0 to 100.</span> <span>NAx is computed similarly to Nx, but based on lengths of aligned blocks instead of contig lengths. Contigs are broken into aligned blocks at misassembly breakpoints. NAx is the minimum block length <b>y</b> such that using blocks of length at least <b>y</b> accounts for at least x% of the bases of the assembly. This metric is computed only if a reference genome is provided.</span>",
    "NGAx"                          : "<span>plot shows the NGAx metric value as x varies from 0 to 100.</span><span>NGAx is computed similarly to NGx, but based on lengths of aligned blocks instead of contig lengths. Contigs are broken at misassembly breakpoints. NGAx is the minimum block length <b>y</b> such that using blocks of length at least <b>y</b> accounts for at least x% of the bases of the reference genome. This metric is computed only if a reference genome is provided.</span>",
    "GC content"                    : "<span>plot shows the distribution of GC percentage among the contigs, i.e., the total number of bases in contigs with such GC content. Typically, the distribution is approximately Gaussian. However, for some genomes it is not Gaussian. For assembly projects with contaminants, the GC distribution of the contaminants often differs from the reference genome and may give a superposition of multiple curves with different peaks.</span>",

    "Duplication ratio"             : "<span>is the total number of aligned bases in the assembly (i.e. <i>Total length</i> - <i>Fully unaligned length</i> - <i>Partially unaligned length</i>), divided by the total number of aligned bases in the reference (see the <b>Genome fraction (%)</b> metric). If the assembly contains many contigs that cover the same regions of the reference, its <i>Duplication ratio</i> may be much larger than 1. This may occur due to overestimating repeat multiplicities and due to small overlaps between contigs, among other reasons.</span>",
    "Largest alignment"             : "<span>is the length of the largest continuous alignment in the assembly. This metric is always equal to the <i>Largest contig</i> metric but it can be smaller if the largest contig of the assembly contains a misassembly event.</span>",
    "Total aligned length"          : "<span>is the total number of aligned bases in the assembly.</span>",
    "Avg contig read support"       : "<span>is the average coverage of contigs that have large unique alignments to the reference. Read coverage is extracted from contig names (SPAdes/Velvet naming scheme only).</span>",

    "# N's"                         : "<span>is the total number of uncalled bases (N's) in the assembly.</span>",
    "# N's per 100 kbp"             : "<span>is the average number of uncalled bases (N's) per 100000 assembly bases.</span>",

    "# mapped"                      : "<span>is the number of reads that mapped to the assembly.</span>",
    "Mapped (%)"                    : "<span>is the percent of reads that mapped to the assembly.</span>",
    "# properly paired"             : "<span>is the number of reads that mapped within expected range and orientation of each other to the assembly.</span>",
    "Properly paired (%)"           : "<span>is the percent of reads that mapped within expected range and orientation of each other to the assembly.</span>",
    "# singletons"                  : "<span>is the number of reads which do not overlap any other reads.</span>",
    "Singletons (%)"                : "<span>is the percent of reads which do not overlap any other reads.</span>",
    "# misjoint mates"              : "<span>is the number of reads with mate mapped to a different contig.</span>",
    "Misjoint mates (%)"            : "<span>is the percent of reads with mate mapped to a different contig.</span>",
    "Avg. coverage depth"           : "<span>is the average depth of coverage.</span>",
    "Coverage >= 1x (%)"            : "<span>is the total number of bases with at least 1x coverage, divided by the total length of the assembly.</span>",

    "# similar correct contigs"     : "<span>is the number of correct contigs similar among > 50% assemblies (see Icarus for visualization).</span>",
    "# similar misassembled blocks" : "<span>is the number of misassembled blocks similar among > 50% assemblies (see Icarus for visualization).</span>"
}

        </div>
    </div>
</div>
</body>
</html>
